<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns:mv="http://macVmlSchemaUri" xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta name=Title content="">
<meta name=Keywords content="">
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List href="OP_S1_files/filelist.xml">
<link rel=Edit-Time-Data href="OP_S1_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Bruno Gonçalves</o:Author>
  <o:LastAuthor>Bruno Gonçalves</o:LastAuthor>
  <o:Revision>1</o:Revision>
  <o:TotalTime>1</o:TotalTime>
  <o:Created>2016-03-08T21:37:00Z</o:Created>
  <o:LastSaved>2016-03-08T21:38:00Z</o:LastSaved>
  <o:Pages>6</o:Pages>
  <o:Words>2288</o:Words>
  <o:Characters>13047</o:Characters>
  <o:Company>IMM</o:Company>
  <o:Lines>108</o:Lines>
  <o:Paragraphs>30</o:Paragraphs>
  <o:CharactersWithSpaces>15305</o:CharactersWithSpaces>
  <o:Version>14.0</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="OP_S1_files/themedata.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>JA</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
   <w:UseFELayout/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="276">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Verdana;
	panose-1:2 11 6 4 3 5 4 4 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-1593833729 1073750107 16 0 415 0;}
@font-face
	{font-family:Verdana;
	panose-1:2 11 6 4 3 5 4 4 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-1593833729 1073750107 16 0 415 0;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1073743103 0 0 415 0;}
@font-face
	{font-family:"Lucida Grande";
	panose-1:2 11 6 0 4 5 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-520090897 1342218751 0 0 447 0;}
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	mso-themecolor:hyperlink;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:purple;
	mso-themecolor:followedhyperlink;
	text-decoration:underline;
	text-underline:single;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Balloon Text Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:9.0pt;
	font-family:"Lucida Grande";
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-ansi-language:DE-CH;}
p.MsoNoSpacing, li.MsoNoSpacing, div.MsoNoSpacing
	{mso-style-priority:1;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Balloon Text";
	mso-ansi-font-size:9.0pt;
	mso-bidi-font-size:9.0pt;
	font-family:"Lucida Grande";
	mso-ascii-font-family:"Lucida Grande";
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:"Lucida Grande";
	mso-bidi-font-family:"Lucida Grande";
	mso-ansi-language:DE-CH;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"ＭＳ 明朝";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
@page WordSection1
	{size:595.0pt 842.0pt;
	margin:72.0pt 89.85pt 72.0pt 89.85pt;
	mso-header-margin:35.45pt;
	mso-footer-margin:35.45pt;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;}
</style>
<![endif]-->
</head>

<body bgcolor=white lang=EN-US link=blue vlink=purple style='tab-interval:36.0pt'>

<div class=WordSection1>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'>OP01 </span></b><span style='mso-bidi-font-size:10.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
text-transform:uppercase;mso-ansi-language:EN-US;mso-fareast-language:PT'>- </span><b
style='mso-bidi-font-weight:normal'><span style='mso-bidi-font-size:10.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-fareast-language:PT'>Nullarbor: Reads To Reports,
Rapidly</span></b><span style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;text-transform:uppercase;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><u><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Torsten Seemann</span></u><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>; Dieter Bulach<sup>1</sup>; Jason Kwong<sup>1</sup>;
Anders Goncalves Da Silva<sup>1</sup>; Benjamin Howden<sup>1</sup></span><span
lang=DE-CH style='mso-bidi-font-size:9.0pt;mso-bidi-font-family:Arial'><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>University of Melbourne</span></i><i style='mso-bidi-font-style:normal'><span
lang=DE-CH style='mso-bidi-font-size:9.0pt;mso-bidi-font-family:Arial'><o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background:</span></b><span
style='font-size:8.0pt;font-family:Verdana;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-US;
mso-fareast-language:PT'> </span><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>The modern public health microbiology laboratory
has embraced genome sequencing as the primary assay for pathogen surveillance
and outbreak analysis. Here we present Nullarbor, a software pipeline for
turning a set of isolate sequence data into a single report summarizing the key
information about each isolate and the relationship between isolates.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods:</span></b><span
style='font-size:8.0pt;font-family:Verdana;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-US;
mso-fareast-language:PT'> </span><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Nullarbor first adaptor clips and quality trims
the reads for each isolate and assesses sequencing depth. Reads are scanned
with Kraken to identify the species and any potential contamination or
mix-ups.. The reads are de novo assembled into contigs using SPAdes or MegaHit
(fast mode) and annotated using Prokka (*). The contigs are used to infer MLST
with the mlst tool (*) without requiring specification of the scheme. ABRicate
(*) is used to determine the full resistome profile using the ResFinder
database (Fig 1). Extra organism-specific in silico modules (*) are available
for N.gonorrhoeae (NG-MAST), N.meningitidis (porA+fetA), L.monocytogenes
(serotyping, binary typing) and S.enterica (MLVA).</span><span
style='font-size:8.0pt;font-family:Verdana;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Snippy (*) is used
to align each isolate reads to a common reference (can be an assembled isolate)
and identify SNPs, and to also compute the core genome alignment, from which a
phylogenetic ML tree is generated using FastTree. &nbsp;A key feature of Snippy
(*) is that adding or removing taxa from the alignment can be done very
efficiently because no unnecessary recomputation is performed. The pan-genome
(Fig 2) is generated by Roary from the annotated assemblies and visualized
using roary2svg (*). The use of the pan-genome augments the core genome SNP
analysis with data on mobile and accessory genetic elements, and is a feature
unique to Nullarbor amongst the publicly available pipelines.</span><span
style='font-size:8.0pt;font-family:Verdana;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;font-family:Verdana;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-US;
mso-fareast-language:PT'> </span><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>The Nullarbor pipeline follows the Unix
philosophy of using and combining existing standalone tools in an efficient
manner and is parallelizable via the standard Make system. This makes is
amenable to provincial public health laboratories with low compute capabilities
and slow upload bandwidth to perform their own analyses. The input is a
spreadsheet-like text file, and the default output is a clean HTML report. It
is currently used by the Microbiological Diagnostics Unit Public Health
Laboratory in Australia routinely for regular reporting and all outbreak
investigations. An example report is available for viewing at the Nullarbor web
site: </span><span lang=DE-CH><a href="https://github.com/tseemann/nullarbor"><span
lang=EN-US style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:windowtext;mso-ansi-language:EN-US;mso-fareast-language:
PT;text-decoration:none;text-underline:none'>https://github.com/tseemann/nullarbor</span></a></span><span
style='font-size:8.0pt;font-family:Verdana;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion:</span></b><span
style='font-size:8.0pt;font-family:Verdana;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-US;
mso-fareast-language:PT'> </span><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Nullarbor is open-source software released under
a GPLv2 licence and runs on Unix systems. Simple installation of Nullarbor and
its dependencies may be achieved via Homebrew </span><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-fareast-language:PT'>Science </span><span
lang=DE-CH><a href="https://github.com/Homebrew/homebrew-science"><span
lang=EN-US style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:windowtext;mso-ansi-language:EN-US;mso-fareast-language:
PT;text-decoration:none;text-underline:none'>https://github.com/Homebrew/homebrew-science</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>. Future <span
style='color:black'>plans include: a virtual machine image for use with Amazon
EC, OpenStack and VirtualBox (Jan 2015); &nbsp;a richer, more interactive and
colourful report generator called QandongR (Feb 2016); the ability to accept
already assembled genomes (Dec 2015); and support for legacy Ion Torrent and
Roche 454 data sets (Apr 2016).</span></span><span style='font-size:8.0pt;
font-family:Verdana;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>(*) denotes
software written by one or more of the authors.</span><span style='font-size:
8.0pt;font-family:Verdana;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:10.0pt;line-height:115%'><span
style='mso-bidi-font-size:10.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-no-proof:yes'><!--[if gte vml 1]><v:shapetype id="_x0000_t75" coordsize="21600,21600"
 o:spt="75" o:preferrelative="t" path="m@4@5l@4@11@9@11@9@5xe" filled="f"
 stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Picture_x0020_2" o:spid="_x0000_i1026" type="#_x0000_t75"
 style='width:333pt;height:168pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="OP_S1_files/image001.png" o:title=""/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=170
src="public/abstracts/Webpages/OP_S1_files/image002.png" v:shapes="Picture_x0020_2"><![endif]></span><span
style='mso-bidi-font-size:10.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:10.0pt;line-height:115%'><span
style='mso-bidi-font-size:10.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-no-proof:yes'><!--[if gte vml 1]><v:shape id="Picture_x0020_3" o:spid="_x0000_i1025"
 type="#_x0000_t75" style='width:333pt;height:125pt;visibility:visible;
 mso-wrap-style:square'>
 <v:imagedata src="OP_S1_files/image003.png" o:title=""/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=127
src="public/abstracts/Webpages/OP_S1_files/image004.png" v:shapes="Picture_x0020_3"><![endif]></span><span
style='mso-bidi-font-size:10.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";text-transform:uppercase;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal style='margin-bottom:10.0pt;line-height:115%'><span
style='mso-bidi-font-size:10.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'>OP02 - </span></b><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Scalable
And User-Friendly Workflows For Molecular Epidemiology Using Enterobase</span></b><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><u><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Martin Sergeant</span></u><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>; Nabil-Fareed Alikhan<sup>1</sup>; Zhemin Zhou<sup>1</sup>;
Mark Achtman<sup>1</sup></span><span lang=DE-CH style='mso-bidi-font-size:9.0pt;
mso-bidi-font-family:Arial'><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>University of Warwick</span></i><i style='mso-bidi-font-style:normal'><span
lang=DE-CH style='mso-bidi-font-size:9.0pt;mso-bidi-font-family:Arial'><o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;text-transform:uppercase;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'> The decreasing cost of
next-generation sequencing can now facilitate the routine sequencing of
pathogens, moving closer to real-time molecular epidemiology. There are over 13
billion bacterial nucleotide base-pairs catalogued within GenBank, while pathogens
such as <i>Salmonella enterica </i>and <i>Escherichia coli</i> have over 40,000
and 16,000 sets, respectively, of sequenced reads available. These data could
provide a basis for a global perspective of microbial pathogens. However, such
analyses are confounded by the unprecedented logistical challenge of
corralling, curating and comparing these data. Furthermore, there is a paucity
of standardized typing approaches that can reflect subtle genetic variation
within outbreaks to deep genealogical structures beyond the species level –
revealing both short-term and long-term transmission trends.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods:</span></b><span
style='font-size:8.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'> Here
we present EnteroBase, which addresses both challenges by providing novel
genotyping methods (cgMLST, CRISPR and SNPs) through a graphical web interface,
accessible to a wide audience of clinicians, epidemiologists and
bioinformaticians.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'>Backed by computing resources
of the Cloud Infrastructure for Microbial Bioinformatics (</span><span
lang=DE-CH><a href="http://www.climb.ac.uk/"><span lang=EN-US style='font-size:
8.0pt;color:windowtext;mso-ansi-language:EN-US;mso-fareast-language:PT;
text-decoration:none;text-underline:none'>http://www.climb.ac.uk/</span></a></span><span
style='font-size:8.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'>),
EnteroBase offers automatic pipelines to type bacterial strains based on short
reads fetched from Sequence Read Archive (SRA) or uploaded by registered users.
Data from different sources are integrated and presented in a centralised
database, which gives the greatest genetic panoramas so far for several
bacterial genera.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'>EnteroBase
serves over 50,000 genome assemblies from the SRA with metadata and genotyping
data (MLST, rMLST and cgMLST), alongside 16,000 legacy records from MLST
databases. These assemblies have been screened for contamination, poor
sequencing quality and misassemblies and are paired with standardised and
curated strain metadata, including geographic, host and temporal information.
All sequencing data are processed in a matter of hours and their results are available
through the user-friendly web interface.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'>EnteroBase is also designed to
be integrated into a data exchange network, in which Enterobase not only offers
well established APIs for automatic machine access, but also communicates data
with public data archives (EBI), traditional MLST sites (MLST Warwick and
pubMLST), and commercial local database interfaces (Bionumerics).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'> Currently EnteroBase supports
the genera <i>Salmonella</i>, <i>Escherichia/Shigella</i>, <i>Moraxella</i> and
the <i>Yersiniae</i> but further development will see EnteroBase as a generic
platform for the curation, calculation and communication of sequencing data for
all organisms of clinical relevance.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'>EnteroBase is accessible
through all modern web browsers, and is available at </span><span lang=DE-CH><a
href="http://enterobase.warwick.ac.uk"><span lang=EN-US style='font-size:8.0pt;
color:windowtext;mso-ansi-language:EN-US;mso-fareast-language:PT;text-decoration:
none;text-underline:none'>http://enterobase.warwick.ac.uk</span></a></span><span
style='font-size:8.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal style='margin-bottom:10.0pt;line-height:115%'><span
style='font-size:8.0pt;line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'>OP03 - </span></b><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Successes
And Challenges In High Throughput Whole Genome Sequencing Of Viruses</span></b><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><u><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Dan Frampton</span></u><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>; Tiziano Gallo Cassarino<sup>1</sup>; Zisis
Kozlakidis<sup>1</sup>; Anne Hoppe<sup>1</sup>; Deenan Pillay<sup>2</sup>; Paul
Kellam<sup>3<o:p></o:p></sup></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Division of Infection and Immunity, UCL; <sup>2</sup>Wellcome Trust Africa
Centre for Health and Population Studies; <sup>3</sup>Wellcome Trust Sanger
Institute<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;text-transform:uppercase;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'> Despite recent advancements
human immunodeficiency virus (HIV) remains a significant challenge to global
public health. It is a highly diverse virus, exhibiting large genetic variation
both at the population level and within individual patients. Importantly, even
for treatment-naïve patients the presence of specific drug-resistant minority
variants within an individual´s viral population can lead to failure of drug
therapy. The ability to both identify and quantify minority variants can thus
be of direct clinical benefit.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'>As opposed to traditional
Sanger sequencing, next generation sequencing (NGS) enables accurate
quantification of minority variants and sequencing of viral sub- and
quasi-species. The advent of affordable NGS technology offers a unique
opportunity to rapidly process large numbers of clinical samples. Linking
genomic and clinical data on this scale should facilitate development of novel
clinical diagnostics, prognostics and optimal treatment strategies.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials | Methods</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'> Within the ICONIC project, we
have developed an automated and customizable virus-agnostic high-throughput
analysis pipeline to generate viral clinical whole genomes, quantify minority
variants and identify drug-resistance mutations (DRMs). Via <i>de novo</i>
assembly of contigs, concatenation and correction by alignment to reference and
iterative remapping of reads, our pipeline derives a consensus sequence and
identifies minority variants across each genome within hours.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'> Here we present results from
analysis of a set of several hundred HIV samples from both the UK and various
cohorts in Africa. &nbsp;Samples were collected and amplified using a pan-HIV1
primer set and sequenced by Illumina MiSeq at the Wellcome Trust Sanger
Institute (Cambridge). Despite no prior filtering of samples by viral load or
amplification success, our pipeline successfully generated genomes for 384 (91%)
of UK samples with a median clinical whole genome coverage across samples of
82%. Pipeline validation was performed by directly comparing <i>pol</i>
sequences derived by traditional Sanger sequencing to those generated by NGS
and our pipeline from the same set of 384 samples.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'> Having applied the pipeline
to additional viruses (influenza, norovirus, hepatitis C, respiratory syncytial
virus, ebola and coronavirus) on UCL’s high performance computing (HPC)
cluster, preliminary results suggest this method is of potential clinical
utility to a broad range of pathogens.<o:p></o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal style='margin-bottom:10.0pt;line-height:115%'><span
style='font-size:8.0pt;line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'>SC01 - </span></b><b style='mso-bidi-font-weight:normal'><span
lang=DE-CH style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-fareast-language:PT'>Whole Genome Sequence
Analysis (wgsa.net). An Exemplar Using <i style='mso-bidi-font-style:normal'>Staphylococcus
aureus</i></span></b><span lang=DE-CH style='mso-bidi-font-size:10.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
text-transform:uppercase;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><u><span lang=DE-CH style='mso-bidi-font-size:9.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-fareast-language:PT'>David Aanensen</span></u><sup><span
lang=DE-CH style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>1,2</span></sup><span
lang=DE-CH style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=DE-CH
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=DE-CH style='mso-bidi-font-size:
9.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-fareast-language:PT'>Department of Infectious Disease
Epidemiology, Imperial College London.<sup>2</sup>Wellcome Trust Sanger
Institute, Kingdom<o:p></o:p></span></i></p>

<span lang=DE-CH style='font-size:10.0pt;mso-bidi-font-size:9.0pt;line-height:
115%;font-family:Arial;mso-fareast-font-family:"Times New Roman";color:black;
mso-ansi-language:DE-CH;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal style='margin-bottom:10.0pt;line-height:115%'><span
lang=DE-CH style='mso-bidi-font-size:9.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'>SO05 - </span></b><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>The
Epiquant Framework For Assessing Epidemiologic And Genetic Concordance: Towards
Improved Use Of Genomic Data In Epidemiological Applications</span></b><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'>.<o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Benjamin Hetman<sup>1</sup>; Steven K. Mutschall<sup>2</sup>;
Victor P. J. Gannon<sup>2</sup>; James E. Thomas<sup>1</sup>; <u>Eduardo N.
Taboada</u><sup>2</sup></span><span lang=DE-CH style='mso-bidi-font-size:9.0pt;
mso-bidi-font-family:Arial'><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Department of Biological Sciences, University of Lethbridge; <sup>2</sup>National
Microbiology Laboratory at Lethbridge, Public Health Agency of Canada</span></i><i
style='mso-bidi-font-style:normal'><span lang=DE-CH style='mso-bidi-font-size:
9.0pt;mso-bidi-font-family:Arial'><o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> <i>Campylobacter
jejuni</i> is an important bacterial foodborne pathogen, with an estimated 400
million cases worldwide each year. In Canada and many industrialized countries,
<i>C. jejuni</i> is the leading cause of bacterially-incited enteritis, with a
yearly per capita incidence as high as 1%. Because most cases of
campylobacteriosis appear to be non-outbreak related, improved methods for
attributing sources of exposure are increasingly critical for public health
interventions to reduce this burden of illness. With recent advances in Whole
Genome Sequencing (WGS), analysis of <i>C. jejuni&nbsp;</i>isolates collected
through human and non-human surveillance programs in Canada using WGS is being
increasingly used in epidemiologic investigations aimed at the prevention and
control of campylobacteriosis.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> We have
analyzed the genomes of over 3,000 <i>C. jejuni </i>isolates from
environmental, animal, and clinically-derived sources, with an aim to develop
analytical methods that extract critical information from WGS data to
facilitate genomic epidemiology for attributing the likely source of human
infections. We have developed a model to derive a quantitative summary
statistic for the epidemiological similarity (EpiSym) of any two isolates based
on parameters derived from basic metadata (date of collection, location,
source) collected in routine surveillance. We have also developed a statistical
framework (EpiQuant) for assessing the concordance between epidemiological and
genomic similarity of isolates in the R environment for statistical computing.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> The EpiSym
statistic allows for an estimate of pairwise similarities between isolates
based on basic epidemiological information. This estimate is used in our
EpiQuant framework for directly assessing the concordance between pairwise
similarity estimates based on epidemiologic data and those derived from genomic
data. We have used the EpiQuant framework to optimize similarity thresholds for
the interpretation of <i>C. jejuni</i> WGS data in the context of
epidemiological investigations so as to maximize recovery of epidemiologically
and genomically concordant clusters for source attribution. We have also used
the EpiQuant framework to identify and investigate cases where genetic
similarity estimates deviate from those expected based on underlying
epidemiology and show how the genetic and epidemiological linkage between <i>C.
jejuni</i> isolates can break down under specific circumstances.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> The EpiSym
statistic and the EpiQuant framework allow for direct comparison of
epidemiologic and genomic similarity estimates between isolates, paving the way
for improved source tracking and source attribution. We are developing an
online toolkit to be used in the development of interpretation criteria for WGS
in epidemiologic investigations of&nbsp;<i>C. jejuni&nbsp;</i>and other
high-risk bacterial pathogens.<o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
line-height:115%;font-family:Arial;mso-fareast-font-family:"Times New Roman";
text-transform:uppercase;mso-ansi-language:EN-US;mso-fareast-language:PT;
mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal style='margin-bottom:10.0pt;line-height:115%'><b
style='mso-bidi-font-weight:normal'><span style='mso-bidi-font-size:10.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;text-transform:uppercase;mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'>OP06 - </span></b><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>CLIMB:
Developing A National Cloud Infrastructure For Microbial Bioinformatics</span></b><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNoSpacing style='text-align:justify;text-justify:inter-ideograph'><u><span
style='font-size:10.0pt;font-family:Arial;color:black;mso-ansi-language:EN-US'>Emily
Richardson</span></u><sup><span style='font-size:10.0pt;font-family:Arial;
color:black;mso-ansi-language:EN-US'>1</span></sup><span style='font-size:10.0pt;
font-family:Arial;color:black;mso-ansi-language:EN-US'>; Nick Loman<sup>1; </sup>Simon
Thompson<sup>1</sup>; Matthew Ismail<sup>2</sup>; Sam Sheppard<sup>3</sup>;
Thomas Connor<sup>4</sup>; Mark Pallen<sup>2</sup> </span><span
style='font-size:10.0pt;font-family:Arial;mso-fareast-font-family:"Times New Roman";
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:10.0pt;mso-bidi-font-family:Arial;color:black;
mso-ansi-language:EN-US'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:10.0pt;mso-bidi-font-family:Arial;color:black;
mso-ansi-language:EN-US'>University of Birmingham; <sup>2</sup>University of
Warwick; <sup>3</sup>University of Swansea; <sup>4</sup>University of Cardiff</span></i><span
style='mso-bidi-font-size:10.0pt;mso-bidi-font-family:Arial;color:black;
mso-themecolor:text1;mso-ansi-language:EN-US'><o:p></o:p></span></p>

<p class=MsoNoSpacing style='text-align:justify;text-justify:inter-ideograph'><span
style='font-family:Arial;color:black;mso-themecolor:text1;mso-ansi-language:
EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing style='text-align:justify;text-justify:inter-ideograph'><span
style='font-size:8.0pt;font-family:Arial;mso-fareast-font-family:"Times New Roman";
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Whole genome
sequencing is transforming public health surveillance of bacterial pathogens. However,
the scale of genomic datasets poses new informatics challenges for storing and
analysing large aggregate datasets. This need is compounded by a lack of
trained bioinformaticians to analyse the data. In 2014 the Medical Research
Council made a ~£50m investment in “big data” to support the development of new
research infrastructures. The £8.5m CLoud Infrastructure for Microbial
Bioinformatics (CLIMB) was the only award to a microbial consortium and is one
the largest investments in microbial genomics bioinformatics ever made. CLIMB
provides a national e-infrastructure to deliver bioinformatics infrastructure
as a service to the UK academic medical microbial community. CLIMB is hosted
across four sites (Birmingham, Cardiff, Swansea and Warwick) and we provide a
single sign-on, distributed computing and storage infrastructure.&nbsp;&nbsp;
The total investment in hardware is £3.6m, providing 7680 virtual CPUs, 500
terabytes of local high performance storage per site and 7000 terabytes (7PB)
of replicated object storage. This facility is sufficient to provide over a
thousand simultaneously running virtual machines to users. The service is
implemented using the open-source OpenStack framework. Data storage is managed
through the CEPH object storage system which also provides redundancy. CLIMB
has been made available to early access users mode for the last 6 months and
have begun to deploy microbial genomics software to cater for user
requirements. In this presentation we demonstrate how CLIMB, and cloud services
like it, may impact on the practice of public health genomic surveillance of
pathogens. To assist training and ease of use, we have deployed the Genomics
Virtual Laboratory system (developed at the University of Melbourne) onto
CLIMB. The GVL can help deliver reproducible workflows and assist training of
new bioinformaticians through the use of an inbuilt Galaxy server and IPython
notebook. This system also reduces much of the burden associated with
installing bioinformatics analysis software. Those using genomics for microbial
surveillance should be aware of the potential opportunities offered by cloud
services for large-scale, integrated data analysis.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=DE-CH><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
