<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns:mv="http://macVmlSchemaUri" xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta name=Title content="">
<meta name=Keywords content="">
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List href="OP_S6_files/filelist.xml">
<link rel=Edit-Time-Data href="OP_S6_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Bruno Gonçalves</o:Author>
  <o:LastAuthor>Bruno Gonçalves</o:LastAuthor>
  <o:Revision>1</o:Revision>
  <o:TotalTime>0</o:TotalTime>
  <o:Created>2016-03-08T21:43:00Z</o:Created>
  <o:LastSaved>2016-03-08T21:43:00Z</o:LastSaved>
  <o:Pages>6</o:Pages>
  <o:Words>2853</o:Words>
  <o:Characters>16267</o:Characters>
  <o:Company>IMM</o:Company>
  <o:Lines>135</o:Lines>
  <o:Paragraphs>38</o:Paragraphs>
  <o:CharactersWithSpaces>19082</o:CharactersWithSpaces>
  <o:Version>14.0</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="OP_S6_files/themedata.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:GrammarState>Clean</w:GrammarState>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>JA</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
   <w:UseFELayout/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="276">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1073743103 0 0 415 0;}
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	mso-themecolor:hyperlink;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:purple;
	mso-themecolor:followedhyperlink;
	text-decoration:underline;
	text-underline:single;}
p.MsoNoSpacing, li.MsoNoSpacing, div.MsoNoSpacing
	{mso-style-priority:1;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;}
span.GramE
	{mso-style-name:"";
	mso-gram-e:yes;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"ＭＳ 明朝";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
@page WordSection1
	{size:595.0pt 842.0pt;
	margin:72.0pt 89.85pt 72.0pt 89.85pt;
	mso-header-margin:35.45pt;
	mso-footer-margin:35.45pt;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;}
</style>
<![endif]-->
</head>

<body bgcolor=white lang=EN-US link=blue vlink=purple style='tab-interval:36.0pt'>

<div class=WordSection1>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP33
- Pan-Genome Approaches To Identify Candidate Genetic Markers In Clinical <i
style='mso-bidi-font-style:normal'>Staphylococcus</i> Species<span
style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNormal><u><span lang=PT style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
PT;mso-fareast-language:PT'>Guillaume Méric</span></u><sup><span lang=PT
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:PT;mso-fareast-language:
PT'>1</span></sup><span lang=PT style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
PT;mso-fareast-language:PT'>; Leonardos Mageiros<sup>1</sup>; Samuel K.
Sheppard<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Swansea University Medical School<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=EN-GB style='mso-bidi-font-family:Arial;
color:black;mso-themecolor:text1;mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Opportunistic pathogens from the genus <i>Staphylococcus</i> are associated
with increased mortality and morbidity, especially in the hospital environment.
In the clinical laboratory, <i>S. aureus</i> and other <i>Staphylococcus sp</i>.
including <i>S. epidermidis </i>are routinely discriminated using growth on
blood agar, latex fixation tests, MALDI-TOF, or coagulase assays. However, it
remains challenging to identify other non-<i>aureus </i>and non-<i>epidermidis </i>staphylococcal
species carried by humans, some of which can also cause disease. The decreasing
cost of high-throughput sequencing and the increasing availability of
user-friendly bioinformatic analysis methods can help solve this problem.
Additionally, there is a need to identify markers that are relevant for human
infection in order to improve diagnostics.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Methods: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>We present here a comparative genomics pipeline based on the gene-by-gene
approach to compare more than 500 whole genome sequences from more than 25
staphylococcal species to characterize core and accessory genome variation, and
find species-specific epidemiological markers. This approach is based on the
creation of a reference pan-genome of more than 21,000 genes from many strains
of all species in our dataset. It provides an alternative to classical <span
class=GramE>methods which</span> fail to capture species-specific markers that
are absent from a single-strain reference genome.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>A robust phylogeny of <span class=GramE>clinically-important</span> <i>Staphylococcus
</i>specieswas obtained and analysed. Additionally, we identified in 14 <i>Staphylococcus
sp.</i> a total of 2942 genes that were species-specific (i.e., present in one
species but absent in all other species), amongst which a total of 1543 were
core to individual species (i.e., present in all isolates from a given species
but absent from all other species).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusions</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>: Possible applications are discussed, including the creation of a gene
“scheme” which can allow the rapid discrimination of most human-carried species
of <i>Staphylococcus </i>after high-throughput sequencing in the clinical
environment.<o:p></o:p></span></p>

<span lang=EN-GB style='font-size:8.0pt;line-height:115%;font-family:Arial;
mso-fareast-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
mso-fareast-language:PT;mso-bidi-language:AR-SA'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='font-size:8.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP34
- Biomarker Discovery For Phenotypic Groups In <i style='mso-bidi-font-style:
normal'>Escherichia coli</i> Using Superphy<span style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNormal><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Chad Laing<sup>1</sup>; <u>Eduardo Taboada</u><sup>1</sup>,
Akiff Manji<sup>2</sup>; Vic Gannon<sup>1 </sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Public Health Agency of Canada; <sup>2</sup>The University of British
Columbia<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-bidi-font-family:Arial;color:black;
mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Shiga-toxin producing <i>Escherichia coli</i> (STEC) are associated with
food- and water-borne human disease outbreaks and sporadic infections. Symptoms
can range from diarrhea to the life-threatening hemolytic uremic syndrome. <span
class=GramE>Among STEC,</span> certain serotypes and lineages within serotypes
are more frequently associated with human disease than others. Further, they
may differ in their ability to colonize animals and survive in the environment.
Despite the importance of STEC, the link between phenotype and genotype is
still largely unknown. The omnilog system by Biolog is a phenotypic microarray
(PM) that tests nutrient utlization and chemical sensitivity of microorganisms,
in a high-throughput manner. We have previously developed SuperPhy, an online
predictive genomics platform (&lt;</span><span lang=DE-CH><a
href="http://lfz.corefacility.ca/superphy/"><span lang=EN-GB style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-GB;mso-fareast-language:PT'>http://lfz.corefacility.ca/superphy/</span></a></span><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>&gt;) for *Escherichia coli*, which can be used to determine statistically
predictive markers for groups of genomes. In this study we examined the
whole-genome sequence and phenotypic profile of 143 STEC of a variety of
serotypes isolated over a period of 20 years, from bovine, human, and
environmental sources.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Materials
and Methods</span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>: Genomes were sequenced using the Illumina
MiSeq at Genome Quebec or the National Microbiology Laboratory in Winnipeg,
Canada. Sequences were assembled using Spades. Comparative genomics were
performed using Panseq. Phylogenetic trees were created using FastTree2. PM
analyses were performed using the Biolog Omnilog system, and the opm package
for R. Statistical analyses were performed using methods from the SuperPhy
platform, in R. The PM data were hierarchically clustered with the Euclidean
distance measure using complete linkage.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>: Hierarchical clustering of the 143 strains based on phenotypic
differences generated a tree that was broadly split into two groups: one
comprised of O157:H7 strains and the other of non-O157<span class=GramE>:H7</span>
strains. Within the non-O157 group, four large clusters were observed, each
with strains of multiple serotypes. Within each of these four large clusters,
smaller serotype-specific clades were identified. The large O157:H7 specific
group was divided into three main clusters that were largely specific to the
three major lineages of the serotype. Phenotypic utilization profiles capable
of statistically predicting the observed clades, as well as serotype-specific
clusters were also identified. The phylogeny based on single nucleotide
polymorphisms (<span class=GramE>SNP)s</span> among the 143 genomes produced
clusters of strains that were largely concordant with those based on the PM
data, showing that most clades were serotype specific with occasional outliers
clustering apart from other strains of the same serotype.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>: The tree based on phenotypic markers recapitulated the groupings of
bacterial strains obtained through SNP whole-genome phylogeny. Known genetic
lineages such as those within serotype O157:H7 were distinguished based on
phenotypic profile and sets of markers predictive of these subgroups were
identified. The results likely reflect a similar physiology among related
strains, which could indicate a similar pathogenicity to humans, or
environmental niche. Potential implications of this work include the
development of selective media for particular serotypes, clades and lineages,
or biochemical tests to identify specific bacterial sub-groups most frequently
associated with human disease.<o:p></o:p></span></p>

<span lang=EN-GB style='font-size:8.0pt;line-height:115%;font-family:Arial;
mso-fareast-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
mso-fareast-language:PT;mso-bidi-language:AR-SA'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='font-size:8.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP35
- Clostridium Difficile - Heterogeneity Of Pathogenicity Locus and
Characterisation Of Atypical Paloc-Replacing Sequences In Non-Toxigenic Strains<span
style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNormal><u><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Sandra Janezic</span></u><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>; Kate Dingle<sup>2</sup>; Derrick W. Crook<sup>2</sup>;
Maja Rupnik<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>National Laboratory for Health, Environment and Food and University of
Maribor, Faculty of Medicine; <sup>2</sup>Oxford University, John Radcliffe
Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-bidi-font-family:Arial;color:black;
mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>A Gram-positive, anaerobic, sporogenic, bacterium<i> Clostridium difficile</i>
is the most common pathogen associated with nosocomial intestinal infections in
humans. The clinical manifestations of <i>C. difficile</i> infection (CDI)
range from mild diarrhoea to pseudomembranous colitis and even death. Although,
majority of CDI are still healthcare-associated, increasing number of studies
describe isolation of <i>C. difficile</i> from community CDI cases, from animal
hosts and from different non-hospital environments, which may represent
important reservoirs. Toxin A and B are responsible for onset of the disease
and are encoded within the 19.6 kb pathogenicity locus (PaLoc) together with
tree accessory genes (<i>tcdC</i>, <i>tcd</i>E and <i>tcd</i>R). In the
majority of toxigenic strains, the PaLoc <span class=GramE>is found to be
inserted</span> in the same chromosomal site and is in non-toxigenic strains
replaced by conserved 115/75 bp non-coding region. Recently, a 7.2 kb sequence
of unknown origin have been found to be inserted at the PaLoc insertion site
and strains with the PaLoc integrated in unusual chromosomal sites have been
described by different authors. Genetic variation of the PaLoc has been
assessed by <span class=GramE>toxinotyping,</span> a PCR-RFLP based typing
method, in which strains with different PaLoc variants are distributed into
different toxinotypes. In this study, we used whole genome sequencing to
explore variations in<i> tcd</i>A and <i>tcd</i>B, and to investigate
phylogenetic relationship of different toxinotypes. Furthermore, we used
genomic sequencing to explore genomic organization in a group of non-toxigenic
strains where the 115 bp sequence could not be PCR-amplified.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Materials/methods:
</span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>Genomes of 25 representative <i>C. difficile</i>
toxinotypes and 12 non-toxigenic isolates (2 human and 10 soil isolates) were
sequenced using Illumina technology and <i>de-novo</i> genome assemblies were
created using Velvet.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Sequence analysis confirmed earlier observations that single nucleotide
polymorphisms (SNPs) are more common in <i>tcd</i>B. Four different patterns of
point mutations were differentiated in TcdB, which correlated well with
phylogenetic lineages and type of cytopathic effect of TcdB on cultured cells,
suggesting evolution of TcdB from different ancestors. The <i>tcd</i>A is more
conserved with mostly deletions and insertions of larger sequences present in
catalytic and binding domain, respectively. In some non-toxigenic strains,
insertions up to 9 <span class=GramE>kb</span> in size were found. <i>In silico</i>
MLST analysis showed that toxinotypes were distributed across all five
toxigenic clades, so far describe for <i>C. difficile</i>. One strain of
toxinotype XXXII (with variant form of PaLoc and atypical PaLoc insertion site)
occupied a distinct branch and was therefore designated as a putative novel
lineage. Strains with atypical insertions replacing the PaLoc were found in
three distinct and highly divergent clades, of which only one was described
previously (C-I) (figure 1).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion</span></b><b
style='mso-bidi-font-weight:normal'><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> Characterization of new <i>C. difficile</i> variants and screening for
changes in toxin coding regions are important in development of molecular
diagnostic tests, vaccines and have also implications in understanding the
evolution of PaLoc variants.<o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='font-size:8.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shapetype
 id="_x0000_t75" coordsize="21600,21600" o:spt="75" o:preferrelative="t"
 path="m@4@5l@4@11@9@11@9@5xe" filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Imagem_x0020_1" o:spid="_x0000_i1025" type="#_x0000_t75"
 alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/77imagem_2560108707_0.jpg"
 style='width:333pt;height:351pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="OP_S6_files/image001.png" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/77imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=353
src="public/abstracts/Webpages/OP_S6_files/image002.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/77imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_1"><![endif]></span><span style='mso-bidi-font-family:
Arial;color:black;mso-themecolor:text1;mso-ansi-language:EN-US'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
color:black;mso-themecolor:text1;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-bidi-font-family:Arial;color:black;
mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP36
- Exploring The Key Factors For The Success Of Clonal Complex 2 (CC2), The Main
<i style='mso-bidi-font-style:normal'>Staphylococcus epidermidis</i> Lineage<span
style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNormal><u><span lang=PT style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
PT;mso-fareast-language:PT'>Diana Espadinha</span></u><sup><span lang=PT
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:PT;mso-fareast-language:
PT'>1</span></sup><span lang=PT style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
PT;mso-fareast-language:PT'>; Rita Sobral<sup>2</sup>; Guillaume Méric<sup>3</sup>;
Samuel Sheppard<sup>3</sup>; Hermínia de Lencastre<sup>4</sup>; Maria Miragaia<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Laboratory of Bacterial Evolution and Molecular Epidemiology, ITQB-UNL; <sup>2</sup>Laboratory
of Molecular Microbiology of Bacterial Pathogens, UCIBIO@REQUIMTE, FCT-UNL; <sup>3</sup>College
of Medicine, Institute of Life Science, Swansea University; <sup>4</sup>Laboratory
of Microbiology and Infectious Diseases, The Rockefeller University<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-bidi-font-family:Arial;color:black;
mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:6.0pt;text-align:left'><b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Background:</span></b><i><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'> Staphylococcus
epidermidis</span></i><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'> is the main colonizer of the human skin and the
most frequent opportunistic pathogen associated to device-related infections.
The most prevalent lineage found in the community and hospital settings is
clonal complex 2 (CC2), a highly genetically diverse lineage. The factors
underlying CC2 success are not well understood yet.<o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:6.0pt;text-align:left'><b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Material/Methods:</span></b><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'> In order to
explore the genetic and phenotypic factors responsible for the success of CC2
lineage, 83 <i>S. epidermidis</i> clinical strains representative of the
population from the hospital and community settings were sequenced using HiSeq.
Whole genome sequencing (WGS) data were searched for virulence and
antimicrobial resistance genes by BLAST analysis. Strains were also
characterized for phenotypic traits such as the ability to form biofilm, growth
in acidic and osmotic stress conditions and resistance to </span><span lang=PT
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:PT;mso-fareast-language:PT'>β</span><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>-lactams (oxacillin).<o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:6.0pt;text-align:left'><b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Results:</span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'> We observed that in a range of pH conditions
(4.5-7.4) and high osmotic stress (2 M NaCl) the growth rates were similar for
CC2 and non-CC2 strains. However, non-CC2 strains showed a longer lag phase and
did not reach optical density values as high as CC2-strains at the stationary
phase. Moreover, whereas the CC2 group presented a variable ability to form
biofilm, none of the non-CC2 strains produced biofilm. No differences were
observed regarding resistance to oxacillin. Additionally, WGS analysis showed
that some hypothetical proteins were exclusively found within CC2 and non-CC2
groups. Genes involved in the formation of biofilm, adhesion and colonization
were significantly more represented in CC2 strains, whereas genes involved in
anaerobiosis and resistance to oxidative stress were exclusively found among
non-CC2 strains.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusions:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> Our results suggest that CC2 strains have a higher fitness than non-CC2
strains under conditions of biological stress and that their success may rely
on specific genetic determinants related to biofilm formation, adhesion and
colonization.<o:p></o:p></span></p>

<span lang=EN-GB style='font-size:8.0pt;line-height:115%;font-family:Arial;
mso-fareast-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
mso-fareast-language:PT;mso-bidi-language:AR-SA'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='font-size:8.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP37
- Insertion Sequences Play A Key Role In The Phenotypic And Genotypic
Plasticity Of The Most Frequent <i style='mso-bidi-font-style:normal'>Staphylococcus
haemolyticus</i> Clonal Lineage<span style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
PT;mso-fareast-language:PT'>Ons Bouchami<sup>1</sup>; Herminia de Lencastre<sup>2</sup>;
<u>Maria Miragaia</u><sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Laboratory of Molecular Genetics, Instituto de Tecnologia Química e
Biológica (ITQB) António Xavier, Oeiras, Portugal and Laboratory of Bacterial
Evolution and Molecular Epidemiology; <sup>2</sup>Laboratory of Molecular
Genetics, Instituto de Tecnologia Química e Biológica (ITQB) António Xavier,
Oeiras, Portugal and Laboratory of Microbiology and Infectious Diseases, The
Rockefeller University, New York, New York, United States of America<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='font-size:8.0pt;mso-ansi-language:
EN-GB;mso-fareast-language:PT;mso-bidi-font-weight:bold'>Keywords: </span><span
lang=EN-GB style='font-size:8.0pt;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Methicillin-resistance, Insertion sequences, Evolution.<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> <i>Staphylococcus haemolyticus</i> is one of the most important nosocomial
human pathogens frequently isolated in bloodstream and medical devices-related
infections. This species is notorious for its multidrug resistance and genome
plasticity, which is believed to result from the multiple insertion sequences
(IS) elements contained in its chromosome. However, the impact of IS in the
evolution and adaptation of <i>S. haemolyticus</i> is not clear. This study was
undertaken to understand how IS modulate chromosomal and phenotypic diversity
in <i>S. haemolyticus</i>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Material and
Methods: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>With this purpose, we analyzed a strain
belonging to the most epidemic <i>S. haemolyticus</i> clonal type (ST3) for
genetic and phenotypic stability after serial passage <i>in vitro</i> (n=34
days). Cultures were characterized by PFGE and Southern hybridization of SmaI
patterns with a probe for IS<i>1272</i> and also for growth, oxacillin and
cefoxitin susceptibility, hemolysis, mannitol fermentation and biofilm
production. To assess variability within the cell population, five colonies at
7 time points during stability assays (n=35 colonies) were analyzed for
phenotypic characteristics and whole genome sequencing (WGS) using MiSeq.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> A high instability was observed both in SmaI PFGE and IS<i>1272</i>
hybridization patterns during serial passage <i>in vitro</i>. Most of the
changes in the SmaI-IS<i>1272</i> patterns paralleled those observed in
SmaI-PFGE profiles and alterations in mannitol fermentation, hemolysis, <span
class=GramE>biofilm</span> formation were also identified. Phenotypic and
genotypic variants were detected within individual colonies taken from the same
population at the same time point. Analysis of single colony WGS data showed
the occurrence of 11 alterations in IS insertion sites. Alterations were
associated with IS<i>1272</i> loss and/or deletion of the regions flanking IS<i>1272</i>
elements. Variant IS sites were in the vicinity of genes related to respiratory
chain, DNA replication, mRNA degradation, iron sequestration, resistance to
nitrosative stress and metal and aminoacid transport. Also, alteration in the
number of viable cells and population growth rates were observed from day to
day.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> Our findings suggest that the high genetic and phenotypic variability
observed in the most epidemic <i>S. haemolyticus</i> clonal type appears to be
promoted by IS transposition. Overall, our results suggest that <i>S.
haemolyticus</i> populations are composed of subpopulations of transposition
variants that might be affected in their growth, gene expression level, stress
resistance, nutrient availability and virulence. The maintenance of
subpopulations of cells in different physiological states might be a strategy
to adapt rapidly to environmental stresses imposed by host or hospital
environment.<o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
color:black;mso-themecolor:text1;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-bidi-font-family:Arial;color:black;
mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP38
- Novel Lineage Of Non-European CC80 Community-Acquired MRSA<span
style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNoSpacing style='text-align:justify;text-justify:inter-ideograph'><span
style='font-size:10.0pt;font-family:Arial;color:black;mso-ansi-language:EN-US'>Sofie
Edslev<sup>1; </sup>Paal Skytt Andersen<sup>1</sup>; Henrik Westh<sup>2</sup>;
Nobumichi Kobayashi<sup>3</sup>; Mette Bartels<sup>2</sup>; Peder Worming<sup>2</sup>;
Robert Skov<sup>1</sup>; Anders Rhod Larsen<sup>1</sup>; <u>Marc Stegger</u><sup>1<o:p></o:p></sup></span></p>

<p class=MsoNoSpacing style='text-align:justify;text-justify:inter-ideograph'><i
style='mso-bidi-font-style:normal'><sup><span style='font-size:10.0pt;
font-family:Arial;color:black;mso-ansi-language:EN-US'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='font-size:10.0pt;font-family:
Arial;color:black;mso-ansi-language:EN-US'>Statens Serum Institut; <sup>2</sup>Hvidovre
Hospital; <sup>3</sup>Sapporo Medical University School of Medicine</span></i><i
style='mso-bidi-font-style:normal'><span lang=EN-GB style='font-family:Arial;
mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=EN-GB style='mso-bidi-font-family:Arial;
color:black;mso-themecolor:text1;mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Keywords:
CA-MRSA, Evolution, <span class=GramE>SCC<i>mec</i></span><i><o:p></o:p></i></span></p>

<p class=MsoNormal><span lang=EN-GB style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:6.0pt;text-align:left'><b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Background: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>We recently reconstructed the evolutionary
history of the PVL-positive European community-acquired <i>Staphylococcus
aureus</i> clonal complex (CC) 80 lineage (Stegger <i>et al</i>. mBio 2014).
This revealed a Sub-Saharan origin with distinct genetic features associated
with the endemic CA-MRSA clone, and with indications that selective use of
clinical antimicrobials had driven the expansion across Europe, the Middle East
and Northern Africa. However, a recent discovery of PVL-negative, fusidic
acid-susceptible MRSA isolates in Bangladesh led us to investigate the genetic
features of these isolates and their presence outside this geographic region.<o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:6.0pt;text-align:left'><b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Materials | Methods: </span></b><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>This study
comprises 231 CC80 isolates; 97 isolates (23 MSSA and 74 MRSA) from an
international collection (Stegger <i>et al</i>.); three MRSA isolates from
Bangladesh (Paul <i>et al</i>. Microb Drug Resist, 2014); and 131 Danish
isolates that include all PVL-negative CC80 MRSA isolates identified in Denmark,
and all identified CC80 MRSA isolates from the greater capital region of
Denmark from 2013 onwards (Bartels <i>et al</i>. Euro Surveill, 2015). All
isolates were whole-genome sequenced using Illumina, and their phylogenetic
relationships were inferred using SNPs identified in the core genome. Genetic
typing of the SCC<i>mec</i> element and encoding of resistance and virulence
genes (<i>lukS/F</i>, <i>mecA</i>, <i>fusB</i> and <span class=GramE><i>tet</i>(</span><i>K</i>)),
was performed using CLCbio’s Genomics Workbench.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Phylogenetic analyses revealed the existence of three distinct clades in
the CC80 complex, including the previously identified basal clade encompassing
sub-Saharan African isolates and the derived clade that as a whole define the
European CA-MRSA clone. The new 3<sup>rd</sup> <span class=GramE>clade,
containing 20 MRSA isolates, include</span> the three Bangladesh isolates and
17 Danish isolates. All isolates in this new unrelated clade carried an SCC<i>mec</i>
type IVa element, contrary to the type IVc found exclusively in the European
CA-MRSA <span class=GramE>clone.No</span> isolates in the new cluster harboured
the PVL-encoding genes <i>lukS </i>and <i>lukF</i>, but all carried </span><span
lang=PT style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:PT;mso-fareast-language:
PT'>Φ</span><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>Sa2 remnants. All isolates in the novel clade
lacked the fusidic acid resistance determinant <i>fusB</i>, whereas four
isolates carried pT181-like plasmids encoding <span class=GramE><i>tet(</i></span><i>K)</i>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Our investigation provides new insight into the emergence and spread of
CA-MRSA. We find that CC80 MRSA isolates are more diverse than previously
thought, and include a novel PVL-negative, fusidic acid-susceptible sublineage
with a distinct SCC<i>mec</i> type IVa element. All isolates in this cluster
carry remnants of the PVL-encoding prophage, indicating that the ancestor of
the CC80 lineage originally carried the prophage. Investigation of the Danish
isolates, showed this clone to be present from 2009 onwards, with &gt;10% of
the isolates as part of this novel sublineage. Preliminary analyses of the
epidemiological information of the patients indicate younger age, skin and
soft-tissue infections and primarily largely related to import from Asia. Combined,
these new data show that two distinct sublineages of CC80 MRSA evolved
simultaneously; i) the European CA-MRSA from a subpopulation originating in
sub-Saharan Africa, and ii) a secondary unrelated acquisition of the SCC<i>mec</i>
element into a PVL-negative CC80 variant unrelated to the sub-Saharan clone.</span><span
lang=EN-GB style='mso-bidi-font-family:Arial;color:black;mso-themecolor:text1;
mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=DE-CH><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
