<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns:mv="http://macVmlSchemaUri" xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta name=Title content="">
<meta name=Keywords content="">
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List href="OP_S5_files/filelist.xml">
<link rel=Edit-Time-Data href="OP_S5_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Bruno Gonçalves</o:Author>
  <o:LastAuthor>Bruno Gonçalves</o:LastAuthor>
  <o:Revision>1</o:Revision>
  <o:TotalTime>0</o:TotalTime>
  <o:Created>2016-03-08T21:42:00Z</o:Created>
  <o:LastSaved>2016-03-08T21:42:00Z</o:LastSaved>
  <o:Pages>6</o:Pages>
  <o:Words>3309</o:Words>
  <o:Characters>18863</o:Characters>
  <o:Company>IMM</o:Company>
  <o:Lines>157</o:Lines>
  <o:Paragraphs>44</o:Paragraphs>
  <o:CharactersWithSpaces>22128</o:CharactersWithSpaces>
  <o:Version>14.0</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="OP_S5_files/themedata.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>JA</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
   <w:UseFELayout/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="276">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1073743103 0 0 415 0;}
@font-face
	{font-family:MingLiU;
	mso-font-alt:細明體;
	mso-font-charset:136;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-1610611969 684719354 22 0 1048577 0;}
@font-face
	{font-family:"Lucida Grande";
	panose-1:2 11 6 0 4 5 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-520090897 1342218751 0 0 447 0;}
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Balloon Text Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:9.0pt;
	font-family:"Lucida Grande";
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-ansi-language:DE-CH;}
p.MsoNoSpacing, li.MsoNoSpacing, div.MsoNoSpacing
	{mso-style-priority:1;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Balloon Text";
	mso-ansi-font-size:9.0pt;
	mso-bidi-font-size:9.0pt;
	font-family:"Lucida Grande";
	mso-ascii-font-family:"Lucida Grande";
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:"Lucida Grande";
	mso-bidi-font-family:"Lucida Grande";
	mso-ansi-language:DE-CH;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"ＭＳ 明朝";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
@page WordSection1
	{size:595.0pt 842.0pt;
	margin:72.0pt 89.85pt 72.0pt 89.85pt;
	mso-header-margin:35.45pt;
	mso-footer-margin:35.45pt;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;}
</style>
<![endif]-->
</head>

<body bgcolor=white lang=EN-US style='tab-interval:36.0pt'>

<div class=WordSection1>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP21
- Molecular Divergence Of Vancomycin-Resistant <i style='mso-bidi-font-style:
normal'>Enterococcus faecium</i> Isolated In Belgium And Denmark Dissected By
Whole Genome Mapping<span style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNormal><u><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Jasmine Coppens</span></u><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>; Basil Britto Xavier<sup>1</sup>; Katherine
Loens<sup>1</sup>; Christine Lammens<sup>1</sup>; Lotte Jakobsen<sup>2</sup>;
Anette M. Hammerum<sup>2</sup>; Herman Goossens<sup>1</sup>; Surbhi
Malhotra-Kumar<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Department of Medical Microbiology, Vaccine &amp; Infectious Disease
Institute, Universiteit Antwerpen; <sup>2</sup>Department of Microbiological
Surveillance and Research, Statens Serum Institut<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-bidi-font-family:Arial;color:black;
mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:6.0pt;text-align:left'><b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Introduction</span></b><b style='mso-bidi-font-weight:normal'><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>:</span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'> Vancomycin-resistant <i>E. faecium</i> (VRE) is
a prominent cause of nosocomial outbreaks worldwide. Molecular strain typing
plays a crucial role in outbreak investigations and hospital epidemiological
surveillance. Here we report, for the first time, a whole genome mapping (WGM)
based-typing approach to study VRE isolated from two different geographic areas
in Europe, Belgium (BE, Belgian National Reference Centre for invasive Group A
Streptococci, n=13) and Denmark (DK, Skejby Aarhus University Hospital, n=14).<o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:6.0pt;text-align:left'><b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Methods</span></b><b style='mso-bidi-font-weight:normal'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> Twenty-seven VRE strains isolated during 2009-2013 from blood (n=8),
catheter (n=2), urine (n= 8), wound (n=2), throat (n=2), faeces (n=1),
abdominal secretions (n=1), bronchial/tracheal secretions (n=1), and from other
undetermined sites (n=2) were typed by MLST, PFGE and WGM. Restriction analysis
for PFGE was performed using <i>SmaI</i> using a 7-band difference cut-off.WGMs
were generated using <i>NcoI</i> restriction and the Argus® system (Opgen,
Gaithersburg, USA), and data were analysed using Bionumerics (Applied Maths
NV).</span><span lang=EN-GB style='font-size:8.0pt;font-family:MingLiU;
mso-bidi-font-family:MingLiU;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'><o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:6.0pt;text-align:left'><b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Results</span></b><b style='mso-bidi-font-weight:normal'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> According to MLST, all typed <i>E. faecium</i> belonged to CC17 and were
divided into 5 ST types. All BE and one DK (217) strains were ST203. The other
13 DK strains were ST18 (n=6), ST192 (n=3), ST117 (n=3) and ST260 (n=1) (Table
1). All BE strains (ST203) were grouped into 1 PFGE type further subdivided
into 2 subtypes, while DK strains were divided into 8 PFGE types and 10
subtypes (Table 1). Twelve of the 14 ST203 strains formed one WGM cluster (95%
similarity co-efficient), while the 2 singleton strains (BE-U0706 and VRE217)
showed only 78.3% similarity to the major cluster (Fig 1a). Using similar
criteria, WGMs of the 13 DK strains were divided into 2 clusters and 7
singletons (Fig 1b). Application of the adjusted Wallace index showed that both
WGM and PFGE types were strongly predictive of MLST types (Wallace co-efficient
of 1.00 for both methods). The Simpson’s diversity index showed that
discriminatory power of WGM (0.871, 95% CI: 0.758-0.984) was slightly higher
than that of PFGE (0.843, 95% CI: 0.716-0.971).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion</span></b><b
style='mso-bidi-font-weight:normal'><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> This study revealed high genomic heterogeneity among the typed CC17<i>
E.faecium</i> clinical isolates. Notwithstanding the obvious advantage of WGM
in allowing comparisons to whole genome sequenced isolates to detect deletions/insertions
in the tested strains, our comparison of typing methods showed WGM to be a
superior approach to PFGE for analyzing rapidly evolving clones like the <i>E.
faecium</i> CC17.<o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='font-size:8.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
mso-no-proof:yes'><!--[if gte vml 1]><v:shapetype id="_x0000_t75" coordsize="21600,21600"
 o:spt="75" o:preferrelative="t" path="m@4@5l@4@11@9@11@9@5xe" filled="f"
 stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Imagem_x0020_48" o:spid="_x0000_i1026" type="#_x0000_t75"
 alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/26imagem_2560108707_1.jpg"
 style='width:333pt;height:247pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="OP_S5_files/image001.png" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/26imagem_2560108707_1.jpg"/>
</v:shape><![endif]--><![if !vml]><img width=335 height=249
src="public/abstracts/Webpages/OP_S5_files/image002.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/26imagem_2560108707_1.jpg"
v:shapes="Imagem_x0020_48"><![endif]></span><span lang=EN-GB style='font-size:
8.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
mso-no-proof:yes'><!--[if gte vml 1]><v:shape id="Imagem_x0020_49" o:spid="_x0000_i1025"
 type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/26imagem_2560108707_0.jpg"
 style='width:333pt;height:395pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="OP_S5_files/image003.png" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/26imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img width=335 height=397
src="public/abstracts/Webpages/OP_S5_files/image004.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/26imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_49"><![endif]></span><span lang=EN-GB style='font-size:
8.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p></o:p></span></p>

<span lang=EN-GB style='font-size:8.0pt;line-height:115%;font-family:Arial;
mso-fareast-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
mso-fareast-language:PT;mso-bidi-language:AR-SA'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='font-size:8.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP22
- Mitigating The Effects Of Sequence Data Quality On Strain Typeability:
Towards The Development Of Robust Core Genome MLST (cgMLST) Schemes<span
style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNormal><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Dillon O.R. Barker<sup>1</sup>; Peter
Kruczkiewicz<sup>2</sup>; James E. Thomas<sup>1</sup>; Chad Laing<sup>2</sup>;
Victor P. J. Gannon<sup>2</sup>; <u>Eduardo N. Taboada<sup>2</sup></u><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Department of Biological Sciences, University of Lethbridge; <sup>2</sup>National
Microbiology Laboratory at Lethbridge, Public Health Agency of Canada<o:p></o:p></span></i></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='mso-bidi-font-family:
Arial;color:black;mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background</span></b><b
style='mso-bidi-font-weight:normal'><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> Core Genome Multi-Locus Sequence Typing (cgMLST) is a sequence typing
method in which the core genome, which is conserved and shared by all members
of a taxonomic group, is used in a conceptual extension of Multi-Locus Sequence
Typing (MLST). By expanding from 7-9 loci used in MLST to hundreds or thousands
of loci, cgMLST promises a dramatic increase in discriminatory power. Moreover,
because it incorporates essential features for public health surveillance such
as stable type definitions that can be centralized, curated, and universally
shared, cgMLST represents a viable approach for global genomic epidemiology
efforts. A current problem hindering the development of robust cgMLST schemes
relates to incomplete loci in draft genome assemblies, which is exacerbated as
the number of genomes analyzed increases. Our aim was to examine approaches to mitigate
the effects of data loss on strain typeability in the design cgMLST schemes.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Materials
and Methods: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>A prototype scheme based on 732 core genes was
designed for the foodborne pathogen <i>Campylobacter jejuni</i>, one of the
leading cause of enteritis worldwide. This scheme was used to examine a set of
2,585 draft genome assemblies. Assignable loci were determined; incomplete loci
due to contig truncations were identified and the location of truncations
catalogued. The global and local information content at all loci was analysed
using Shannon Entropy. A subset of genomes with complete data at all loci
(n=1,464) was used to examine the effect of incomplete loci on strain
typeability by introducing varying levels of synthetic contig truncations.
Three different strategies for mitigating the effects of incomplete loci were
evaluated: loci based on optimised partial gene sequences; imputation of
incomplete loci based on linkage disequilibrium; and use of reduced numbers of
target loci.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>The overall levels of incomplete loci observed in the dataset were
approximately 3.5%. Although most genomes did not present contig truncations,
713 genomes (27.6%) had one or more incomplete loci; a significant number of
genomes (n=107; 4.1%) had greater than 1% incomplete loci. Similarly, although
most loci possessed low probability of contig truncation, some loci (n=19;
2.6%) had incomplete data in 1% or more of the genomes analyzed. Such loci can
be utilized in a cgMLST scheme provided that partial sequences avoiding regions
of high truncation probability and optimised for high information content are
used. For most loci with low levels of incomplete data, data imputation based
on linkage disequilibirum and knowledge of allelles at flanking loci could
correctly estimate the allele with above 95% accuracy despite the high
recombinogenic potential of&nbsp;<i>C. jejuni</i>. Moreover, 95% of cgMLST
schemes based on 200-250 randomly selected loci were able to retain most of the
information content captured by the assay based on 732 loci while concomitantly
decreasing exposure to incomplete loci.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusions:
</span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>Our results suggest that empirically observed
levels of incomplete loci in draft WGS data can have a significant impact on
the typeability of strains. However, the application of one or more strategies
tested in this study can be used to mitigate these adverse effect and can help
inform the development of reliable and robust approaches to cgMLST.<o:p></o:p></span></p>

<span lang=EN-GB style='font-size:8.0pt;line-height:115%;font-family:Arial;
mso-fareast-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
mso-fareast-language:PT;mso-bidi-language:AR-SA'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='font-size:8.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP23
- Development Of A Rapid Diagnostic Test For The Identification Of Pathogenic <i
style='mso-bidi-font-style:normal'>Campylobacter jejuni </i>Isolates<span
style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNormal><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Cody Buchanan<sup>1</sup>; Peter Kruczkiewicz<sup>2</sup>;
Steven K. Mutschall<sup>2</sup>; Benjamin Hetman<sup>3</sup>; James E. Thomas<sup>3</sup>;
Victor P. J. Gannon<sup>2</sup>; G. Douglas Inglis<sup>4</sup>; <u>Eduardo N.
Taboada</u><sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Animal Diseases Research Institute, Canadian Food inspection Agency; <sup>2</sup>National
Microbiology Laboratory at Lethbridge, Public Health Agency of Canada; <sup>3</sup>Department
of Biological Sciences, University of Lethbridge; <sup>4</sup>Lethbridge
Research Centre, Agriculture and Agri-Food Canada<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-bidi-font-family:Arial;color:black;
mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:6.0pt;text-align:left'><b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Background: </span></b><i><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Campylobacter
jejuni</span></i><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'> is an important bacterial foodborne pathogen,
with an estimated 400 million cases worldwide each year. In Canada and many
industrialized countries, <i>C. jejuni</i> is the leading cause of
bacterially-incited enteritis, with a yearly per capita incidence as high as
1%. Deducing the epidemiology of campylobacteriosis towards the development of
efficacious prevention and control measures is complicated by the fact that
most cases of campylobacteriosis appear to be sporadic in nature. <i>C. jejuni </i>has
a broad ecological distribution and high prevalence in wild and domesticated
animals, including in most companion and food production animals, and
environmental sources such as water, and soil, complicating efforts in source
tracking and source attribution. Evidence from previous molecular epidemiology
efforts by our group suggests that many genetic lineages are rarely associated
with disease in humans (i.e. “non-clinically associated” or NCA). Moreover,
these data also suggest that a small number of genetic lineages is responsible
for a disproportionate number of human campylobacteriosis cases (i.e.
“clinically associated” or CA). Our aim was to perform a genome-wide association
study to identify genetic markers associated with CA lineages in order to
develop a molecular assay for rapidly identifying strains likely to pose a risk
to public health.<o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:6.0pt;text-align:left'><b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Methods and Materials: </span></b><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>The Canadian
National Comparative Genomic Fingerprinting database, which contains subtyping
data for &gt;20,000 <i>C. jejuini&nbsp;</i>isolates was used to select a set of
166 isolates representative of prominent CA and NCA lineages in Canada for
whole genome sequencing. Comparative genomic analysis of gene content was used
to identify genes with statistically significant differences in carriage
between CA and NCA lineages. Candidate genes &nbsp;were subjected to primary
validation <i>in silico</i> against a panel of over 2,000 publicly-available <i>C.
jejuni </i>genomes. Leading candidate markers were subjected to secondary
validation against a set of 376 isolates, including cohorts of 192 CA and 184
NCA isolates, and tertiary validation against a cohorts of 557 CA and 356 NCA
isolates.<o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:6.0pt;text-align:left'><b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Results: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>From a total of 730 statistically significant
genes identified in the first stage of the analysis, a total of 33 high quality
markers were identified based on primary <i>in silico </i>validation against
the expanded set of <i>C. jejuni</i> genomes. Among these, 11 markers
maintained extreme differences in carriage (p &lt; 10<sup>-20</sup>) between CA
and NCA isolates after tertiary validation. These included operons for iron
acquisition and vitamin B<sub>5 </sub>biosynthesis, and several putative and
hypothetical open reading frames. These markers, when used in combination were
present in 97% versus 46% of strains from CA and NCA lineages, respectively and
a subset of 5 markers has been used to develop of a PCR assay for rapid and
high-throughput identification of potentially pathogenic strains of <i>C.
jejuni</i>.<o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:6.0pt;text-align:left'><b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Conclusions :</span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>Effective mitigation strategies will require
identification of reservoirs with the highest prevalence of strains posing an
increased risk to human health. Employing a comparative genomic approach on CA
(i.e. “pathogenic”) and NCA (i.e. “non-pathogenic”) lineages, we have developed
a PCR-based risk assessment tool for public health authorities to rapidly
identify <i>C. jejuni</i> strains that are potentially pathogenic to the human
population.<o:p></o:p></span></p>

<span lang=EN-GB style='font-size:8.0pt;line-height:115%;font-family:Arial;
mso-fareast-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
mso-fareast-language:PT;mso-bidi-language:AR-SA'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='font-size:8.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP24
- Proteotyping Of <i style='mso-bidi-font-style:normal'>Streptococcus pneumoniae</i>,
Using Tandem Mass Spectrometry For Identification Of Biomarkers For Species And
Strain Differentiation<span style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNormal><u><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Hedvig Engström Jakobsson</span></u><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>; Lucia Gonzales Siles<sup>1</sup>; Roger
Karlsson<sup>2</sup>; Fredrik Boulund<sup>3</sup>; Francisco Salvà-Serra<sup>1</sup>;
Erik Kristiansson<sup>3</sup>; Edward R.B. Moore<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal align=left style='text-align:left'><i style='mso-bidi-font-style:
normal'><sup><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>1</span></sup></i><i style='mso-bidi-font-style:
normal'><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Department of Infectious diseases, Gothenburg University; <sup>2</sup>Department
of Infectious diseases, Gothenburg University and Nanoxis Nanoxis Consulting
AB; <sup>3</sup>Department of Mathematical Statistics, Chalmers University of
Technology<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background: </span></b><i><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Streptococcus pneumoniae</span></i><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'> (pneumococcus) is
the leading cause of community-acquired pneumonia and a major cause of
morbidity and mortality worldwide. <i>S. pneumoniae</i> belongs to the <i>S.
mitis</i>-group (viridans streptococci) and is phenotypically and genotypically
similar to commensal species of the upper respiratory tract of the <i>S. mitis</i>-group
such as <i>S. mitis, S. oralis</i>, and <i>S. pseudopneumoniae</i>. This fact
causes problems of identification in clinical microbiology laboratories. In
this project, we are applying state-of-the-art proteomics techniques for <i>Streptococcus</i>
spp. proteotyping; to detect and characterize expressed protein biomarkers for
species-level identification, determine antibiotic resistance and virulence
factors and perform strain typing for epidemiological analyses.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Material and
methods: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>The proteins of intact bacteria or cell
fractions are bound to a membrane surface, using patented (WO2006068619)
Lipid-based Protein Immobilization (LPITM) technology. Peptides are generated
from the bound proteins using enzymatic digestion and then separated and
analysed using liquid chromatography-tandem mass spectrometry (LC-MS/MS). The
mass spectra profiles are compared to a database of reference peptide
sequences, consisting of all complete genomes of the NCBI Reference Sequence
(RefSeq) Database. In this study, the type strains of the closely-related mitis
complex species <i>S. pneumoniae</i> (CCUG 28588<sup>T</sup>), <i>S. mitis</i>
(CCUG 31611<sup>T</sup>), <i>S. oralis</i> (CCUG 13229<sup>T</sup>), <i>S.
psedopneumoniae</i> (CCUG 49455<sup>T</sup>) and the more distantly-related <i>S.
pyogenes</i> (CCUG 4207<sup>T</sup>) were analysed individually and in
mixtures, to demonstrate the capabilities of proteotyping to differentiate
closely related species. Additionally, mixes containing different <i>S.
pneumoniae</i> strains were analysed.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Using proteotyping protocols, it was possible to detect and correctly
identify <i>S. pneumoniae</i> from the closely related bacterial species, <i>S.
mitis, S. oralis, S. psedopneumoniae</i> and <i>S. pyogenes</i>, as well as
different strains of <i>S. pneumoniae</i> by identification of unique
discriminatory peptides. For successful proteotyping, a comprehensive and
accurate genomic database is the key to obtaining reliable proteotyping data.
Importantly, since there is a high rate of misclassified genomes in the public
databases, the taxonomic classification of the genome sequences should be
verified for accurate identification of biomarkers. In this study, hundreds of
genomes of the <i>S. mitis</i>-group in GenBank were analysed against type
strain genomes by calculating the Average Nucleotide Identity using BLAST
(ANIb). <i>S. pneumoniae</i> were well classified, while almost half the <i>S.
mitis</i> were misclassified. Also, <i>S. mitis</i>-group strains that could
not be identified to the species level, using standard genotypic and phenotypic
approaches, were characterized by proteotyping and whole genome sequencing to
describe their taxonomy and to improve the database matching.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>&nbsp;Proteotyping enabled the differentiation and identification of
pneumococcus from its closely related species as well as sub-species-level
strain discrimination, all from single MS analyses. The whole method will
enhance the identification and characterization of microorganisms, allowing
high-resolution discrimination of closely related species through the confident
identification of new biomarkers, ultimately for cultivation-independent
analyses of clinical samples.<o:p></o:p></span></p>

<span lang=EN-GB style='font-size:8.0pt;line-height:115%;font-family:Arial;
mso-fareast-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
mso-fareast-language:PT;mso-bidi-language:AR-SA'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='font-size:8.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP25
- The Evaluation Of Whole Genome Sequencing For The Epidemiological Typing Of <i
style='mso-bidi-font-style:normal'>Legionella pneumophila</i><span
style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNormal><u><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Sophia David</span></u><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>; Martin Aslett<sup>1</sup>; Rediat Tewolde<sup>2</sup>;
Simon Harris<sup>1</sup>; Massimo Mentasti<sup>2</sup>; Baharak Afshar<sup>2</sup>;
Anthony Underwood<sup>2</sup>; Julian Parkhill<sup>1</sup>; Timothy Harrison<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Wellcome Trust Sanger Institute; <sup>2</sup>Public Health England<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background</span></b><b
style='mso-bidi-font-weight:normal'><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> Human infections with <i>Legionella pneumophila</i> are acquired directly
from a contaminated environmental source. The characterisation of clinical and
epidemiologically linked environmental isolates is crucial for locating the
source and determining the extent of infection. The cost and turn-around time of
whole genome sequencing (WGS) has decreased significantly in recent years
making it a viable tool for outbreak investigations by public health reference
laboratories. We compared a range of WGS-based methods for the epidemiological
typing of <i>L. pneumophila </i>and compared their performance to that of
current typing techniques.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Materials
&amp; Methods: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>A number of methods for analysis of WGS data
were tested including SNP/mapping-based, kmer-based, gene
presence/absence-based, and scaled-up multi-locus sequence typing (MLST)
approaches. Five different MLST schemes were created using 53 ribosomal genes
and 50, 100, 500 and 1455 core genes. These different approaches were tested
using WGS data from the European Society for Clinical Microbiology Study Group
on Legionella Infections (ESGLI) standard typing panel comprising 106 isolates
and 17 sets of epidemiologically related isolates. A further 206 isolates,
comprising five major disease-associated sequence types (STs) as defined by the
current gold standard 7-gene MLST, were used to further test the discriminatory
power of the different methods. A number of isolates were also sequenced
multiple times at different sequencing centres, using different platforms, and
in the same and different sequencing runs, to establish the reproducibility of
the methods.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>All WGS-based methods provide high levels of discrimination when applied to
epidemiologically unrelated isolates from the ESGLI typing panel. When only a
single difference is required for isolates to be considered different “types”,
the WGS-based methods achieved indices of discrimination of between 0.980 and
1. However, the most discriminatory methods also detect differences between
epidemiologically related isolates. For each method, we therefore defined a higher
number of differences permitted for isolates to be considered the same “type”
in order to maintain epidemiological concordance.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>The
WGS-based methods mostly offered high discrimination within some of the common
disease-associated STs (e.g. ST1). However, the clonal nature of some <i>L.
pneumophila</i> strains means that isolates belonging to some STs (e.g. ST47)
remain difficult to differentiate, even using the most discriminatory methods.
Indeed many epidemiologically unrelated isolates are genetically identical and
thus total differentiation between epidemiologically related and unrelated
isolates is not achieved with any method.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Finally, we
show that reproducibility is intrinsically linked to sequence data quality and
that high quality sequence data provides reproducibility scores approaching
100% for all methods.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>WGS-based methods achieve higher discrimination than current typing
techniques and will be particularly useful when investigating outbreaks
involving common strains such as ST1. While the different WGS-based methods
have their own advantages and disadvantages, we conclude that a typing scheme
based on a scaled-up MLST approach offers the best means of achieving an
appropriate balance between discrimination and epidemiological concordance.
Importantly, it is also amenable to standardisation and the easy exchange of
data between laboratories.<o:p></o:p></span></p>

<span lang=EN-GB style='font-size:8.0pt;line-height:115%;font-family:Arial;
mso-fareast-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
mso-fareast-language:PT;mso-bidi-language:AR-SA'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='font-size:8.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP26
- Tracking A <i style='mso-bidi-font-style:normal'>Mycobacterium tuberculosis</i>
MDR Strain In Equatorial Guinea Based On ASO-PCR Targeting Specific SNPs
Obtained From WGS Data<span style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNoSpacing style='text-align:justify;text-justify:inter-ideograph'><span
lang=PT style='font-size:10.0pt;font-family:Arial;mso-fareast-font-family:"Times New Roman";
color:black;mso-fareast-language:PT'>Laura Pérez-Lago<sup>1; </sup>Griselda
Tudó<sup>2</sup>; Iñaki Comas<sup>3</sup>; María Carcelén<sup>1</sup>; Marta
Herranz<sup>1</sup>; Santiago Izco<sup>1</sup>; Maria Jesus Ruiz Serrano<sup>1</sup>;
Juliá Gonzalez<sup>2</sup>; Juan Eyene<sup>4</sup>; Emilio Bouza<sup>1</sup>; <u>Darío
García de Viedma</u><sup>1<o:p></o:p></sup></span></p>

<p class=MsoNoSpacing style='text-align:justify;text-justify:inter-ideograph'><i
style='mso-bidi-font-style:normal'><sup><span lang=PT style='font-size:10.0pt;
font-family:Arial;mso-fareast-font-family:"Times New Roman";color:black;
mso-fareast-language:PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
lang=PT style='font-size:10.0pt;font-family:Arial;mso-fareast-font-family:"Times New Roman";
color:black;mso-fareast-language:PT'>Instituto de Investigación Sanitaria
Gregorio Marañón; <sup>2</sup>Servei de Microbiologia-CDB. Hospital Clínic de
Barcelona-ISGlobal. Universitat de Barcelona; <sup>3</sup>FISABIO; <sup>4</sup>MINISTERIO
DE SANIDAD Y BIENESTAR SOCIAL de Guinea Ecuatorial. Programa Nacional de
Tuberculosis (PNLT-L)<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='mso-bidi-font-size:10.0pt;mso-ansi-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=PT style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
PT;mso-fareast-language:PT'>Funding: Plan Estatal I+D+I 2013-2016, ISCIII
(13/01207; 15/01554) cofunded by the Fondo Europeo de Desarrollo Regional
(FEDER): “Una manera de hacer Europa”. </span><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Fondation
Merieux (DM/CL/cb15). Grant for International Cooperation Projects by IiSGM (I
and II-COOP-INT 2015).Miguel Servet research grant CP15/00075 (to L.P.), Ramón
y Cajal and MINECO research grants (RYC-2012-10627, SAF2013-43521-R) and the
European Research Council (ERC) (638553-TB-ACCELERATE) (to I.C). The Spanish
Network for Research in Infectious Diseases (REIPI, RD12/0015 to GT).<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Background:</span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'> Molecular epidemiology studies in TB are
generally based on the application of the same fingerprinting tool, namely
MIRU-VNTR, to all the <i>Mycobacterium tuberculosis</i> (MTB) isolates in a
population. In some cases, specific PCRs have been developed to track selected
strains, but this was only possible when a specific genetic feature made it
possible to target them. Whole genome sequencing (WGS) enables us to identify
singular features (strain-specific SNPs) for any MTB strain of interest and to
develop allele-specific-oligonucleotide (ASO)-PCRs tailored to detect relevant
strains. We evaluated the efficiency and flexibility of this new strategy to
track prevalent multidrug-resistant (MDR) MTB strains.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Materials and Methods:</span></b><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'> We focused on a
MDR MTB strain (EG-MDR) suspected to be prevalent in Equatorial Guinea (EG).
One isolate from an immigrant case infected with this strain, identified by
MIRU-VNTR, was analyzed by WGS to identify strain-specific SNPs. An allele-specific
PCR (ASO-PCR) was designed to target 4 of these strain-specific SNPs. The
ASO-PCR was prospectively applied on the left-overs of the diagnostic molecular
tests (GenXpert or Genotype MDRplus) performed on the specimens obtained in EG
in the period February-April 2015, which were delivered to our laboratory in
Spain.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Results:</span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'> The ASO-PCR was applied to 161 positive
specimens for MTB and a result was obtained in 139. The EG-MDR strain was
identified in 18 isolates. All the isolates identified as EG-MDR strain by
ASO-PCR were confirmed by MIRU-VNTR analysis and all corresponded to resistant
strains according to the molecular diagnostic tests. The EG-MDR isolates
constituted 51.4% of all the resistant strains detected in EG. To track
retrospectively the presence of the EG-MDR strain in EG, the ASO-PCR was
applied on a sample of stored isolates obtained in EG in 1999-2001, as part a
molecular epidemiology study. The analysis was performed directly on the crude
boiled extracts of 93 strains and the EG-MDR strain was identified in two
cases. Both corresponded to MDR cases, one of them a new case and the other a
previously treated case.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Conclusion:</span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'> PCRs targeting strain-specific SNPs is a novel
strategy for monitoring relevant MTB strains. It is based on low-cost, rapid,
and transferable tests tailored to the challenges of different populations. Its
prospective application allowed to track a MDR strain, which is responsible for
half of the current MDR cases in EG, and to document retrospectively its presence
in the country 15 years ago.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=DE-CH><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
