<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns:mv="http://macVmlSchemaUri" xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta name=Title content="">
<meta name=Keywords content="">
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List href="OP_S2_files/filelist.xml">
<link rel=Edit-Time-Data href="OP_S2_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Bruno Gonçalves</o:Author>
  <o:LastAuthor>Bruno Gonçalves</o:LastAuthor>
  <o:Revision>1</o:Revision>
  <o:TotalTime>0</o:TotalTime>
  <o:Created>2016-03-08T21:38:00Z</o:Created>
  <o:LastSaved>2016-03-08T21:38:00Z</o:LastSaved>
  <o:Pages>6</o:Pages>
  <o:Words>2581</o:Words>
  <o:Characters>14715</o:Characters>
  <o:Company>IMM</o:Company>
  <o:Lines>122</o:Lines>
  <o:Paragraphs>34</o:Paragraphs>
  <o:CharactersWithSpaces>17262</o:CharactersWithSpaces>
  <o:Version>14.0</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="OP_S2_files/themedata.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:GrammarState>Clean</w:GrammarState>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>JA</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
   <w:UseFELayout/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="276">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:"ＭＳ ゴシック";
	mso-font-charset:78;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1791491579 18 0 131231 0;}
@font-face
	{font-family:Verdana;
	panose-1:2 11 6 4 3 5 4 4 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-1593833729 1073750107 16 0 415 0;}
@font-face
	{font-family:Verdana;
	panose-1:2 11 6 4 3 5 4 4 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-1593833729 1073750107 16 0 415 0;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1073743103 0 0 415 0;}
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{mso-style-priority:99;
	mso-style-link:"Footer Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	tab-stops:center 216.0pt right 432.0pt;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;}
p.MsoNoSpacing, li.MsoNoSpacing, div.MsoNoSpacing
	{mso-style-priority:1;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;}
span.FooterChar
	{mso-style-name:"Footer Char";
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:Footer;
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Arial;
	mso-ansi-language:PT;}
p.IMMEMl, li.IMMEMl, div.IMMEMl
	{mso-style-name:IMMEM_l;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"IMMEM_l Caráter";
	margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:1;
	font-size:12.0pt;
	font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	font-weight:bold;}
p.IMMEMr, li.IMMEMr, div.IMMEMr
	{mso-style-name:IMMEM_r;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"IMMEM_r Caráter";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:right;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:1;
	font-size:12.0pt;
	font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	font-weight:bold;}
span.IMMEMlCarter
	{mso-style-name:"IMMEM_l Caráter";
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:IMMEM_l;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Arial;
	mso-bidi-font-family:Arial;
	font-weight:bold;}
span.IMMEMrCarter
	{mso-style-name:"IMMEM_r Caráter";
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:IMMEM_r;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Arial;
	mso-bidi-font-family:Arial;
	font-weight:bold;}
span.GramE
	{mso-style-name:"";
	mso-gram-e:yes;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"ＭＳ 明朝";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
@page WordSection1
	{size:420.0pt 595.0pt;
	margin:42.55pt 1.0cm 1.0cm 1.0cm;
	mso-header-margin:14.2pt;
	mso-footer-margin:14.2pt;
	mso-gutter-margin:14.2pt;
	mso-title-page:yes;
	mso-even-header:url(":OP_S2_files:header.htm") eh1;
	mso-header:url(":OP_S2_files:header.htm") h1;
	mso-footer:url(":OP_S2_files:header.htm") f1;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
@page WordSection2
	{size:595.0pt 842.0pt;
	margin:72.0pt 89.85pt 72.0pt 89.85pt;
	mso-header-margin:35.45pt;
	mso-footer-margin:35.45pt;
	mso-even-header:url(":OP_S2_files:header.htm") eh1;
	mso-header:url(":OP_S2_files:header.htm") h1;
	mso-footer:url(":OP_S2_files:header.htm") f1;
	mso-paper-source:0;}
div.WordSection2
	{page:WordSection2;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;}
</style>
<![endif]-->
</head>

<body bgcolor=white lang=EN-US style='tab-interval:36.0pt'>

<div class=WordSection1>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'>OP07 - </span></b><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Resolving
Genotype-Phenotype Maps Of Opportunistic Pathogenicity In <i style='mso-bidi-font-style:
normal'>Staphylococcus epidermidis</i> To Identify Functional Markers Of
Infection</span></b><span style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;text-transform:uppercase;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><u><span lang=PT style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
PT;mso-fareast-language:PT'>Guillaume Méric</span></u><sup><span lang=PT
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:PT;mso-fareast-language:
PT'>1</span></sup><span lang=PT style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
PT;mso-fareast-language:PT'>; Leonardos Mageiros<sup>1</sup>; Samuel K.
Sheppard<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Swansea University Medical School<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Effective
treatment of nosocomial infections is difficult, as most severe infections can
be caused by antibiotic-resistant isolates from bacterial species naturally
colonising humans asymptomatically. Under certain circumstances, these
commensal organisms can become opportunistic pathogens. An example is <i>Staphylococcus
epidermidis</i>, which lives commensally on human skin and mucous membranes.
Although <i>S. epidermidis</i> is asymptomatically carried, hospitalised
patients are often at risk of developing an infection, typically through the
contamination of catheters or other surgical implants, and the mechanisms
involved in the emergence of pathogenicity in this organism are currently not
fully understood. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> We present and
interpret results of a comparative analysis on 415 whole-genome sequences from
genetically diverse <i>S. epidermidis</i> from clinical infections as well as
asymptomatic isolates. Whole-genome alignments were obtained using a
gene-by-gene approach and robust phylogenetic trees were constructed. Groups of
genetically related isolates where identified to create a suitable dataset to
be analysed using a <span class=GramE>recently-developed</span> pangenome-wide
association study (pan-GWAS) pipeline combined with <i>in vitro </i>quantitative
phenotyping (immune response in blood and skin cells, biofilm formation,
methicillin resistance and cell toxicity).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> A list of
candidate genetic elements statistically associated with opportunistic
pathogenicity was obtained. The association signal was then filtered by
correlating each element with quantitative scores for phenotypes tested <i>in
vitro</i> to identify <span class=GramE>phenotypically-filtered</span> genetic
markers of infection.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> In this study, we
present an original and novel approach to identify and filter functionally
genetic elements associated with complex ecological lifestyles (opportunistic
pathogenicity). The association signal obtained <i>in silico</i> is then
functionally correlated with <i>in vitro </i>quantitative pathogenicity assays.
This work provides a clearer understanding of the evolutionary mechanisms
underlying the emergence of opportunistic pathogenicity in <i>S. epidermidis</i>,
as well as a novel and powerful method to resolve genotype-phenotype maps in
bacteria.<o:p></o:p></span></p>

</div>

<span style='font-size:8.0pt;font-family:Verdana;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-US;
mso-fareast-language:PT;mso-bidi-language:AR-SA'><br clear=all
style='page-break-before:always;mso-break-type:section-break'>
</span>

<div class=WordSection2><br clear=all style='mso-special-character:line-break;
page-break-before:always'>

<p class=MsoNormal style='margin-bottom:10.0pt;line-height:115%'><span
style='font-size:8.0pt;line-height:115%;font-family:Verdana;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:"Times New Roman";color:black;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'>OP08 - </span></b><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Multiomic
Approach To Delineate Intraclonal Diversity Of CG14 And CG15 <i
style='mso-bidi-font-style:normal'>Klebsiella pneumoniae</i></span></b><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=DE-CH style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT'>Carla Rodrigues<sup>1</sup>; Val F Lanza<sup>2</sup>; Clara Sousa<sup>3</sup>;
Elisabete Machado<sup>4</sup>; João A Lopes<sup>5</sup>; Fernando Baquero<sup>2</sup>;
Luísa Peixe<sup>1</sup>; Ângela Novais<sup>1</sup>; Teresa M Coque<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><sup><span lang=DE-CH style='mso-bidi-font-size:9.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-fareast-language:PT'>1</span></sup><span lang=DE-CH
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>REQUIMTE@UCIBIO,
Faculdade de Farmácia da Universidade do Porto; <i style='mso-bidi-font-style:
normal'><sup>2</sup>Hospital Universitario Ramón y Cajal and Instituto Ramón y
Cajal de Investigación Sanitaria (IRYCIS)/CIBERESP; <sup>3</sup>Center of
Biological Engineering, University of Minho; <sup>4</sup>REQUIMTE@UCIBIO,
Faculdade de Farmácia da Universidade do Porto/FP-ENAS/CEBIMED. Faculdade de
Ciências da Saúde, Universidade Fernando Pessoa; <sup>5</sup>iMed, Departamento
de Farmácia Galénica e Tecnologia Farmacêutica, Faculdade de Farmácia,
Universidade de Lisboa<o:p></o:p></i></span></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Project supported by a FEMS Research Grant.<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Population
structure of <i>Klebsiella pneumoniae</i> (Kp) cannot be accurately delineated
by traditional analysis of MLST data. Recent genome wide genotyping identified
14 clonal groups (CG), with MDR clones largely represented by CG258, CG14 and
CG15. Whereas CG258 has been extensively studied, only a few isolates of CG14
and CG15 have been comprehensively analysed at genome level. Hospital and
community acquired infections caused by Kp ST14 and ST15 isolates are frequent
in Portugal since 2003, and their characterization by molecular genotyping
(PFGE, MLST, <i>wzi</i>-based capsule typing) and spectroscopic methods
(Fourier transform infrared spectroscopy-FTIR) unveiled congruently the
circulation of 1 ST14 and 4 ST15 clones exhibiting specific capsular types. We
aimed to delineate the intraclonal diversity of CG14 and CG15 by comparing the
genomes from representative circulating clones in Portugal and those available
in public databases. Moreover, a&nbsp;comparative analysis of&nbsp;genomics
(NGS),&nbsp;proteomics (MALDI-TOF MS)&nbsp;and metabolomics (FTIR) was&nbsp;performed
to&nbsp;elucidate differential resolution potential for
typing&nbsp;of&nbsp;Kp&nbsp;CG14 and CG15.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials|Methods:
</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Whole-genome sequencing (WGS) of 9 Kp isolates with different capsular
types ST15 (5 K24, 1 K19, 1 K60, 1 <i>wzi89</i>) and ST14 (1 K16) from
different origins and variable PFGE-types were obtained using llumina MiSeq
(2x300bp pair-ended runs, ~6Gb/genome, coverage<span class=GramE>:100x</span>).
WGS of ST15 (n=21; USA/Italy/Nepal/Taiwan; 2008-14) and ST14 (n=13;
USA/Argentina/Korea/South Africa; 1986-2014) isolates available on NCBI (as
from November 2015, all from human infections) were included in the analysis.
Assembly was performed using SPAdes and the core genome (&gt;90% similarity and
coverage) of Kp isolates was defined using home Perl scripts. Core genes were
concatenated and aligned using MUSCLE, and SNPs extraction was performed with
Harvest tools. The results were output in a SNP matrix, which was used to
generate a phylogenetic tree using the maximum likelihood method (100
bootstraps) in R 3.2.2 (Phangorn package). Spectra (obtained from colony and/or
bacterial extracts) were acquired in MALDI-TOF MS and FTIR-ATR, and analysed by
PCA/PLSDA using MATLAB 8.3 (PLS toolbox 7.5).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>
Phylogenetic analysis of a 30832 core genome SNPs identified 5 clusters among
ST15 and 2 clusters among ST14 isolates, each one of them comprising isolates
from different origins that harbour the same capsular type (K19, K24, K60, <i>wzi</i>89
or <i>wzi</i>118 for ST15, and K2 or K16 for ST14) (Fig.1). Phylogenetic
relationships among isolates inferred from this tree corroborate previous
clonal assignments based on molecular genotyping and FTIR (Fig.2). Within each
cluster, a high diversity in ESBL and/or carbapenemase content was observed,
suggesting the ability of these lineages to acquire a variable pool of MDR
plasmids. In contrast with genomics- or metabolomics-based methods, ST15 or
ST14 subpopulations were not discriminated by MALDI-TOF, probably reflecting
the high degree of conservation of their corresponding ribosomal proteins.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> WGS and
metabolomics are reliable tools to delineate the intraclonal diversity of Kp
CG14 and CG15, whereas current MLST scheme or MALDI-TOF <span class=GramE>are</span>
not suitable for an accurate differentiation. Within these groups, particular
widespread subpopulations are linked to specific capsular types, probably
reflecting variable host adaptation or pathogenicity potential.<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shapetype
 id="_x0000_t75" coordsize="21600,21600" o:spt="75" o:preferrelative="t"
 path="m@4@5l@4@11@9@11@9@5xe" filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Imagem_x0020_37" o:spid="_x0000_i1026" type="#_x0000_t75"
 alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/144imagem_2560108707_0.jpg"
 style='width:333pt;height:221pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="OP_S2_files/image001.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/144imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img width=335 height=223
src="public/abstracts/Webpages/OP_S2_files/image002.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/144imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_37"><![endif]></span><span lang=PT style='mso-bidi-font-size:
10.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
text-transform:uppercase;mso-ansi-language:PT;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Imagem_x0020_36" o:spid="_x0000_i1025" type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/144imagem_2560108707_1.jpg"
 style='width:291pt;height:163pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="OP_S2_files/image003.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/144imagem_2560108707_1.jpg"
  croptop="20707f" cropbottom="18911f"/>
</v:shape><![endif]--><![if !vml]><img width=293 height=165
src="public/abstracts/Webpages/OP_S2_files/image004.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/144imagem_2560108707_1.jpg"
v:shapes="Imagem_x0020_36"><![endif]></span><b style='mso-bidi-font-weight:
normal'><span style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></b></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
line-height:115%;font-family:Arial;mso-fareast-font-family:"Times New Roman";
text-transform:uppercase;mso-ansi-language:EN-US;mso-fareast-language:PT;
mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal style='margin-bottom:10.0pt;line-height:115%'><b
style='mso-bidi-font-weight:normal'><span style='mso-bidi-font-size:10.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;text-transform:uppercase;mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'>OP09 - </span></b><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Evolution
Of The Skin Commensal <i style='mso-bidi-font-style:normal'>Staphylococcus
epidermidis</i> In The Hospital Environment: Insights From An Early Collection</span></b><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><u><span lang=DE-CH style='mso-bidi-font-size:9.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-fareast-language:PT'>Joana Rolo</span></u><sup><span
lang=DE-CH style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>1</span></sup><span
lang=DE-CH style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>; Rita Sobral<sup>2</sup>;
Jesper Boye Nielsen<sup>3</sup>; Peder Worning<sup>3</sup>; Rory Bowden<sup>4</sup>;
Samuel Sheppard<sup>5</sup>; Guillaume Méric<sup>5</sup>; Robert Skov<sup>6</sup>;
Henrik Westh<sup>3</sup>; Hermínia de Lencastre<sup>7</sup>; Maria Miragaia<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=DE-CH
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=DE-CH style='mso-bidi-font-size:
9.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-fareast-language:PT'>Laboratory of Molecular Genetics,
Instituto de Tecnologia Química e Biológica António Xavier; Laboratory of
Bacterial Evolution and Molecular Epidemiology, Instituto de Tecnologia Química
e Biológica António Xavier; <sup>2</sup>Laboratory of Molecular Genetics,
Instituto de Tecnologia Química e Biológica António Xavier ; Laboratory of
Molecular Microbiology of Bacterial Pathogens, UCIBIO-REQUIMTE, Faculdade de
Ciências e Tecnologia - Universidade Nova de Lisboa; <sup>3</sup>Department of
Clinical Microbiology, Hvidovre University Hospital; <sup>4</sup>Wellcome Trust
Centre for Human Genetics, University of Oxford; <sup>5</sup>College of
Medicine, Swansea University, Institute of Life Science; MRC CLIMB Consortium,
Swansea University, Institute of Life Science; <sup>6</sup>Statens Serum
Institut; <sup>7</sup>Laboratory of Molecular Genetics, Instituto de Tecnologia
Química e Biológica António Xavier; Laboratory of Microbiology and Infectious
Diseases, The Rockefeller University<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=DE-CH style='mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> One of the
strategies of adaptation of <i>Staphylococcus epidermidis</i> to the hospital
environment has been related with the acquisition and diversification of the
staphylococcal cassette chromosome <i>mec</i> (SCC<i>mec</i>) that carries <i>mecA</i>,
the genetic determinant of resistance to methicillin. However, it is not known
when SCC<i>mec</i> was first introduced in the genome of <i>S. epidermidis </i>and
what additional evolutionary mechanisms could have been associated to the
adaptation of these bacteria to the nosocomial setting and to a more pathogenic
lifestyle. In this study, we aimed to identify the molecular events that
allowed for the recent emergence of <i>S. epidermidis</i> as a pathogen,
through the study of its genomic evolutionary history.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials&amp;Methods:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> We compared
the whole genomes of 21 early <i>S. epidermidis</i> strains isolated in a
hospital in Denmark in 1965 with the genomes of 14 contemporary nosocomial <i>S.
epidermidis </i>collected in the same country in 1997/1998. Genomes were sequenced
using HiSeq technology and reads were assembled <i>de novo</i> with VELVET.
Multilocus sequence types were extracted from the contigs. In addition, the
core genome of all strains was predicted with Mugsy and aligned with Mauve.
Finally, screening of virulence genes was performed by BLAST analysis and the
presence of intact prophages, insertion sequences and antibiotic resistance
genes in the contigs was assessed using online databases of specific bacterial
sequences.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> We found
that the great majority of early (67%) and contemporary (79%) strains belonged
to the main <i>S. epidermidis</i> clonal lineage, clonal complex 2 (CC2). In
addition, our analysis revealed that the core genome of early and contemporary
isolates was conserved but their mobilome showed distinctive features. Early
isolates carried genes associated to resistance to several different
antibiotics commonly used in clinical practice in the 1960s; some were
extremely prevalent (<i>blaZ, </i>67%, <i>fosA</i>, 90%; <i>norA</i>, 100%) and
others were found in a low frequency (<i>aaD</i>, 9.5%; <i>ermA</i>, 9.5%, <i>fusB</i>,
9.5%; <i>cat</i>, 14.3%, <span class=GramE><i>tet(</i></span><i>K)</i>, 14.3%).
Of note, a single isolate carried the <i>mecA</i> gene, disrupted by IS<i>431</i>,
within <span class=GramE>a</span> SCC<i>mec</i> IV-like element. In addition,
we found that early strains carried biofilm-associated factors, such as <span
class=GramE><i>ica</i></span> (5%), <i>bhlp </i>(28.5%) and <i>aap</i> (38%)
and the arginine catabolic mobile element, ACME (types I and II equally
represented, 38%). <span class=GramE>Moreover, a high frequency of intact
prophages (95%) and as many as 10 different insertion sequences (IS) were found
in early genomes.</span> In contrast, resistance determinants (including SCC<i>mec
</i>IV, 57%), biofilm-associated genes (<i>ica</i>, 21%; <i>aap</i>, 64%) and
ACME-I (57%) were found in higher frequency among contemporary isolates.
Moreover, additional genes associated to biofilm production (<i>bap</i>, 7%; <i>sdrF</i>,
14%) and IS, presumably originated in other bacterial species, were found in
contemporary isolates. However, the frequency of prophages decreased
dramatically (to 50%) in this population.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Our results
suggest that the accumulation of genetic determinants associated to antibiotic
resistance, colonization, biofilm formation and genome plasticity as well as
phage loss have contributed to the adaptation of <i>S. epidermidis</i> to the
hospital environment and to its emergence as a major nosocomial pathogen. Also,
our data suggest that SCC<i>mec</i> IV acquisition and spread in <i>S.
epidermidis</i> population was a relatively recent event (&lt;50 years).</span><b
style='mso-bidi-font-weight:normal'><span style='mso-bidi-font-size:10.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
text-transform:uppercase;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></b></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
line-height:115%;font-family:Arial;mso-fareast-font-family:"Times New Roman";
text-transform:uppercase;mso-ansi-language:EN-US;mso-fareast-language:PT;
mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal style='margin-bottom:10.0pt;line-height:115%'><b
style='mso-bidi-font-weight:normal'><span style='mso-bidi-font-size:10.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;text-transform:uppercase;mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'>OP10 - </span></b><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Evolution
And Adaptation Of <i style='mso-bidi-font-style:normal'>Pseudomonas aeruginosa</i>
Within Patients With Cystic Fibrosis<span style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNormal><u><span style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Rasmus Lykke Marvig</span></u><sup><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup><span style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Rigshospitalet, Copenhagen University Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Bacterial whole
genome sequencing of longitudinally collected isolates enables the
investigation of evolutionary trajectories, which may inform both the
prevention and treatment of human-associated pathogen infections.&nbsp;Chronic
infection with Pseudomonas aeruginosa is a major cause of morbidity and
mortality in cystic fibrosis (CF) patients, and a more complete understanding
of P. aeruginosa within-host genomic evolution, transmission, and population
genomics may provide a basis for improving intervention strategies.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>By genome
sequencing of &gt;600 P. aeruginosa isolates sampled over 40 years from 50
patients, we elucidated the within-host evolution of clonal lineages in each
individual patient. Many of the identified mutations were located in
pathoadaptive genes associated with host adaptation, and we correlated
mutations with changes in CF-relevant phenotypes such as antibiotic
resistance.&nbsp;Considerable genome reduction was detected, suggesting that
host adaptation is characterized by the reduction of genomic repertoire rather
than acquisition of novel functions.&nbsp;Furthermore<span class=GramE>,the</span>
genomic analysis revealed transmission of P. aerguinosa between
patients,&nbsp;and we show that patient-to-patient transmission may be
facilitated by temporal overlap in the respective patients´ visits to the
hospital.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Comparison of the
genomes of multiple isolates from the same patients showed extensive
within-patient genomic diversification; the populations were composed of
different sub-lineages that had coexisted for many years since the initial
colonization of the patient&nbsp;and occupied distinct niches. Thus, our
results suggest that the spatial heterogeneity in CF airways plays a major role
in relation to the generation and maintenance of population diversity and
emphasize that a single isolate in sputum may not represent the entire pathogen
population in the infected individual.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Although there are
other studies encompassing several hundreds of whole genome sequenced isolates
of bacterial pathogens, our study is unique in its focus on an opportunistic
pathogen causing long-term infections.&nbsp;In addition, whereas many other
studies are focusing on single clonal lineages or complexes of a pathogenic
species, our collection compares the within-host evolution of genotypically
different strains of the same species. This is an important issue, as mutations
or genes associated with pathogenicity in one bacterial strain&nbsp;might not
be predictive of pathogenicity in other strains.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>In conclusion, we
present the largest collection of whole genome sequenced P. aeruginosa isolated
from CF patients, and together with both phenotypic and clinical information
this dataset facilitates a more detailed understanding of P. aeruginosa
within-host genomic evolution, transmission, and population genomics.<o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
line-height:115%;font-family:Arial;mso-fareast-font-family:"Times New Roman";
text-transform:uppercase;mso-ansi-language:EN-US;mso-fareast-language:PT;
mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal style='margin-bottom:10.0pt;line-height:115%'><b
style='mso-bidi-font-weight:normal'><span style='mso-bidi-font-size:10.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;text-transform:uppercase;mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'>OP11 - </span></b><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Genomic
Diversity Of Carbapenemase-Producing <i style='mso-bidi-font-style:normal'>Klebsiella
pneumoniae</i> In Germany<span style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNormal><u><span lang=DE-CH style='mso-bidi-font-size:10.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-fareast-language:PT'>Laura Becker</span></u><sup><span
lang=DE-CH style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>1</span></sup><span
lang=DE-CH style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>; Matthias
Steglich<sup>1</sup>; Yvonne Pfeifer<sup>1</sup>; Martin Kaase<sup>2</sup>;
Stephan Fuchs<sup>1</sup>; Ulrich Nübel<sup>3</sup>; Guido Werner<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNoSpacing style='text-align:justify;text-justify:inter-ideograph'><i
style='mso-bidi-font-style:normal'><sup><span lang=EN-GB style='font-size:10.0pt;
font-family:Arial;mso-fareast-font-family:"Times New Roman";color:black;
mso-ansi-language:EN-GB;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=EN-GB style='font-size:10.0pt;
font-family:Arial;mso-fareast-font-family:"Times New Roman";color:black;
mso-ansi-language:EN-GB;mso-fareast-language:PT'>Robert Koch Institute, FG13
Nosocomial Pathogens and Antibiotic Resistance; <sup>2</sup>National Reference
Laboratory for multidrug-resistant gram-negative bacteria, Department for
Medical Microbiology, Ruhr-University Bochum; <sup>3</sup>Leibniz Institute
DSMZ<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Background: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Carbapenem-resistant <i>Klebsiella pneumoniae</i>
are dreaded nosocomial pathogens. In Germany, the carbapenemases KPC-2, KPC-3,
OXA-48 and NDM-1 display the highest prevalence in carbapenem-resistant <i>K.
pneumoniae</i>. As a basis for future next generation sequencing (NGS)-based
outbreak analyses and the elucidation of dynamics of epidemic strains, we
analysed the genomic diversity of carbapenemase-producing <i>K. pneumoniae</i>
occurring in German hospitals. In cooperation with the National Reference
Laboratory for multidrug-resistant gram-negative bacteria, a representative
collection of more than 100 carbapenem-resistant isolates from both outbreaks
and single cases which had been collected between 2009 and 2014 allover Germany
was included in our analysis.<b style='mso-bidi-font-weight:normal'><o:p></o:p></b></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Material/Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>All isolates were characterised previously by
antimicrobial susceptibility testing and by determination of the carbapenemase
gene. For whole genome sequencing, bacterial DNA was extracted using the DNeasy
Blood &amp; Tissue Kit (Qiagen). Sequencing libraries were prepared using the
Nextera XT DNA Sample Preparation Kit. Sequencing was performed on a Miseq
machine (Illumina). Sequencing reads were trimmed (Trimmomatic) and mapped
(BWA-SW) onto a reference genome. Based on single nucleotide polymorphisms (SNPs)
a maximum likelihood tree was constructed. Further, whole genome sequence data
were used to determine the content of virulence and resistance genes.<b
style='mso-bidi-font-weight:normal'><o:p></o:p></b></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Whole genome sequence analysis revealed the
presence of four major clusters, representing several carbapenem-resistant <i>K.
pneumoniae </i>multilocus sequence types (ST) known to be distributed
internationally: KPC-3-producing ST512, KPC-2-/KPC-3-producing ST258,
OXA-48-producing ST101 and OXA-48-producing ST147. Genomic diversity within
these groups was extremely low, reflecting the clonal spread of distinct
carbapenem-resistant <i>K. pneumoniae </i>strains in Germany.<b
style='mso-bidi-font-weight:normal'><o:p></o:p></b></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusion: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Our baseline genomic sequence dataset provides a
comprehensive insight in the diversity of carbapenemase-producing <i>K.
pneumoniae </i>in Germany and will enable rapid identification and phylogenetic
clustering of emerging strains.<o:p></o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal style='margin-bottom:10.0pt;line-height:115%'><span
style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'>SC02 - </span></b><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Genomics
Of Campylobacter Adaptation To The Host And Man-Made Environments</span></b><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><u><span lang=DE-CH style='mso-bidi-font-size:9.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-fareast-language:PT'>Samuel K Sheppard</span></u><sup><span
lang=DE-CH style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>1,2,3</span></sup><span
lang=DE-CH style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='background:white'><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-bidi-font-size:9.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US'>Department of Zoology, University of
Oxford, Oxford, UK; <sup>2</sup>College of Medicine, Institute of Life Science,
Swansea University, Swansea, UK; <sup>3</sup>MRC CLIMB Consortium, Institute of
Life Science, Swansea University, Swansea, UK.<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=DE-CH><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
