<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns:mv="http://macVmlSchemaUri" xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta name=Title content="">
<meta name=Keywords content="">
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List href="OP_S9_files/filelist.xml">
<link rel=Edit-Time-Data href="OP_S9_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Bruno Gonçalves</o:Author>
  <o:LastAuthor>Bruno Gonçalves</o:LastAuthor>
  <o:Revision>1</o:Revision>
  <o:TotalTime>1</o:TotalTime>
  <o:Created>2016-03-08T21:46:00Z</o:Created>
  <o:LastSaved>2016-03-08T21:47:00Z</o:LastSaved>
  <o:Pages>7</o:Pages>
  <o:Words>2112</o:Words>
  <o:Characters>12040</o:Characters>
  <o:Company>IMM</o:Company>
  <o:Lines>100</o:Lines>
  <o:Paragraphs>28</o:Paragraphs>
  <o:CharactersWithSpaces>14124</o:CharactersWithSpaces>
  <o:Version>14.0</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="OP_S9_files/themedata.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:GrammarState>Clean</w:GrammarState>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>JA</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
   <w:UseFELayout/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="276">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1073743103 0 0 415 0;}
@font-face
	{font-family:"Lucida Grande";
	panose-1:2 11 6 0 4 5 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-520090897 1342218751 0 0 447 0;}
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	mso-themecolor:hyperlink;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:purple;
	mso-themecolor:followedhyperlink;
	text-decoration:underline;
	text-underline:single;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Balloon Text Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:9.0pt;
	font-family:"Lucida Grande";
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-ansi-language:DE-CH;}
p.MsoNoSpacing, li.MsoNoSpacing, div.MsoNoSpacing
	{mso-style-priority:1;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Balloon Text";
	mso-ansi-font-size:9.0pt;
	mso-bidi-font-size:9.0pt;
	font-family:"Lucida Grande";
	mso-ascii-font-family:"Lucida Grande";
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:"Lucida Grande";
	mso-bidi-font-family:"Lucida Grande";
	mso-ansi-language:DE-CH;}
span.GramE
	{mso-style-name:"";
	mso-gram-e:yes;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"ＭＳ 明朝";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
@page WordSection1
	{size:595.0pt 842.0pt;
	margin:72.0pt 89.85pt 72.0pt 89.85pt;
	mso-header-margin:35.45pt;
	mso-footer-margin:35.45pt;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;}
</style>
<![endif]-->
</head>

<body bgcolor=white lang=EN-US link=blue vlink=purple style='tab-interval:36.0pt'>

<div class=WordSection1>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>SC07
- Whole-Genome Linkage Analysis Reveals Co-Evolutionary Mechanisms Of
Antibiotic Resistance In The Pneumococcus<span style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNormal><u><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Jukka Corander</span></u><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;text-transform:uppercase;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Department of Mathematics and Statistics, University of Helsinki<o:p></o:p></span></i></p>

<p class=MsoNormal align=left style='text-align:left'><b style='mso-bidi-font-weight:
normal'><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal align=left style='text-align:left'><b style='mso-bidi-font-weight:
normal'><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:9.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:
AR-SA'><br clear=all style='mso-special-character:line-break;page-break-before:
always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:9.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP04
- How To Compare And Cluster Every Known Genome In About An Hour<span
style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNormal><u><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Adam M. Phillippy</span></u><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>; Brian D. Ondov<sup>2</sup>; Todd J. Treangen<sup>2</sup>;
Sergey Koren<sup>1</sup><br>
<i style='mso-bidi-font-style:normal'><sup>1</sup>National Human Genome
Research Institute, National Institutes of Health; <sup>2</sup>National
Biodefense Analysis and Countermeasures Center</i></span><i style='mso-bidi-font-style:
normal'><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;text-transform:uppercase;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>The rapid growth of genomic data has begun to outpace traditional methods
for sequence clustering and search. Given a massive collection of sequences, it
is infeasible to perform pairwise alignment for basic tasks like sequence
clustering and search. To address this problem, we demonstrate that the MinHash
technique, first applied to clustering web pages, can be applied to biological
sequences with similar effect, and extend this idea to include biologically
relevant distance and significance measures. Our new tool, Mash, uses
MinHashing to reduce large sequences to a representative sketch and rapidly
estimate pairwise distances between genomes or metagenomes. Using Mash, we
explored several use cases, including a ~7,000-fold size reduction and
clustering of all ~55,000 NCBI RefSeq genomes in 46 CPU hours. The resulting 93
MB sketch database includes all RefSeq genomes, effectively delineates known
species boundaries, reconstructs approximate phylogenies, and can be searched
in seconds using assembled genomes or raw sequencing runs from Illumina,
Pacific Biosciences, and Oxford Nanopore. For metagenomics, Mash scales to
thousands of samples and can replicate Human Microbiome Project and Global
Ocean Survey results in a fraction of the time. Other potential applications
include any problem where an approximate, global sequence distance is
acceptable, e.g. to triage and cluster outbreak genomes, assign species labels,
identify mis-tracked samples, and rapidly search massive genomic databases. In
addition, the Mash distance metric is based on simple set intersections, which
are compatible with privacy-preserving tests via homomorphic encryption
schemes. To facilitate integration with other software, Mash is implemented as
a lightweight C++ toolkit and freely released under a BSD license at </span><span
lang=DE-CH><a href="https://github.com/marbl/mash"><span lang=EN-US
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:windowtext;mso-ansi-language:EN-US;mso-fareast-language:PT;
text-decoration:none;text-underline:none'>https://github.com/marbl/mash</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>.<o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:9.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:
AR-SA'><br clear=all style='mso-special-character:line-break;page-break-before:
always'>
</span></b>

<p class=MsoNormal style='margin-bottom:10.0pt;line-height:115%'><b
style='mso-bidi-font-weight:normal'><span style='mso-bidi-font-size:9.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP45
- Whole Genome Sequence Analysis Suggests That A Remarkable Spread Of Anthrax
Occurred In France During The Hundred Years War With England<span
style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNoSpacing style='text-align:justify;text-justify:inter-ideograph'><u><span
lang=FR style='font-size:10.0pt;font-family:Arial;color:black;mso-ansi-language:
FR'>Gilles Vergnaud</span></u><sup><span lang=FR style='font-size:10.0pt;
font-family:Arial;color:black;mso-ansi-language:FR'>1; </span></sup><span
lang=FR style='font-size:10.0pt;font-family:Arial;color:black;mso-ansi-language:
FR'>Guillaume Girault<sup>2</sup>; Simon Thierry<sup>2</sup>; Christine Pourcel<sup>1</sup>;
Nora Madani<sup>2</sup>; Yann Blouin<sup>3</sup></span><u><span lang=FR
style='font-size:10.0pt;font-family:Arial;mso-fareast-font-family:"Times New Roman";
color:black;mso-ansi-language:FR;mso-fareast-language:PT'><o:p></o:p></span></u></p>

<p class=MsoNoSpacing style='text-align:justify;text-justify:inter-ideograph'><i
style='mso-bidi-font-style:normal'><sup><span style='font-size:10.0pt;
font-family:Arial;color:black;mso-ansi-language:EN-US'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='font-size:10.0pt;font-family:
Arial;color:black;mso-ansi-language:EN-US'>University Paris-Saclay; <sup>2</sup>ANSES,
University Paris-Est; <sup>3</sup>DGA</span></i><i style='mso-bidi-font-style:
normal'><u><span lang=FR style='font-size:10.0pt;font-family:Arial;mso-fareast-font-family:
"Times New Roman";color:black;mso-ansi-language:FR;mso-fareast-language:PT'><o:p></o:p></span></u></i></p>

<p class=MsoNormal><span lang=FR style='mso-bidi-font-size:10.0pt;color:black;
mso-themecolor:text1;mso-ansi-language:FR'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background: </span></b><i><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Bacillus anthracis</span></i><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>, the highly
dangerous zoonotic bacterial pathogen species responsible for anthrax is
naturally present in many places worldwide. In France, previous investigations
have demonstrated that the majority of sporadic strains belong to the so-called
A.Br.011/009 lineage. Whole genome sequencing of more than 120 french strains
further demonstrated that the A.Br.011/009 strains defined a very remarkable
polytomy, i.e. a star-shape pattern of evolution with six branches. Here we
explore the significance of this polytomy by comparing the French <i>B.
anthracis</i> lineages to worldwide lineages.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Materials
and methods: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>We take advantage of whole genome sequence data
previously determined for 122 French strains and included publicly available
sequence data from 45 strains of various origins. <span class=GramE>Single
nucleotide polymorphisms (SNPs) were identified by mapping each genome on the
Ames strain genome sequence</span>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>A total of 6690 SNPs was identified among the available dataset and used to
draw the phylogeny. The data clearly demonstrates that the currently
predominant <i>B. anthracis</i> lineage in North America, called WNA for
Western North American, and another lineage frequently encountered in former
French colonies in West Africa are both derived from one of the six branches of
the A.Br.011/009 polytomy predominant in France. The position of the split
along the French lineage, together with the consideration of French history can
then be used to provide a tentative dating for the most recent common ancestor
(MRCA) of the A.Br.011/009 polytomy. The Hundred Years period, 1350-1450,
appears to be the most likely candidate. However, the wide range of observed
substitution rates along the different <i>B. anthracis</i> lineages illustrated
by the present data, with ratios up to forty, complicates such dating attempts.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>The present work extends the range of observed substitution rate
heterogeneity within <i>B. anthracis</i>, in agreement with its ecology and in
contrast with some other pathogens. The population structure of A.Br.011/009 in
France, diversity of branch length, and comparison with two derived lineages,
suggests that the MRCA of the six branches polytomy identified in France dates
back to the Hundred Years´ war between France and England started in the
mid-fourteenth century. These events were associated in France with deadly
epidemics and major economic and social changes. This suggestion will need to
be confirmed by sequencing additional A.Br.011/009 of diverse origins.<o:p></o:p></span></p>

<span lang=EN-GB style='font-size:8.0pt;line-height:115%;font-family:Arial;
mso-fareast-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
mso-fareast-language:PT;mso-bidi-language:AR-SA'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='font-size:8.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP46
- MLST And CGF40 For Characterization Of <i style='mso-bidi-font-style:normal'>Campylobacter
jejuni</i> Isolates From The Poultry Chain Production And From Clinical Cases
In France<span style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNoSpacing style='text-align:justify;text-justify:inter-ideograph'><u><span
lang=FR style='font-size:10.0pt;font-family:Arial;color:black;mso-ansi-language:
FR'>Katell Rivoal</span></u><sup><span lang=FR style='font-size:10.0pt;
font-family:Arial;color:black;mso-ansi-language:FR'>1; </span></sup><span
lang=FR style='font-size:10.0pt;font-family:Arial;color:black;mso-ansi-language:
FR'>Amandine Thépault<sup>1</sup>; Valérie Rose<sup>1</sup>; Ségolène Quesne<sup>1</sup>;
Francis Mégraud<sup>2</sup>; Marianne Chemaly<sup>1<o:p></o:p></sup></span></p>

<p class=MsoNoSpacing style='text-align:justify;text-justify:inter-ideograph'><i
style='mso-bidi-font-style:normal'><sup><span style='font-size:10.0pt;
font-family:Arial;color:black;mso-ansi-language:EN-US'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='font-size:10.0pt;font-family:
Arial;color:black;mso-ansi-language:EN-US'>Anses; <sup>2</sup>CNR
Campylobacters and Hélicobacters</span></i><i style='mso-bidi-font-style:normal'><span
lang=FR style='font-size:10.0pt;font-family:Arial;color:black;mso-ansi-language:
FR'><o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=FR style='mso-bidi-font-size:10.0pt;color:black;
mso-themecolor:text1;mso-ansi-language:FR'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Intestinal campylobacteriosis is the leading bacterial zoonosis in Europe
and worldwide. The main causative agent of this foodborne acute gastroenteritis
is <i>Campylobacter jejuni</i>. <i>C. <span class=GramE>jejuni</span></i>
contaminated poultry meat and meat products are considered the most important
sources of disease in humans. In France, <span class=GramE>76% of chicken meat
at the retail level is contaminated by <i>Campylobacter</i></span>. The aim of
this study is to trace by Multilocus Sequence Typing (MLST) and comparative
genomic fingerprinting (CGF) using a 40-gene assay (CGF40) <i>C. jejuni</i>
strains circulating along the French poultry chain production from farms to
retail stores and to determine a link with human cases. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Materials
and Methods: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>Isolates from farms (176) and slaughterhouses
(295) were collected during a survey representative of the French broiler
production (Rivoal et al., 2011; Guyard-Nicodème et al., 2015). Retail meat
isolates (174) were collected in 10 French departments representing the most
significant consumption patterns over one year. Human isolates (143) were
collected during the same year from patients living in the same areas in order
to study the overlap between broiler and human isolates. All the 788 isolates
were typed using both MLST and CGF40 according to Taboada et al. </span><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>(2012). </span><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>For the CGF40, isolates were categorized into types based on more than 90%
CGF40 fingerprint similarity (CGF-90%). The results were analyzed using
Bionumerics software and the genetic diversity of the different strain
populations was evaluated using the Simpson Index of Similarity (ID).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>It appears that the CGF40 presents a good congruence with MLST and a higher
discriminating power. A high genetic diversity was observed with 90 different
CGF-90% types and 82 Clonal Complex (CC) among the 788 isolates typed,
corresponding to indices of 0.957 and 0.925 respectively. Within the clinical
isolates, a lower diversity in <i>C. jejuni</i> population was observed with
indices of 0.909 (CGF40) and 0.843 (MLST). Nevertheless, almost 50% of human
isolates were divided into few CGF40 types or CC all found in chicken isolates.
These results confirmed the strong link between poultry and campylobacteriosis
(Sheppard et al., 2009; Strachan et al. 2012; Abay et al., 2014). Nevertheless
some isolates highly prevalent among the poultry isolates were not isolated
from clinical isolates supporting thus the hypothesis of different virulent
potential between poultry isolates (Schouls et al., 2003; Guyard-Nicodème et
al., 2015). On the other hand, 9% and 6% of <i>C. jejuni</i> isolates issued
from human campylobacteriosis cases found respectively by CGF40 and MLST were
not present in chicken population supporting in that case the implication of
other sources of campylobacteriosis.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>This work reveals the great diversity among <i>C. jejuni</i> isolates and
the implication of the poultry meat production in human contamination. However,
other animal reservoirs must be investigated to elucidate the part not
attributed to poultry.<o:p></o:p></span></p>

<span lang=EN-GB style='font-size:8.0pt;line-height:115%;font-family:Arial;
mso-fareast-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
mso-fareast-language:PT;mso-bidi-language:AR-SA'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='font-size:8.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP47
- TAXONOMER, An Integrated Ultra-Fast Tool For Metagenomic Sequence Analysis<span
style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNoSpacing style='text-align:justify;text-justify:inter-ideograph'><u><span
style='font-size:10.0pt;font-family:Arial;color:black;mso-ansi-language:EN-US'>Robert
Schlaberg</span></u><sup><span style='font-size:10.0pt;font-family:Arial;
color:black;mso-ansi-language:EN-US'>1; </span></sup><span style='font-size:
10.0pt;font-family:Arial;color:black;mso-ansi-language:EN-US'>Guochun Liao<sup>2</sup>;
Stephen Flygare<sup>2</sup>; Evan Buss<sup>2</sup>; Keith Simmon<sup>3</sup>;
Chase Miller<sup>1</sup>; Yi Qiao<sup>1</sup>; EJ Osborne<sup>1</sup>; Karen
Eilbeck<sup>1</sup>; Gabor Marth<sup>1</sup>; Mark Yandell<sup>1</sup></span><u><span
style='font-size:10.0pt;font-family:Arial;mso-fareast-font-family:"Times New Roman";
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></u></p>

<p class=MsoNoSpacing style='text-align:justify;text-justify:inter-ideograph'><i
style='mso-bidi-font-style:normal'><sup><span style='font-size:10.0pt;
font-family:Arial;color:black;mso-ansi-language:EN-US'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='font-size:10.0pt;font-family:
Arial;color:black;mso-ansi-language:EN-US'>University of Utah; <sup>2</sup>DbyDNA
Inc.; <sup>3</sup>Institute for Clinical and Experimental Pathology,</span></i><span
style='font-size:10.0pt;font-family:Arial;color:black;mso-ansi-language:EN-US'>
<i style='mso-bidi-font-style:normal'>ARUP Laboratories<o:p></o:p></i></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='mso-bidi-font-size:
10.0pt;color:black;mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Metagenomics,
the genomic analysis of a population of microorganisms, makes possible the
profiling of microbial communities in the environment and the human body at
unprecedented depth and breadth. Enrichment-independent, high-throughput
sequencing allows for unbiased, hypothesis-free detection and molecular typing
of a theoretically unlimited number of common and unusual pathogens. Rapid,
accurate, and user-friendly data analysis is limiting adoption.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Taxonomer is
an integrated, ultra-fast tool for metagenomic sequence analysis. Taxonomer
enables novel analysis modalities of unmatched complexity in an easy-to-use
format including: (1) comprehensive panmicrobial detection and discovery, (2)
host-response profiling, (3) interactive result visualization, and (4) access
through an IOBIO-based web user interface, which eliminates the need for
specialized hardware or expertise.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Taxonomer
computes at speeds comparable to the fastest existing search tools, but
supports both nucleotide and protein-based classification using a single
integrated algorithmic framework using the same k-mer weighting-based approach
and classification algorithm. The result is greater tolerance for sequencing
errors, greater sensitivity, more accurate microbial abundance estimates, and
execution times exceeding the fastest published protein search tools. This
speed and breath of functionality is crucial, as many samples contain complex
mixtures of bacterial, fungal and viral taxa.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Here we
present analyses of patient samples harboring viral, bacterial, and fungal
pathogens, including pathogens&nbsp;of great public health
concern,&nbsp;demonstrating that Taxonomer provides effective means for rapid
pathogen detection for patient care and discovery in public health emergencies.
As costs and turn-around times for high-throughput sequencing continue to fall,
Taxonomer will enable a rapidly growing number of laboratories with access to
sequencing instruments to analyze data in a meaningful timeframe without having
to invest in computational infrastructure or bioinformatics expertise.<o:p></o:p></span></p>

<span lang=EN-GB style='font-size:8.0pt;line-height:115%;font-family:Arial;
mso-fareast-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
mso-fareast-language:PT;mso-bidi-language:AR-SA'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='font-size:8.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>OP48
- Tracking Resistance Determinants Encoded Within The Chromosomes Of <i
style='mso-bidi-font-style:normal'>Pasteurellaceae</i> Species<span
style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNormal><u><span style='mso-bidi-font-size:9.0pt;mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US'>Stephen Douthwaite</span></u><sup><span
style='mso-bidi-font-size:9.0pt;mso-bidi-font-family:Arial;color:black;
mso-ansi-language:EN-US'>1; </span></sup><span style='mso-bidi-font-size:9.0pt;
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US'>Benoit
Desmolaize<sup>1</sup>; Anders S. Olsen<sup>1</sup>; Michal Beker<sup>1</sup>;
Simon Rose<sup>1</sup></span><u><span style='mso-bidi-font-size:9.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></u></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-bidi-font-family:Arial;color:black;
mso-ansi-language:EN-US'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-bidi-font-family:Arial;color:black;
mso-ansi-language:EN-US'>University of Southern Denmark<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Backgound</span></b><b
style='mso-bidi-font-weight:normal'><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>The bacterial pathogens <i>Mannheimia haemolytica</i> and <i>Pasteurella
multocida</i> cause respiratory disease in a range of hosts including poultry,
swine and cattle. Treatment with one of several groups of antibiotics is
generally effective, although therapeutic regimens become complicated in cases
where the pathogens have acquired resistance (1-4). Tracking the occurrence of
resistance in field isolates together with the mechanisms of resistance and
their means of dissemination are important not only for the farming industry
but also for human health.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Materials
&amp; Methods:</span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'> Independent field isolates of <i>P. multocida</i>
and <i>M. haemolytica </i>were screened for resistance to a range of
antibiotics. Over 40 resistant strains were analyzed by a combination of next
generation sequencing and multiplex PCR approaches. The occurrence and
composition of integrative conjugative elements (ICEs) were determined.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> One or more of the resistance determinants <span class=GramE><i>erm(</i></span><i>42),
msr(E) </i>and<i> mph(E) </i>were found in the genomes of over 40 strains and
conferred resistance to different sets of macrolide antibiotics. The macrolide
resistance genes were confined to ICEs within the <i>M. haemolytica </i>and <i>P.
multocida</i> chromosomes, and were often associated with genes conferring
resistance to aminoglycosides and less frequently with genes conferring
resistance to sulfonamides, chloramphenicol, tetracyclines and beta-lactams. In
a subset of strains that were highly-resistant to macrolides, ICE sequences
were absent, and in these cases, macrolide resistance was shown to be conferred
by mutations in all six copies of the <i>rrn</i> operons encoded 23S rRNA.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion</span></b><b
style='mso-bidi-font-weight:normal'><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Resistance to macrolides, sulfonamides, chloramphenicol, tetracyclines and
beta-lactams was associated with chromosomally encoded ICEs sequences, and the
composition of ICEs varies greatly between isolates. Macrolide resistance can
additionally arise from 23S rRNA mutations (Figure) in the absence of an ICE.
The data have been used to design more comprehensive multiplex PCR tools for
screening for resistance determinants in the <i>Pasteurellaceae</i> including
the human pathogen <i>Haemophilus influenzae</i>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>References:</span></b><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'> 1. Desmolaize
B <i>et al.</i> (2011) <i>Mol. Microbiol.</i> <b>80:</b> 184-94; 2. Desmolaize
B <i>et al.</i> (2011) <i>Antimicrob. Agents Chemother.</i> <b>55:</b> 4128-33;
3. Michael GB <i>et al.</i> (2012) <i>J. Antimicrob. Chemother.</i> <b>67:</b>
84-90; 4. Olsen AS <i>et al</i>. (2015). </span><i><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>J.</span></i><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> <i>Antimicrob. Chemother. </i><span class=GramE><b>70:</b> 420-423.</span><o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='font-size:8.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shapetype id="_x0000_t75"
 coordsize="21600,21600" o:spt="75" o:preferrelative="t" path="m@4@5l@4@11@9@11@9@5xe"
 filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Imagem_x0020_6" o:spid="_x0000_i1025" type="#_x0000_t75"
 alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/195imagem_2560108707_0.jpg"
 style='width:333pt;height:197pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="OP_S9_files/image001.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/195imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=199
src="public/abstracts/Webpages/OP_S9_files/image002.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/195imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_6"><![endif]></span><span style='mso-bidi-font-size:
10.0pt;color:black;mso-themecolor:text1;mso-ansi-language:EN-US'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT;mso-bidi-font-weight:
bold'>Figure legend:</span><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'> The <span class=GramE>macrolide binding</span>
site in the bacterial ribosome&nbsp;(grey)&nbsp;showing the orientations of the
16-membered ring compounds tylosin (yellow) and tildipirosin (green).
Resistance to both drugs is conferred in <i>P. multocida</i> and <i>M.
haemolytica </i>by mutations at the 23S rRNA nucleotides A2058 or A2059 (4), or
by methylation at A2058 (red sphere) by <span class=GramE>Erm(</span>42) (1).<o:p></o:p></span></p>

<span lang=EN-GB style='font-size:8.0pt;line-height:115%;font-family:Arial;
mso-fareast-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
mso-fareast-language:PT;mso-bidi-language:AR-SA'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='font-size:8.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-bidi-font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>SC07
- Whole-Genome Linkage Analysis Reveals Co-Evolutionary Mechanisms Of
Antibiotic Resistance In The Pneumococcus<span style='text-transform:uppercase'><o:p></o:p></span></span></b></p>

<p class=MsoNormal><u><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Jukka Corander</span></u><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;text-transform:uppercase;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Department of Mathematics and Statistics, University of Helsinki<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=DE-CH><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
