<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns:mv="http://macVmlSchemaUri" xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta name=Title content="">
<meta name=Keywords content="">
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List href="PO_S1_files/filelist.xml">
<link rel=Edit-Time-Data href="PO_S1_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Bruno Gonçalves</o:Author>
  <o:LastAuthor>Bruno Gonçalves</o:LastAuthor>
  <o:Revision>1</o:Revision>
  <o:TotalTime>0</o:TotalTime>
  <o:Created>2016-03-08T21:48:00Z</o:Created>
  <o:LastSaved>2016-03-08T21:48:00Z</o:LastSaved>
  <o:Pages>23</o:Pages>
  <o:Words>9889</o:Words>
  <o:Characters>56369</o:Characters>
  <o:Company>IMM</o:Company>
  <o:Lines>469</o:Lines>
  <o:Paragraphs>132</o:Paragraphs>
  <o:CharactersWithSpaces>66126</o:CharactersWithSpaces>
  <o:Version>14.0</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="PO_S1_files/themedata.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>JA</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
   <w:UseFELayout/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="276">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-520092929 1073786111 9 0 415 0;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1073743103 0 0 415 0;}
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	mso-themecolor:hyperlink;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:purple;
	mso-themecolor:followedhyperlink;
	text-decoration:underline;
	text-underline:single;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"ＭＳ 明朝";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
@page WordSection1
	{size:595.0pt 842.0pt;
	margin:72.0pt 89.85pt 72.0pt 89.85pt;
	mso-header-margin:35.45pt;
	mso-footer-margin:35.45pt;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;}
</style>
<![endif]-->
</head>

<body bgcolor=white lang=EN-US link=blue vlink=purple style='tab-interval:36.0pt'>

<div class=WordSection1>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO1 - Oligonucleotide-based
Fast Clustering Of Bacterial Genomes For Epidemiology And Phylogenetics<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Rosario San Millan<sup>1</sup>; Ilargi Martinez-Ballesteros<sup>2</sup>;
Lorena Laorden<sup>2</sup>; Javier Garaizar<sup>2</sup>; Joseba Bikandi<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University of the Basque Country (EHU/UPV). Faculty of
Medicine and Odontology, <sup>2</sup>University of the Basque Country
(EHU/UPV). Faculty of Pharmacy<o:p></o:p></span></i></p>

<p class=MsoNormal align=left style='margin-bottom:6.0pt;text-align:left'><span
lang=EN-GB style='mso-bidi-font-family:Arial;color:black;mso-themecolor:text1;
mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Comparing
the oligonucleotide frequencies of genomes to find phylogenetics relationships
has a long history. In most cases, very short oligonucleotides, often
tetranucleotides, have been researched. Similar frequencies were observed in
phylogenetically-related genomes, such as members of the same species or
family. In this work, we selected some types of frequencies and distances from
the literature to compare oligonucleotide frequencies. We then studied their
behavior in different situations. We wanted to know whether oligonucleotides
over four bases long could be used to cluster together phylogenetically-related
genomes, and whether the clustering method could be applied to members of the
same species for epidemiology. <o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:6.0pt;text-align:left'><b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials |
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Sequences and other related files were obtained from NCBI or&nbsp;<i>Ensembl
Genomes</i>. Tools to assess the best comparison algorithm were&nbsp;developed
with PHP running on an Apache server with a Linux operating system. An online
service was also developed that uses C programs for some computationally
intensive tasks.&nbsp;The basic code used in the service is provided. <o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:6.0pt;text-align:left'><b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Different
statistical methods were searched to determine which would yield the best
clustering results for oligonucleotides. The best results were obtained by
comparing longer oligonucleotides (octanucleotides) and hamming distance. The
selected procedure was applied to several complete sequenced genomes from&nbsp;<i>Ensembl
Genomes</i>.&nbsp;The proposed method was able cluster together
phylogenetically-related genomes. In addition, epidemiologically-related
genomes (as, for example, members of the same&nbsp;<i>Salmonella</i>&nbsp;or&nbsp;<i>E.
coli</i>&nbsp;serotypes) clustered together. <o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:6.0pt;text-align:left'><b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Usage of
oligonucleotide frequencies for fast comparison and clustering of complete
bacterial genomes has great potential for both phylogenetics and epidemiology.
This method may be used for many species.&nbsp;A&nbsp;freely accessible website
was developed for evaluation of the </span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-fareast-language:PT'>proposed comparison methods (</span><span
lang=DE-CH><a href="http://gscompare.ehu.eus"><span lang=EN-US
style='font-size:8.0pt;mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
mso-fareast-language:PT'>http://gscompare.ehu.eus</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>). The
service&nbsp;allows uploading of sequences and comparisons of them or with
sequences already in our database of more than 20,000 complete genomes.</span><span
lang=EN-GB style='mso-bidi-font-family:Arial;color:black;mso-themecolor:text1;
mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<span lang=EN-GB style='font-size:10.0pt;mso-bidi-font-size:11.0pt;font-family:
Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
color:black;mso-themecolor:text1;mso-ansi-language:EN-GB;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'><br clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO2 - Russian Phage
Roulette In Reference Genomes<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Jonas T. Björkman<sup>1</sup>; Kristoffer Kiil<sup>1</sup>; Eva Møller
Nielsen<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Statens Serum Institut<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>For outbreak
analysis, you often would like the maximum discriminatory power, resulting in
that you resort to SNP analysis with a closely related strain as mapping
reference. We have analysed outbreaks, clusters and closely related isolates of
<i>Listeria monocytogenes</i> from a ten-year period in Denmark and assessed
the impact on the cluster analysis when using reference genome with, or without
prophages.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Prophages were
identified using PHAST and by looking at SNP density over the genome. In some
isolate clusters, certain positions in the genome were always occupied by a
phage. The phages did not necessary have a common phylogenetic origin with the
host strain, but all the phages were related enough to contribute SNPs, that
our standard method of removing horizontally transferred genetic material did
not catch.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>We found there
could be a large variation of related phages in the same genomic site. The
individual phage variants showed very few internal SNPs but the different phage
variants had enough similarity to cause a large impact on the tree creation and
cluster analysis.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The effect of
using the reference genome with and without removal of phage varied from no
impact at all, to relatively small with mostly elongated branches, and finally
to a strong impact, where the phages accounted for more than 90 % of the
variation and caused rearrangements of the trees.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Our recommendation
is that you (at least for <i>Listeria</i>) should not use a reference genome
with a possibly mobilisable prophage present. Since the task of properly
identifying all prophages in a genome might not be trivial, ideally this should
be done thoroughly once on reference genomes, that subsequently could be shared
throughout the world.</span><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p></o:p></span></p>

<span lang=EN-GB style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:
115%;font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:
minor-latin;color:black;mso-themecolor:text1;mso-ansi-language:EN-GB;
mso-fareast-language:EN-US;mso-bidi-language:AR-SA'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='mso-bidi-font-family:Arial;
color:black;mso-themecolor:text1;mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO3 - SCCmec Typing Of
MRSA Genomes<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Peder Worning<sup>1</sup>; Jesper Boye Nielsen<sup>1</sup>; Heidi Gumpert<sup>1</sup>;
Mette Bartels<sup>1</sup>; Henrik Westh<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Hvidovre University Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal align=left style='text-align:left;line-height:115%'><span
lang=EN-GB style='mso-bidi-font-family:Arial;color:black;mso-themecolor:text1;
mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='text-align:left;line-height:115%'><span
lang=DE-CH style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT;mso-bidi-font-weight:bold'>Keywords</span><span lang=DE-CH style='font-size:
8.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>: SCCmec Type, MRSA, Whole Genome
Sequencing.<o:p></o:p></span></p>

<p class=MsoNormal align=left style='text-align:left;line-height:115%'><span
lang=DE-CH style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: Since 2013
all methicillin resistant Staphylococcus aureus (MRSA) isolates from the
Capital Region of Denmark&nbsp; have been whole genome sequenced on an Illumina
MiSeq. We have made a SCCmec typing tool as a part of our MRSA pipeline, which
also includes spa-, dru- and MLST-typing. We have determined SCCmec types for
3,052 different isolates (by October 2015). The 3,052 isolates contains 337
different spa-types and 109 different MLST types. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: The main
SCCmec types (I – XI) are determined by blasting the DNA sequence in question
against a list of known SCCmec genes to determine which versions of the SCCmec
genes the sequence contains. We use mecA, mecR, mecI and mecC from the mec
complex, ccrA1-4, ccrB1-4,6, ccrC from the ccr complex plus the two IS elements
IS431 and IS1272. To find SCCmec type XI we use alternative versions of the
mec- and the ccr-genes, namely mecC, mecRx1, mecIx1, ccrAx1 and ccrBx3. For
SCCmec type IV we use six sets of primers and a virtual PCR routine to
determine the sub-type. Here we can resolve the subtypes IVa, IVb, IVd, IVg,
and IVh, but we cannot separate IVc from IVe, so we report them as IVc/e.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: By far we
see SCCmec type IV the most at 76%, followed by type V at 13.2%, type IX at 1.6
% and type II at 1.4 %. We perform subtyping of type IV into six different
subtypes: IVa, IVb, IVc/e, IVd, IVg and IVh, where IVa and IVc/e are the most
common. We will bring a computer with the software installed, so guests at
IMMEM XI can bring their favorite MRSA genome on a stick in fasta format to get
it typed. The SCCmec typer is a work in progress and we are very interested in
collaboration to make the determinations of the SCCmec types more complete. It
is our plan to host the validated SCCmec typer online.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: We can
type the 11 main SCCmec types and six SCCmec IV sub-types, from DNA sequences.
We se by far most SCCmec type IV 76 %, followed by type SCCmec V at 13%. Our
dataset contains 3,052 isolates representing 337 spa types and 109 MLST types.</span><span
lang=EN-GB style='mso-bidi-font-family:Arial;color:black;mso-themecolor:text1;
mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<span lang=EN-GB style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:
115%;font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:
minor-latin;color:black;mso-themecolor:text1;mso-ansi-language:EN-GB;
mso-fareast-language:EN-US;mso-bidi-language:AR-SA'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=EN-GB style='mso-bidi-font-family:Arial;
color:black;mso-themecolor:text1;mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO4 - Towards NGS As Frontline
Tool For Routine Typing Of <i style='mso-bidi-font-style:normal'>S. aureus</i>
At The German</span></b><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'> <b style='mso-bidi-font-weight:normal'>Reference Center For Staphylococci
And Enterococci?</b><o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Birgit Strommenger<sup>1</sup>; Sophie Ziegler<sup>1</sup>; Stephan Fuchs<sup>1</sup>;
Jennifer Bender<sup>1</sup>; Franziska Layer<sup>1</sup>; Guido Werner<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Robert Koch Institute<o:p></o:p></span></i></p>

<p class=MsoNormal style='line-height:115%'><span lang=EN-GB style='mso-bidi-font-family:
Arial;color:black;mso-themecolor:text1;mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> The rapid
progress of Next generation sequencing (NGS) technology has dramatically
changed our vision of clinical microbiological diagnostics, strain
characterization and typing during recent years. Although the constant decrease
in cost nowadays enables the implementation of NGS technology in molecular
surveillance activities, detailed bioinformatic analysis of genome data still
remains a challenge. Therefore this study was initiated to evaluate feasibility
and benefit of NGS for the staphylococcal molecular typing service provided by
the National Reference Centre (NRC) for staphylococci and enterococci in
comparison to typing tools used to date.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials and
methods: W</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>e selected a total of 282 <i>S. aureus</i> isolates sent to the NRC
throughout February 2014. All isolates passed the routine typing workflow at
our laboratory including phenotypic susceptibility testing, <i>spa</i>-typing
(Sanger sequencing) and PCR-based <i>mecA</i>-detection. Depending on the
initial strain report additional resistance and virulence determinants were
investigated by phenotypic and genotypic methods, respectively. Sequencing was
carried out on a MiSeq instrument aiming at a theoretical coverage of app.
120-fold. Initial data analysis was performed using the Ridom SeqSphere 2.4
pipeline including Velvet as assembler. Based on the finished assemblies
SeqSphere extracted <i>spa</i>-type and MLST from the whole genome data for
each isolate. For in-depth-typing a predefined core genome MLST scheme (cgMLST)
was used which analyzes sequence diversity of 1861 <i>S. aureus</i> core genome
loci. To extract further information concerning the presence of a variety of
relevant resistance and virulence determinants, user-defined sets of
corresponding genetic loci were added to the SeqSphere queries.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Out of 282
genome datasets 278 yielded satisfactory results (≥95% of the 1861 MLST<sup>+</sup>
target genes with good quality) in the SeqSphere pipeline, thus highlighting
the importance of high quality raw data and an average coverage of at least
70-fold. Typing results extracted from NGS data were overall concordant with
those generated by currently used genotypic methods (<i>spa</i>-typing, MLST).
However, concordance with phenotypic methods (e.g. antibiotic resistance) was
lower than expected (app. 75% in initial analyses). Molecular typing based on
cgMLST data, as expected, yielded significantly higher discrimination,
especially for endemic <i>S. aureus </i>lineages and enabled the support of
outbreak investigations in several institutions.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>In
conclusion, whole genome sequencing was successfully established for molecular
typing purposes at the NRC. Future work will focus on the clarification of
contradictory results and on the comparison of results obtained from the
combination of different available bioinformatic tools.<o:p></o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO5 - Accnet: New Tool
For Accessory Genome Analysis Using Bipartite Networks.<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Val Fernandez Lanza<sup>1</sup>; Irene Rodriguez<sup>1</sup>; Ana P. Tedim<sup>1</sup>;
María de Toro<sup>2</sup>; Fernando de la Cruz<sup>2</sup>; Rafael Cantón<sup>1</sup>;
Fernando Baquero<sup>1</sup>; Teresa M. Coque<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>CIBER-ESP. Hospital Universitario Ramón y Cajal and
Instituto Ramón y Cajal de Investigación Sanitaria IRYCIS, Instituto de
Biomedicina y Biotecnología de Cantabria IBBTEC UC-SODERCAN-CSIC<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Lateral gene
transfer plays an essential role in the bacterial evolution The genomes of most
opportunistic pathogens differ in their accessory genome which include
determinants associated with virulence, colonization or antibiotic resistance,
among other adaptive traits. It exists a wide range of bioinformatics tools for
typing, core genome analysis or phylogenetics but only a few tools for
analysing the accessory genomes. We present AccNET (Accesory Constelation
Network)), a new tool, non-computational expensive, based on bipartite networks
that is able to analyse simultaneously tens of genomes. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Material &amp;
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>AccNET uses the kClust software to customly detect highly-related
homologous genes, by modifying the identity threshold. The core genome, defined
as the subset of proteins present in all genomes, is removed from the analysis.
Then, AccNET builds a bipartite network of the accessory genome with two kind
of nodes: genomes and proteins. The edges that link the nodes represent the
presence of a protein in a given genome. The edge-weight is determined as a
phylogenetic distance between the proteins included in the cluster. To
calculate such distance, we use a pipeline: First, we align all the proteins
belonged to the cluster using MUSCLE. Then, we optimize the multiple alignment
using Trimal and finally we calculate the distance matrix using ProtDist,
(PHYLIP package). For each protein the phylogenetic distance for edge-weight is
determined by the mean of the distance with all of the remaining proteins.
Final output network is compatible with most of the network viewer software
such as Cytoscape or Gephi. We used ForceAtlas2 layout from Gephi to arrange
networks. To compare the structure of the accessory distribution with the
phylogenetic relationships, we build the core-genome phylogenetic tree using
the previous defined core-genome. First, we aligned each one of the core genes
using MUSCLE. Then, we concatenated all the multiple alignments and extract the
SNPs using HarvestTools. Finally, we built a Maximum Likelihood phylogenetic
tree using FastTree2 with the resulting SNPs. To demonstrate AccNET
capabilities, we have used two datasets that comprises genomes of the
polyclonal group ST131 of <i>E. coli</i> (n=37) and the phylogenomic group BAPS
2.1 of <i>E. faecium</i> (n=25). Both include specific clonal variants
worldwide disseminated.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> The
resulting network from <i>E. coli </i>dataset comprises 6973 nodes and 41585
edges while that of <i>E. faecium</i> has 3326 nodes and 20041 edges. A high
correlation was noted between the clusters in the network of accessory genomes
and the branches in the phylogenetic tree of the core genomes. However, the
networks disclose specific interactions between genome clusters that are remote
in the phylogenetic tree. Besides, both network shown specific protein clusters
that belong to genome clusters associated with epidemic multidrug resistant
variants such as <i>E. coli </i>ST131-fimH30 or <i>E. faecium </i>ST117. Fig1
show a snapshot of the network arrangements obtained by using AccNET and the
core-phylogenetic tree applied to the E. coli dataset.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> We have
developed an specific software tool for the analysis of accessory genome. This
software is able to analysis tens of genomes with high precision. User friendly
environment as Cytoscape allow us comprehensively exploring the accessory
genome, and identifying specific genes/gene clusters of interest in
evolutionary biology and epidemiological surveillance.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='mso-ansi-language:
EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shapetype id="_x0000_t75"
 coordsize="21600,21600" o:spt="75" o:preferrelative="t" path="m@4@5l@4@11@9@11@9@5xe"
 filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Imagem_x0020_7" o:spid="_x0000_i1029" type="#_x0000_t75"
 alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/155imagem_2560108707_0.jpg"
 style='width:333pt;height:251pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S1_files/image001.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/155imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=253
src="public/abstracts/Webpages/PO_S1_files/image002.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/155imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_7"><![endif]></span><span style='mso-bidi-font-family:
Arial;color:black;mso-themecolor:text1;mso-ansi-language:EN-US'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
color:black;mso-themecolor:text1;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-bidi-font-family:Arial;color:black;
mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO6 - The Contaminated
Genome<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Peder Worning<sup>1</sup>; Heidi Gumpert<sup>1</sup>; Jesper Boye Nielsen<sup>1</sup>;
Henrik Westh<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Department of Clinical Microbiology, Copenhagen
University Hospital, Hvidovre Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-font-weight:bold'>Keywords<b>:</b></span><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Contaminated
Samples, Mixed Assembly, Whole Genome Sequencing</span><span style='mso-ansi-language:
EN-US'><o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Whole
genome sequencing is employed for every methicillin-resistant <i>Staphylococcus
aureus </i>(MRSA) and vancomycin-resistant <i>Enterococcus faecium </i>(VREfm) isolate
that passes through the Department of Clinical Microbiology at Hvidovre
Hospital (KMA-Hvidovre) as part of the daily routine. Now and then we observe
assemblies that contain more DNA, i.e. a larger genome, than we would expect
for these species. The larger assembly size is interpreted as a sign of a
contaminated sample. Yet, questions such as how different do mixed isolates
have to be to make a larger assembly, and would a mixture of two different
isolates of the same species result in an assembly which is significantly
larger than a clean sample, remain to be answered.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> We have
made combined assemblies of reads from different isolates and analyzed the size
of the combined assembly as a function of the SNP distance between the involved
isolates. Sequences were mixed in a ratio of 1:1 with an equal coverage of
about 50 fold. The isolates have been sequenced on an Illumina MiSeq using 150
bp paired-end reads. Reads have been assembled using Velvet and
VelvetOptimiser. Reference genomes for calculating SNP-distances used are
USA300-THC1415 (NC_010079) for the MRSA samples and AUS004 (NC_017022) for the
VREfm samples. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> We observe
that the size of the combined assemblies increases linearly with the SNP
distance, and we see a significantly larger assembly already when the distance
between the two isolates is as small as 1,000 SNPs. 1,000 SNPs is equal to the
diameter of a typical <i>S. aureus</i> clonal complex (CC). The size of the
combined assemblies saturates abruptly when the size is equal to the sum of the
sizes of the two isolates. This happens at a SNP distance on the order of
35-45,000 SNPs. This means that when we mix reads from two genomes with a SNP
distance larger than 45,000 SNPs, we get the two complete sequences from our
assembly. 45,000 SNPs are further apart than many CC are in <i>S. aureus</i>,
but still inside the same species.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>When we compare
the two species included in this study, we observe that the size of the
combined assembly increases faster with the SNP distance for VREfm than for
MRSA. The size of the combined VREfm assemblies saturates at a SNP distance of
approximately 35,000 SNPs, while the size of the MRSA assemblies saturates at
approximately 45,000 SNPs.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Contaminated
samples shows up as to large genomes when the reads are assembled into contigs.
The size of the contaminated assembly increases linearly with the SNP distance
and significantly larger assemblies are observed when the distance between the
two isolates is as small as 1000 SNPs. This means that contamination with
isolates form a different clonal complex of the same species will result in
enlarged assembled genomes.<o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
color:black;mso-themecolor:text1;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-bidi-font-family:Arial;color:black;
mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO7 - Genome-based
Epidemiological Surveillance Of <i style='mso-bidi-font-style:normal'>Listeria monocytogenes</i>
At A Global Scale<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Alexandra Moura<sup>1</sup>; Hannes Pouseele<sup>2</sup>; Alexis Criscuolo<sup>1</sup>;
Mylène Maury<sup>1</sup>; Marie Touchon<sup>1</sup>; Alexandre Leclercq<sup>1</sup>;
Viviane Chenal-Francisque<sup>1</sup>; Hélène Dieye<sup>1</sup>; Thomas
Cantinelli<sup>1</sup>; Louis Jones<sup>1</sup>; Elise Larsonneur<sup>1</sup>;
Cheryl Tarr<sup>3</sup>; Heather Carleton<sup>3</sup>; Zuzana Kucerova<sup>3</sup>;
Lee Katz<sup>3</sup>; Steven Stroika<sup>3</sup>; Jonas T. Larsson<sup>4</sup>;
Aleisha Reimer<sup>5</sup>; Matthew Walker<sup>5</sup>; Celine Nadon<sup>5</sup>;
Eva M. Nielsen<sup>5</sup>; Tim Dallman<sup>6</sup>; Kathie Grant<sup>6</sup>;
Peter Gerner-Smidt<sup>3</sup>; Bruno Pot<sup>2</sup>; Marc Lecuit<sup>1</sup>;
Sylvain Brisse<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Institut Pasteur <sup>2</sup>Applied-Maths <sup>3</sup>Centers
for Disease Control and Prevention <sup>4</sup>Statens Serum Institut, <sup>5</sup>Public
Health Agency of Canada,<sup>6</sup>Public Health England<o:p></o:p></span></i></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bidi-font-family:
Arial;color:black;mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:115%'><i><span style='font-size:8.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Listeria monocytogenes</span></i><span
style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'> (<i>Lm</i>) outbreaks have been reported worldwide, causing numerous
deaths and reinforcing the need for discriminatory typing methods for
surveillance and outbreak investigations. Here we report a standardized
genome-wide genotyping system based on core genome multi-locus sequencing
typing (cgMLST) of 1748 highly conserved genes<i>,</i> representing 62% of the <i>Lm</i>
genome. The <i>Lm</i> cgMLST scheme was developed and validated based on a
large set of genomes (<i>n=</i>957) representative of the clonal diversity
within <i>Lm</i> lineages. Robustness of genotyping was tested using
assembly-free and assembly-based allele detection methods and different
assembly pipelines. Epidemiological and phylogenetic relevance of cgMLST
profiles was assessed using genomes collected within the frame of national
surveillance programs in USA, Canada, England, Denmark and France (<i>n=</i>1696).
cgMLST showed highly reproducibility of profiles using both assembly-free and
assembly-based calls as well as different assemblers. Allelic profiles
reflected the phylogenetic relationships between isolates and were highly
conserved within single outbreaks. Moreover, the discriminatory power of cgMLST
was significantly higher compared to the current standardised pulsed-field gel
electrophoresis (PFGE) typing method, allowing to narrow clusters of closely
related isolates with identical PFGE profiles and to improve epidemiological
investigations. To facilitate global communication, a bioinformatics platform
and a unified nomenclature of sublineages and cgMLST types were implemented.
This study shows that cgMLST is a practical and robust approach for
international collaboration on research and public health surveillance of <i>Lm</i>,
enhancing disease surveillance and response efforts.<o:p></o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO8 - Fimtyper; A Web
Tool For <i style='mso-bidi-font-style:normal'>E. coli</i> fimH Genotyping<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Evgeni Sokurenko<sup>1</sup>; Veronika Tchesnokova<sup>1</sup>; Mariya
Muradova<sup>1</sup>; Sujay Chattophadhay<sup>1</sup>; Johanne Ahrenfeldt<sup>2</sup>;
Rosa Allesøe<sup>2</sup>; Frank Hansen<sup>3</sup>; Anette Hammerum<sup>3</sup>;
Henrik Hasman<sup>3</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University of Washington,<sup>2</sup>Technical
University of Denmark,<sup>3</sup>Statens Serum Institut<o:p></o:p></span></i></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='mso-bidi-font-family:
Arial;color:black;mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Background:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> The <i>fimH</i> gene is part of a <i>fim</i>
operon which encodes a surface organelle called Type 1 fimbriae found in most <i>E.
coli</i> strains. The FimH protein is localized at the tip of the fimbrial
structure and serves as a D-mannose specific adhesin, which aids to immobilize
the bacterium on both biotic and abiotic surfaces. Studies have shown great
allelic variation in the <i>fimH</i> gene, which have an influence on binding
characteristics of the fimbriae. Until now, almost 500 different alleles of <i>fimH</i>
have been identified. In addition to being used to study binding abilities of
the FimH protein variants, these <i>fimH </i>alleles can also be used for
subtyping of different <i>E. coli</i> clonal variants. The relevance of this <i>fimH</i>
sub-typing has been especially emphasized within the highly virulent sequence
type (ST) 131 clonal clade, where resistant and multi-resistant sub-clades
carrying<i> fimH30</i> allele have been identified. As <i>E. coli</i> ST131 is
the most dominant human pathogenic clonal group, the need to be able to perform
<i>fimH</i> sub-typing is increasing. Traditionally, typing of the <i>fimH</i>
allele has been performed by specific <i>fimH</i> PCR followed by Sanger sequencing
and alignment to a <i>fimH</i> allele database containing 477 different <i>fimH</i>
alleles. With the rapid increase in the use of Whole Genome Sequencing (WGS) to
type and analyze bacteria, a need to develop a system to handle WGS data in
relation to <i>fimH</i> typing has emerged.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Materials and Methods:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> The Centre for Genomic Epidemiology has been
developing freely available stand-alone web-tools for handling WGS information
for outbreak investigation, epidemiological surveillance and diagnostics based
on BLAST analysis towards standardized and curated gene databases. A new
stand-alone web-tool called FimTyper was developed. Here, a FASTA database
containing all 477 <i>fimH</i> alleles is used to perform BLAST searches within
WGS data. These WGS data can either originate directly from sequencing
platforms such as Illumina, Ion Torrent or 454, or they can consist of <i>de
novo</i> assembled draft (or complete) genomes. The best hit to a given <i>fimH
</i>allele from the database is given as output including length of the hit and
percent identity between the hit in the genome and in the database. To test the
database, 245 Extended-Spectrum Beta-Lactamase (ESBL) producing <i>E. coli</i>
draft genomes originating from blood infections and submitted to Statens Serum
Institut in 2014 as part of the routine surveillance were analyzed </span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>using the FimTyper tool
available at </span><span lang=DE-CH><a
href="https://cge.cbs.dtu.dk/services/FimTyper-1.0/"><span lang=EN-US
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>https://cge.cbs.dtu.dk/services/FimTyper-1.0/</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> The FimTyper tool successfully analyzed all 245
draft genomes. FimTyper was able to identify a <i>fimH</i> allele (more than
80% identity to any target in the database) in 232 of the 245 genomes giving a
typability of 95%. In total, 33 different alleles were identified in the
dataset including 2 new alleles with the most dominant being the <i>fimH30</i>
allele (38%).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusion:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> We have developed a fully functional and freely
available web tool called FimTyper to identify <i>fimH</i> alleles in WGS data
from <i>E. coli </i>genomes, thus enabling researchers and primary
investigators to rapidly detect the <i>fimH</i> allele in their WGS datasets.
The service is available at </span><span lang=DE-CH><a
href="https://cge.cbs.dtu.dk/services/FimTyper-1.0/"><span lang=EN-US
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>https://cge.cbs.dtu.dk/services/FimTyper-1.0/</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>. It is the
aim to include this in a newly developed web-based WGS pipeline combining all
CGE tools for microbial typing including Serotyping, resistance gene typing and
Multilocus sequence typing (MLST) already available at </span><span lang=DE-CH><a
href="http://www.genomicepidemiology.org"><span lang=EN-US style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-fareast-language:PT'>www.genomicepidemiology.org</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>.<o:p></o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO9 - Examining
Gene-by-gene Diversity Across Thousands Of Whole Genomes To Determine Suitable
Targets<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Sofia Hauck<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University of Oxford<o:p></o:p></span></i></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='mso-bidi-font-family:
Arial;color:black;mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>With the
rise of high throughput whole genome sequencing, thousands of genomes for the
most epidemiologically important bacterial species have become available. While
many of these genomes will have poor coverage or will not assemble well, in
combination they are a useful resource for exploring the diversity of a species
at a gene-by-gene level. This allows for the discovery of both highly conserved
and highly diverse genes, a key step in choosing markers for epidemiological
studies.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials &amp;
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Whole genomes from public archives such as the Sequence Read Archive (SRA)
were added to the PubMLST database, following assembly via a Velvet optimiser
pipeline. This database runs the BIGSdb (Bacterial Isolate Genome Sequence
database) software, which allows for genome annotation based on a set of
reference sequences that defines genes, and an iterative scanning and
definition process for new alleles of those genes. A set of simple Perl scripts
then finds measures of diversity from the length, count of unique alleles (in
both nucleotide and amino acid format), and ratio of variable sites for each
defined locus, using the mean for the genome as the point of comparison<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>This method
was first tested on a dataset of 7681 <i>Mycobacterium tuberculosis</i>
genomes, using the nearly four thousand defined genes in the H37Rv annotation
as the reference sequences. This allowed us to identify genes both in the
conserved and diverse ends of the spectrum, on all measures. Of special
interest were genes in the ESAT-6 family which were genetically diverse (a high
number of alleles found) but functionally conserved (a low ratio of amino acid
to nucleotide unique sequences), suggesting perhaps two stages of evolution
with different pressures. We are currently running the same methods on <i>Staphylococcus
aureus</i>, <i>Neisseria gonorrhoeae</i> and&nbsp;<i>Salmonella enterica</i>
data sets of similar sizes, and hope to have results ready for presentation in
March. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>High
throughput whole genome sequencing allows for thousands of genomes to be
gathered and analysed together. Together with annotated reference genomes, this
means genes can be explored as &quot;gene pools&quot; that can be compared to
one another in order to find markers according to need. For example, if a
highly reliable species identification is necessary, a highly conserved gene
can be chosen. If typing down to individual strains is more important, a often
present but highly diverse gene might be prefered. This method allows to simple
and rapid evaluation of gene targets across an entire genome using a large
number of isolates so as to avoid sampling bias.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO10 - Core Genome
Typing: A Useful Tool For Micro-epidemiological Investigations Involving <i
style='mso-bidi-font-style:normal'>Staphylococcus aureus?</i><o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Bruno Pichon<sup>1</sup>; Michel Doumith<sup>1</sup>; Neil Woodford<sup>1</sup>;
Angela M. Kearns<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Public Health England<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-bidi-font-family:Arial;color:black;
mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Recent
advances in sequencing technologies provide unprecedented insights into the
relatedness of bacteria such as <i>S. aureus</i>. SNP analyses based on whole
genome sequence (WGS) data are commonly used to infer the phylogenetic
relatedness, but results are dependent on the reference genome used, the
bioinformatics pipeline and user expertise in phylogenetic interpretation. We
have explored an alternative approach based on sequence analysis of core genome
loci.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conserved
gene sequences (n=1334) in 54 complete genomes available from Genbank were
identified by BLAST search. Combinations of allelic profiles were assigned to
an identifier dubbed a core genome cluster (CGC) using an in-house hierarchical
clustering approach based upon pairwise homology, and were populated in a
reference database. Further validation was performed on 114 study genomes,
newly sequenced, and representing i) a broad range of lineages (human and
animal clades); ii) suspected clusters (2 HA-MRSA ST22 and one CA-MRSA ST59);
iii) a representative of <i>S. argenteus</i> (ST2250). Genomes were assembled <i>de
novo</i> and loci sequences were extracted by BLAST. New alleles were confirmed
using mapping-based approach before being added to the database. Whole genome
variant calling pipeline used BWA-mem and GATK2 using ST22MRSA H050960412 and
SA40 Asian Pacific Clone as reference strains for respectively the ST22 and
ST59 case cluster analysis.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Result: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Fifty four
allelic profiles were identified from the analysis of published genomes. The
number of alleles per locus ranged from 2 to 30 (mean of 15.8). Newly sequenced
genomes were grouped into 97 CGCs and corroborated with MLST lineages. Core
genome typing grouped those with known epidemiological links and differentiated
them from the background population belonging to the same MLST lineage. Genomes
belonging to the same CGC differed by 0 to 5 WGS SNPs. CGCs of linked isolates
differed from those representing background isolates by 34 or more alleles
compared to 50 or more WGS SNPs by phylogenetic analysis.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>We designed
and applied a core genome typing approach and applied it to a broad range of S.
aureus lineages including <i>S. argenteus</i>. The technique was able to assign
a CGC to all reference and study genomes. Core genome clustering was able to resolve
the molecular epidemiology with a discriminatory power comparable, to a certain
extent, to SNP-based phylogenetic analyses without requiring the construction
and interpretation of a phylogenetic tree. Core genome clustering is a
promising approach for micro-epidemiological investigations involving
MSSA/MRSA.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO11 - Genetic
Diversity Of <i style='mso-bidi-font-style:normal'>Salmonella typhimurium</i>
And <i style='mso-bidi-font-style:normal'>Salmonella enteritidis</i> Isolated
From Ducks And Chickens Using Random Amplified Polymorphic DNA (RAPD)<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Frederick Adzitey<sup>1</sup>; Courage Kosi Setsoafia Saba<sup>2</sup>;
Raja Arief Deli<sup>3</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University for Development Studies, Animal Science
Department, <sup>2</sup>University for Development Studies, Department of
Biotechnology,<sup>3</sup>Food Technology Research Centre, Malaysian
Agriculture Research and Development Institute<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-bidi-font-family:Arial;color:black;
mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: Genetic
characterization of <i>Salmonella species</i> of food animal origin is
important for determining their relationships. Such relationships are used to
establish the similarities and differences between bacterial isolates. This
study investigated the genetic diversity between <i>Salmonella </i>Typhimurium <i>Salmonella</i>
Enteritidis strains isolated from chicken and ducks using Random Amplified
Polymorphic Deoxyribonucleic Acid (RAPD)-PCR.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials |
Methods:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'> DNA of 36 pure <i>Salmonella</i> colonies of chicken and duck origins were
extracted using Wizard® Genomic DNA Purification Kit following the
manufacturers’ instructions. The C-05 (10-mer) primer 5</span><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>׳</span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>-GATGACCGCC-3</span><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>׳</span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> was
selected for RAPD-PCR after a panel of 8 random primers (designed and
manufactured by 1st BASE, Singapore) had been screened. The PCR and
Amplifications were performed according to Adzitey et al. (2013).Clustering was
defined at a coefficient of 0.85. <i>Salmonella</i> serovars not belonging to
any particular cluster were referred to as singletons (single isolates).
Discriminatory index was calculated according to Hunter and Gaston (1988) based
on the number of clusters and singletons identified.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Analysis of
the <i>Salmonella</i> strains by RAPD-PCR produced DNA bands that ranged from
242 to 3189 bp <i>Salmonella</i> Typhimurium and 252 to 2756 bp for <i>Salmonella</i>
Enteritidis. Cluster analysis at a coefficient of 0.85 grouped the <i>Salmonella</i>
Typhimurium into 4 clusters and 1 singleton at a discriminatory index of 0.85. <i>Salmonella</i>
Enteritidiswere grouped into 2 clusters and 2 singletons at a discriminatory
index of 0.64. One <i>Salmonella</i> Typhimurium isolated from chicken carcass
was not characterized as the RAPD-PCR employed failed to produce any DNA band
from that isolate. It was evident from this study that some <i>Salmonella</i>
strains circulating between chickens and ducks were similar while others were
not. Characterizing <i>Salmonella</i> serovars from different sources is
important to determine their genetic relatedness, and source of contamination
and spread.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>This study
compared the genetic relatedness of chicken and duck <i>Salmonella </i>serovars
using RAPD-PCR. RAPD-PCR analysis of the <i>Salmonella</i> serovars resulted in
the characterization of the various <i>Salmonella</i> strains isolated from
chickens and ducks and provided a means of determining the genetic relatedness
among the serovars. Some <i>Salmonella</i> strains were genetically closely
related but not others.<o:p></o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO12 - Application Of
PCR-RFLP Provides As Discrimination As Total FLAA Sequence Analysis Among <i
style='mso-bidi-font-style:normal'>C. jejuni</i> Isolates Of Clinical Origin<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Mahdi Ghorbanalizadgan<sup>1</sup>;BitaBakhshi2; ShahinNajar-Peerayeh<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Department of Bacteriology, Faculty of Medical
Sciences, TarbiatModares University, Tehran, IR Iran,<sup>2</sup>Department of
Bacteriology, Faculty of Medical Sciences,Baqiyatallah University, Tehran,IR
Iran<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-bidi-font-family:Arial;color:black;
mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-font-weight:bold'>Keywords<b>:</b></span><i><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Campylobacter
jejuni,flaA</span></i><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>sequence, typing</span><span style='mso-bidi-font-family:
Arial;color:black;mso-themecolor:text1;mso-ansi-language:EN-US'><o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-bidi-font-family:Arial;color:black;
mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The aim of this
study was to determinethegenetic relatedness amongtwenty clinical <i>Campylobacterjejuni</i>isolated
from children with diarrhea in Iran,and to introduce the best discriminatory
method based on <i>fla</i>A sequence divergence.A total of 400 stool specimens
were obtained from children under five years of age during July 2012 to June
2013. Primers were designedbased on conserved sequences flanking the <i>fla</i>A
gene which encompass and amplify entire <i>fla</i>Agenefollowed bysequencing
and analysis with MEGA version 6.0.6 software.Ninetyamino acid versus 560
nucleotidepolymorphic sequenceswere detectedwithin 1681bp of <i>flaA</i>
sequence among which 43 (2.5%( and 12 (0.7%) were singleton.New repeat boxes
within <i>flaA</i> sequences were found in this study.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>UPGMA dendrogram
based on nucleotides of full length flaAgene,flaA SVR gene and in silicoflaA
phylogenic tree of DdeI RFLP profilesproduced very similarclustering with
diversity indexof 0.86 for each of three methods. It can be concluded that <i>fla</i>A
typing based on <i>DdeI</i> RFLP of the PCR products, can be applied as cheap,
rapid and reliable method for epidemiological study of <i>C. jejuni</i>
isolates of clinical origin in resource limiting regions or in large scale
population<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO13 - A Unified
Database For <i style='mso-bidi-font-style:normal'>Acinetobacter baumannii</i>
Genomic And MLST Sequences Enables Comparison Of Pasteur And Oxford 7-gene MLST
Nomenclatures<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Raffaele Zarrilli<sup>1</sup>;Virginie Passet<sup>2</sup>; Eliana Pia
Esposito<sup>1</sup>; Keith A Jolley<sup>3</sup>; Sylvain Brisse<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Department of Public Health, University of Naples
“Federico II”, Naples, <sup>2</sup>Italy,Institut Pasteur, Microbial
Evolutionary Genomics, Paris, France,<sup>3</sup>Department of Zoology,
University of Oxford, Oxford, UK<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> <i>Acinetobacter
baumannii</i> is a Gram-negative bacteria responsible for a large number of
hospital-acquired infections, which are usually caused by a limited number of
multidrug-resistant epidemic clones assigned to distinct clonal lineages The
population structure of <i>A. baumannii</i> has been studied using two MLST
nomenclatures, which complicates advances of the epidemiological and population
biology knowledge of this important pathogen. The aim of the present study was
to create a unified nomenclature of <i>A. baumannii</i> clones by taking
advantage of publicly available genome sequences. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> The <i>Acinetobacter
baumannii</i> PubMLST website hosted at the University of Oxford and powered by
the Bacterial Isolate Genome Sequence Database (BIGSdb) software was used to
integrate the MLST databases based on the ´´Oxford´´ scheme and the ´´Pasteur´´
scheme into a single database. The <i>Acinetobacter</i> isolates database was
populated with genome data from 765 strains of the <i>Acinetobacter
calcoaceticus-baumannii</i> (Acb) complex and automated tagging and assignment
of alleles for both MLST schemes was performed. Whole genome sequence (WGS) assemblies
of 196 out of 765 <i>Acinetobacter</i> isolates were uploaded to the isolates
database. <i>A. baumannii</i> ATCC 17978 and ACICU WGS were selected as
reference genomes. The universal ribosomal MLST scheme based on 53 loci (Jolley
et al., Microbiology 158, 1005, 2012) was used to assign rMLST genotypes to all
genome sequences.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> ST
assignments for a large number of genomes generated a convenient and expandable
table of correspondence between the two MLST schemes. MLST-based species
identification was performed using the “Pasteur” MLST scheme by
neighbor-joining tree analysis of concatenated allele sequence alignments and
comparison with reference STs. The species field was imported in to the PubMLST
database for all STs with no ambiguous clustering (&gt; 98% of them). The
´´Pasteur´´ scheme was less discriminant, but possibly more appropriate for
population biology. On the other hand, the ´´Oxford´´ scheme was able to
identify additional genotypes and to differentiate isolates belonging to
international clone II into two distinct clonal lineages, but suffered from
elevated variability due to high recombination at the gpi and gdhB loci. rMLST
showed improved resolution over both 7-gene MLST schemes and provided a useful
subtyping approach. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> The availability
of a single database hosting both widely-used classical MLST schemes will
facilitate epidemiology and population biology research of <i>Acinetobacter</i>.
The publicly accessible database can be expanded to allow rapid extraction of
biologically and epidemiologically relevant information from <i>Acinetobacter</i>
genome sequences. It may also host in the future, a genome-wide MLST scheme
that will provide the improved resolution needed for small scale epidemiology
and for precise strain classification into<i> A. baumannii</i> clonal groups.<o:p></o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO14 - PHYLOViZ Online:
Web-based Phylogenetic Data Analysis And Visualization For Allelic Profiles And
SNP Data<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Bruno Ribeiro-Gonçalves<sup>1</sup>; Alexandre Francisco<sup>2</sup>; Cátia
Vaz<sup>3</sup>; Mário Ramirez<sup>1</sup>; João André Carriço<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>Instituto de Microbiologia, Instituto de Medicina
Molecular, Faculdade de Medicina, Universidade de Lisboa, <sup>2</sup>Instituto
Superior Técnico, Universidade de Lisboa, <sup>3</sup>Instituto Superior de
Engenharia de Lisboa <sup>4</sup>Instituto Politécnico de Lisboa<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
PT;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Next Generation
Sequencing methods gave rise to a paradigm shift in microbial typing. The
ability to sequence the genomes of hundreds of strains created the need for
effective ways to represent the phylogenetic relationships between isolates
that are scalable and robust. Single Nucleotide Polymorphism (SNP) analysis and
core/whole genome MLST approaches create profiles with up to thousands of loci
that can be used for outbreak investigation, surveillance and bacterial
population or evolutionary studies. The analysis of these profiles can be done
with traditional phylogenetic algorithms or Minimum Spanning Tree (MST) like
approaches, the latter being in particularly suited to cope with the increasing
number of isolates that are used in each study. The software PHYLOViZ allows
using epidemiological data to annotate the resulting tree, but lacks options to
share visualizations between users or provide public access to a given dataset.
<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>With the aim of
overcoming these limitations, we have developed PHYLOViZ Online, a
user-friendly web application for profile based data analysis, visualization
and sharing.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>PHYLOViZ
Online is an online application written in node.js in a modular perspective,
separating visualization from data processing. Input files formats can be
FASTA, tab separated text or Newick. The goeBURST algorithm is used to define
the MST that is displayed in a force directed-layout on webGL rendering engine,
which allows displaying simultaneously thousands of isolates on a tree in a web
browser.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Novel features of
this online version are: 1) an interactive heat map visualization of distances
of selected nodes on the tree, where information and heat map ordering can be
selected according to available auxiliary data; and 2) N Locus Variant (NLV)
graph analysis, which generates clusters by modifying the MST and linking all
profiles up to a given distance, allowing a different visualization of how
large datasets are related.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>PHYLOViZ Online
also offers user-specific areas where users can store private datasets for
future access while providing a way to share datasets through the creation of
public links, allowing other researchers to access – but not modify – private
data. A RESTful API provides authenticated programmatic access to dataset
creation, which can then be accessed using the web interface.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>PHYLOViZ
Online allows the online analysis of allelic profile or Single Locus
Polymorphism (SNP) through the use of a web-browser. With the integration of
auxiliary data and by analyzing the distances in a heat map, users can
determine which factors can be relevant in an epidemiological perspective.
Through the user-specific area, researchers have now the ability to share data,
facilitating the exchange of relevant epidemiological information. The RESTful
API facilitates the integration with novel visual and data analysis
applications. This allows the seamless integration of PHYLOViZ Online with
existing databases and other online resources. All data can be exported in CSV
or PDF format and images can be exported in PNG format.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The PHYLOViZ
Online web-application provides a resource allowing MST analysis of profiles
and SNP data without the need to install specific software. PHYLOViZ Online can
be </span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>found
at </span><span lang=DE-CH><a href="http://node.phyloviz.net"><span lang=EN-US
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>http://node.phyloviz.net</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> and the
source code is freely available at </span><span lang=DE-CH><a
href="https://github.com/bfrgoncalves/Online-PhyloViZ"><span lang=EN-US
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>https://github.com/bfrgoncalves/Online-PhyloViZ</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>.<o:p></o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO15 - Comparative
Analysis Sequencing-based Typing Methods For Epidemiological Study Of <i
style='mso-bidi-font-style:normal'>C. trachomatis</i><o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Zhanna Amirbekova<sup>1</sup>; Dmitriy Babenko<sup>1</sup>; Anar
Turmuhambetova<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Karaganda State Medical University<o:p></o:p></span></i></p>

<p class=MsoNormal align=left style='text-align:left'><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Chlamydia
trachomatis is a leading cause of infectious blindness and sexually transmitted
disease worldwide (Gerbase et al. 1998) causing in a variety of body sites
including the eye and the urogenital tract and leading to life-threatening
conditions (Peipert JF, 2003).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Several genetic
approach for typing C.trachomatis have been developed including MLST, MLVA,
PFGE, PCR-based omp-gene typing and others which have advanced from
conventional typing using antibodies (Pedersen et al., 2009).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The aim of present
study was the comparison of sequence-based typing methods for C.trachomatis.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials |
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>The collection of 127 C.trachomatis genomes have been downloaded from </span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Genbank (</span><span
lang=DE-CH><a href="http://www.ncbi.nlm.nih.gov/genome"><span lang=EN-US
style='font-size:8.0pt;mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
mso-fareast-language:PT'>http://www.ncbi.nlm.nih.gov/genome</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>) and have
been analyzed with SeqSphere v.3 (Ridom Software) to extract alleli for MLST
(for Chlamidiales and Uppsala scheme), wgMLST and MOMP gene complex. Diversity
and concordance between typing methods were calculated using </span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Simpson index and
adjusted Wallace coefficients (</span><span lang=DE-CH><a
href="http://www.comparingpartitions.info"><span lang=EN-US style='font-size:
8.0pt;mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:
PT'>http://www.comparingpartitions.info</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>).
Comparison of dendrogram have been performed with dendextend packages (R
statistics).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Using 127
whole genomes of C. trachomatis two MLST types (7 alleli for MLST Chlamydiales
and 5 alleli for Uppsala scheme), wgMLST (933 genes) and alleli for MOMP gene
complex (ompH, ompB and omcB genes) have been determined. Characteristics such
as resolution power and concordance of these typing are presented in table1.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Table 1.
Resolution ability and concordance between sequence-based typing methods for
C.trachomatis.<o:p></o:p></span></p>

<table class=MsoNormalTable border=0 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;mso-yfti-tbllook:1184;mso-padding-alt:0cm 2.85pt 0cm 2.85pt'>
 <tr style='mso-yfti-irow:0;mso-yfti-firstrow:yes;height:15.6pt'>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>Typing method<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'># Different types<o:p></o:p></span></p>
  </td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>Simpson´s index<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>Adj.Rand coefficient<o:p></o:p></span></p>
  </td>
  <td nowrap valign=bottom style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'></td>
  <td nowrap valign=bottom style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'></td>
  <td nowrap valign=bottom style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'></td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>Wallace coefficient<o:p></o:p></span></p>
  </td>
  <td nowrap valign=bottom style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'></td>
  <td nowrap valign=bottom style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'></td>
 </tr>
 <tr style='mso-yfti-irow:1;height:20.4pt'>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:20.4pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>Discriminatory index [95% CI]<o:p></o:p></span></p>
  </td>
  <td nowrap valign=bottom style='padding:0cm 2.85pt 0cm 2.85pt;height:20.4pt'></td>
  <td nowrap valign=bottom style='padding:0cm 2.85pt 0cm 2.85pt;height:20.4pt'></td>
  <td nowrap valign=bottom style='padding:0cm 2.85pt 0cm 2.85pt;height:20.4pt'></td>
  <td nowrap valign=bottom style='padding:0cm 2.85pt 0cm 2.85pt;height:20.4pt'></td>
  <td nowrap valign=bottom style='padding:0cm 2.85pt 0cm 2.85pt;height:20.4pt'></td>
 </tr>
 <tr style='mso-yfti-irow:2;height:15.6pt'>
  <td nowrap valign=bottom style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'></td>
  <td nowrap valign=bottom style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'></td>
  <td nowrap valign=bottom style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'></td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>MLST&nbsp;<o:p></o:p></span></p>
  </td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>MLST&nbsp;<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>wgMLST<o:p></o:p></span></p>
  </td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>MOMP&nbsp;<o:p></o:p></span></p>
  </td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>MLST&nbsp;<o:p></o:p></span></p>
  </td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>MLST&nbsp;<o:p></o:p></span></p>
  </td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>wgMLST<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:3;height:15.6pt'>
  <td nowrap valign=bottom style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'></td>
  <td nowrap valign=bottom style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'></td>
  <td nowrap valign=bottom style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'></td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>(Chlamydiales)<o:p></o:p></span></p>
  </td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>(Uppsala scheme)<o:p></o:p></span></p>
  </td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>complex<o:p></o:p></span></p>
  </td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>(Chlamydiales)<o:p></o:p></span></p>
  </td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>(Uppsala scheme)<o:p></o:p></span></p>
  </td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'></td>
 </tr>
 <tr style='mso-yfti-irow:4;height:15.6pt'>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>MOMP complex<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>23<o:p></o:p></span></p>
  </td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.888&nbsp;<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.603<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.344<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.01<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>1<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.653<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.257<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.006<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:5;height:15.6pt'>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>[0.861 - 0.915]<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:6;height:15.6pt'>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>MLST (Chlamydiales)<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>33<o:p></o:p></span></p>
  </td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.886&nbsp;<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>1<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.363<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.01<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.642<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>1<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.268<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.005<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:7;height:15.6pt'>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>[0.852 - 0.921]<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:8;height:15.6pt'>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>MLST (Uppsala scheme)<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>47<o:p></o:p></span></p>
  </td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.961&nbsp;<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.344<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>1<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.03<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.74<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.785<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>1<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.016<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:9;height:15.6pt'>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>[0.948 - 0.974]<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:10;height:15.6pt'>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>wgMLST<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>123<o:p></o:p></span></p>
  </td>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.999&nbsp;<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.01<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>0.03<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>1<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>1<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>1<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>1<o:p></o:p></span></p>
  </td>
  <td rowspan=2 style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>1<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:11;mso-yfti-lastrow:yes;height:15.6pt'>
  <td style='padding:0cm 2.85pt 0cm 2.85pt;height:15.6pt'>
  <p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
  style='font-size:6.0pt;font-family:Calibri;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:"Times New Roman";color:black;mso-ansi-language:EN-GB;
  mso-fareast-language:EN-GB'>[0.998 - 1.0]<o:p></o:p></span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Clustering rates
for MOMP complex, MLST (chlamydiales), MLST (Uppsala) and wgMLST were 0.795,
0.74, 0.63 and 0.031 respectively. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Clonal complex
structure based on burst algorithm showedthat there was 1 complex with 119
samples for MLST Uppsala, 1 CC with 127 Samples for MOMP gene set, 3CC with 115
Samples for MLST Chlamydiales and 12 CC with 106 Samples for wgMLST.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Dendrogram
comparing and entanglement parameter is shown in figure 1.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>wgMLST as
perspective typing approach based on whole genome sequence showed the ultimate
level of discriminatory power – 99,9%. Furthermore, wgMLST is able to predict
well the other sequence type (Wallace coef was 100%).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Generally, there
was a poor/moderate concordance between typing methods using several
genes/alleli – no more 75%.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='mso-ansi-language:
EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape id="Imagem_x0020_8" o:spid="_x0000_i1028"
 type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/206imagem_2560108707_0.jpg"
 style='width:333pt;height:236pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S1_files/image003.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/206imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=238
src="public/abstracts/Webpages/PO_S1_files/image004.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/206imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_8"><![endif]></span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO16 - Appraisal Of
Complementary Strategies For Refinement Of A Core Genome Multi-locus Sequence
Typing Scheme For Typing Of <i style='mso-bidi-font-style:normal'>Acinetobacter
baumannii</i><o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Mickael Silva<sup>1</sup>; João André Carriço<sup>1</sup>; Nir Gilad<sup>2</sup>;
Michal Gordon<sup>2</sup>; Mario Ramirez<sup>1</sup>; Vered Chalifa-Caspi<sup>2</sup>;Jacob
Moran-Gilad<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>Faculdade de Medicina, Instituto de Microbiologia,
Instituto de Medicina Molecular, Universidade de Lisboa, <sup>2</sup>Surveillance
and Pathogenomics Israeli Centre of Excellence, National Institute for
Biotechnology in the Negev, Ben-Gurion University of the Negev<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
PT;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Objectives</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: <i>Acinetobacter</i>
<i>baumannii </i> is a therapeutic and infection control challenge worldwide.
Microbial whole genome sequencing (WGS) currently allows bacterial typing based
on a gene-by-gene approach using a large set of core genome genes as compared
to only seven housekeeping genes in traditional MLST, thus achieving a more
refined population structure analysis and increased discriminatory power. Here
we propose a novel core genome MLST (cgMLST) typing scheme for <i>A. baumannii</i>
complex.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: For the
construction of the wgMLST scheme we downloaded from NCBI, 16 closed <i>A.baumannii</i> 
genomes. Processing via an in-house bioinformatics pipeline identified a set of
single copy genes, present in at least one strain. To validate the scheme, a
total of 1042 genomes were analyzed. This included the majority of strains
annotated in ENA as A.<i>baumannii</i> and sequenced by Illumina technologies
(987 reads sets). Draft assemblies were generated using SPAdes . An allele
calling algorithm was developed and applied to define standardized alleles for
the 1042 genomes. Considering the issue of missing data and possible assembly problems,
a method was developed to identify the core genome by removing strains with
defined levels of missing data. The algorithm considers an exclusion threshold
(<i>et</i>) at which a genome with loci with missing data over the <i>et</i>
will be excluded by analyzing loci present in over 99% of the whole set of
genomes. This algorithm runs iteratively for each <i>et</i> in order to
maximize the number of cgMLST loci. Higher <i>et</i>s will provide a higher
typeability, with more included genomes and less cgMLST loci, while lower <i>et</i>s
will provide a higher discriminatory power, with less genomes included in the
analysis but more cgMLST loci.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: The
initial wgMLST gene set derived from reference genomes consisted of 5,266 loci.
Running the algorithm to identify the core genome we were able to generate two
schemes. A restrictive core (<i>RS</i>) genome scheme at a <i>et</i> of 60 and
an inclusive (<i>IS</i>) at <i>et</i> of 190. <i>RS</i> classified 974 strains
with 718 unique profiles and using 942 loci, whereas <i>IS</i> classified 997
strains into 593 unique profiles using 551 loci.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: Both
proposed cgMLST schemes offered a high discrimination between <i>A.baumannii</i> 
strains and allowed the differentiation of strains belonging to other A.<i>baumannii</i>
complex species, wrongly submitted to public databases as A.<i>baumannii</i>.
However each scheme demonstrated its own advantages and caveats that have to be
considered prior to their usage. The cgMLST approaches can allow standard
common allelic nomenclature and  easy implementation in automated analysis
pipelines, and is thus expected to enable the adoption of draft genome
sequencing as a mainstream procedure in clinical microbiology.<o:p></o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO17 - MLVA For The
Genotyping Of <i style='mso-bidi-font-style:normal'>Acinetobacter baumannii</i>:
Towards A Consensus For Interpretation Of Data And The Development Of
International Databases<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Christine Pourcel<sup>1</sup>; Yolande Hauck<sup>1</sup>; Charles SOLER<sup>2</sup>;
Gilles Vergnaud<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University Paris-Saclay, <sup>2</sup>Percy military
hospital<o:p></o:p></span></i></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: Infections
by <i>A. calcoaceticus</i>-<i>A. baumannii</i> (ACB) complex isolates represent
a serious threat for wounded and burn patients. Three international
multidrug-resistant (MDR) clones (EU clone I-III) are responsible for a large
proportion of nosocomial infections with <i>A. baumannii</i> but other emerging
strains with high epidemic potential also occur. Different genotyping
techniques that not always allow differentiation of strains inside clones have
been described. Multiple locus VNTR analysis (MLVA) based upon variable number
of tandem repeats (VNTR) polymorphism is among the most promising methods owing
to its low cost, discriminatory power, and the possibility to create shared
databases over the internet. Since the initial report of VNTRs in Acinetobacter
baumanii in 2011, and the proposition of an MLVA assay, hundreds of additional
strains have been genotyped worldwide, and the corresponding data has been made
public. In parallel, additional strains have been fully sequenced. We here take
advantage of this context to update the available internet MLVA database and
MLVA assay.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials and methods: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>The </span><span
lang=DE-CH><a href="http://mlva.u-psud.fr"><span lang=EN-US style='font-size:
8.0pt;mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:
PT'>http://mlva.u-psud.fr</span></a></span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> resource and
associated tools was used to create a MLVA cooperative database, from published
and unpublished data set. The experimental genotyping data set was complemented
by <i>in silico</i> deduced MLVA genotypes using available complete genome
sequences (downloaded from sequence databases accessed via the NCBI resources).
PubMed was searched to identify MLVA publications.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Five studies
using MLVA to genotype <i>Acinetobacter baumanii</i> published since the
initial 2011-2012 MLVA reports were identified. Altogether, more than 800
strains were genotyped, allowing the critical assessment of MLVA. The full
genome from more than 30 strains was in silico analysed to deduce their MLVA
genotype. The data was used, together with data from the current MLVA database
based upon the 2012 published data to create a new cooperative database
comprising seven elementary databases for a total of more than 1000 strains. Different
combinations of markers have been proposed by different groups, they are all
represented in the new database.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>MLVA assays,
such as MLVA8<sub>Orsay</sub> which employs 4 VNTRs with low level of
variability (the L markers) and 4 VNTRs with a high level of polymorphism (the
S marker) are fast and cheap and can be applied to different epidemiological
investigations made necessary by the increasing occurrence of <i>A. baumannii</i>
infections in the hospital. The new version of the MLVA database which will be presented
should facilitate such investigations.<o:p></o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO18 - Straintracer-db:
A Flexible Analysis And Persistent Storage System For Sequence And
Epidemiological Data<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-bidi-font-size:9.0pt;mso-bidi-font-family:
Arial;mso-ansi-language:EN-US'>Jonas Evensen<sup>1</sup>; Sveinung Gundersen<sup>2</sup>;
Anja B. Kristoffersen<sup>3</sup>; <u>Karin Lagesen</u><sup>4</sup></span><span
style='mso-bidi-font-size:9.0pt;mso-bidi-font-family:Arial;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-bidi-font-family:Arial;mso-ansi-language:
EN-US'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-bidi-font-family:Arial;mso-ansi-language:
EN-US'>Department of Informatics, University of Oslo; <sup>2</sup>ELIXIR.NO /
Department of Informatics, University of Oslo; <sup>3</sup>Norwegian Veterinary
Institute; <sup>4</sup>Norwegian Veterinary Insitute/Department of Informatics,
University of Oslo<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Traditionally,
the transmission pathways of infectious diseases have been tracked using
epidemiological data, such as time, location and contact information. Today,
new technologies allow for direct analysis of the genome sequence of an
infectious agent. Based on differences detected between the genomes of
different isolates, the transmission pathways of the agent can potentially be
elucidated. In addition, inferences of how an agent moves in time and space can
be greatly strengthened by combining epidemiological data with sequence
information. Systems for managing and analysing such data need to satisfy a
range of requirements. For instance, it may not be desirable or permissible to
share the data outside of the host institution; hence the system should be
locally installable and usable. Also, bioinformatics analyses are often very
resource intensive, thus such a system should be able to run on compute
clusters. In addition, it should allow for storing already processed results to
enable easy comparison with new results. The system should also allow its users
to create new analyses to enable them to explore the data in novel ways.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods and
Materials: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>In this project we combine Galaxy a web-based
bioinformatics analysis platform, with a postgres database for persistent
storage of epidemiological data, sequence information, analysis metadata and
results. Galaxy gives users access to many different tools and allows users to
install new tools themselves. Tools can be chained into reusable workflows,
thus making results from different runs comparable. Additionally, Galaxy can be
installed and used on a local PC as well as on large compute clusters.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>StrainTracer-DB
is a flexible analysis and persistent storage system for sequence and
epidemiological data. StrainTracer-DB is a collection of tools inside Galaxy
that allows the combination of many useful features of Galaxy with a database
designed to store data produced throughout all steps necessary for an analysis.
StrainTracer-DB provides tools for uploading sequence data for specific
biological agents. Within the tool, the metadata that should be associated with
the sequence information is specified. A Galaxy workflow is also included in
the tool, and once that is run, the results including run information is stored
in the database. The database storing the results is designed to be very
flexible. Thus the workflows processing the data can provide SNP data, “shared
gene” data, allele data and more as their results. StrainTracer-DB includes
tools that allow for accessing previous results from the database, so that
results based on new data can be compared to those already processed. All tools
and data sets are under access control, so that only authorised users have
access and modification rights to specific tools and workflows, and so that
different groups of users can share the same system.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>StrainTracer-DB
offers an easy-to-use system for managing and analysing sequence and
epidemiological data. We envision that the system will make it easier and
faster to trace the transmission pathways and origins of infectious diseases.<o:p></o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO19 - MicrobesNG: A
Microbial Sequencing Facility Designed To Integrate Myriad Of Sequenced Whole
Genomes With Their Samples<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Pablo Fuentes Utrilla<sup>1</sup>;Emily Richardson<sup>1</sup>; Joshua
Quick<sup>1</sup>; Jennie Law<sup>1</sup>; Andrew Smith<sup>1</sup>; Ian
Henderson1; Roy Chaudhuri<sup>2</sup>; Nick Loman<sup>1</sup>; Tim Wells<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University of Birmingham, University of Sheffield<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-font-weight:
bold'>Improvements in sequencing technology have driven down the cost of
sequencing bacterial genomes. However, it remains time consuming and expensive
to establish efficient high-throughput sequencing in non-specialist
laboratories. MicrobesNG is a BBSRC-funded facility to provide sequencing of
microbial genomes. The facility aims to i) advance sequencing protocols to
improve data quality ii) reduce cost and decrease processing time, iii) setup
up a repository for the storage and delivery of bacterial strains, iv) develop
pipelines for processing sequencing data and providing basic bioinformatics v)
develop tools with user friendly interfaces that allow scientists to not only
analyse their own data but to see it in the context of all relevant publicly
available strains.</span><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-font-weight:
bold'>MicrobesNG accepts either genomic DNA or individual bacterial strains,
deposited on bead tubes. We have established laboratory automation protocols
that are capable of generating 480 samples per week using the Hamilton STAR
robotics platform running a modified Nextera XT protocol, with libraries
sequenced on 250bases paired-end on Illumina HiSeq/MiSeq instruments. Efforts
have been expended in ensuring the longest possible insert sizes are produced
with this protocol in order to give the most contiguous assemblies. These
protocols have also been optimised for speed and are available for others to
reuse. In order to encourage sharing and reuse of data, to access the service
at its lowest subsidised price we mandate a policy of strain and data sharing,
with all data being uploaded to the European Nucleotide Archive within one year
of sending strains for sequencing. We have established bioinformatics protocols
for quality control of data.&nbsp;</span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-font-weight:
bold'>MicrobesNG currently utilises three main pipelines, QC, assembly and
variant calling. The QC pipeline uses Kraken to identify the closest reference
genome, to which the reads are then aligned to. The Kraken output is also used
to assess contamination. The assembly pipeline uses SPAdes to assemble the
genomes and PROKKA to annotate. The variant pipeline uses BWA-MEM to align to
the reference, samtools to process the alignment, Varscan to call variants and
SnpEff to add annotation context to the results. Two types of variant calling
are performed, sensitive (variant present in &gt;90% of depth) and specific
(variant present in &gt;10% depth). In order to visualise variants, a custom
interface has been built using the Dalliance browser (figure 1).</span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-font-weight:
bold'>So far we have sequenced 4000 isolates using this system. Currently we
are able to sequence a microbial genome for a consumable cost of &lt; £45
($65). Through further anticipation we anticipate that this cost can be halved
by the end of 2016.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='mso-ansi-language:
EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape id="Imagem_x0020_9" o:spid="_x0000_i1027"
 type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/216imagem_2560108707_0.jpg"
 style='width:333pt;height:260pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S1_files/image005.png" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/216imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=262
src="public/abstracts/Webpages/PO_S1_files/image006.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/216imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_9"><![endif]></span><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:8.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
"Times New Roman";color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO20 - A Scripting
Language For Standardized Evaluation Of Quality Metrics In Galaxy And
Command-line Driven Workflows<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Damion M. Dooley<sup>1</sup>; Aaron J. Petkau<sup>2</sup>; Franklin Bristow<sup>2</sup>;
Gary Van Domselaar<sup>2</sup>; William W.L. Hsiao<sup>3</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Department of Pathology, University of British
Columbia, <sup>2</sup>National Microbiology Laboratory, Public Health Agency of
Canada, <sup>3</sup>Department of Pathology, University of British Columbia
&amp; BC Public Health Microbiology and Reference Laboratory<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT'>Keywords</span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>: </span><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT'>Quality control - Metrics - Pipeline - Workflow - Whole genome sequencing<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Background: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Quality control (QC) metrics in a genomics
assembly and annotation pipeline are essential for routine lab sample
processing, and for comparing the performance of pipeline optimizations or
alternatives. Ideally sequence run read metrics and other assembly or
annotation stats can be collected and analyzed in real-time to enable low QC
scores to stop otherwise unnecessary processing and manual analysis. &nbsp;QC
reporting can also enable retrospective tracing of problems that arise.
Unfortunately QC metrics are a complicated affair influenced by the subject
organism, sample quality, sequencing technology and preparation, and the mix of
software components that are brought together in a workflow. The challenge is
to transform QC reporting from a manual process of perusing a workflow’s
disparate and often opaque application log files, into an intermediate and
automated system&nbsp;of reporting and decision making that can control workflow
execution, and that can be adjusted by researchers and system administrators
who are not expert programmers.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> We have developed a general purpose text-mining
and reporting application called Report Calc for Quality Control (RCQC) that
works directly within command-line scripts, or as a tool in Galaxy (an
interactive bioinformatics platform and workflow engine). &nbsp;Our test case
is an assembly and annotation pipeline used in the IRIDA.ca (Interactive Rapid
Infectious Disease Analysis) project, one that combines command-line Flash,
Spades, and Prokka tools. &nbsp;For a given workflow job, RCQC extracts QC
variables from various pipeline log files, and it implements a variety of rules
that can appropriately trigger a warning or failure status in the workflow at
large.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusions:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> The Report Calc for Quality Control system
provides a transparent and adjustable ruleset that reduces the skill needed to
make concise reports and process adjustments based on QC metrics. &nbsp;We were
able to standardize many aspects of QC reporting and job handling while
remaining within the limits of Galaxy’s workflow engine. A few standard
assembly pipeline QC metrics are introduced which provide a blueprint
for&nbsp;the way QC components could be shared amongst NGS sequencing pipelines.
&nbsp;Further information, including source code, is available at </span><span
lang=DE-CH><a href="https://github.com/Public-Health-Bioinformatics/rcqc"><span
lang=EN-US style='font-size:8.0pt;mso-bidi-font-family:Arial;mso-ansi-language:
EN-US;mso-fareast-language:PT'>https://github.com/Public-Health-Bioinformatics/rcqc</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='mso-ansi-language:
EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape id="Imagem_x0020_10"
 o:spid="_x0000_i1026" type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/220imagem_2560108707_0.jpg"
 style='width:333pt;height:200pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S1_files/image007.png" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/220imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=202
src="public/abstracts/Webpages/PO_S1_files/image008.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/220imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_10"><![endif]></span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='mso-ansi-language:
EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape id="Imagem_x0020_11"
 o:spid="_x0000_i1025" type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/220imagem_2560108707_1.jpg"
 style='width:333pt;height:200pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S1_files/image009.png" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/220imagem_2560108707_1.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=202
src="public/abstracts/Webpages/PO_S1_files/image010.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/220imagem_2560108707_1.jpg"
v:shapes="Imagem_x0020_11"><![endif]></span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO21 - GTFinder
Workflow To Detect Exclusively Conserved Region Across A Large Genome Dataset: <i
style='mso-bidi-font-style:normal'>Salmonella</i> spp. Study<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Arnaud Felten<sup>1</sup>; Laurent Guillier<sup>1</sup>; Fabrice Touzain<sup>2</sup>;
Michel-Yves Mistou<sup>1</sup>; Renaud Lailler<sup>1</sup>; Nicolas Radomski<sup>1</sup>;
Sabrina Cadel Six<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Université PARIS-EST,Anses, Laboratory for food
safety, <sup>2</sup>Anses, Genomic platform<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Keywords: exclusively conserved regions, <i>de novo</i> assembly, k-mer
clustering, <i>Salmonella</i>, bacterial genomics.<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Background:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> The modern molecular microbiologists are today
in front of massive databases of DNA sequences produced by high-throughput
sequencing, and need to develop their own <i>in silico</i> tools according to
their specific objectives. In a context of food safety, the <i>Salmonella</i>
serotyping method based on sero-agglutination is cost prohibitive, and leads to
propose less expensive molecular targets to identify <i>Salmonella</i>
serotypes by PCR-based methods. Consequently, we propose in the present
manuscript a workflow called ‘Genome Target Finder’ (GTFinder) which has been
designed to detect exclusively conserved k-mers across a large genome dataset
of <i>Salmonella</i> serovars.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> Based on stringent and tolerant clustering
algorithms implemented in the CD-HIT programs, the GTFinder workflow includes
Python scripts driving k-mers through redundancy, specificity, sensitivity, and
combination filtrations, successively. Among 526 genomes from NCBI, as well as
211 newly sequenced and assembled genomes, 90 serovars of <i>Salmonella</i>
spp. were taken into account, including 48 and 31 serovars represented by only
one genome and more than four genomes, respectively. Concerning the 31 serovars
represented by more than four genomes, unique k-mers were detected as
exclusively conserved into 18 serovars, and combined k-mers were selected to
identify the 13 remaining serovars.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusion:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> The unique and combined k-mers identified by
GTFinder workflow as specific and sensitive into 31 serovars, will be used to
design PCR primers in order to check their <i>in vitro</i> specificity and
sensitivity by the Access Array system from Fluidigm. The GTFinder workflow
could also be applied to other bacterial genus, species, and subspecies in
order to discover regions distinguishing between genotypes or phenotypes.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO22 - MOST: A Modified
MLST Typing Tool Based On Short Read Sequencing<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Rediat Tewolde<sup>1</sup>; Tim Dallman<sup>1</sup>; Ulf Schaefer<sup>1</sup>;
Carmen Sheppard<sup>1</sup>; Philip Ashton1; Bruno Pichon<sup>1</sup>; Craig
Swift<sup>1</sup>; Jonathan Green<sup>1</sup>; Anthony Underwood<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Public Health England<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Multilocus
sequence typing (MLST) is one of the most commonly used typing methods used for
surveillance and identification of outbreaks. The conventional MLST technique
involves Polymerase Chain Reaction (PCR)&nbsp;amplification of 5-10
housekeeping genes followed by Sanger DNA sequening.&nbsp;Public Health England
(PHE) is in the process of replacing the conventional MLST methodology used by
the reference services with Whole Genome Sequencing (WGS). In order for PHE to
replace the conventional method, validation tests should demonstrate
concordance between the WGS ouputs and the conventional assay in conjunction
with quantitative quality metrics for ongoing assurance purposes.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials |
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>This presentation compares the reliability and quality of MLST results
derived from WGS data using de novo assembly/blast-based and mapping based
approaches to MLST results derived from the conventional Sanger sequencing
method. Sensitivity and specificity of the two WGS based methods were further
investigated with a set of mixed and low coverage genomic data. The initial set
of isolates used for the validation included 120 <i>Campylobacter sp. ( Coli
and jejuni)</i> ,&nbsp; 121&nbsp; <i>Staphylococcus aureus</i> and 99 <i>Streptococcus
pneumoniae</i> isolates. These were all characterized by conventional and WGS
based MLST methods . A further 26 mixed strain samples and 29 low coverage
samples were used to investigate sensitivity and specificity of the WGS based
MLST methods.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The assembly
and mapping approaches obtained a full MLST profile more often than the
conventional method. The concordance between the conventional and both WGS
methods was 100%.&nbsp;For both mixed strain and low coverage samples the
mapping based methodology obtained a full MLST profile more often than the
assembly approach.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>In
conclusion, deriving MLST from WGS is sensitive as well as being less labour
intensive than the conventional method. In addition, the mapping based approach
assigns quality metrics that flag mixed and low sequence coverage genomic data
from bacterial populations, something that is difficult to determine using both
conventional methodologies and WGS assembly based approaches.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO23 - The <i
style='mso-bidi-font-style:normal'>Brucella</i> MLVA Database<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Gilles Vergnaud<sup>1</sup>; Yolande Hauck<sup>1</sup>; Christine Pourcel<sup>1</sup>;
Isabelle Jacques<sup>2</sup>; Axel Cloeckaert<sup>2</sup>; Michel Zygmunt<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University Paris-Saclay, INRA</span></i><span
lang=EN-GB style='mso-bidi-font-family:Arial;color:black;mso-themecolor:text1;
mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Polymorphic tandem
repeats have proved to be sufficiently numerous in the <i>Brucella</i> genome
to allow the development of Multiple Loci VNTR (Variable Number of Tandem
Repeats) Analysis (MLVA) assays. MLVA assays are interesting because they rely
on widely available technologies, i.e. PCR amplification and DNA fragment
length measurement. They do not require the manipulation of the live agent, and
can be run at low cost in any laboratory. DNA fragment lengths are subsequently
converted into a number of repeat units and the resulting genotypes can be
easily stored, compared, shared, and made accessible in databases via the
internet.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The most widely
used <i>Brucella</i> MLVA assay is composed of 16 loci, eight with large repeat
units which can be analysed with many different equipment including regular
agarose gels, and eight with shorter repeat units. The 16 loci can be typed in
four multiplex PCRs if the appropriate equipment is available.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-fareast-language:PT'>An MLVA database can be
accessed at </span><span lang=DE-CH><a href="http://mlva.u-psud.fr"><span
lang=EN-US style='font-size:8.0pt;mso-bidi-font-family:Arial;mso-ansi-language:
EN-US;mso-fareast-language:PT'>http://mlva.u-psud.fr</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>. It
contains genotypes from 2500 strains, organized in nine contributing databases
contributed by different laboratories.<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>We will present a new version of the database, with extensive improvements
of the underlying software and inclusion of new data from approximately 2000
strains. Representatives from the newly described <i>Brucella</i> species are
included. The new version contains genotypes from more than 4000 strains. This
central repository also enables the naming of new genotypes in a coordinated
manner.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=DE-CH><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
