<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns:mv="http://macVmlSchemaUri" xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta name=Title content="">
<meta name=Keywords content="">
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List href="PO_S2_files/filelist.xml">
<link rel=Edit-Time-Data href="PO_S2_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Bruno Gonçalves</o:Author>
  <o:LastAuthor>Bruno Gonçalves</o:LastAuthor>
  <o:Revision>1</o:Revision>
  <o:TotalTime>1</o:TotalTime>
  <o:Created>2016-03-08T21:49:00Z</o:Created>
  <o:LastSaved>2016-03-08T21:50:00Z</o:LastSaved>
  <o:Pages>12</o:Pages>
  <o:Words>5884</o:Words>
  <o:Characters>33541</o:Characters>
  <o:Company>IMM</o:Company>
  <o:Lines>279</o:Lines>
  <o:Paragraphs>78</o:Paragraphs>
  <o:CharactersWithSpaces>39347</o:CharactersWithSpaces>
  <o:Version>14.0</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="PO_S2_files/themedata.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:GrammarState>Clean</w:GrammarState>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>JA</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
   <w:UseFELayout/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="276">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1073743103 0 0 415 0;}
@font-face
	{font-family:"Lucida Grande";
	panose-1:2 11 6 0 4 5 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-520090897 1342218751 0 0 447 0;}
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	mso-themecolor:hyperlink;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:purple;
	mso-themecolor:followedhyperlink;
	text-decoration:underline;
	text-underline:single;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Balloon Text Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:9.0pt;
	font-family:"Lucida Grande";
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-ansi-language:DE-CH;}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Balloon Text";
	mso-ansi-font-size:9.0pt;
	mso-bidi-font-size:9.0pt;
	font-family:"Lucida Grande";
	mso-ascii-font-family:"Lucida Grande";
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:"Lucida Grande";
	mso-bidi-font-family:"Lucida Grande";
	mso-ansi-language:DE-CH;}
span.GramE
	{mso-style-name:"";
	mso-gram-e:yes;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"ＭＳ 明朝";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
@page WordSection1
	{size:595.0pt 842.0pt;
	margin:72.0pt 89.85pt 72.0pt 89.85pt;
	mso-header-margin:35.45pt;
	mso-footer-margin:35.45pt;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;}
</style>
<![endif]-->
</head>

<body bgcolor=white lang=EN-US link=blue vlink=purple style='tab-interval:36.0pt'>

<div class=WordSection1>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO24 - Comparative
Genomic Analysis Of Two NetF-positive <i style='mso-bidi-font-style:normal'>Clostridium
perfringens</i> Isolates Associated With Foal And Canine Necrotizing Enteritis<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Iman Mehdizadeh Gohari<sup>1</sup>; Andrew M. Kropinski<sup>1</sup>; Scott
J. Weese<sup>1</sup>; Valeria R. Parreira<sup>1</sup>; Ashley E. Whitehead<sup>2</sup>;
Patrick Boerlin<sup>1</sup>; John F. Prescott<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Department of Pathobiology, University of Guelph<span
class=GramE>,<sup>2</sup>Department</span> of Veterinary Clinical and
Diagnostic Sciences, University of Calgary<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Clostridium
perfringens</span></i><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> type A-associated diarrheal and enteric disease
in foals and dogs is not well characterized. However, recently, our group
described three novel putative toxin genes encoding proteins related to the
pore-forming Leukocidin/Hemolysin Superfamily; these were designated <i>netE</i>,
<i>netF</i>, and <i>netG</i>. NetF has been implicated as the primary virulence
factor of foal necrotizing enteritis and canine haemorrhagic gastroenteritis.
This study presents the complete genome sequence of two <i>netF</i>-positive
strains, JFP55 and JFP838, which were recovered from cases of foal necrotizing
enteritis and canine haemorrhagic gastroenteritis, respectively. Work is
continuing on sequencing 30 additional <i>netF</i>-positive strains from dogs
and horses that will be used for epidemiological analysis.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials and
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>The genomic DNA of the samples was extracted using a modified version of
the Qiagen bacterial DNA extraction protocol. Single Molecule, Real-Time (SMRT)
technology-PacBio and Illumina Hiseq2000 PE100 were used for both samples. <i>De
novo </i>assembly was done using DNASTAR’s SeqMan NGen12 software.
Subsequently, the <span class=GramE>complete chromosome sequences were
annotated by the prokaryotic genome annotation pipeline</span>. In addition,
complete plasmid sequences were annotated by MyRAST software, the next
generation of Rapid Annotation using Subsystem Technology. Sequencing of an
additional 30 strains is proceeding with Illumina Hiseq2000 PE100.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The JFP55
and JFP838 genomes include a single 3.34 <span class=GramE>Mb</span> and 3.53
Mb chromosome, respectively, and both genomes include five circular plasmids.
The <i>C. perfringens</i> JFP55 chromosome carries 2,825 protein-coding genes
whereas the chromosome of JFP838 contains 3,014 protein-encoding genes.
Comparison of these two chromosomes with three available reference <i>C.
perfringens</i> chromosome sequences identified 48 (~ 247 kb) and 81(~ 430 kb)
regions unique to JFP55 and JFP838, respectively. Sixteen of these unique
chromosomal regions (~69 kb) were shared between the two isolates. Five of
these shared regions formed a mosaic of plasmid-integrated segments, suggesting
that these elements were acquired early in a clonal lineage of <i>netF</i>-positive
<i>C. perfringens</i> strains. These shared chromosomal regions could possibly
represent epidemiology markers for this clonal lineage. In addition, plasmid
annotation revealed that both <i>netF</i>-positive <i>C. perfringens</i>
strains, JFP55 and JFP838, harbour three plasmids in common, including a
NetF/NetE toxins-encoding plasmid, a CPE/CPB2 toxins-encoding plasmid and a
putative bacteriocin-encoding plasmid. The common plasmid profiles may also
represent valuable epidemiological markers for <i>netF</i>-positive <i>C.
perfringens</i> strains. Sequencing of an additional 30 canine and equine <i>netF</i>-positive
using JFP55 and JFP838 as scaffolds will be used for epidemiologic purposes to
examine whether there are canine and equine specific strains, the presence of
conserved plasmids, and the evolution of this lineage(s) of <i>C. perfringens</i>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> These
results provide potential useful molecular markers for <i>netF</i>-positive <i>C.
perfringens</i> by identifying unique chromosomal regions shared by
netF-positive strains as well as common plasmids profiles.<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO25 - Population
Profiling Of <i style='mso-bidi-font-style:normal'>Enterococcus faecium</i>: A
Comparative Study Of Diversity And In-vivo Gene Transfer Events Within The
Human Bowel<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>S. A. Ballard<sup>1</sup>; S. Xie<sup>1</sup>; A. Mahony<sup>1</sup>; A.
Buultjens<sup>2</sup>; E. A. Grabsch<sup>3</sup>; M. L. Grayson<sup>1</sup>; T.
Seemann<sup>4</sup>; B. P. Howden<sup>5</sup>; T. Stinear<sup>6</sup>; P. D. R.
Johnson<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Department of Infectious Diseases, Austin Health<span
class=GramE>,<sup>2</sup>Department</span> of Microbiology and Immunology,
Doherty Institute of Infection and Immunity,<sup>3</sup>Department of
Microbiology Austin Health,<sup>4</sup>VLSCI, The University of Melbourne,<sup>5</sup>MDUPHL,
Doherty Institute for Infection and Immunity,<sup>6</sup>Department of
Microbiology and Immunology, Doherty Institute for Infection and Immunity<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Using <span
class=GramE>large scale</span> whole genome sequencing we have shown previously
that acquisition of vancomycin resistant <i>E. faecium</i> (VREfm) is
frequently driven by <i>de-novo</i> acquisition of the Tn<i>1549</i> element
and that these events may obscure the true rate of nosocomial transmission of
VREfm. Moreover, the population of <i>E. faecium,</i> <i>a priori,</i> to these
events appears to be of significance and likely drives the evolution of
clinically important vancomycin resistant lineages. This longitudinal study
uses genomics to explore <i>E. faecium</i> diversity within and between
patients, to provide insights into the frequency and diversity of vancomycin
susceptible populations within the fecal microbiota in attempt to identify
factors driving the development of new VREfm strains and their expansion.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials &amp;
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>A large prospective cohort of patients at risk of acquisition of VRE was
recruited from Austin Health, Heidelberg, Australia. Faecal samples were
collected on admission and up to seven occasions looking for patient conversion
to VREfm. Faeces were screened for VREfm and without selection for vancomycin
sensitive <i>E. faecium</i> (VSEfm) both directly and by enrichment broth to
improve the isolation of minority populations. Subsequently all isolates were
subjected to whole genome sequencing, their resistance phenotype was confirmed,
an <i>in silico</i> determination of MLST made and the core genome phylogeny of
isolates examined.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Of the 186
patients recruited, 25 transitioned to VRE carriage; 1/25 demonstrated probable
<i>de novo</i> generation of VREfm via transfer of <i>vanB</i> between
enterococci. A total of 1188 <i>E. faecium</i> isolates were obtained along with
2 <i>vanB</i>-Tn<i>1549</i> anaerobes. VREfm isolates represented 7 different
MLSTs. In contrast there was exceptional diversity in MLST amongst VSEfm with
92 different STs and up to 30 uncharacterised STs, highlighting the potential
depth of minority populations of VSEfm. This diversity was also observed within
patients, with up to 7 different VSEfm clones isolated from a single patient
specimen. Complex antibiotic exposure and chemotherapy lead to loss of
diversity and acquisition of the dominant hospital clone.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>In
conclusion, acquisition of VRE from the hospital environment is still an
important mechanism of transmission and antibiotic selection pressure appears
to play an important role in the acquisition of hospital adapted VREfm.
However, whilst periodic selection may place limits on genetic diversity of
hospital populations of <i>E. faecium</i> it does not prevent divergence nor
limit diversity. The cloud of VSEfm clones within the human bowel, the
plasticity of their genome and the potential for lateral gene transfer likely
contribute significantly to the evolution of new VRE strains.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO26 - Elucidation Of
Transmission Events Of Vancomycin Resistant <i style='mso-bidi-font-style:normal'>Enterococcus
faecium</i> Isolates In Intensive Care Units Of A German Tertiary-care Hospital<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Jennifer Bender<sup>1</sup>; Carola Fleige<sup>1</sup>; Torsten Semmler<sup>1</sup>;
Sandra Schneider<sup>2</sup>; Petra Gastmeier<sup>2</sup>; Guido Werner<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University Medicine Berlin<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-font-style:italic'>Background</span></b><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'> According to the recent EARS-Net report, Germany is one of the few
European countries where numbers of vancomycin resistant enterococci (VRE)
isolated from bloodstream infections have constantly increased since 2002.
Also, data from the German National Nosocomial Surveillance System report a
significant increase of VRE frequency in intensive care units (ICUs) (Gastmeier
<i>et al</i>., doi: 10.1093/jac/dku035). Further molecular typing of isolates
received by the National Reference Centre for Staphylococci and Enterococci at
the Robert Koch Institute revealed distinct sequence types highly prevalent in
German hospitals. However, little is known about the clonal relatedness and
success of dominant lineages in ICUs with a persistent burden of VRE. Thus, we
set out to characterize 176 VRE derived from a study on Surveillance of
Antibiotic Usage and Bacterial Resistance (SARI-IQ) from 6 ICUs with respect to
population structure and dynamics.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-font-style:italic'>Methods:</span></b><i><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Enterococcus
faecium </span></i><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>isolates were collected by the Charité hospital between 2014 and 2015 and
sent to the NRC for detailed molecular analyses. The entire set of strains was
analyzed by the rep-PCR-based method DiversiLab® and whole genome sequencing
(WGS) utilizing Illumina technology. WGS data were used to infer phylogenies by
means of a mapping and SNP-based approach, a comparison of the maximum common
genome and by utilizing the commercially available software SeqSphere (Ridom,
Münster, Germany).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-font-style:italic'>Results:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Initial MLST
analyses proved to be insufficient to allocate specific isolates to entry or
transmission events across ICUs due to limited discriminatory power.
Conflicting results were obtained by analyzing the population structure derived
from DiversiLab® results. As an example, several smaller (n&lt;10) or larger (n&gt;25)
DL clusters combined obviously different VRE isolates demonstrating <i>vanA</i>
and <i>vanB </i>genotypes, varying presence of virulence determinants and
different antibiotic susceptibility profiles. In addition, some DL clusters
contained isolates with up to 4 different MLST types. Hence, whole genome
sequencing was applied for maximal resolution to discriminate entry,
transmission and persistence events (ongoing analyses).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-font-style:italic'>Conclusion</span></b><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'> WGS provides maximal discriminatory power to monitor entry and/or
transmission events between hospital <span class=GramE>wards which</span> is of
utmost importance in order to reduce the possibility of VRE infections in high
risk patients.</span><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO28 - Lack Of Genetic
Isolation By Distance And Persistence Of Monomorphic Genotypes In Generalist <i
style='mso-bidi-font-style:normal'>Campylobacter jejuni</i> Strains<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Mirko Rossi<sup>1</sup>; Ji Zhang<sup>1</sup>; Ann-Katrin Llarena<sup>1</sup>;
Minna Vehkala<sup>1</sup>; Niko Välimäki<sup>1</sup>; Jukka Corander1<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University of Helsinki<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> <span
class=GramE>Identifying spatiotemporal genetic changes in the pathogen
populations circulating in a given area is</span> critical for forecasting the
epidemiological relationship between isolates. <i>C. <span class=GramE>jejuni</span></i>
ST-45 clonal complex (cc) is a genotype frequently detected in human infections
and colonize chicken flocks worldwide. However, little is known on the
population stability over time and space. Thus, the aim of this study was to
explore spatiotemporal patterns of genetic diversity within ST-45cc by
phylogenomic and genome-wide association studies (GWAS).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> The genomes
of a total of 340 isolates obtained from four European countries (Finland, UK,
Lithuania and Estonia) during 14 years were selected. Genealogy was
reconstructed using maximum likelihood phylogenetic analysis (RAxML) of 1,043
core genes after deleting recombination regions using BRATNEXGEN. Population
structure was inferred with BAPS. RAxML was also used to cluster strains based
on distribution of 1,281 accessory <span class=GramE>genes.GWAS</span> was
facilitated by first running a distributed string mining method on both core
and accessory genomic elements, and then testing for association among the
resulting genetic features. Pangenome allele profile of a single BAPS lineage
was generated using Genome Profiler and goeBURST was used to infer
microevolution of isolates. The effect of time on the genetic diversity was
estimated using BEAST v1.8.2.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> We
identified two major clades of ST-45cc characterized by a large variation in
r/m ratio (from 0.9 up to 22). With few exceptions, STs did not form
monophyletic lineages and were composed by numerous BAPS populations
characterized by a clonal structure, limited genetic variation and presence or
absence of specific features regardless origin or year of isolation. However,
when accessory genome was included in the analysis, GWAS identified short
genetic signals over-represented in either Finnish or British isolates.
Supporting the results of GWAS, RAxML tree based on distribution of accessory
genes clustered isolates with identical core sequence according to origin,
regardless the year of isolation. Nevertheless, microevolutionary analysis with
goeBURST of a single BAPS lineage, having the highest Shannon and Simpson
diversity index in term of countries and year of isolation, did not allow a
clear separation of the isolates according to origin even at lower LV values.
We further attempted to estimate divergence times between Finnish isolates
belonging to a single BAPS lineage sampled serially between 2000 and 2012. A
linear regression of the root to tip distance versus date of isolation was not
possible (R<sup>2 </sup>= 0.12), suggesting absence of temporal signal.
Moreover, calibrating the clock with mutation rate calculated using MLST data
(3.2x10<sup>-5</sup>; Wilson et al., 2009), we estimated the time for TMRCA to
be 2011, clearly not reflecting the true sampling time of the isolates.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Our genomic
study supported the theory of homogeneous global distribution of <i>C. jejuni</i>
genotypes. Contrary to what expected, several lineages within ST45cc appear to
be genetically monomorphic pathogens. They evolved slower than previously
thought and were persistently detected over the years independently from
geographical origin. We observed a lack of genetic isolation by distance
suggesting that proximity-by-mobility due to rapid animal and/or human movement
eroded the effect of geographical separation on the fine-scale genetic
structure of <i>C. jejuni </i>ST-45cc.</span><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO29 - Virulence
Differences Among Type AI <i style='mso-bidi-font-style:normal'>Francisella
tularensis</i> Strains Linked To Genetic Mutations Found By Whole Genome
Sequencing<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Caroline Öhrman<sup>1</sup>; Jeannine Petersen<sup>2</sup>; Luke Kingry<sup>2</sup>;
Kiersten Kugeler<sup>2</sup>; Claudia Molins<sup>2</sup>; Laurel
Respicio/Kingry<sup>2</sup>; Andreas Sjödin<sup>1</sup>; Mats Forsman<sup>1</sup>;
Dawn Birdsell<sup>3</sup>; Paul Keim<sup>3</sup>; Dave Wagner<sup>3</sup>;
Anders Johansson4<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Swedish Defence Research Agency (FOI) Centers for
Disease Control and Prevention Northern Arizona University (NAU)<span
class=GramE>,<sup>2</sup></span> Umeå University<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Background:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> <i>Francisella tularensis</i> subspecies <i>tularensis</i>
is one of three subspecies of <i>Francisella tularensis</i> causing the
potentially severe disease tularemia. This subspecies, which is also known as
type A and&nbsp;found in <span class=GramE>North-America</span>,&nbsp;is the
most virulent and was historically stockpiled as a biological weapon. In 2004,
multiple locus variable-number <span class=GramE>tandem</span> repeat analysis
(MLVA) identified two major genetic subpopulations among type A strains (AI and
AII) and in 2009, analysis by pulsed-field gel electrophoresis (PFGE)
identified a further genetic subdivision of AI strains into A1a and A1b.
Strikingly, comparison with epidemiological surveillance data showed there <span
class=GramE>were</span> pronounced differences in clinical outcomes in
infections by different type A genetic subpopulations. Despite that all type A
strains shares an extensive genetic similarity, mortality among humans was
strongly associated with A1b strains. Here, we used whole-genome sequencing to
identify mutations that may explain virulence differences among type AI
strains.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Material/Methods:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> Whole genome sequencing with paired end
Illumina technology was used to sequence AI strains (n=13) with a subset of
strains (n=6) selected for long pair inserts, so called mate pair sequencing.
The genomes were assembled and single nucleotide polymorphisms (SNPs),
insertion deletion events (INDELs), and genomic rearrangements were identified.
Using this genomic data, we recently set up a typing system based on assaying
canonical whole genome SNPs (canSNPs) identifying three distinct genetic
subpopulations (A.I.3, A.I.8 and A.I.12) among F. tularensis AI strains and in
this work, typing results were analyzed in relation to clinical outcomes and
whole genome differences. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Typing of 106 clinical strains from infected
humans by canSNP assays revealed there was 27 percent mortality in infections
from strains assigned to the genetic subpopulations A.I.3 (n=4/15) or A.I.8
(n=6/22) but only 6 percent mortality from A.I.12 strains (n=4/69). For two
strains from each of the three subpopulations, the genomes were completed
including determination of gene orders across insertion sequence elements.
Comparing the three subpopulations, twenty-seven SNPs and two INDELs were
unique to the A.I.3 group and nine SNPs and two indels to the AI.8 group. Five
SNPs were shared between A.I.12 and A.I.8 strains that thereby formed sister
clades in a phylogenetic whole-genome analysis. Thirty-eight SNPs, seven
INDELS, and one 23,100 bp genomic inversion were unique to the A.I.12
subpopulation. The genomic inversion fully explained the unique banding pattern
previously denoted A1a by PFGE typing using the restriction enzyme <i>PmeI</i>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusions:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> The analysis suggests virulence differences
among type AI <i>F. tularensis</i> strains evolved by a decrease in virulence
along the A.I.12 whole-genome phylogenetic tree branch and we have identified
46 mutation candidates in the genome to explain this. <span class=GramE>Gene
order differences caused by one genomic inversion event explains</span> the
previously observed distinct PFGE pattern for type AI strains with decreased
virulence.</span><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO31 - Using Whole
Genome Sequencing To Understand Host-nontuberculous Mycobacteria Interaction<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Sonia Faria<sup>1</sup>; Vitor Borges<sup>2</sup>; Catarina Carneiro<sup>2</sup>;
Luis Vieira<sup>2</sup>; Joao Paulo Gomes<sup>2</sup>; Luisa Jordao2<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>Instituto Nacional de Saúde Dr. Ricardo
Jorge/Instituto de Higiene e Medicina Tropical (IHMT/UNL) Instituto Nacional de
Saúde Dr. Ricardo Jorge<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-ansi-language:PT;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Background:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> Nontuberculous mycobacteria (NTM) are a large
group of <i>Mycobacterium </i>species that don’t belong to the <i>Mycobacterium
tuberculosis </i>(Mtb) complex<i>.</i> These bacteria are mostly environmental
being regarded as etiological agents of opportunistic infection in humans
mainly immunocompromised. Distinguishing NTM from Mtb is still a challenge and
identification at the species level is essential for an accurate diagnostic and
effective treatment. The growth rate of NTM is very important for the onset of
treatment being these bacteria divided into rapidly growing mycobacteria (RGM)
and slowly growing mycobacteria (SGM). <i>Mycobacterium fortuitum, M. abscessus
</i>and the model organism <i>M. smegmatis </i>are RGM whereas <i>M. avium </i>is
SGM that take more than 7 days to form CFU on growth media. The knowledge of
NTM infections is still reduced being needed more studies to understand the
host-pathogen interaction during the infectious process in order to establish
more effective therapeutic schemes. Recently our group conducted a study using
human alveolar macrophages as a model. The obtained data showed that both RGM
(747/08) and SGM (60/08) were able to persist and even replicate within this
macrophages whereas others do not (<i>M.smegmatis</i> and <span class=GramE><i>M.abscessus</i>)<sup>1</sup></span>.
Here we used NGS as a tool to identify bacterial factor responsible for the
observed outcome.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Materials and Methods:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> Three reference strains (<i>M.fortuitum </i>ATCC6841<i>,
M.avium </i>ATCC25291<i>, M.smegmatis </i>ATCC700084) and 3 strains from
Ricardo Jorge mycobacterial collection isolated from patients <i>(M.fortuitum
747/08, M. avium 60/08 and </i>Mtb70/09<i>)</i> were used. For DNA extraction
bacteria were grown in Middlebrook 7H9 supplemented with 10% OADC and 0.05%
Tween80.<sup>2<o:p></o:p></sup></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Full genome
sequence was performed using NGS platform MiSeq (Illumina Inc., San Diego, CA,
USA) according to the manufacturer’s instructions.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-fareast-language:PT'>Data analysis: RAST (</span><span
lang=DE-CH><a href="http://www.rast.nmpdr.org"><span lang=EN-US
style='font-size:8.0pt;mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
mso-fareast-language:PT'>www.rast.nmpdr.org</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>) and MAUVE
platforms were used for annotation and multiple alignments, respectively.<sup>3</sup><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> Data analysis suggest a link between
mycobacteria growth rate and genome size with RGM (6.7 million bp) having
longer genomes than SGM (4.8 million bp) what might reflect bacteria adaptation
to the host(s). Mtb with an exclusive host has a shorter genome (4.3 million
bp) than NTM which exhibit a wider host tropisms and the ability to persist
within the environment Therefore, we can find genes associated to the Lactate
fermentation like MAV_2543 and Methanogenesis xfp that cannot be found in Mtb.
The two NTM (747/08 and 60/08) strains sequenced and that show intramacrophage
persistence displayed also a longer size, which is associated to the
persistence related genes.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>In specific gene
subsystems were observed intra-species differences between clinical and reference
NTM. Fatty acid metabolism cluster and cell envelope related genes subsystems
illustrate this result.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusion:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> The preliminary results suggest a link between
genome size and intracellular persistence and support the fact that clinic NTM
share virulence factors with Mtb. However, only a transcriptional analysis
could ensure the evolvement of these genes in intracellular persistence.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>References: </span></b><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>Sousa S <i>et
al</i>. <i>Int J Mycobacteriology</i>. 2015;4(1):36-43.&nbsp;</span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Benjak A <i>et
al</i>. <i>Methods Mol Biol</i>. 2015<span class=GramE>;1285:1</span>-16.
Darling ACE <i>et al. Genome Res</i>. 2004<span class=GramE>;14</span>(7):1394-1403.<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<i style='mso-bidi-font-style:normal'><span style='font-size:10.0pt;mso-bidi-font-size:
11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:Cambria;
mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></i>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><i style='mso-bidi-font-style:normal'><span style='mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></i></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO32 - Adaptive
Evolution Of <i style='mso-bidi-font-style:normal'>Burkholderia cenocepacia</i>
Within The Lungs Of A Chronically Infected Cystic Fibrosis Patient<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Sandra C. dos Santos<sup>1</sup>; Carla P. Coutinho<sup>1</sup>; Marcus
Dillon<sup>2</sup>; Ana S. Moreira<sup>1</sup>; Rita F. Maldonado<sup>1</sup>;
Vaughn S. Cooper<sup>2</sup>; Isabel Sá-Correia1<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>iBB – Institute for Bioengineering and Biosciences,
Department of Bioengineering, Instituto Superior Técnico, Universidade de
Lisboa Department of Microbiology and Molecular Genetics, University of
Pittsburgh School of Medicine<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Chronic
pulmonary infections are the prominent cause of high morbidity and mortality in
cystic fibrosis (CF). <i>Burkholderia cepacia </i>complex (Bcc) bacteria are
particularly threatening since they are associated with a worse prognosis, are
inherently resistant to most clinical antibiotics and antimicrobials of the
immune system, and are able to evolve and adapt to multiple selective pressures,
significantly hindering their eradication. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods and
Materials: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>In order to uncover key features in the
pathogenic potential, persistence and virulence of <i>B. cenocepacia</i> during
progressive lung disease, a comparative genomic analysis was performed for 11
sequential clonal variants, obtained from a single patient followed at the
major Portuguese CF Center at Hospital de Santa Maria (HSM) in Lisbon [1]. This
work is anchored on a 2 decade-long collaboration with this CF Centre for
microbiological surveillance of Bcc respiratory infections, resulting in a
collection of over 800 Bcc clinical isolates with accompanying patient clinical
data [2]<span class=GramE>.Sputum</span> samples were collected from the onset
of infection until the patient’s death 3.5 years later with a fatal necrotizing
pneumonia known as cepacia syndrome. The complete genome of all isolates was
sequenced using Illumina to generate short paired-end reads, while the first
isolate collected and believed to have initiated the infection (IST439) was
sequenced with PacBio, allowing the assembly of a complete draft genome to be
used as reference. Different computational approaches were used for the
identification of indels and major genomic rearrangements in the short-read
sequencing data, specifically Breseq (made available by the Barrick Lab), and
an in-house computational pipeline developed jointly at the Cooper Lab.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>IST439 has a
total genome size of 7.63 <span class=GramE>Mb</span> arranged in 3
chromosomes. Overall, 59 single nucleotide polymorphisms were detected (8
synonymous, 46 non-synonymous, 5 intergenic), as well as 11 deletion events.
The altered genes were analyzed regarding biological function, and include
encoded proteins involved in response to low iron conditions, oxidative and
heat stress, antimicrobial treatment, lipopolysaccharide biosynthesis,
microaerophilic growth, and membrane transport. Most remarkably, one isolate
recovered near the end of the infection exhibited <i>in vivo </i>loss of
chromosome 3. Additional fitness and virulence studies are consistent with the
suggested role of this chromosome as a virulence encoding and stress response
non-essential megaplasmid.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Comparative
genomic studies possess a paramount importance in the identification of genomic
modifications underlying the dynamics of adaptive evolution that promote
successful colonization and long-term survival of pathogenic bacteria in a
given host. Here we present a number of specific mechanisms of convergent
evolution and adaptation of Bcc bacteria in the CF lung, representing potential
targets for inactivation to enhance antimicrobial activity and to limit
persistent infections by these and other Gram-negative bacteria.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>References: </span></b><span
lang=FR style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:FR;mso-fareast-language:
PT'>[1] Coutinho et al., Infection and Immunity 79(7): 2950-60, 2011.&nbsp;[2]
Coutinho et al., Front Cell Inf. </span><span class=GramE><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Microbio,
1:1-11, 2011.</span></span><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-font-style:italic'>Acknowledgements</span></b><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'> This work was supported by FCT funding received for iBB
(UID/BIO/04565/2013) and Post-Doctoral and PhD fellowships. The contribution of
C. Barreto, L. Pereira, A. Correia, P. Azevedo, L. Lito, M. Salgado and J.
Melo-Cristino at HSM is gratefully acknowledged.</span><span style='mso-ansi-language:
EN-US;mso-fareast-language:PT'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
<o:p></o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO33 - Comparative
Genomics Of An Endemic Strain Of Multidrug-resistant <i style='mso-bidi-font-style:
normal'>Pseudomonas aeruginosa</i> In A Pediatric Cystic Fibrosis Population<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Ruth Ann Luna<sup>1</sup>; Jennifer K. Spinler<sup>1</sup>; Anthony M. Haag<sup>1</sup>;
Miriam Balderas<sup>1</sup>; Prapaporn Boonma<sup>1</sup>; Abria R. Magee<sup>1</sup>;
Peter W. Hiatt<sup>1</sup>; James Versalovic1<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Baylor College of Medicine/ Texas Children´s Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Multi-year
molecular epidemiological surveillance of the multidrug-resistant <i>Pseudomonas
aeruginosa</i> (MRPA) isolates obtained from a pediatric cystic fibrosis (CF)
population led to the identification of an endemic strain (Houston-1).
Acquisition of the Houston-1 strain correlated strongly with recent
hospitalization, and subsequent infection with the Houston-1 strain was
associated with higher hospitalization rates and greater declines in lung
function and nutritional status compared to infection with other MRPA strains.
Improvements in infection control procedures and compliance have successfully
mitigated ongoing transmission of the Houston-1 strain, but it persists in the
pediatric population as well as in the patients transitioning to local adult CF
care centers. Comparative genomics between this endemic MRPA strain with other
MRPA strains isolated from our pediatric CF population as well as publicly
available <i>P. aeruginosa</i> genomes will enable further investigation into
underlying factors related to potential increased transmissibility and
virulence of the Houston-1 strain.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Eight
archived MRPA isolates obtained from patients of the pediatric CF center and
previously classified via molecular strain typing were selected based on their
clonal group as well as their antibiotic susceptibility profiles. The genomes
of these eight MRPA isolates, which included a total of 5 different MRPA
strains (4 Houston-1 and 4 unique isolates), were obtained via next-generation
sequencing (Roche 454). Circular BLASTn-based representations of the MRPA
genomes were constructed using BLAST Ring Image Generator. The newly generated
clinically-relevant genomes were compared to eight other publicly available
genomes from <i>P. aeruginosa</i> isolates associated with CF. Multi-locus
sequence typing was carried out using the <i>Pseudomonas aeruginosa</i> MLST
database. In addition, selected reaction monitoring (SRM) was implemented to
determine the rate at which these MRPA strains degrade five classes of
antimicrobials.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The genome
sizes of the newly sequenced MRPA genomes range from 6-7 Mbp and contain an
average of 6,500 genes per genome. The four isolates identified as the
Houston-1 strain distinctly clustered into a clade separate from other MRPA
isolates, including the other four MRPA isolates/strains obtained from the same
pediatric CF population. Based on MLST data, the Houston-1 strain represents a
novel sequence type. Antibiotic degradation monitoring by SRM also indicated
rapid degradation of penicillin class antibiotics by the Houston-1 strains as
compared to unique MRPA strains.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Comparative
genomics analysis has emphasized the distinct evolution of the Houston-1 strain
that was found to be endemic in our pediatric CF population. Further
characterization of these strains will highlight virulence determinants
important for their evolution and persistence. Longitudinal SRM monitoring of
MRPA isolates in this patient population may allow for the rapid detection of
antibiotic resistance and assist in effective clinical management and minimized
risk of newly emergent antibiotic resistant strains. In addition, investigation
into genetic variations identified in the Houston-1 strain may provide
additional insight into the perceived increased transmissibility of this MRPA
strain.</span><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO34 - Genomic
Epidemiology Of The Gaza Community-acquired Methicillin-resistant <i
style='mso-bidi-font-style:normal'>Staphylococcus aureus</i> Clone: A Unique ST22
Clone<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Qiuzhi Chang<sup>1</sup>; Asaf Biber<sup>2</sup>; Izzeldin Abualaish<sup>3</sup>;
Hanaa Jaber<sup>2</sup>; Galia Rahav<sup>2</sup>; William P Hanage<sup>1</sup>;
Gili Regev-Yochay2<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Harvard School of Public Health <sup>2</sup>Sheba
Medical Center <sup>3</sup>Toronto University<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Introduction:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>&nbsp;Community-acquired
methicillin-resistant&nbsp;<i>Staphylococcus aureus</i>&nbsp;(CA-MRSA) has
become a major global problem. We recently reported an abnormally&nbsp;high
carriage prevalence of CA-MRSA in healthy children and adults in the Gaza. A
particular ST22 clone, associated with HA-MRSA EMRSA-15, accounted for over 60%
of all MRSA isolates. Using whole genome sequencing we provide detail on
virulence properties, the unique resistance pattern, and the evolutionary
history of the Gaza MRSA clone.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods:&nbsp;</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>During 2009,
a cross-sectional surveillance of <i>S. aureus</i> nasal carriage was conducted
among healthy individuals in the Gaza, in the framework of the
Palestinian-Israeli Collaborative Research (PICR). Isolates collected were
assessed for antibiotic susceptibility using the VITEK2 system. Any unexpected
susceptibility patterns were further assessed using disk diffusion and E-test.
High quality draft Genomes of&nbsp;69 ST22 <i>S. aureus</i> isolates were
assembled from Illumina short read data using Velvet. Together with 175
additional ST22 genomes from global sources, we reconstructed the phylogeny of
the ST22 lineage to define the origin and evolution of the Gaza clone.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:&nbsp;</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The Gaza
isolates did not group with typical HA ST22s and were in fact two distinct
closely related clades, one of which was characterized by the presence of
staphylococcal cassette chromosome&nbsp;<i>mec</i>&nbsp;element&nbsp;(SCC<i>mec</i>)
IV and the other by SCC<i>mec</i>V. Nearly all Gaza isolates in both clades
contained the toxic shock syndrome toxin 1 (<i>tsst-1</i>) gene but the two
clades had distinct antibiotic resistance genes. &nbsp;Only SCC<i>mec</i>IV
clade isolates contained the co-trimoxazole resistant gene <i>dfrA</i>. These
isolates showed a contradictory resistance phenotype, detected as resistant by
automated machines, but susceptible by disk-diffusion, E-test and broth
microdilution. &nbsp;The diversity of both clades is not consistent with recent
emergence and shows evidence of changes in population structure.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Discussion:&nbsp;</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The ‘Gaza
clone’ is in fact two clones that group with the diverse and predominantly
methicillin-susceptible&nbsp;<i>S. aureus</i>&nbsp;ST22 isolates from which
EMRSA15 emerged. This&nbsp;suggests two separate acquisitions of resistance by
the same lineage, followed by diversification in other resistance determinants.
In the absence of phenotypic testing, the <i>dfrA</i> variant would have been
erroneously flagged as highly resistant, showing the importance of accurately
linking phenotype to genotype.</span><span style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO35 - Next Generation
Sequencing: The Key To Understood <i style='mso-bidi-font-style:normal'>Klebsiella
pneumoniae </i>Biofilms?<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Luisa Jordão<sup>1</sup>; Vitor Borges<sup>1</sup>; Luis Vieira<sup>1</sup>;
João Paulo Gomes<sup>1</sup>; Aida Duarte2<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>Instituto Ricardo Jorge Faculdade de Farmácia ULisboa<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-ansi-language:PT;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Background:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> The incidence of healthcare-associated
infections (HAI) is determined by underlying disease conditions and exposure to
<span class=GramE>high risk</span> medical interventions. In Portugal since
1980s <i>K. pneumoniae</i> is a recognized etiological agent of epidemic and
endemic infections in healthcare units. An increasing rate of <i>K. pneumoniae</i>
strains resistant either to extended cephalosporins or carbapenems has been
observed and one of the mechanisms responsible for the emergence of drug
resistance could be the biofilm assembly. The capacity of <i>K. pneumoniae</i>
to form biofilm was first described in the 1980s for abiotic surfaces and ten
years later on biotic surfaces. The antibiotic failure to penetrate through the
biofilm layers, the emergence of mutations which might be easily transferred
horizontally, and quorum sensing have been pointed as responsible for the
increased antibiotic resistance of bacteria within biofilms. The main objective
was to study the biofilm structure and the kinetic assembly associated to
antibiotic resistance profile in <i>K. pneumoniae </i>strainscapsulate or not,
and identify the genes involved in biofilm assembly on full genome sequencing
of studied strains.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Material and Methods:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> Three <i>K. pneumoniae</i> isolates collected
in 1980 (Kp45, Kp703) and one (Kp2948) in 2011 were studied.&nbsp; Kp703 was
encapsulated and the remaining had capsular type K<span class=GramE>:2</span>.
The bacterial ability to assemble biofilms on cell culture plates was
evaluated. For SEM analysis, biofilms were allowed to form on six wells cell
culture plates (Nunc) for 12h at 37ºC. DNA was extracted using QiAamp DNA mini
kit following the manufactures instructions. Full genome sequence was performed
using next-generation sequencing platform MiSeq (Illumina Inc., San Diego, CA,
USA) according to the manufacturer’s instructions. The RAST platform was used
for annotation and MAUVE platform for multiple alignments.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> The three isolates were able to assemble
biofilms although following different kinetics. <i>K. <span class=GramE>pneumoniae</span></i>
strains Kp703 and Kp45 followed similar kinetics with identical biomass
increase nevertheless these bacteria differ in capsule expression. <span
class=GramE>Full-sequencing</span> and annotation of genomes of isolates was
performed in order to explain the differences found in biofilm assembly.
Preliminary data already revealed that the <i>K. pneumoniae</i> strains
displaying enhanced biofilm-forming ability is genetically different from the
others, and, in particular, present some specific features enrolling genomic
regions believed to be biofilm-related in other bacteria (an intact prophage,
genes coding for a filamentous haemoagglutinin, a haemolysin expression
modulating protein and an YdeA protein).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusion:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> <i>K. pneumoniae</i> lacking capsule, regarded
as less virulent, have a better performance as biofilm assembler and exhibited
the highest increase in antibiotic resistance when organized within biofilms.
The analysis of the full genome sequence will allow reachingon <i>K.pneumoniae</i>biofilms
and provide novel opportunities to exploit the overall fitness of<i>K. <span
class=GramE>pneumoniae<span style='font-style:normal'>under</span></span></i>
antibiotic stress.</span><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO36 - Whole Genome
Sequencing Of Clonal Complex 8 MRSA Isolates Shows Several Clusters Of USA300-like
Isolates<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Mette Damkjær Bartels<sup>1</sup>; Peder Worning<sup>1</sup>; Jesper Boye
Nielsen<sup>1</sup>; Henrik Westh1<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Hvidovre University Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: MRSA
strains belonging to clonal complex (CC) 8 are common worldwide. Among the CC8 clones
is the USA300 <span class=GramE>clone which</span> usually is<i> spa</i> type
t008 and contains both Panton Valentine Leukocidin (PVL) genes and the arginine
catabolic mobile element (ACME). In 2014, t008 was the sixth most common <i>spa</i>
type in Denmark. Another CC8 clone, t024, PVL negative and ACME positive, was
spreading in nursing homes and local hospitals of the Capital Region of Denmark
in the beginning of the millennium. We have previously shown that PFGE could
not distinguish between t008 and t024 (1). In a study on sequence polymorphisms
within 112 housekeeping loci, we could divide 174 geographically and time
spread <i>S. aureus</i> isolates into 9 clusters (2) thereby separating t008
from t024. Whole genome sequencing (WGS) of MRSA isolates has been a routine
analysis in our laboratory since 2013, and the purpose of this study was to
obtain a more detailed insight into the evolution and spread of CC8 in the
Capital Region of Denmark.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Material and
methods</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>: In the present study we included 263 CC8 isolates from the Capital Region
of Denmark. All isolates were whole genome sequenced on an Illumina MiSeq.
Isolates were sequenced with 2 x 150 bp paired-end reads. An in-house script
analysed the genomes for <i>mec</i>A, <i>mec</i>C, <i>nuc</i>, <i>ccr</i>
genes, <i>spa</i>-type, MLST, <i>dru</i> types, PVL, ACME and SCC<i>mec</i>.
Reads were assembled using Velvet and Velvet optimizer. The reads were mapped
to a reference genome (US300_TCH1516) using Stampy and single nucleotide
polymorphisms (SNPs) were called using Samtools.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>A distinct
cluster of t024/ST8/ACME+/PVL- isolates was seen as well as a large cluster of
t008/ST8/ACME+/PVL+ isolates. In addition, a minor cluster of 28 ACME-/PVL+
isolates was seen (yellow isolates in Fig. 1, a subtree of 149 isolates). The
ACME-/PVL<span class=GramE>+,</span> isolates were <i>spa</i> type t008 except
for one isolate that was a <i>spa</i> type t024. Among the 28 isolates some
isolates had less than 10 SNPs between them, but an epidemiological link was
not always present. However, the majority of the 28 ACME-/PVL+ isolates had
more than 100 SNPs between them indicating that it was not a large outbreak. In
most cases, there was no information on travel abroad and the majority of
patients were ethnic Danes. Finally, among the t008/ ACME+/PVL+ isolates, SNP
analysis revealed previously unrecognized relations between several patients,
e.g. two mothers that have given birth few days apart and had overlapping
hospitalization in the same department.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>WGS gives us
a detailed insight into the evolution and spread of CC8 MRSA isolates in the
Capital Region of Denmark. The SNP analysis shows that the typical USA300 clone
has diversified into a subgroup of ACME negative isolates. Whether the loss of
ACME changes the pathogenicity of this clone is unknown. SNP analysis also
revealed unrecognized spread of MRSA in some cases.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>References: </span></b><span
lang=FR style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:FR;mso-fareast-language:
PT'>Larsen AR et al, J Clin Microbiol. </span><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>2009 Nov;47(11):3765-8. </span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Strommenger
B et al, J Antimicrob Chemother. </span><span lang=DE-CH style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-fareast-language:PT'>2014 Mar;69(3):616-22.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shapetype
 id="_x0000_t75" coordsize="21600,21600" o:spt="75" o:preferrelative="t"
 path="m@4@5l@4@11@9@11@9@5xe" filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Imagem_x0020_12" o:spid="_x0000_i1025" type="#_x0000_t75"
 alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/91imagem_2560108707_0.jpg"
 style='width:333pt;height:272pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S2_files/image001.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/91imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=274
src="public/abstracts/Webpages/PO_S2_files/image002.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/91imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_12"><![endif]></span><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO37 - <i
style='mso-bidi-font-style:normal'>Mycobacterium tilburgii</i>: The First
Sequenced Genome<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Jan Zoll<sup>1</sup>; Jakko van Ingen, Saskia Kuipers, and Willem Melchers<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Radboud University Medical Center</span></i><span
lang=EN-GB style='mso-bidi-font-family:Arial;color:black;mso-themecolor:text1;
mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal align=left style='text-align:left;line-height:115%'><span
lang=EN-GB style='font-size:8.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
mso-bidi-font-family:Arial;color:black;mso-themecolor:text1;mso-ansi-language:
EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Background:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> <i>Mycobacterium tilburgii</i> is a recently
discovered <i>Mycobacterium</i> species causing serious infections with high
mortality in immunocompromised hosts, specifically in North-West Europe. <i>M. <span
class=GramE>tilburgii</span></i> cannot be grown in vitro. Therefore,
resistance cannot be determined phenotypically and proper treatment of the
infection is hampered. Until now, <i>M. tilburgii</i> could only be diagnosed
by microscopy and 1S6 rDNA sequence analysis.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Materials and methods:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> In this study, we determined the full genome of
<i>M. tilburgii</i> directly from an autopsy sample of the spleen from a
patient with disseminated <i>M. tilburgii</i> disease. Depletion of human DNA
was a prerequisite for successful determination of the <i>Mycobacterium</i>
genome. Therefore, human cells were selectively lysed and treated with
nucleases using the Molzym Molysis system. The <i>Mycobacterium</i> DNA
isolated from the samples was used for whole genome sequencing using Illumina
NextSeq500 technology.<o:p></o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results and
conclusion:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> <i>De novo</i> assembly using CLC Genomic
Workbench resulted in 173 contigs ranging from 500 to 220.000 bp, covering
almost the entire bacterial genome. Phylogenetic analysis showed a close
relationship with <i>Mycobacterium simiae</i>. Whole genome sequencing was
required to obtain extra information on antibiotic susceptibility. Genomic
characteristics and genetic relationships with other <i>Mycobacterium</i>
species will be presented.</span><span lang=EN-GB style='mso-bidi-font-family:
Arial;color:black;mso-themecolor:text1;mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=DE-CH><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
