<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns:mv="http://macVmlSchemaUri" xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta name=Title content="">
<meta name=Keywords content="">
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List href="PO_S6_files/filelist.xml">
<link rel=Edit-Time-Data href="PO_S6_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Bruno Gonçalves</o:Author>
  <o:LastAuthor>Bruno Gonçalves</o:LastAuthor>
  <o:Revision>1</o:Revision>
  <o:TotalTime>1</o:TotalTime>
  <o:Created>2016-03-08T21:52:00Z</o:Created>
  <o:LastSaved>2016-03-08T21:53:00Z</o:LastSaved>
  <o:Pages>9</o:Pages>
  <o:Words>3354</o:Words>
  <o:Characters>19118</o:Characters>
  <o:Company>IMM</o:Company>
  <o:Lines>159</o:Lines>
  <o:Paragraphs>44</o:Paragraphs>
  <o:CharactersWithSpaces>22428</o:CharactersWithSpaces>
  <o:Version>14.0</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="PO_S6_files/themedata.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>JA</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
   <w:UseFELayout/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="276">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1073743103 0 0 415 0;}
@font-face
	{font-family:MingLiU;
	mso-font-alt:細明體;
	mso-font-charset:136;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-1610611969 684719354 22 0 1048577 0;}
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	mso-themecolor:hyperlink;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:purple;
	mso-themecolor:followedhyperlink;
	text-decoration:underline;
	text-underline:single;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"ＭＳ 明朝";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
@page WordSection1
	{size:595.0pt 842.0pt;
	margin:72.0pt 89.85pt 72.0pt 89.85pt;
	mso-header-margin:35.45pt;
	mso-footer-margin:35.45pt;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;}
</style>
<![endif]-->
</head>

<body bgcolor=white lang=EN-US link=blue vlink=purple style='tab-interval:36.0pt'>

<div class=WordSection1>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO1 - Comparative
Genomics Provides Insight In The Evolution Of Antimicrobial Resistance In
Canine Methicillin-resistant <i style='mso-bidi-font-style:normal'>Staphylococcus
pseudintermedius<o:p></o:p></i></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Birgitta Duim<sup>1</sup>; Jaap A. Wagenaar<sup>1</sup>; Mirlin Spaninks<sup>1</sup>;
Arjen Timmerman<sup>1</sup>; Els M. Broens<sup>1</sup>; Aldert Zomer<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Utrecht University/Faculty of Veterinary Medicine<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: <i>Staphylococcus
pseudintermedius</i> is a significant pathogen in dogs and cats and is
occasionally associated with infections in humans. Methicillin-resistant
strains (MRSP), often resistant to three or more antimicrobial classes
(multi-drug resistance; MDR) have emerged since 2004. Recent studies in our lab
showed that the majority of MDR MRPS belonged to clonal complex (CC) CC71 and
to a lesser extend to CC258 or CC45, and that since 2011 less resistant
isolates were obtained from dogs. The aim of this study was to investigate how
recombination and mobile genetic elements shape the resistance dynamics in
different MRSP lineages. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: At the
Veterinary Microbiological Diagnostic Centre (VMDC) of Utrecht University, 478
MDR MRSP strains were isolated from canine infections from 2004 to 2014. Fifty
MRSP strains, all with a unique MDR profile, with resistances to maximum 8
antimicrobial classes were selected for whole genome sequencing using Illumina
Miseq. Genomes were annotated; orthologous relations between genes were
determined using Orthagogue. The Harvest suite was used for core-genome
alignment and reconstruction single-nucleotide polymorphism (SNP) based
phylogeny. Recombination and gene flow was studied using Gubbins.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>:
Phylogenetic analysis showed highly conserved genomes of MRSP belonging to CC71
and CC45. Recombination was detected in CC258 and hardly in CC71 and CC45. In
all strains mobile genetic elements associated with resistance genes were
present. Regions of horizontal transfer were identified and all relevant
recombination events will be presented.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Whole
genome sequencing identifies the evolution of antimicrobial resistance in MRSP
and may provide insight in how antimicrobials can co-select for multi-resistant
strains.</span><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO2 - Comparison Of
Fully Sequenced Genomes From Clinical And Non-clininal <i style='mso-bidi-font-style:
normal'>Enterococcus faecium</i> Strains<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Young-Hee Jung<sup>1</sup>; Taesoo Kwon<sup>1</sup>; Won Kim<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Korea National Institute of Health,<sup>2</sup> Seoul
National University<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Keywords: comparative genomics, candidate of therapy, difference between
clinic and nonclinic<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Background:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> Enterococci habit in gastrointestinal tract of
humans and animals and have been known as an opportunistic pathogen in chronic
patients. Most of vancomycin-resistant enterococci (VRE) are <i>E. faecium</i>
and are resistant to high-level vancomycin. In recent, vaccine candidates have
been under development to treat VRE. Therefore, we investigated the difference
between clinical and nonclinical strains in genome level to search the
candidate genes for VRE treatment.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Materials and methods:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> Four completed genome sequences of <i>E.
faecium</i> (one clinical VSE, two clinical VRE, one food-born VSE) were
download from NCBI and were analyzed using genome sequences of E. faecium DO as
a reference sequence. Multiple sequence alignment analysis were performed on
the proteins under 80% homology.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> We divided the proteins into six groups
according to the homology. Two proteins have 0~10% homology, 10~70% in seven
proteins, 70~80% in four proteins, 80~90% in ten proteins, paralog in 97
proteins, over 90% in 267 proteins, and the rest of the proteins have
difference among the clinical genomes. Twelve proteins were selected as the
candidates by multiple sequence alignment analysis. The proteins under 80%
homology are related to ABC transporter, beta-lactamase, cell wall or carbohydrate
metabolism.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusion:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> We analyzed four completed genome sequences of
one clinical <i>E. faecium</i>, two clinical VRE, and one food-born <i>E.
faecium</i> and we suggested the candidates of therapeutic agents to treat
enterococcal infection including VRE using the difference of clinical and
nonclinical genomes.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO3 - Phylogeny Of <i
style='mso-bidi-font-style:normal'>Mycobacterium leprae</i> Using Whole Genome
Sequencing<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Andrej Benjak<sup>1</sup>; Charlotte Avanzi<sup>1</sup>; Chloé Loiseau<sup>1</sup>;
Philippe Busso<sup>1</sup>; Pushpendra Singh<sup>2</sup>; Stewart Cole<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Global Health Institute, EPFL,<sup>2</sup> Louisiana
State University - School of Veterinary Medicine<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Keywords: <i>Mycobacterium leprae</i>, phylogeny, SNPs</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><i><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></i></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Mycobacterium
leprae</span></i><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'> strains have remarkably conserved genomes. Our objective is to
characterize the <i>M. leprae</i>´s genomic variability by whole genome
sequencing and to better understand the microevolution of this species. The
biggest challenge to this effort is the <i>in vitro</i> uncultivability of <i>M.
leprae</i> and the inadequate amounts of <i>M. leprae</i>’s DNA obtained from
skin biopsies using standard DNA extraction protocols. As consequence, genomic
resources for this species are very scarce.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><i><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials |
Methods: </span></i></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Total DNA was extracted from mouse food-pad
samples while a host-depletion step was applied on most biopsy samples prior
DNA extraction. Samples with low abundance of <i>M. leprae</i> DNA were
additionally enriched using whole-genome array capture. Illumina libraries were
sequenced either on HiSeq or MiSeq platforms. Reads were adapter-trimmed and
quality-trimmed and mapped against the <i>M. leprae </i>reference genome
sequence. High quality SNPs were inferred from the alignments by applying
stringent coverage and quality thresholds. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><i><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></i></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Thanks to
our considerably improved DNA extraction and enrichment methods, we have
successfully sequenced over 150 <i>M. leprae </i>strains deriving from
different locations around the world, using high-throughput sequencing. We have
identified 2,810 SNP loci, which enabled an accurate phylogenetic analysis and
a more refined SNP-based classification of strains. Correlation patterns were
observed for geographic locations and drug resistance.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><i><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></i></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>We are
establishing a genomic database representing the global diversity of <i>M.
leprae</i> strains. The new genomic resource will facilitate studies on <i>M.
leprae</i>, with the ultimate goal of eradicating this pathogen.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO4 - <i
style='mso-bidi-font-style:normal'>In Silico</i> Comparison Serotyping And
Genotyping Methods For <i style='mso-bidi-font-style:normal'>C.trachomatis.</i><o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Zhanna Amirbekova<sup>1</sup>; Dmitriy Babenko<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Karaganda State Medical University<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Chlamydia
trachomatis is a cause of the most prevalent sexually transmitted bacterial
infections with 89 million cases annually worldwide (Gaydos et al., 2004) and
is responsible for &nbsp;both asymptomatic&nbsp; and wide spectrum of
clinically significant diseases (Manavi , 2006).&nbsp; <b><o:p></o:p></b></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Based on
serological typing using antibodies against the major outer-membrane protein
(MOMP) there are 19 prototypic serovars of C.trachomatis. &nbsp;Compared to
immunotyping, the genotyping methods have some advances being more precise, sensitive
and specific for typing of C. trachomatis (Molano et al., 2004).&nbsp;The aims
of the present study were to compare phenotypic and genotypic methods for
subspecies identification of C.trachomatis in term of discriminatory power and
concordance.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials |
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>The datatable of Chlamydia trachomatis isolates with identified </span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>serovars/genovars and
sequence types (ST) was used for analysis (</span><span lang=DE-CH><a
href="http://pubmlst.org/"><span lang=EN-US style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
mso-fareast-language:PT'>http://pubmlst.org/</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>). Clonal
relationships of ST defined with minimum spanning tree using BURST algorithms
(MLVAcompare (Ridom GmbH)).&nbsp; Diversity and concordance between typing
methods were calculated using </span><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
mso-fareast-language:PT'>Simpson index and adjusted Rand and Wallace
coefficients (</span><span lang=DE-CH><a
href="http://www.comparingpartitions.info"><span lang=EN-US style='font-size:
8.0pt;mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:
PT'>http://www.comparingpartitions.info</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>There were
3691 strains of C.trachomatis in datatable. For 3242 isolates serovar/genovar
were determined. Sequence type based on 7 loci (MLST Chlamydiales scheme) and 5
loci (MLST Uppsala scheme) were identified for 512 and 3277 isolates
respectively. Discriminant power of each typing method and concordance are
presented in table1.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Table 1.
Resolution ability and concordance between serotyping (MOMP)/genotyping (ompA
gene) and MLST methods.<o:p></o:p></span></p>

<table class=MsoNormalTable border=0 cellpadding=0 style='mso-cellspacing:1.5pt;
 mso-yfti-tbllook:1184;mso-padding-alt:0cm 5.4pt 0cm 5.4pt'>
 <tr style='mso-yfti-irow:0;mso-yfti-firstrow:yes'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-fareast-language:PT'>Typing
  method<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-fareast-language:PT'>#
  Different types<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-fareast-language:PT'>Simpson
  index<o:p></o:p></span></p>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-fareast-language:PT'>(95%
  CI)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span style='font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
  mso-fareast-language:PT'>Adjusted Rand´s coefficient&nbsp;</span><span
  style='font-size:8.0pt;font-family:MingLiU;mso-bidi-font-family:MingLiU;
  mso-ansi-language:EN-US;mso-fareast-language:PT'><br>
  </span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>in
  comparison with&nbsp;</span><span style='font-size:8.0pt;font-family:MingLiU;
  mso-bidi-font-family:MingLiU;mso-ansi-language:EN-US;mso-fareast-language:
  PT'><br>
  </span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>serovar/genovar<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:1'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=left style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:left'><span style='font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
  mso-fareast-language:PT'>Serovar/genovar (n=3242)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span style='font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
  mso-fareast-language:PT'>27<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>0.822 [0.814 - 0.831]<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=right style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:right'><span style='font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
  mso-fareast-language:PT'>1<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:2'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=left style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:left'><span style='font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
  mso-fareast-language:PT'>MLST (Uppsala) (n=3277)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span style='font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
  mso-fareast-language:PT'>519<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span style='font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
  mso-fareast-language:PT'>0.974 [0.972 - 0.976]<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=right style='text-align:right'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'><br>
  0.171 (n=3104)<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:3;mso-yfti-lastrow:yes'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=left style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:left'><span style='font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
  mso-fareast-language:PT'>MLST (Chlamydiales) (n=512)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span style='font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
  mso-fareast-language:PT'>58<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>0.903 [0.893 - 0.914]<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=right style='text-align:right'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>0.609 (n=49)<o:p></o:p></span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Adjusted Wallace
coefficients were 0.096 &nbsp;for W1(serovar/genovar -&gt; MLST (Uppsala)) and
0.793 &nbsp;for W2 (MLST (Uppsala) -&gt; serovar/genovar). In the case with
serovar/genovar and MLST (Chlamydiales) the W1 was 0.758, and W2 was
0.508.&nbsp;Clonal complexes (CC) of MLST and their characteristics including
pairwise agreement between CC and serovars/genovars are presented in figure 1.<o:p></o:p></span></p>

<p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Our results
showed the higher level of discriminatory power of MLST in comparison with
serotyping The highest resolution ability (97.4%) had Uppsala scheme MLST using
5 loci. In spite of weak/moderate degree of global congruence between
serovars/genovars and sequence types (adj.Rand in table 1.) there was greater
concordance for clonal complexes with more than 90% successive prediction using
serovars/genovars (W1 coefficients in figure1). Interestingly, MLST typing
method demonstrated clonal population structure of C.trachomatis. <o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shapetype
 id="_x0000_t75" coordsize="21600,21600" o:spt="75" o:preferrelative="t"
 path="m@4@5l@4@11@9@11@9@5xe" filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Imagem_x0020_17" o:spid="_x0000_i1026" type="#_x0000_t75"
 alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/205imagem_2560108707_0.jpg"
 style='width:333pt;height:251pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S6_files/image001.png" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/205imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=253
src="public/abstracts/Webpages/PO_S6_files/image002.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/205imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_17"><![endif]></span><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p></o:p></span></b></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO5 - Investigation Of
The Austrian LA-MRSA Population By Whole Genome Sequence Based Typing<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Sarah Lepuschitz<sup>1</sup>; Burkhard Springer<sup>1</sup>; Alexander
Indra<sup>1</sup>; Franz Allerberger<sup>1</sup>; Werner Ruppitsch<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Austrian Agency for Health and Food Safety<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> In Austria,
the National Reference Laboratory for Staphylococci (NRLS) at the Austrian
Agency for Health and Food Safety (AGES) observed a steady increase of
infections caused by human CC398 LA-MRSA isolates from 0.2% in 2004 to 7.9% in
2015.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Since the majority
of Austrian LA-MRSA isolates harbor <i>spa-</i>type t011 an arbitrary
collection of isolates was analyzed by next generation sequencing in order to
evaluate the performance and discriminatory power of NGS.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Material/Methods: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Genomic DNA
was isolated using theMagAttract HMW DNA Kit according to the instructions of
the manufacturer (Qiagen, Hilden, Germany) from an arbitrary collection of 153
LA-MRSA isolates (14 animal, 20 food, 20 environment, 99 human) collected in
Austria from 2004 to 2015 comprising <i>spa-</i>types t011 (n=134), t034 (n=14)
and t108 (n=5). The fragment library was constructed with the NexteraXT Kit and
2x300 basepair fragments were sequenced on a MiSeq (Illumina Inc., USA). Raw
reads were de-novo assembled using Velvet version 1.1.04. Contigs were filtered
for a minimum coverage of 5 and a minimum length of 200&nbsp;bp. A core genome
(cg)MLST scheme comprising 1,862 target genes and a cluster threshold (CT) of
15 was used for NGS data interpretation using SeqSphere<sup>+</sup> (Ridom,
Germany) (1). <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Core genome
(cg)MLST based typing distinguished 15 different complexes and 41 isolates not
assignable to a complex (Figure). Isolates belonging to <i>spa</i>-type t011
differed from isolates of <i>spa</i>-type t034 by at least 60 alleles and from
isolates of <i>spa</i>-type t108 by at least 86 alleles. Isolates belonging to <i>spa</i>-type
t034 showed from <i>spa</i>-type t108 an allelic difference of 26. The allelic
differences were up to 71 between <i>spa</i>-type t011 isolates, 92 between <i>spa</i>-type
t034 isolates and 36 between <i>spa</i>-type t108 isolates. Closely related
human-human, pig-human, and environmental-human isolates were identified by
cgMLST based NGS.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>In
conclusion NGS based typing allows the differentiation of the highly clonal
LA-MRSA population in Austria as determined by <i>spa</i>-typing. This
improvement in typing will now allow to monitor the evolution of strains, the
identification of potential transmission routes, the identification of
outbreaks and the identification of resistance and virulence determinants and
as a consequence the emergence of more virulent clones.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Thus, the
application of NGS based typing is highly recommended for the public health
sector to improve MRSA surveillance.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>References: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Leopold S,
et al. (2014) J Clin Microbiol 52:2365–2370.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='mso-ansi-language:
EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape id="Imagem_x0020_18"
 o:spid="_x0000_i1025" type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/208imagem_2560108707_0.jpg"
 style='width:333pt;height:196pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S6_files/image003.png" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/208imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=198
src="PO_S6_files/image004.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/208imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_18"><![endif]></span><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p></o:p></span></b></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO7 - Water On The Lungs
- Characterisation Of Pathogens In Pleural Effusion By The Use Of Metagenomic
Sequencing<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Anna Maria Malberg Tetzschner<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Department of Systems Biology, The technical
university of Denmark<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Lung
effusion is a common complication for patients hospitalized with pneumonia.
Fast and adequate treatment is important for fast recovery and survival, and it
relies solely on rapid characterization of the pathogen causing the disease.
Unfortunately, the clinical microbiology practice is labor intensive and it can
take days or even weeks to determine the pathogen.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>In this study we
aim to develop a new method to improve and accelerate the detection and
characterization the infectious pathogen in patients suffering from lung
effusion by the use of direct sequencing on clinical sample.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Methods<span style='mso-bidi-font-weight:bold'>: </span></span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>At the
hospital we collect pleural fluid by drainage from patients with pleural
effusion. DNA is purified directly from the clinical samples, and several steps
are used to remove the human contamination prior to sequencing.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>After sequencing
the raw reads are trimmed and then mapped against predefined databases by the </span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>use of the MGmapper
software (</span><span lang=DE-CH><a
href="https://cge.cbs.dtu.dk/services/MGmapper/"><span lang=EN-US
style='font-size:8.0pt;mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
mso-fareast-language:PT'>https://cge.cbs.dtu.dk/services/MGmapper/</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>). Firstly
the raw reads are mapped against a human genome, thereby removing any human
contamination. The remaining unmapped reads are mapped against several
databases containing complete and draft bacterial, fungal protozoan and viral
genomes. The remaining reads that do not match anything will be mapped against
the complete nucleotide database.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>In order to obtain
detailed characteristics of the pathogenic strain, the raw reads are mapped
against databases containing virulence and resistance genes.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Finally a report
will be generated containing information for diagnosis to be made. This will
include abundance profiles of the strains, virulence factors and resistance
markers.</span><span style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO8 - Examination Of
Metagenomic Analysis For Typing Of <i style='mso-bidi-font-style:normal'>Campylobacter</i>
In A Cultureless Future Using Present-day Technology<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Katrine Grimstrup Joensen<sup>1</sup>; Søren Persson<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Statens Serum Institut<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Hospitals
are increasingly using PCR diagnostics directly on clinical samples instead of
culturing, due to lower cost as well as higher sensitivity and speed. The lack
of culturing presents a problem in the public health surveillance of pathogens
that traditionally depends on isolates for molecular typing methods, including
WGS. One approach to maintain public health surveillance without culturing
would be metagenomics sequencing of patient samples.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials/methods:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Human and
chicken fecal samples (negative for <i>Campylobacter </i>by culturing and PCR)
were spiked with 10-fold dilutions of 10^5 to 10^8 cfu/g of <i>C. jejuni</i>
SC181. We sequenced each of the sub-samples, including non-spiked controls, on
an Illumina MiSeq, using 250bp paired-end reads and 1/8 of a flowcell for each.
The sequences were analysed using Kraken </span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-fareast-language:PT'>(</span><span lang=DE-CH><a
href="https://www.eventgest.com/GEST/%C2%B4%C2%B4https:/ccb.jhu.edu/software/kraken/%C2%B4%C2%B4"><span
lang=EN-US style='font-size:8.0pt;mso-bidi-font-family:Arial;mso-ansi-language:
EN-US;mso-fareast-language:PT'>https://ccb.jhu.edu/software/kraken/</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>) and plasmid and phage
sequences were filtered out. The <span style='color:black'>sequences matching <i>Campylobacter</i>
in each of the sub-samples were compared using blastn to estimate the size of
the overlap.<o:p></o:p></span></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> We detected
446 and 58 <i>Campylobacter </i>reads in the human 10^7 and 10^8 sub-samples,
respectively. Of these, we were able to assign 64% and 53%, respectively, of
the reads to the SC181 strain. We detected 5118 and 254 <i>Campylobacter </i>reads
in the 10^6 and 10^5 chicken sub-samples, respectively, of which 8% and 7% were
specific for strain SC181. In the human control sample no <i>Campylobacter </i>reads
were detected, whereas a single read was detected in the chicken control
sample. We compared the SC181 sequences detected in the different sub-samples
and found overlaps ranging from 4101bp to 183027bp (see table).<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>&nbsp;<i>Table: Number of basepairs overlapping between Campylobacter reads
in samples spiked with different levels of Campylobacter. Probability of
observing an individual SNP in parenthesis.</i><o:p></o:p></span></p>

<table class=MsoNormalTable border=0 cellpadding=0 style='mso-cellspacing:1.5pt;
 mso-yfti-tbllook:1184;mso-padding-alt:0cm 5.4pt 0cm 5.4pt'>
 <tr style='mso-yfti-irow:0;mso-yfti-firstrow:yes'>
  <td style='padding:.75pt .75pt .75pt .75pt'></td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Human, 10<sup>8</sup>
  cfu/g<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Human, 10<sup>7</sup>
  cfu/g<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Chicken, 10<sup>6</sup>
  cfu/g<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Chicken, 10<sup>5</sup>
  cfu/g<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:1'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Human, 10<sup>8</sup>
  cfu/g<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>-<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'></td>
  <td style='padding:.75pt .75pt .75pt .75pt'></td>
  <td style='padding:.75pt .75pt .75pt .75pt'></td>
 </tr>
 <tr style='mso-yfti-irow:2'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Human, 10<sup>7</sup>
  cfu/g<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>4101 (1:365)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>-<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'></td>
  <td style='padding:.75pt .75pt .75pt .75pt'></td>
 </tr>
 <tr style='mso-yfti-irow:3'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Chicken, 10<sup>6</sup>
  cfu/g<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>127598 (1:12)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>19278 (1:78)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>-<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'></td>
 </tr>
 <tr style='mso-yfti-irow:4;mso-yfti-lastrow:yes'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Chicken, 10<sup>5</sup>
  cfu/g<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>15039 (1:100)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>2615 (1:574)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>183027 (1:8)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>-<o:p></o:p></span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>&nbsp;<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> This shows
that we will be able to identify a known outbreak strain directly in fecal
samples containing a load of infection above 10^5 cfu/g. Another issue is if we
will be able to compare the identified sequences from the metagenomics samples,
and estimate how different the strains are. With the loads and sequencing
depths we have investigated in this study, it would not be enough for outbreak
detection, although with the detection of 1/8 of the genome it is getting
close.</span><span style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO9 - The Role Of
Surface Adhesins In Epidemic Hypervirulent <i style='mso-bidi-font-style:normal'>Clostridium
difficile</i> BI/NAP1/027 Strain And Non-epidemic Strain Virulence And Bio<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>SOZA BABAN<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University Of Nottingham<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><i><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Clostridium difficile</span></i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>&nbsp;is a
major cause of healthcare-associated infection and inflicts a considerable financial
burden on healthcare systems worldwide.Disease symptoms range from
self-limiting diarrhoea to fatal pseudomembranous colitis.Whilst&nbsp;<i>C.difficile</i>&nbsp;has
two major virulence factors, toxin A and B,it is generally accepted that other
virulence components of the bacterium contribute to disease.<i>C.difficile</i>&nbsp;colonises
the gut of humans and hence the processes of adherence and colonisation are
essential for disease onset.Bacteria within biofilms are protected from
multiple stresses,including immune responses and antimicrobial agents.
Increased antibiotic resistance and chronic recurrent infections have been
attributed to the ability of bacterial pathogens to form biofilms.While
biofilms have been well studied for several gut pathogens, little is known
about biofilm formation by anaerobic gut species.Numerous cell surface proteins
and surface structures such as flagella have been shown to be important for
biofilm formation in this Gram-positive pathogen. This study investigated for
the first time the role of surface adhesion genes in virulence of two important
<i>C.difficile</i> isolates; non-epidemic historical <i>C.difficile</i>
PCR-ribotype012 and newly emerged hypervirulent <i>C.difficile</i>BI/NAP1/027
strain which caused large outbreaks ofCDAD worldwide.Non-motile, paralyzed-flagellated
and non-flagellated mutants were created using ClosTron and Allelic-exchange
mutagenesis technologies.Our work demonstrates the ability of epidemic pathogen
to form complex biofilms and the involvement of important clostridial pathways
in biofilm development.Importantly,thebacterial sensitivity to antibiotics is
reduced in clostridial biofilms. Biofilm formation may be a mechanism employed
by <i>C.difficile </i>to survive in environment such as human gut.We compared
the flagellated wild-type to a mutant with a paralyzed flagellum and also to
non-flagellated&nbsp;<i>in vitro</i>&nbsp;and&nbsp;<i>in vivo</i>.This study
provides the first strong evidence that by disabling the motor of the
flagellum, the structural components of the flagellum rather than active
motility, is needed for adherence and colonisation by hypervirulent strain.The
hypervirulent epidemic flagellar mutants adhered less than the parental
strain&nbsp;<i>in vitro</i>, whereas we saw the opposite in 630</span><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>Δ</span><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>erm</span></i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>. In the
later strain we showed that flagella and motility are not needed for successful
colonisation&nbsp;<i>in vivo</i>. Finally we demonstrated that in in epidemic
strain, flagella do play a role in colonisation and adherence and that there
are striking differences between&nbsp;strains. The latter emphasises the
overriding need to characterize more than just one strain before drawing
general conclusions concerning specific mechanisms of pathogenesis.In addition,
the hypervirulent strain, form structured biofilms <i>in vitro</i>.Biofilm
formation by <i>C.difficile</i> is a complex multifactorial process and may be
a crucial mechanism for clostridial persistence in the host.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO10 - <i
style='mso-bidi-font-style:normal'>Bifidobacterium bifidum</i> YIT 10347
Suppress Gut Microbiota Dysbiosis And Alleviate Severe Diarrhea In <i
style='mso-bidi-font-style:normal'>Helicobacter pylori</i> Eradication Therapy<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Toshifumi Ohkusa<sup>1</sup>; Takao Osaki<sup>2</sup>; Hideo Yonezawa<sup>2</sup>;
Shigeru Kamiya<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Department of gastroenterology and Hepatology, The
Jikei University Kashiwa Hospital,<sup>2</sup> Department of Infectious
Diseases, Kyorin University School of Medicine</span></i><span lang=EN-GB
style='mso-bidi-font-family:Arial;color:black;mso-themecolor:text1;mso-ansi-language:
EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> An
antibiotics-associated diarrhea (AAD) following <i>Helicobacter pylori</i>
eradication therapy is most side effects and may be due to gut microbiota
dysbiosis. However, it has not been a metagenomic analysis study about diarrhea
in <i>Helicobacter pylori</i> eradication therapy and gut microbiota dysbiosis.
We conducted a randomized double-blind placebo-controlled clinical trial to
examine the effect of probiotic <i>Bifidobacterium bifidum</i> YIT 10347 on <i>H.
pylori</i> eradication, diarrhea, and gut microbiota dysbiosis.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Patients and
Methods:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'> 228 Japanese patients 228 patients were divided into active drug and
placebo groups at random, and ingested 100 ml of either fermented milk
containing <i>B. bifidum</i> YIT 10347 (active group) or non-fermented milk
(placebo group) once daily for 17 weeks. Triple therapy of lansoprazole 30 mg
bid, amoxicillin 750 mg bid, and clarithromycin 200 mg bid (LAC) was treated at
week 8 for 1 week to eradicate <i>H. pylori</i>. Side effects and rate of
eradication were evaluated. Gut microbiota of 105 participants (active group:
49 subjects, placebo group: 56 subjects) selected randomly were analyzed on 8,
9, and 17 weeks by 16S rDNA metagenomics sequencing and the UniFrac index was
used to evaluate gut microbiota dysbiosis.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> <i>H. pylori</i> eradication rate and the
occurrence of side effects did not significantly differ between the groups,
except that the occurrence of severe diarrhea (4 times a day or more of
diarrhea) was significantly lower in the active group. Gut microbiota dysbiosis
occurred in both the group, but the increase in the UniFrac index on 17 weeks
was significantly suppressed in the active group. <i>B. bifidum</i> YIT 10347
suppressed decrease of Bifidobacteriaceae and increase of Bacteroidaceae and
alleviated decrease of Firmicutes phylum like Lachnospiraceae on 9 weeks.
Recovery of gut microbiota was significantly earlier in the active group than
in the placebo group on 17 weeks.<o:p></o:p></span></p>

<p class=MsoNormal><b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Conclusion:</span></b><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> In the <i>H.
pylori</i> eradication LAC therapy, ingestion of probiotic <i>B. bifidum</i>
YIT 10347 alleviated of severe AAD by suppression of gut microbiota dysbiosis.</span><span
lang=EN-GB style='mso-bidi-font-family:Arial;color:black;mso-themecolor:text1;
mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=DE-CH><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
