<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns:mv="http://macVmlSchemaUri" xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta name=Title content="">
<meta name=Keywords content="">
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List href="PO_S3_files/filelist.xml">
<link rel=Edit-Time-Data href="PO_S3_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Bruno Gonçalves</o:Author>
  <o:LastAuthor>Bruno Gonçalves</o:LastAuthor>
  <o:Revision>1</o:Revision>
  <o:TotalTime>0</o:TotalTime>
  <o:Created>2016-03-08T21:50:00Z</o:Created>
  <o:LastSaved>2016-03-08T21:50:00Z</o:LastSaved>
  <o:Pages>8</o:Pages>
  <o:Words>4378</o:Words>
  <o:Characters>24955</o:Characters>
  <o:Company>IMM</o:Company>
  <o:Lines>207</o:Lines>
  <o:Paragraphs>58</o:Paragraphs>
  <o:CharactersWithSpaces>29275</o:CharactersWithSpaces>
  <o:Version>14.0</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="PO_S3_files/themedata.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>JA</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
   <w:UseFELayout/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="276">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1073743103 0 0 415 0;}
@font-face
	{font-family:"Lucida Grande";
	panose-1:2 11 6 0 4 5 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-520090897 1342218751 0 0 447 0;}
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Balloon Text Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:9.0pt;
	font-family:"Lucida Grande";
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-ansi-language:DE-CH;}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Balloon Text";
	mso-ansi-font-size:9.0pt;
	mso-bidi-font-size:9.0pt;
	font-family:"Lucida Grande";
	mso-ascii-font-family:"Lucida Grande";
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:"Lucida Grande";
	mso-bidi-font-family:"Lucida Grande";
	mso-ansi-language:DE-CH;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"ＭＳ 明朝";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
@page WordSection1
	{size:595.0pt 842.0pt;
	margin:72.0pt 89.85pt 72.0pt 89.85pt;
	mso-header-margin:35.45pt;
	mso-footer-margin:35.45pt;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;}
</style>
<![endif]-->
</head>

<body bgcolor=white lang=EN-US style='tab-interval:36.0pt'>

<div class=WordSection1>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO38 - Description Of A
Fatal Outbreak In A Neonatal Intensive Care Unit (NICU) Caused By The
Paediatric Clone (ST5-IVa) Of MRSA<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Nuno Alexandre Faria<sup>1</sup>; Maria Miragaia<sup>1</sup>; Edgar
Botelho-Moniz<sup>2</sup>; M. José Espinar<sup>2</sup>; Susana Pissarra<sup>2</sup>;
Carlos Alves<sup>2</sup>; M. Manuela Ribeiro<sup>2</sup>; Hermínia de Lencastre3<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>Instituto de Tecnologia Química e Biológica António
Xavier,<sup>2</sup> Centro Hospitalar São João,<sup>3</sup> Laboratory of
Microbiology and Infectious Diseases, The Rockefeller University<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-ansi-language:PT;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Staphylococcus
aureus</span></i><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'> is a major nosocomial pathogen worldwide and one of the main causes of
outbreaks in neonatal intensive care units (NICU). In Portuguese hospitals,
MRSA prevalence reached 47.4% in 2014, the second highest rate in Europe, and a
single clone, EMRSA-15 accounts for more than 70% of nosocomial infections
caused by MRSA. In this study we characterized MRSA isolates responsible for a
large outbreak in a NICU, defined their clonal structure and origin and
established their relation with the known Portuguese MRSA epidemiology.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Between
November 2013 and April 2014, a MRSA outbreak occurred at the NICU of a major
Portuguese hospital that caused two deaths in newborn babies. A total of 35
isolates were recovered from 12 patients and one healthcare worker. Isolates
from four patients were recovered from infection (hemoculture and sputum, <i>n</i>=10)
and colonization (nasal, perineal and umbilical swabs, <i>n</i>=9), while
isolates from eight patients and one healthcare worker were collected from
colonization (<i>n</i>=16). All isolates were characterized by PFGE.
Representative isolates of each PFGE subtype per patient were characterized by <i>spa</i>
typing and MLST. Detection of Panton-Valentine leukocidin (PVL) and
staphylococcal cassette chromosome <i>mec</i> (SCC<i>mec)</i> typing was
performed by PCR. Representative isolates were screened for vancomycin
resistance by population analysis profiles (PAP). Whole genome sequencing (WGS)
was performed for key isolates to establish outbreak relation with hospital
flora.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>During the
NICU outbreak, 33 MRSA and 2 MSSA were recovered. Molecular characterization
identified two clonal types. The majority of the isolates belong to PFGE type A
(subtype A1), <i>spa</i> type t311, ST5-SCC<i>mec</i> IVa (n=31) (Paediatric
clone related); to subtype A3, <i>spa</i> type t586, ST5-MSSA (n=1); and to
subtype A4, <i>spa</i> type t010, ST5-MSSA (n=1). The remaining isolates belong
to PFGE B1, <i>spa</i> type t032, ST22-IVh (n=2) (EMRSA-15), and considered as
outbreak independent. All isolates were PVL negative. Preliminary PAP results
demonstrate that strains belonging to ST5-IVa contain subpopulations with
decreased susceptibility to vancomycin, having the capacity to grow up to 12.5 </span><b><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>μ</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>g/ml of
vancomycin (MIC, R&gt;2 mg/L). The Paediatric clone was detected in four
infected babies, colonizing other four babies and in a healthcare worker, the
majority at the end of the study period, suggesting cross transmission and
persistence of the clone in the NICU environment and the HCW as possible
vehicle of MRSA dissemination.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The Paediatric
clone and EMRSA-15 were previously described in this hospital as part of a
national surveillance performed in 2011. Back then EMRSA-15 was the major
clone, accounting for more than 50% of the MRSA population, whereas the
Paediatric clone was a sporadic isolate. WGS analysis is being performed to
assess relations between outbreak and previous minor clones, and to evaluate
the role of the HCW in the transmission chain during the outbreak.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>This is the
first description of an outbreak caused by the Paediatric clone in a NICU in
Portugal. Decreased susceptibility to vancomycin might have been the reason
behind therapeutic failure and newborn deaths. This can be indicative of the
clonal replacement of EMRSA15 by the Paediatric clone in the hospital and
directs attention to the potential clinical relevance of minor clones in
high-risk wards and the need of continuous surveillance of MRSA and vancomycin
resistance.</span><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><br clear=all style='mso-special-character:line-break;page-break-before:
always'>
<o:p></o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO39 - Application Of
Whole Genome Sequencing To Investigate A Two Year’s <i style='mso-bidi-font-style:
normal'>Pseudomonas aeruginosa</i> Outbreak<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Bárbara Magalhães<sup>1</sup>; Mohamed M. H. Abdelbary<sup>1</sup>;
Frédéric Tissot<sup>1</sup>; Patrick Basset<sup>1</sup>; Mette Berger<sup>2</sup>;
Yok-Ai Que<sup>2</sup>; Philippe Eggimann<sup>2</sup>; Guy Prod´Hom<sup>3</sup>;
Gilbert Greub<sup>3</sup>; Giorgio Zanetti<sup>1</sup>; Laurence Senn<sup>1</sup>;
Dominique S. Blanc1<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Service of Hospital Preventive Medicine, Lausanne
University Hospital,<sup>2</sup> Intensive Care Service, Lausanne University
Hospital,<sup>3</sup> Institue of Microbiology, Lausanne University Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:12.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>From 2010 to
2012, an increase in <i>P. aeruginosa</i> incidence was observed in the ICUs at
the University Hospital of Lausanne. A total of 689 isolates from 254 patients
were typed using Double Locus Sequence Typing (DLST), and subsequently grouped
into 46 DLST clusters. Cluster DLST 1-18 affected the highest number of
patients (24 out of 254), mostly hospitalized due to burn injuries. Isolates of
the same DLST type were found in the environment of the burn unit’s
hydrotherapy room, suggesting this location was the source of the outbreak. To
investigate the suspected transmission events, and to infer the outbreak
isolates’ phylogeny, we decided to complement this investigation with the high
discriminatory power of whole genome sequencing (WGS).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:12.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Material and
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>DLST cluster 1-18 incorporates 106 isolates, including clinical specimens
of 24 patients, as well as environmental isolates. In this preliminary study,
we performed WGS on ten clinical isolates randomly selected between March 2010
and October 2012. The isolates’ sequence type (ST) was assigned from the short
reads data. Core genome alignment of all ten isolates was acquired with Snippy.
This alignment was subsequently used as an input for maximum likelihood tree
construction with Gubbins, excluding regions of high SNP density suggestive of
recombination.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:12.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>All ten
sequenced isolates belonged to <i>P. aeruginosa</i> ST 1076. Phylogenetic
analysis demonstrated the occurrence of a major ICU clade, clearly separated
from one isolate retrieved from a patient possibly infected before its
admission to the ICU. The ICU clade was further divided into subclade A,
including six isolates from patients hospitalized in ICU 3 (burn unit), and
subclade B, which comprises three isolates from other ICUs (ICU 2, 4, and 6).
WGS results confirmed the occurrence of two suspected epidemiological links in
the burn unit (hospitalized at the same time), while determined the close
relatedness of two isolates retrieved two years apart, suggesting an
environmental source.<o:p></o:p></span></p>

<p class=MsoNormal><b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Preliminary
results showed a clear differentiation between <i>P. aeruginosa</i> DLST 1-18
isolates present in the burn unit, and isolates retrieved in other ICUs or
hospital units. This suggests that different routes of transmission may have
occurred during this outbreak. These results prompt us to further investigate
this outbreak by sequencing isolates from other patients and from the
environment in order to have a broader representation of transmission events
and probable outbreak sources.<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO40 - KPC-2
Carbapenemase Producing <i style='mso-bidi-font-style:normal'>Enterobacteriacae</i>
At A Multi-species Nosocomial Outbreak: Genomic Insights Of A Novel IncN
Plasmid And Its Dissemination<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Yancheng Yao<sup>1</sup>; Can Imirzalioglu<sup>1</sup>; Yalda Rezazadeh<sup>1</sup>;
Konrad Gwozdzinski<sup>1</sup>; Linda Falgenhauer<sup>1</sup>; Jörg Overmann<sup>2</sup>;
Boyke Bunk<sup>2</sup>; Alexander Goesmann<sup>3</sup>; Soeren Gatermann<sup>4</sup>;
Martin Kaase<sup>4</sup>; Torsten Hain<sup>1</sup>; Eugen Domann<sup>1</sup>;
Trinad Chakraborty1<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Institute for Medical Microbiology,
Justus-Liebig-University Giessen and German Centre for Infection Research
(DZIF), partner site Giessen-Marburg-Langen, Compus Giessen, Germany,<sup>2</sup>
Leibniz Institute DSMZ- Bioinformatics, German Collection of Microorganisms and
Cell Cultures, Braunschweig, and German Center for Infection Research (DZIF),
partner site Hannover-Braunschweig, Germany,<sup>3</sup> Bioinformatics
Computational Biology, Justus-Liebig-University Giessen and German Centre for
Infection Research (DZIF), partner site Giessen-Marburg-Langen, Compus Giessen,
Germany,<sup>4</sup> Institute for Hygiene and Medical Microbiology, Ruhr
University, Bochum, Germany<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>
Carbapenem-resistant <i>Enterobacteriaceae</i> are an increasing cause of
nosocomial infections and hospital outbreaks. An outbreak involving 132
patients occurred in a single hospital in Germany from October 2013 to
September 2014. The outbreak was unusual as it involved many different
bacterial species and sub-species <i>Enterobactericeae</i> all exhibiting <i>KPC-2</i>
carbapenem-resistance. It could be demonstrated that all isolates harboring
a&nbsp;<i>bla</i><sub>KPC-2</sub> carrying plasmid. We describe the
determination and comparison of the genomes and plasmids of the carbapenemase
encoding isolates to discover horizontal carbapenemase transfer between
bacteria of different species, as well as to find evidence for dissemination of
a distinct&nbsp;<i>bla</i><sub>KPC-2</sub> carrying plasmid in Germany.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Material and
Methods</span></b><b style='mso-bidi-font-weight:normal'><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> We analysed
36 outbreak isolates, including <i>Citrobacter freundii</i> (n=13), <i>Escherichia
coli</i> (n=7), <i>Klebsiella oxytoca</i> (n=6), <i>K. pneumoniae</i> (n=3), <i>Enterobacter
aerogenes</i> (n=3), <i>C. amalonaticus</i>, <i>C. braakii</i>, <i>C. koseri</i>
and <i>E. cloacae</i> from patients and the outbreak hospital as well as 7 non
outbreak isolates from three other hospitals by whole genome sequencing
(Illumina MiSeq). A subset of 10 isolates was additionally sequenced with
single-molecule-real-time sequencing (PacBio RSII) for complete genome
assembly. S1-nuclease digestion followed by pulsed-field gel electrophoresis
was carried out to investigate the plasmids. Phylogenetic and comparative
analysis was performed.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> The phylogenetic
analysis and <i>in silico</i> MLST typing revealed that the isolates of <i>K.
oxytoca</i> belonged to a distinct clone, while all other outbreak related
species such as <i>C. freundii</i>, <i>E. coli,</i> <i>K. pneumoniae</i> and <i>E.
aerogenes</i> were not clonally related and displayed different ST´s. The
non-outbreak isolates were phylogenetically not related to the outbreak
isolates. The carbapenem resistence gene <i>bla</i><sub>KPC-2</sub> was located
adjacent to <i>bla</i><sub>TEM-1</sub> in a 9,571 bp unique genetic environment
on an IncN plasmid. One isolate harboured <i>bla</i><sub>KPC-2</sub> on an
IncFI plasmid. The IncN plasmids encoded CDS required for replication,
conjugation transfer and a multi drug resistance (MDR) region, but varied
considerably in size, from 58,447 to 78,849 bp mainly due to variations in the
MDR region. The most frequently occurring plasmid comprises 78,021 bp encoding
100 CDS and including 13 different antibiotic resistance genes in a single
multi drug resistance region of 40,501 bp. Plasmids were transferrable to <i>E.
coli</i> J53 by conjugation but also showed size variation following transfer
indicating that both mobility and genetic variability are inherent properties
of these plasmids.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Complete genome
sequence analysis supports the finding of promiscuous plasmid transmission
between the carbapenem-resistant multi-species isolates involved in the
described nosocomial outbreak. The outbreak was caused by the efficient and
dynamic transfer of a novel IncN plasmid. This plasmid does not only occur
locally as in the hospital outbreak but could be demonstrated in independent
isolates from different regions in Germany, hinting towards an endemic
situation in healthcare settings with the potential for new outbreaks. The
presence of the unique <i>bla</i><sub>KPC-2</sub> in direct genetic vicinity
with<i> bla</i><sub>TEM-1</sub> in all outbreak strains and non-outbreak
strains can be used as a unique molecular probe for detecting this plasmid in
outbreak situations and epidemiologic investigations.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO41 - Real-time
Whole-genome Sequencing For Investigating A Vancomycin-resistant <i
style='mso-bidi-font-style:normal'>Enterococcus faecium</i> Outbreak At A
Tertiary Care Hospital<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Mohamed M. H. Abdelbary<sup>1</sup>; Claire Bertelli<sup>2</sup>; Gilbert
Greub<sup>2</sup>; Delphine Hequet<sup>1</sup>; Giorgio Zanetti<sup>1</sup>;
Laurence Senn<sup>1</sup>; Dominique S. Blanc1<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Service of Hospital Preventive Medicine, Lausanne
University Hospital,<sup>2</sup> Institute of Microbiology, Lausanne University
Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Between
January and March 2015, an outbreak of vancomycin-resistant <i>Enterococcus
faecium</i> (VREfm) occurred at the University Hospital of Lausanne. We
performed whole-genome sequencing (WGS) to determine characteristic genetic
signatures in the outbreak isolates for developing a specific multiplex PCR
that could be implemented in the outbreak control measures. Furthermore, we
investigated the relatedness, molecular features and the transmission events of
the outbreak isolates. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials and
Methods:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'> In total, twenty-six VREfm isolates that were retrieved from clinical
samples and rectal swabs were sequenced using the Illumina MiSeq platform.
Genome sequences were subjected to <i>in silico </i>multi-locus sequence typing
(MLST) and the identification of genetic determinants of antibiotic resistance.
Comparative genome analysis of the assembled ordered draft genomes was
performed using the Mauve software. To infer the isolates’ phylogeny, all
paired-end reads were mapped to the most closely related reference genome and,
subsequently, a maximum likelihood tree was constructed using single nucleotide
polymorphisms (SNPs) from a core genome alignment.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> The <i>in
silico </i>MLST analysis revealed that twenty-five of the sequenced isolates
represented the multilocus sequence type (ST) 17, whereas the remaining isolate
belonged to ST80. Using the comparative genomics approach, we identified
genetic loci responsible for polysaccharide biosynthesis that were specific to
all ST17 outbreak isolates. These loci were used to develop a multiplex PCR to
differentiate between outbreak and non-outbreak isolates, which preceded and
followed the outbreak period. This specific PCR revealed that three additional
patients were part of the outbreak; two of them were hospitalized in November
2014, while the remaining patient was admitted in late June 2015. Furthermore,
we were able to exclude four patients from the outbreak.<br>
The phylogenetic analysis demonstrated that outbreak isolates were very closely
related with only 0-4 SNPs differences. It also revealed that none of our
isolates could be considered as the initial source of the outbreak, suggesting
that previous transmission events occurred in the hospital.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> WGS
analysis provided comprehensive evidence of a clonal outbreak of VREfm, and was
able to precisely distinguish outbreak from non-outbreak isolates. Furthermore,
it suggests that there were some undiscovered cases prior to the index case;
and that VREfm was introduced from a point source with secondary transmissions.
WGS demonstrates the prospect to substitute currently used phenotypic and
molecular typing approaches to investigate outbreaks.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO42 - Fast Update,
Supported On ASO-PCR Based On WGS Of The Prevalence Of A <i style='mso-bidi-font-style:
normal'>Mycobacterium tuberculosis</i> Outbreak-strain 20 Years After Its
Emergence<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Laura Pérez-Lago<sup>1</sup>; Estefanía Abascal<sup>2</sup>; Diana Maricela
Herrera<sup>2</sup>; Iñaki Comas<sup>3</sup>; Sofía Samper<sup>4</sup>; María
Isolina Campos-Herrero<sup>5</sup>; Rodolfo Copado<sup>6</sup>; Laura Sante<sup>7</sup>;
Emilio Bouza<sup>1</sup>; Darío García de Viedma1<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>Instituto de Investigación Sanitaria Gregorio Marañón,<sup>2</sup>
CIBER Enfermedades Respiratorias, CIBERES,<sup>3</sup> Instituto de
Investigación Sanitaria Gregorio Marañón,<sup>4</sup> Genomics and Health Unit,
FISABIO Public Health,<sup>5</sup> CIBER Salud Pública CIBERESP,<sup>6</sup>
Instituto de Investigación Sanitaria Aragón,<sup>7</sup>Hospital Universitario
Miguel Servet,<sup>8</sup> CIBER Enfermedades Respiratorias, CIBERES,<sup>9</sup>
Hospital Universitario de Gran Canaria Dr. Negrín,10_ Hospital Dr. José Molina
Orosa,1<sup>1</sup> Hospital Universitario de Canarias<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-ansi-language:PT;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Standard
genotypic analysis allowed the identification of a severe outbreak of
tuberculosis in Gran Canaria (GCI), Canary Islands, Spain. It involved a
Beijing strain (GCI strain), which was imported by a Liberian immigrant in 1993
and it spread until being responsible for 25% of the total number of cases in
GCI. Since 2008, no new data about the prevalence of the GCI strain in the
island have been obtained. A recent tracking of the GCI strain in Madrid
allowed to identify it from two immigrant cases recently diagnosed (2014) after
arriving from Guinea Conakry, a country sharing borders with Liberia, which was
the origin of the first importation, suggesting that the GCI strain might be
prevalent in West Africa. Our study aimed to i) update the current situation of
the GCI strain in the Canary Islands using a simple, fast and low cost tool
based on targeting strain-specific SNPs detected by whole genome sequencing
(WGS) and ii) characterize the potential microevolution of this strain.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials/Methods:
</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>WGS was used to identify the specific SNPs for the GCI strain. We optimized
a multiplex allele-specific PCR (ASO-PCR) targeting 4 strain-specific SNPs. It
was applied directly on boiled crude extracts from the stored collections
(period 2013-2014) from 3 hospitals which collect strains from four of the
seven Canary Islands, Gran Canaria, La Palma, Lanzarote and Tenerife (these
islands are representative of the two provinces of the archipelago). MIRU-VNTR
was applied to track the emergence by microevolution of clonal variants of the
GCI strain.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The ASO-PCR
specific for the GCI strain was applied to 145 isolates from 4 of the 7 Canary
Islands. ASO-PCR results were available within the same day in which the
isolates were received. The amplification pattern expected for the GCI strain
was identified in 32 isolates (22%). Among them, a pattern consistent with a
co-infection involving a GCI strain and another strain was found in two cases.
4 clonal variants, with variations in a single MIRU-VNTR locus each, were
detected.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>We could
fastly update the situation of the GCI outbreak strain by applying a simple,
fast and low-cost strategy based on ASO-PCR targeting strain-specific SNPs.
This strategy also allowed to identify mixed infections involving the GCI
strain and another non-GCIstrain. 22% of the current TB cases in the Canary
Islands are still infected by this strain. The high prevalence of the GCI
strain is not restricted to the island where the outbreak was described, Gran
Canaria, but it was also found in other islands. Several microevolved clonal
variants have emerged from the parental outbreak strain and are being also
transmitted.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=ES style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:ES;mso-fareast-language:
PT'>Funding:</span></b><span lang=ES style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
ES;mso-fareast-language:PT'> Plan Estatal I+D+I 2013-2016, ISCIII (13/01207;
15/01554) cofunded by the Fondo Europeo de Desarrollo Regional (FEDER): “Una
manera de hacer Europa”. </span><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Miguel Servet research grant CP15/00075 (to
L.P.), Ramón y Cajal and MINECO research grants (RYC-2012-10627,
SAF2013-43521-R) and the European Research Council (ERC) (638553-TB-ACCELERATE)
(to I.C).</span><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO43 - Microevolution
And Persistence Of <i style='mso-bidi-font-style:normal'>Listeria monocytogenes</i>
In The Environment And Links To A Foodborne Outbreak<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Vítor Borges<sup>1</sup>; Carla Maia<sup>1</sup>; Maria João Barreira<sup>1</sup>;
Leonor Silveira<sup>1</sup>; Catarina Silva<sup>1</sup>; Silvia Duarte<sup>1</sup>;
Luis Vieira<sup>1</sup>; Luis Lito<sup>2</sup>; Mónica Oleastro<sup>1</sup>;
João Paulo Gomes<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>National Institute of Health, Portugal,<sup>2</sup>
Laboratory of Microbiology, Department of Clinical Pathology, Centro Hospitalar
de Lisboa Norte<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Listeria
monocytogenes</span></i><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> is a foodborne human pathogen associated with a
high mortality rate. Given the remarkable ability of this bacterium to persist
in food processing facilities, ready-to-eat food products are common vehicles
of <i>L. monocytogenes. </i>In the present study, we report the use of
whole-genome sequencing (WGS) to investigate the diversity among <i>L.
monocytogenes</i> isolates associated with a hospital outbreak of listeriosis
occurred in Portugal.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials |
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-font-weight:bold'>To investigate the potential source of 3 cases of
</span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>hospital-acquired <i>L. monocytogenes </i>infections, we performed WGS (<span
style='mso-bidi-font-weight:bold'>on a</span> MiSeq Illumina instrument) on
strains isolated from: <span style='mso-bidi-font-weight:bold'>patient (n= 3);
environment, collected by swabbing surfaces in the establishment that produces
the foodstuffs for the hospital (n = 20); and ready-to-eat foods prepared in
this establishment and served in the hospital (n = 4). Samples from the
environment included </span>surfaces of the processing area and equipment,<span
style='mso-bidi-font-weight:bold'> and were collected spaced by 2 weeks, with
cleaning and disinfection procedures performed during this interval. To infer
the genetic relatedness among all the strains isolated, we performed
reference-based mapping followed by the identification of bona fide core Single
Nucleotide Polymorphisms (SNPs) (using samtools/bcftools). SNP-based p</span>hylogenetic
relationships among the isolates were visually depicted in a minimum spanning
tree constructed using the goeBURST algorithm implemented in the PHYLOViZ
platform.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-font-weight:
bold'>L. monocytogenes&nbsp;</span></i><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT;mso-bidi-font-weight:bold'>infections were
confirmed in&nbsp;3 patients admitted to a central hospital in Portugal, being
the meals prepared in a food establishment and distributed in the hospital the
presumable pathogen´s vehicle. WGS-based investigation of the relatedness
between clinical isolates and multiple isolates collected from that
establishment and ready-to-eat foods (Figure 1) revealed:</span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-font-weight:
bold'>i) a high degree of genetic homogeneity between clinical and
environmental isolates, with a maximum SNP distance among all isolates of only
12 SNPs;</span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-font-weight:
bold'>ii) 14 unique sequences profiles, indicating that at least 14 distinct
clones coexisted in the&nbsp;outbreak´ source; </span><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-font-weight:
bold'>iii) that 2 clinical cases genetically match to several clones collected
from diverse locations in the&nbsp;establishment that produces the foodstuffs
for the hospital, whereas the third patient isolate was closely linked with a
single location;</span><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-font-weight:
bold'>iv) </span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>a repertoire of loci likely involved in microevolutionary events (where
non-synonymous mutations are overrepresented) potentially driving <span
style='mso-bidi-font-weight:bold;mso-bidi-font-style:italic'>L. monocytogenes</span><span
style='mso-bidi-font-weight:bold'> environmental</span> persistence;<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-font-weight:
bold'>v) that clones isolated from ready-to-eat foods perfectly matched to
clones circulating in the surfaces</span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> of the processing
area and equipments<span style='mso-bidi-font-weight:bold'>, corroborating that
cross-contamination may still occur. </span><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-font-weight:
bold'>Further analyses are underway to investigate the potential existence of
mobile genetic elements in order to increase the resolution power, and perhaps,
to establish the genetic link between the two indistinguishable clinical
isolates and a particular location in the outbreak source.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>In this
study, we show how the high discriminatory power of WGS may identify potential
links within outbreak scenarios and unveil how <i><span style='mso-bidi-font-weight:
bold'>L. monocytogenes </span></i><span style='mso-bidi-font-weight:bold'>microevolution
may pose some challenge in attributing meaningful similarity thresholds for
food source attribution. It can also contribute to identify genes targeted by
microevolution that could contribute to the <i>L. monocytogenes</i>persistencein
food processing facilities.<o:p></o:p></span></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-font-weight:
bold'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='mso-ansi-language:
EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shapetype id="_x0000_t75"
 coordsize="21600,21600" o:spt="75" o:preferrelative="t" path="m@4@5l@4@11@9@11@9@5xe"
 filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Imagem_x0020_34" o:spid="_x0000_i1025" type="#_x0000_t75"
 alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/228imagem_2560108707_0.jpg"
 style='width:335pt;height:308pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S3_files/image001.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/228imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img width=337 height=310
src="public/abstracts/Webpages/PO_S3_files/image002.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/228imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_34"><![endif]></span><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO44 - Tracing Of
Influenza A H3N2 Outbreaks In Two Nursing Homes And A Hospital In Bavaria By
Whole Genome Sequencing Within The Bavarian Influenza Sentinel<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Dominik Meinel<sup>1</sup>; Susanne Heinzinger<sup>2</sup>; Ute Eberle<sup>2</sup>;
Helmut Blum<sup>3</sup>; Katharina Schönberger<sup>2</sup>; Andreas Sing<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Unispital Basel,<sup>2</sup> Bavarian Health and Food
Safety Authority,<sup>3</sup> Gene Center<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Annually the
influenza epidemics affect several thousand people in Europe. Morbidity,
mortality and associated healthcare costs are high. Many countries conduct
surveillance programs collecting epidemiological data about the influenza
season, however only very little large-scale genomic data is available
regarding the molecular epidemiology of an influenza season helping to
understand transmission routes. Here we describe a fast and straight forward
tool to investigate transmission routes of Influenza A in surveillance programs
by NGS.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>We
established a fast RT-PCR strategy starting directly from patient material
(swabs) to generate influenza A whole genome sequences. The patient swabs were
obtained within the Bavarian influenza Sentinel and extracted using a routine
laboratory Hamilton robotic device. The entire Influenza A virus genome was
amplified in one multiplex RT-PCR reaction with an RT and a PCR step. DNA was
sequenced using an Illumina MiSeq with 50x multiplexing of samples. Data
analysis was carried out using a local Galaxy server, R and Mega 6.0. The
analysis included quality trimming, mapping (Bowtie 2.0), variant calling (Varscan)
and phylogenetic analysis with various algorithms (<i>e.g. </i>Neighborhood
joining).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>We report
the genomic sequences for 50 influenza A viruses from the H3N2 type which was
the dominating virus in the season 2014/15 with a very high coverage (&gt; 350x
fold). The dataset included randomly sampled isolates from all over Bavaria
(Germany) and spiked in isolates, which were derived from three suspected
influenza A H3N2 outbreaks in two nursing homes and one hospital in 2014/15.
The samples were temporarily and spatially matched. We were able to identify
highly robust the outbreak isolates, which were spiked into the randomly
selected dataset, based on the very high sequence identity within the isolates
belonging to each outbreak. We show that WGS is superior in resolution compared
to analysis of single segments, such as HA or NA. Furthermore, we found one of
the suspected outbreak isolates to be genomically unrelated to the other
samples of the same outbreak, suggesting an independent infection and illustrating
the high discriminatory power of WGS. Additionally, we detected manifestation
of substantial amounts of viral quasispecies in several patients, carrying
mutations varying from the dominant virus in each patient.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Our rapid
whole genome sequencing approach for influenza A virus shows that WGS can
effectively be used to detect and understand outbreaks in communities.
Additionally, the genome data provides details about the circulating strains
within a community and will help to detect mutations of the virus within one
season thus contributing to our in depth understanding of transmission pathways
and evolution of the Influenza virus.</span><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO45 - Large-scale
Survey Of <i style='mso-bidi-font-style:normal'>Bacillus cereus</i>-induced
Food-borne Outbreaks: Epidemiological Data And Genetic Diversity<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Benjamin Glasset<sup>1</sup>; Anne Brisabois<sup>1</sup>; Sabine Herbin<sup>1</sup>;
Jacques-antoine Hennekinne<sup>1</sup>; Nalini Ramarao<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>ANSES,<sup>2</sup> INRA</span></i><span lang=EN-GB
style='mso-bidi-font-family:Arial;color:black;mso-themecolor:text1;mso-ansi-language:
EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-bidi-font-family:Arial;color:black;mso-themecolor:text1;mso-ansi-language:
EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Bacillus
cereus</span></i><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'> is a ubiquitous spore-forming bacterium responsible for food-borne
outbreaks (FBOs)<i>. B. cereus</i> can induce diarrhoeal and emetic syndromes
depending on the type of strains, the first due to production of enterotoxins
(diarrhoeal strains) and the second to ingestion of cereulide (emetic strains)<i>.</i>
Historically, <i>B. cereus </i>has often been regarded as a surface
contaminant, but the increasing number and gravity of suspected <i>B. cereus</i>
FBOs highlighted the importance of a thorough characterisation of strains in
relation to the corresponding epidemiological data. Indeed, <i>B. cereus</i> is
now estimated as the fourth agent responsible for FBOs in the European Union
and the second in France.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials &amp;
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>The aim of this study was therefore to identify and characterise <i>B.
cereus </i>strains from a unique national collection of 564 strains closely
related to 140 FBOs that occurred in France during the 2007-2014 period in
order to improve the monitoring and investigation of <i>B. cereus</i> FBOs,
assess the risk of emerging clusters of strains and identify strain
variability. The epidemiological data related to each FBO were collected
through interviews or questionnaires by local health authorities.
Characterization of the strains was based on phenotypic and genotypic methods.
Diversity of isolates in samples within a same FBO,was studied using M13-PCR
derived from an RAPD technique.An extensive molecular characterisation study
was conducted on already known virulence markers: the genes encoding cereulide,
diarrhoeic toxins (Nhe, Hbl, CytK1 and CytK2) and Haemolysin HlyII, as well as <i>panC</i>
phylogenetic classification.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Starchy food
and vegetables were the most frequent food vehicles identified and 82% of human
cases occurred in institutional catering contexts. A significant difference was
observed for incubation periods before the onset of symptoms according to the
type of strain. The combination of the presence or absence of the virulence
markers allowed to cluster the strains into 12 “genetic signatures” (GS), GS1
to GS12. The most frequent GS encountered in the collection was GS1, which
accounted for 22% of strains. Several genetic signatures defined in this study
were associated to a single <i>panC</i> phylogenetic group ie: GS2 (IV), GS3
(III) GS7 (VII), GS8 (VI), GS11 (III) and GS12 (II). 84% of the strains
belonged in GS1 to GS6 thus showing that there were main profiles of <i>B.
cereus</i> strains involved in food poisoning. That allowed us clustering
strains into several features and determined the virulence marker combinations
of the strains responsible for FBOs.<o:p></o:p></span></p>

<p class=MsoNormal><b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>This study
highlighted the virulence potential of each strains and the genetic diversity
among <i>B. cereus</i> in FBOs. Indeed, certain toxin gene profiles were
significantly more encountered and seem to play a greater role in food poisoning.
Taken together, this study provided an accurate and extensive report of FBOs
associated with <i>B. cereus </i>and provided insights into molecular
characterisation and genetic diversity of the <i>B. cereus </i>strains
responsible for FBOs. We show that a scheme should be followed by food safety
authorities to better investigate the FBOs suspicion associated with <i>B.
cereus,</i> which is a real food pathogen with a strong impact on human health.
Further studies are now necessary to precisely characterized the virulence
determinants able to predict the pathogenicity and virulence of strains
isolated by food companies to avoid risks of FBOs.</span><span lang=EN-GB
style='mso-bidi-font-family:Arial;color:black;mso-themecolor:text1;mso-ansi-language:
EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=DE-CH><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
