<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns:mv="http://macVmlSchemaUri" xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta name=Title content="">
<meta name=Keywords content="">
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List href="PO_S5_files/filelist.xml">
<link rel=Edit-Time-Data href="PO_S5_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Bruno Gonçalves</o:Author>
  <o:LastAuthor>Bruno Gonçalves</o:LastAuthor>
  <o:Revision>1</o:Revision>
  <o:TotalTime>0</o:TotalTime>
  <o:Created>2016-03-08T21:52:00Z</o:Created>
  <o:LastSaved>2016-03-08T21:52:00Z</o:LastSaved>
  <o:Pages>20</o:Pages>
  <o:Words>10419</o:Words>
  <o:Characters>59390</o:Characters>
  <o:Company>IMM</o:Company>
  <o:Lines>494</o:Lines>
  <o:Paragraphs>139</o:Paragraphs>
  <o:CharactersWithSpaces>69670</o:CharactersWithSpaces>
  <o:Version>14.0</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="PO_S5_files/themedata.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>JA</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
   <w:UseFELayout/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="276">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1073743103 0 0 415 0;}
@font-face
	{font-family:"Lucida Grande";
	panose-1:2 11 6 0 4 5 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-520090897 1342218751 0 0 447 0;}
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	mso-themecolor:hyperlink;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:purple;
	mso-themecolor:followedhyperlink;
	text-decoration:underline;
	text-underline:single;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Balloon Text Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:9.0pt;
	font-family:"Lucida Grande";
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-ansi-language:DE-CH;}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Balloon Text";
	mso-ansi-font-size:9.0pt;
	mso-bidi-font-size:9.0pt;
	font-family:"Lucida Grande";
	mso-ascii-font-family:"Lucida Grande";
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:"Lucida Grande";
	mso-bidi-font-family:"Lucida Grande";
	mso-ansi-language:DE-CH;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"ＭＳ 明朝";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
@page WordSection1
	{size:595.0pt 842.0pt;
	margin:72.0pt 89.85pt 72.0pt 89.85pt;
	mso-header-margin:35.45pt;
	mso-footer-margin:35.45pt;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;}
</style>
<![endif]-->
</head>

<body bgcolor=white lang=EN-US link=blue vlink=purple style='tab-interval:36.0pt'>

<div class=WordSection1>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO51 - Evaluation Of A
Core Genome MLST Scheme For Typing Of Vancomycin-resistant Enterococci Using
Pulsed-field Gel Electrophoresis As The Reference Method<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Meyke Gillis<sup>1</sup>; Johanna Tien<sup>1</sup>; Danuta Stefanik<sup>1</sup>;
Bozica Licanin<sup>1</sup>; Daniela Tacke<sup>2</sup>; Maria J.G.T. Vehreschild<sup>3</sup>;
Paul G. Higgins<sup>1</sup>; Harald Seifert<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Institute for Medical Microbiology, Immunology and
Hygiene, University of Cologne,<sup>2</sup> Department of Anesthesiology and
Intensive Care Medicine, University Hospital of Cologne,<sup>3</sup> Department
I for Internal Medicine, University Hospital of Cologne<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Colonization
and infection with vancomycin-resistant enterococci (VRE) is an emerging
problem in patients suffering from hematological malignancies and among ICU patients.
To better control the nosocomial spread of VRE transmission analysis among VRE
colonized and infected patients is of fundamental importance. However, VRE has
shown a propensity for recombination which makes whole-genome sequencing (WGS)
and single nucleotide polymorphism (SNP) analysis difficult to interpret. A
core genome MLST (cgMLST) scheme for typing of <i>Enterococcus faecium </i>has
been recently published (doi:10.1128/JCM.01946-15) and the objective of this
study was to evaluate it among VRE isolates presenting with similar
PFGE-patterns.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials |
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>VRE isolates were collected from colonized patients during active
surveillance (rectal swabs/stool specimens) at the University Hospital Cologne.
According to PFGE patterns, VRE isolates were selected for WGS and cgMLST
analysis. Overall we evaluated 16 VRE isolates from 4 different patients:
Patients 1 and 3 had 5 isolates each. Three isolates were derived from the same
sample (i.e., 3 VRE colonies were picked from the same agar plate). One isolate
was collected at a distinct time point. The remaining isolate was obtained from
the initial culture and sub-cultured over a 4-week period at one-week
intervals. Patients 2 and 4 had 3 isolates each taken at different time points.
Patients 1, 2 and 3 displayed an identical pulsotype (pulsotype A1). Patient 4
presented with a slightly different pulsotype (pulsotype A2; i.e., 4-5 band
differences). To identify clonally related VRE isolates cgMLST using SeqSphere<sup>+</sup>
version 3.0.0 was interpreted by the following criteria; 0 allelic differences
= isolates indistinguishable; 1-20 differences = closely related; 21-40
differences = possibly related; &gt;40 differences = most likely unrelated.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results of
cgMLST are summarized in figure 1. In intra-individual isolates, no allele
differences were detected at the same time point, at different time points and
after being sub-cultured. With regard to VRE isolates from different patients,
isolates from patients 1 and 2 showed 7 allele differences, whereas isolates
from patient 3 exhibiting identical PFGE patterns to those from patient 2
displayed 31 allele differences when compared to isolates from patient 2. Of
note, isolates from patient 4 were separated by only 11 allelic differences
from those of patient 3, although displaying a slightly variant pulsotype
compared to the isolates recovered from other patients. Moreover, cgMLST
divided isolates into 2 different cluster types based on allelic differences:
Isolates from patients 1 and 2 belonged to cluster type 20, while isolates from
patients 3 and 4 corresponded to cluster type 28.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Intra-individually,
cgMLST correlated well with PFGE patterns. However, when comparing both methods
inter-individually, allelic differences seen by cgMLST were not necessarily
reflected by different PFGE patterns: VRE isolates were interpreted as only
possibly related according to the number of allelic differences but showed
indistinguishable PFGE patterns. We therefore conclude that the cgMLST scheme
evaluated in our study allows us to further subdivide VRE isolates within the
same pulsotype and therefore might potentially provide a more accurate analysis
of clonal relatedness among clinical VRE when compared to conventional PFGE analysis.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='mso-ansi-language:
EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shapetype id="_x0000_t75"
 coordsize="21600,21600" o:spt="75" o:preferrelative="t" path="m@4@5l@4@11@9@11@9@5xe"
 filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Imagem_x0020_36" o:spid="_x0000_i1030" type="#_x0000_t75"
 alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/59imagem_2560108707_0.jpg"
 style='width:333pt;height:251pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S5_files/image001.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/59imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img width=335 height=253
src="public/abstracts/Webpages/PO_S5_files/image002.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/59imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_36"><![endif]></span><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO52 - Real-time
Identification Of Carbapenem-resistant <i style='mso-bidi-font-style:normal'>Klebsiella
pneumoniae</i> Outbreak Isolates By Fourier Transform Infrared (FTIR)
Spectroscopy<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Liliana Silva<sup>1</sup>; Carla Rodrigues<sup>2</sup>; Clara Sousa<sup>3</sup>;
Agostinho Lira<sup>4</sup>; Mariana Leão<sup>4</sup>; Margarida Mota<sup>4</sup>;
Paulo Lopes<sup>4</sup>; Angelina Lameirão<sup>4</sup>; Gabriela Abreu<sup>4</sup>;
João Lopes<sup>5</sup>; Ângela Novais<sup>2</sup>; Luísa Peixe6<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>UCIBIO@REQUIMTE, Laboratório de Microbiologia,
Faculdade de Farmácia, Universidade do Porto, Porto, Portugal,<sup>2</sup>
ESALD, Instituto Politécnico de Castelo Branco, Castelo Branco, Portugal,<sup>3</sup>
UCIBIO@REQUIMTE, Laboratório de Microbiologia, Faculdade de Farmácia,
Universidade do Porto, Porto, Portugal.,<sup>4</sup> CEB-Centro de Engenharia
Biológica, Universidade do Minho, Braga, Portugal.,<sup>5</sup> CHVNG/E,
Serviço de Patologia Clínica, Centro Hospitalar de Vila Nova de Gaia/Espinho,
Vila Nova de Gaia, Portugal,<sup>6</sup> iMed, Departamento de Farmácia
Galénica e Tecnologia Farmacêutica, Faculdade de Farmácia, Universidade de
Lisboa, Lisboa, Portugal.,<sup>7</sup> UCIBIO@REQUIMTE, Laboratório de
Microbiologia, Faculdade de Farmácia, Universidade do Porto, Porto, Portugal<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The rapid
emergence and spread of multidrug resistant (MDR) bacteria causing
hospital-acquired infections demands a rapid and accurate control of nosocomial
outbreaks. DNA-based approaches (mainly PFGE, MLST or whole genome sequencing)
are commonly used for bacterial typing, nonetheless these methods are still
expensive and/or time-consuming. We recently demonstrated that FTIR-ATR
(attenuated total reflectance) spectroscopy coupled with multivariate data
analysis (MDA) has a very good potential for strain discrimination in different
species, including in <i>K. pneumoniae </i>[1]. Thus, we aimed to assess the
ability of this methodology to assist infection control in real-time, in the
context of a nosocomial outbreak of carbapenemase-producing <i>K. pneumoniae.<o:p></o:p></i></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials/Methods:
</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>We received 24 <i>K. pneumoniae</i> from infected or colonized patients of
different wards of a Portuguese hospital. Clonal relationships were established
by FTIR-ATR spectroscopy and further compared with reference methods (<i>Xba</i>I-PFGE,
MLST). Isolates were grown on Mueller-Hinton agar (37ºC, 18h) and directly
applied on the ATR crystal. Spectra comparison (each other and with those
included in an in-house MDR <i>K. pneumoniae</i> clones database) was performed
by principal component analysis (PCA), considering the
phospholipids/DNA/RNA/carbohydrates region (1500-900 cm<sup>-1</sup>).Carbapenemase
production was assessed by Blue-Carba, and confirmed by PCR and sequencing.
Genetic context of&nbsp;<i>bla</i><sub>KPC</sub>, plasmid profile and capsule
typing (<i>wzi</i>) were investigated by PCR and sequencing. Antimicrobial
susceptibility patterns were assessed by disc diffusion/agar microdilution
methods, according to the EUCAST/CLSI guidelines.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Using FTIR
spectroscopy analysis we were able in less than 48h to assess clonal
relatedness of isolates involved in the outbreak: 22 <i>K. pneumoniae</i>
isolates and the international ST147 clone clustered together, whereas the
remaining 2 strains were unrelated. These results were corroborated by PFGE
(identical PFGE-patterns) and MLST (ST147), with all outbreak isolates carrying
<i>wzi</i>64 (capsule type K14.K64). Additional characterization revealed that
ST147 isolates produced KPC-3 (linked to Tn<i>4401d</i> transposon variant and
IncFIA plasmid). The other two clones depicted by FTIR spectroscopy
corresponded to a <i>K. pneumoniae</i> ST336 carrying <i>wzi</i>150 producing
KPC-3 (linked to Tn<i>4401b</i> transposon variant) and a <i>K. pneumoniae </i>ST348
harboring <i>wzi</i>94 which is a non-carbapenemase producer (Blue-Carba
negative). All isolates exhibited MDR phenotypes and were susceptible, at
least, to amikacin and colistin.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> In this
study, a complete concordance was observed between FTIR-ATR spectroscopy
coupled with MDA and the reference typing methods (MLST, PFGE) for strain
discrimination. Our results suggest that this methodology is suitable for a
reliable, quick and low cost identification and typing of MDR <i>K. pneumoniae</i>
clones that could revolutionize the speed of bacterial typing in the context of
nosocomial outbreaks and hospital surveillance, and consequently have a strong
impact on the efficiency of infection control measures and therapeutic choices.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Reference: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Rodrigues C,
Branquinho R, Sousa C, Andrade LN, Machado E, Darini AL, Novais Â, Peixe L.
Identification of diverse capsular types in multidrug-resistant (MDR) <i>Klebsiella
pneumoniae</i> clones using <i>wzi</i> </span><span lang=DE-CH>‪</span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>sequencing
and FTIR. 6th Congress of European Microbiologists (FEMS 2015), Maastricht,
2015.</span><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO53 - New MLVA Schemes
For <i style='mso-bidi-font-style:normal'>Streptococcus pyogenes</i> And <i
style='mso-bidi-font-style:normal'>Streptococcus anginosus</i> Typing.<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Katarzyna Obszańska<sup>1</sup>; Izabella Kern-Zdanowicz<sup>1</sup>;
Izabela Sitkiewicz<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Institute of Biochemistry and Biophysics, Polish
Academy of Sciences,<sup>2</sup> National Medicines Institute<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Background</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>: Molecular typing methods – from general, to
methods of high resolution, are very important epidemiological tools. They can
be used to track spreading of global pathogenic clones, as well as clones
within outbreaks. One of the high resolution methods is PCR based detection of
variable number tandem repeats (VNTR) such as MLVA/MLVF method. Number of tandem
repeats detected for each of several loci is characteristic treat of a strain
and analysis of multiple loci can be used to determine relation between
strains. It has been successfully adopted for both Gram-positive and
Gram-negative bacteria. The analysis is easy to perform, fast and
interpretation of the results is rather straightforward.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Many streptococcal
species, including <i>Streptococcus pneumoniae</i>, <i>Streptococcus pyogenes</i>,
<i>Streptococcus agalactiae</i> and <i>Streptococcus anginosus</i> are human
pathogens which cause life threatening infections. For <i>S. pneumoniae</i> and
<i>S. agalactiae</i>, MLVA/MLVF methods were already introduced. The aim of
this work was development of <i>S. pyogenes</i> and <i>S. anginosus</i>
MLVA/MLVF typing methods.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Methods:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> Genomes of <i>S. pyogenes</i> and of <i>S. anginosus</i>,
available at NCBI and provided by our sequencing laboratory, were explored
using Tandem Repeats Finder </span><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
mso-fareast-language:PT'>(</span><span lang=DE-CH><a
href="https://www.eventgest.com/CLIENTES/veranatura/%C2%B4%C2%B4http:/tandem.bu.edu/trf/trf.html%C2%B4%C2%B4"><span
lang=EN-US style='font-size:8.0pt;mso-bidi-font-family:Arial;mso-ansi-language:
EN-US;mso-fareast-language:PT'>http://tandem.bu.edu/trf/trf.html</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>). Search generated over
250 putative loci for each species that <span style='color:black'>could be
polymorphic. Putative VNTR loci were screened <i>in silico</i> to fulfill set
criteria (longer than 15 bp and more than 2 repeated units). As a next step we
screened by PCR our collections of <i>S. pyogenes</i> and of <i>S. anginosus</i>
to experimentally detect loci exhibiting size variants. After PCR screen we
selected seven and nine primer pairs, for <i>S. pyogenes</i> and <i>S.
anginosus</i>, respectively, to compose multiplex PCR reactions. Using created
multiplexes we examined over 700 <i>S. pyogenes</i> and 90 <i>S. anginosus</i>
strains to assess the method resolution.<o:p></o:p></span></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> Each of selected primer pairs detected from two
to fourteen size variants. Theoretically, based on typing using seven and nine
loci, we are able to uncover over forty thousand and over half a million
different patterns, for <i>S. pyogenes</i> and for <i>S. anginosus</i>,
respectively. Practically, we found almost 150 patterns among <i>S. pyogenes</i>
and 70 patterns among <i>S. anginosus</i> strains. We detected distinct
patterns among quite homogenous groups of strain such as M3 <i>S. pyogenes</i>
isolates, and also different variants of M1 strains that typed as identical
using other methods. <i>S. anginosus</i> strains, that were similar as shown by
RFLP-PFGE patterns, also clustered into similar groups using MLVF analysis.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusion:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> We developed simple, little time-consuming and
interpretation-friendly molecular typing methods for two pathogenic
streptococcal species, <i>S. pyogenes</i> and <i>S. anginosus</i>. Designed
methods have discriminatory power as high as or even higher than RFLP-PFGE.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The work was
supported by grant N N401 535940 and grant N N401 536140 from National Science
Center (NCN), Internal funding (DS 5.67 and DS 5.82), NPOA-Moduł1 grant from
Ministry of Health and SPUB-MIKROBANK2 grant from Ministry of Science and
Higher Education for strain collection and maintenance.</span><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO54 - Use Of An Early
Access MinION To Determine The Context Of Resistance Genes In <i
style='mso-bidi-font-style:normal'>Escherichia coli</i> And <i
style='mso-bidi-font-style:normal'>Acinetobacter baumannii</i><o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Jane Turton<sup>1</sup>; Claire Perry<sup>1</sup>; Katie Hopkins<sup>1</sup>;
Neil Woodford<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Public Health England<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Resistance
genes are often in close proximity to insertion sequences and transposons that
are found in multiple places in the genome, making determination of their
context difficult using PCR and short-read sequencing methods. Nanopore
sequencing provides long reads (often ≥10 kb and up to 65 kb or more) giving
the potential to definitively link the elements in resistance islands or
plasmids and sometimes sequence across the whole element in a single read. Here
we used an early access minION to determine the context of&nbsp;<i>bla</i><sub>OXA<i>-</i>48</sub>
in representatives of&nbsp;<i>Escherichia coli</i> belonging to ST38 and
of&nbsp;<i>bla</i><sub>OXA-23</sub> in representatives of <i>Acinetobacter
baumanii</i> belonging to ´OXA-23 clone 1´, a sub-lineage of international clone
II, both of which are currently prevalent types carrying these genes in the
United Kingdom.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials |
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Libraries were prepared from genomic DNA as recommended by Oxford Nanopore
Technologies using a SQK-MAP005 Genomic Sequencing kit and New England Biolabs
reagents and loaded onto an early access minION. Reads were basecalled by
Metrichor<sup>TM</sup> analysis and the ´2D´ pass reads combined with available
Illumina reads in a SPAdes 3.1.1 hybrid assembly. The position of resistance
genes was determined from the contigs generated and from individual long reads.
The context of these genes in further representatives was determined using
primers designed from chromosome/resistance element junctions of sequenced
isolates.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The nanopore
sequencing runs provided 2D pass yields of up to 1108 Mb in a single 48 h run
with read lengths of up to 65 kb. Contigs generated from hybrid minION/Illumina
assemblies containing the resistance genes sought were up to 317 kb in length.
&nbsp;It was clear from the long reads that the two representatives of <i>E.
coli</i> ST38 sequenced carried IS1R-mediated chromosomally integrated
fragments of the commonly found pOXA-48a IncL/M plasmid, rather than the
plasmid itself. PCR of the chromposome/plasmid fragment junctions in further
representatives (n=43) confirmed that the insertion site and length of plasmid
fragment varied between isolates, but was consistent among many representatives
of a homogenous cluster that had originally been identified by pulsed-field gel
electrophoresis and confirmed by SNP analysis and had been referred from 25
hospital laboratories across the UK; there appeared to be a variable extent of
deletion of <i>E. coli</i> sequence on one side of the insertion site in many
of these isolates. A representative of OXA-23 clone 1 of&nbsp;<i>A. baumannii</i>
from 2005 carried an AbaR4 type island containing <i>bla</i><sub>OXA-23</sub>
in&nbsp;Tn<i><span style='background:white'>2006</span></i> inserted in&nbsp;<i>comM</i>,
but there was an additional 8 kb of sequence between two copies of a gene
coding for a hypothetical protein that had not been identified from an earlier
determination of the sequence using shorter read technology (Roche 454
sequencing) and PCR. An isolate from 2015 carried AbaR4 type islands in both <i>comM</i>
and on a plasmid.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The long
reads generated by the minION nanopore sequencing technology have been critical
in allowing us to determine the sequence and context of resistance islands and
transposons carrying resistance genes in <i>A. baumannii</i> and <i>E. coli</i>.
This work has highlighted that &nbsp;the successful ST38 lineage of <i>E. coli</i>,
which is the main type carrying <i>bla</i><sub>OXA<i>-</i>48</sub>&nbsp;in the
UK, has chromosomally integrated fragments of the pOXA-48a plasmid, and that
recent isolates of OXA-23 clone 1 of&nbsp;<i>A. baumannii</i>&nbsp;carry&nbsp;<i>bla</i><sub>OXA-23</sub>
in both chromosomal and plasmid locations.</span><span style='mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO55 - Metabolic
Profiles Of ST131 <i style='mso-bidi-font-style:normal'>Escherichia coli</i>
Subclones Depicted By Fourier-transform Infrared Spectroscopy Unveil
Variability Among The H30 Subclone<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Angela Novais<sup>1</sup>; Carla Rodrigues<sup>1</sup>; Clara Sousa<sup>2</sup>;
Ana Constança Mendes<sup>3</sup>; João A Lopes<sup>4</sup>; Teresa M Coque<sup>5</sup>;
Luísa Peixe<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>REQUIMTE@UCIBIO. Faculdade de Farmácia da Universidade
do Porto,<sup>2</sup> Center of Biological Engineering, University of Minho,<sup>3</sup>
REQUIMTE@UCIBIO.</span></i><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'> <i style='mso-bidi-font-style:normal'>Faculdade de
Farmácia da Universidade do Porto,<sup>4</sup> Serviço de Microbiologia, Centro
Hospitalar do Porto,<sup>5</sup> iMed, Departamento de Farmácia Galénica e Tecnologia
Farmacêutica, Faculdade de Farmácia, Universidade de Lisboa,<sup>6</sup>
Departamento de Microbiología, Hospital Universitario Ramón y Cajal, Instituto
Ramón y Cajal de Investigación Sanitaria (IRYCIS)<o:p></o:p></i></span></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: <i>Escherichia
coli </i>ST131 has increasingly been reported in the last decade and became the
predominant <i>E. coli</i> lineage isolated from human extraintestinal
infections at global level. Comprehensive analysis of large collections of
isolates by molecular techniques including whole-genome sequencing approaches,
revealed a polyclonal population structure, some subclones differing in
virulence and antibiotic resistance profiles as well as fimH alleles exhibiting
variable epidemiological features (e.g. H30 subclone, containing allele 30 of <i>fimH</i>).
The potential of Fourier-Transform Infrared (FTIR) spectroscopy coupled with
multivariate data analysis (MDA) to discriminate ST131 subclones is evaluated
in this study. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials and
Methods</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>: Sixty-four well characterized ST131 <i>E. coli</i> isolates from Portugal
and Spain (1996-2014) were tested. These isolates were responsible for
community or hospital-acquired infections in different clinical institutions or
were identified as colonizers of the gastrointestinal tract of healthy volunteers
or household patients and represent main <i>E. coli</i> subclones <i>fimH30</i>
(n=46), <i>fimH22</i> (n=6), <i>fimH21</i> (n=6) and<i> fimH41</i> (n=6).
Spectra acquired from individual colonies in FTIR-ATR (attenuated total
reflectance) spectroscopy were compared by partial least squares discriminant
analysis (PLSDA)using MATLAB 8.3 and the PLS toolbox 7.5 versions, considering
the phospholipids/DNA/carbohydrates region (1500-900 cm<sup>-1</sup>).
Additional characterization included identification of <i>gyrA</i>/<i>parC</i>
mutations, <i>bla</i><sub>ESBL</sub> and ExPEC virulence genes by PCR or PCR
and sequencing.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: In the
first PLSDA-based model (model 1), we were able to discriminate all strains
belonging to the <i>fimH41</i> subclone (variable <i>gyrA</i>/<i>parC</i>
mutations and <i>bla</i><sub>CTX-M-1/-14</sub> carriage) and a subset of
strains of the <i>fimH30</i> subclone (n=12, including strains causing
infection or colonizing humans which were identified in Portugal between 2007
and 2014), hereby designated as fimH30-A. Most <i>fimH22</i> isolates (n=5/6)
were subsequently discriminated from <i>fimH21</i> (variable <i>gyrA</i>/<i>parC</i>
alleles and <i>bla</i><sub>CTX-M-14/TEM-4/-24</sub> carriage) and the remaining
<i>fimH30</i> isolates (n=34, hereby designated as fimH30-B) (model 2).
Finally, <i>fimH21</i> isolates were discriminated from <i>fimH30</i>-B (model
3). Both fimH30-A and fimH30-B isolates show common mutations in GyrA (S83L,
D87N) and ParC (S80I, E84V) typical from the H30 lineage, and differ in the
content of ESBL (<i>bla</i><sub>CTX-M-15,</sub> <i>bla</i><sub>CTX-M-14 </sub>or
<i>bla</i><sub>CTX-M-27</sub>)<sub> </sub>or virulence genes. Interestingly,
metabolomics data revealed two fimH30 subclones (H30-A, H30-B) not yet depicted
by genomic-based approaches.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: We
demonstrate for the first time the high discriminatory power of FTIR
spectroscopy coupled with MDA for subclonal typing of ST131 <i>E. coli</i>,
which constitutes a reliable and cost-efficient alternative for tracking
clinically relevant bacteria in clinical or epidemiological contexts.
Interestingly, this methodology unveiled metabolically distinct <i>fimH30</i>
variants that need to be further investigated by complementary approaches (e.g.
whole genome sequencing and/or proteomics approaches).</span><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
<o:p></o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO56 - Recovering
Full-length Human Papillomavirus Genomes From Cervical Swab Samples Using
Benchtop Next Generation Sequencing.<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Joël Mossong<sup>1</sup>; Jessica Tapp<sup>1</sup>; Ardashel Latsuzbaia<sup>1</sup>;
Steven Weyers<sup>2</sup>; Marc Arbyn<sup>3</sup>; Trung Nguyen<sup>1</sup>; Marc
Fischer<sup>1</sup>; Dag Harmsen<sup>4</sup>; Catherine Ragimbeau<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Laboratoire National de Santé,<sup>2</sup> Ghent
University Hospital,<sup>3</sup> Scientific Institute of Public Health,<sup>4</sup>
University Münster<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Cervical
cancer is caused by persistent infection of oncogenic high risk human
papillomavirus (HPV) genotypes. While detection and genotyping of HPV in
cervical swab samples is mostly performed using validated commercial PCR-based
methods, benchtop next generation sequencing has the potential to be a
cost-efficient alternative means to detect genotypes not covered by commercial
assays and to recover full-length viral genomes for monitoring emergence of
potential vaccine escape variants. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The objective of
our study was to test the feasibility of next generation sequencing for detecting
179 HPV types and for recovering full-length genomes of multiple types present
in cervical swabs.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials &amp;
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>The study sample consisted of 6 cervical swab samples concordantly positive
for high-risk papillomavirus by 2 genotyping assays (EuroImmune EuroArray,
Seegene Anyplex II HPV28) following DNA extraction. By the Anyplex and
EuroArray assays, 3 samples were concordantly positive for single infection by
type 16 and 3 samples were concordantly positive for multiple HPV infection
(types 18-58, 16-18-53-58-66, 16-40-42-43-56, respectively).<b> </b><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Following rolling
circle amplification which amplifies circular viral DNA against background
linear human DNA, samples were sequenced using the Nextera XT library
preparation kit on a Illumina Miseq benchtop sequencer. Raw reads were mapped
using the BWA-SW algorithm to 179 known full length HPV genomes downloaded from
the PAVE database and were <i>de novo</i> assembled using VELVET. All
bioinformatic analysis was performed using SeqSphere+ software (Ridom, Münster,
Germany) on a standard Microsoft Windows 7 desktop PC (16GB memory, i5 Core).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Five of the
6 samples yielded reads mapping to HPV reference genomes. In these 5 samples,
the median proportion of reads mapping to HPV reference genome collection was
5% (range 2.1%-23.7%). NGS detected all the genotypes that were concordantly
detected by the 2 commercial assays with a median coverage of 151x (range
2x-2178x). NGS detected additional genotypes 34, 34-90 and 90-91 in the 3
samples with multiple infections. <i>De novo</i> assembly recovered 9
full-length HPV genomes of genotypes 16, 18, 34, 53, 58, respectively, from a
total of 20 HPV genotypes detected (full length genome recovery rate: 45%).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Benchtop
next generation sequencing provides a feasible alternative for detecting HPV
genotypes that are not detectable by current commercial assays, while
simultaneously enabling the recovery of full length HPV genomes. Further wet
and dry lab work is required to enhance recovery of viral metagenomic DNA and
optimise bioinformatics tools. The significance of full length genome
information in relation to cervical disease and vaccine impact remains to be investigated.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO58 - INNUENDO: A
Cross-sectorial Platform For The Integration Of Genomics In Surveillance Of
Food-borne Pathogens<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Mirko Rossi<sup>1</sup>; João Carriço<sup>2</sup>; Javier Garaizar<sup>3</sup>;
Friederike Hilbert<sup>4</sup>; Saara Salmenlinna<sup>5</sup>; Marjaana
Hakkinen<sup>6</sup>; Monica Oleastro<sup>7</sup>; Mihkel Mäesaar<sup>8</sup>;
Aivars Bërziņš<sup>9</sup>; INNUENDO consortium10<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University of Helsinki,<sup>2</sup> Univeristy of
Lisbon,<sup>3</sup> Universidad del Pais Vasco/Euskal Herriko Unibertsitatea,<sup>4</sup>
University of Veterinary Medicine, Vienna,<sup>5</sup> National Institute for
Health and Welfare,<sup>6</sup> Finnish Food Safety Authority,<sup>7</sup>
Instituto Nacional de Saúde Dr. Ricardo Jorge,<sup>8</sup> Veterinary and Food
Laboratory,<sup>9</sup> Pārtikas drošības, dzīvnieku veselības un vides
zinātniskais institūts,10_ na<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Multinational
outbreaks of food-borne pathogens cause considerable threats to European public
health. An increased level of cooperation between the local, national and
European competent authorities by dedicated training, development of a common
pathogen database and validation of new approaches in microbial characterization
is warranted. Implementing whole genome sequencing (WGS) in routine
surveillance and outbreak investigations is becoming a strategic goal for many
public health authorities all over the world. However, small countries with
limited resources might not be able to succeed in reaching this goal in the
near future, putting several EU member states in a condition of inferior
capabilities for outbreak detection and investigation. Therefore, to guarantee
the reinforcement of European capacities to ensure protection of citizens
against cross border health threats, EU must enable wider access to the new
methodologies.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The potential of
widespread, routine use of WGS analysis for public health protection is
essentially restricted by the absence of accessible IT framework, and the
limited skills of public health microbiologists in handling these novel
methodologies. With this in mind we developed an initiative aiming to deliver a
cross-sectorial framework of bacterial WGS integration in routine surveillance
and epidemiological investigations. The aim of this project is aligned with the
EFSA mission to promote the development and validation of new approaches in
microbial characterization throughout coordinate efforts between all public
health and food safety stakeholders. To design an affordable and sustainable
diagnostic infrastructure, our consortium includes governmental organisations,
authorities and research institutes from the food, veterinary and human
sectors, from small countries such as Finland, Estonia, Latvia, Portugal,
Basque Autonomous Community in Spain and Austria. Allowing project stakeholder
to assess design choices early on in the development cycle, this multinational
collaboration in the ‘One-health’ context will ensure that the planned
infrastructure will address the requirement for the integration of WGS in
routine analysis in the food chain.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Standardization
and calibration of process and simplification of data analysis and
interpretation are the two basic conditions allowing the transition from the current
diagnostic paradigm to a full WGS consolidation. Moreover, the lack of
accessible informatics infrastructure for data processing and integration is
still the major obstacle to implementation of WGS. Thus, it is now essential
that advances in bioinformatics and bacterial genomics encounter the needs of
the public health microbiology. This goal is reachable only throughout an
integration of competencies across different disciplines and professions.
Therefore, throughout active cooperation between experts in bacterial genomics,
evolution, bioinformatics, epidemiology and specialists in validation, food
control and public health, our consortium will use a cross-sectorial approach
in developing a common framework for maximizing the benefit to use WGS in food
safety.</span><span style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO59 - Comparison Of
Typing Methods For Hospital-associated <i style='mso-bidi-font-style:normal'>Acinetobacter
baumanni</i>i Clones Isolated In Greece And India<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Basil Britto Xavier<sup>1</sup>; Jasmine Coppens<sup>1</sup>; Lavanya
Vanjari<sup>2</sup>; Konstantina Dafopoulou<sup>3</sup>; Julia Sabirova<sup>1</sup>;
Christine Lammens<sup>1</sup>; Olympia Zarkotou<sup>4</sup>; Trevor Wagner<sup>5</sup>;
Athanassios Tsakris<sup>3</sup>; Spyros Pournaras<sup>6</sup>; Herman Goossens<sup>1</sup>;
Surbhi Malhotra-Kumar<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University of Antwerp,<sup>2</sup> National Institute
of Tuberculosis,,<sup>3</sup> Department of Microbiology, University of
Athens,,<sup>4</sup> Department of Microbiology, Tzaneio General Hospital,,<sup>5</sup>
OpGen, Inc.,,<sup>6</sup> Department of Microbiology, Medical School, National
and Kapodistrian University of Athens<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Introduction: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Hospital
outbreaks caused by multidrug-resistant&nbsp;<i>Acinetobacter baumannii</i>&nbsp;epidemic
global clones such as GC1 and GC2 are a major problem. We utilized three
different &nbsp;typing methods pulsed-field gel electrophoresis (PFGE),
multilocus sequence typing (MLST), and whole genome mapping (WGM) to understand
the genetic diversity of hospital-associated<i> A. baumannii</i>&nbsp; isolated
from two geographically distinct regions (Greece and India) over seven years.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>
Twenty-three <i>A. baumanni</i> were isolated from patients admitted to ICU
(n=19), surgery (n=2) and medical (n=2) wards in Greece (Tzaneio Hospital n=13,
General hospital Serres n=1, and University hospital Larissa n=2), and in India
(Nizam’s Institute of Medical Sciences n=7) during 2008-2014. These were
screened for resistance to 21 antibiotics (</span><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>β</span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>-lactams with and
without </span><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT'>β</span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>-lactamase inhibitors and carbapenems), by disk diffusion and results were
interpreted according to CLSI cut-offs (Table 1). All strains were typed by
MLST (PubMLST and Pasteur), PFGE and WGM. Restriction analysis for PFGE was
performed using <i>ApaI</i> using a 7-band and a 3-band difference cut-off to
define types and subtypes, respectively.Strains were subjected to WGM (Argus®
system, Opgen Inc, Gaithersburg, USA) using <i>NcoI</i>. Maps were edited and
compared using Bionumerics v7.1 (Applied Maths, Belgium).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Result: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Antibiotic
susceptibility test showed that multi-drug resistance (resistance to </span><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>β</span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>-lactams
with and without </span><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT'>β</span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>-lactamase inhibitors)was detected among all 23 <i>A. baumannii</i>
strains. Of the 16 Greek strains, <i>bla</i><sub>OXA-23 </sub>was detected in 7
and <i>bla</i><sub>OXA-58 </sub>in 9 strains. <i>Bla</i><sub>NDM-1</sub> was
detected in 1 Indian strain, Nab5. MLST (Pasteur) assigned 19 of the 23 strains
to ST2 (GC2) and the other 4 isolates to 4 different STs (645, 85, 646 and 82).
On the other hand, PubMLST assigned the 23 strains to 13 STs. </span><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>ST2 (Pasteur)
strains were assigned to 10 different PubMLST types. </span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>PFGE results
showed four different types. Based on a ≥ 7-band difference, all Greek and 4
Indian strains (Nab7, Nab9, Nab12 and Nab16) belonged to 1 PFGE type and were
divided into 9 subtypes (1a-1i, ≥ 3-band difference). The remaining three Indian
strains (Nab5, Nab15 and Nab17) were singletons. Based on a 95% similarity
co-efficient, WGMs of the 16 Greek strains were divided into 3 clusters and 5
singletons. Overall similarity between the Greek strains was 71.3% (Figure 1a).
On the other hand, WGMs of Indian strains isolated over a 5-month period during
2013-14 and from a single hospital were highly diverse. These 6 strains from
one cluster and 4 singletons with an overall similarity of 36.2%, (Figure 1b).
STs assigned by Pasteur MLST correlated well with PFGE. Application of the
adjusted Wallace index showed that WGM clusters were strongly predictive of
both MLSTs and PFGE types (Wallace co-efficient of 1.00 for both) (Figure 2).
Simpson’s diversity index showed that discriminatory power of WGM (1.000, 95%
CI: 0.976-0.99) was higher than that of PFGE (0.893, 95% CI: 0.801-0.986) and
PubMLST (0.913, 95% CI: 0.857-0.970)<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: Comparison
of the high resolution typing technique WGM to other typing methods (MLST and
PFGE) showed its superior discriminatory power for analysis of rapidly evolving
hospital-associated <i>A. baumannii</i> global clones. The high diversity among
the Indian strains studied here was noteworthy.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='mso-ansi-language:
EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape id="Imagem_x0020_44"
 o:spid="_x0000_i1029" type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/87imagem_2560108707_0.jpg"
 style='width:333pt;height:209pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S5_files/image003.png" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/87imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=211
src="public/abstracts/Webpages/PO_S5_files/image004.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/87imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_44"><![endif]></span><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='mso-ansi-language:
EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape id="Imagem_x0020_45"
 o:spid="_x0000_i1028" type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/87imagem_2560108707_1.jpg"
 style='width:333pt;height:206pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S5_files/image005.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/87imagem_2560108707_1.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=208
src="public/abstracts/Webpages/PO_S5_files/image006.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/87imagem_2560108707_1.jpg"
v:shapes="Imagem_x0020_45"><![endif]></span><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO61 - Development And
Evaluation Of Double Locus Sequence Typing For Molecular Epidemiological
Investigations Of <i style='mso-bidi-font-style:normal'>Clostridium difficile</i><o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Miloš Stojanov<sup>1</sup>; Bàrbara Magalhaes<sup>2</sup>; Valery Terletsky<sup>1</sup>;
Patrick Basset<sup>1</sup>; Guy Prod´hom<sup>1</sup>; Gilbert Greub<sup>1</sup>;
Laurence Senn<sup>1</sup>; Dominique Blanc<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Lausanne University Hospital,<sup>2</sup> Valery
Terletsky<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Despite the
development of novel typing methods based on whole genome sequencing, most
laboratories still rely on classical molecular methods for outbreak
investigation or surveillance. Reference methods for <i>Clostridium difficile </i>include
ribotyping and pulse-field gel electrophoresis, which are band-comparing
methods often difficult to establish and which require reference strain
collections. Here we present the double locus sequence typing (DLST) scheme as
a tool to analyse <i>C. difficile</i> isolates. Using a collection of clinical <i>C.
difficile</i> isolates recovered during a one-year period, we evaluated the
performance of DLST and compared the results to multilocus sequence typing
(MLST), a sequence-based method that has been used to study the structure of
bacterial populations and highlight major clones. DLST had a higher
discriminatory power compared to MLST (Simpson´s index of diversity of 0.979
versus 0.965) and successfully identified all isolates of the study (100%
typeability). Previous studies showed that discriminatory power of ribotyping
was comparable to that of MLST, thus DLST might be more discriminatory than
ribotyping. DLST is easy to establish and provides several advantages,
including absence of DNA extraction (PCR is performed on colonies), no specific
instrumentation, low cost and unambiguous definition of types. Moreover,
implementation of DLST typing scheme on an Internet database, such previously
done </span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>for
<i>Staphylococcus aureus</i> and <i>Pseudomonas aeruginosa</i> (</span><span
lang=DE-CH><a href="http://www.dlst.org"><span lang=EN-US style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-fareast-language:PT'>http://www.dlst.org</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>), will
allow users to easily obtain the DLST type by submitting directly sequencing
files and will avoid problems associated with multiple databases.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO62 - Mass
Spectrometry Proteotyping For Detection, Identification Characterization And
Diagnostics Of Infectious Bacteria In Clinical Respiratory Samples<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Lucia Gonzales Siles<sup>1</sup>; Roger Karlsson<sup>1</sup>; Chantal van
Houten<sup>2</sup>; Louis Bont<sup>2</sup>; Fredrik Boulund<sup>3</sup>; Erik
Kristiansson<sup>3</sup>; Edward Moore<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University of Gothenburg,<sup>2</sup> University
Medical Center Utrecht,<sup>3</sup> Chalmers University<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Lower
respiratory tract infection (LRTI) is the leading cause of childhood deaths in
most developing countries and the world, and are the most common causes of
hospital and out-patient visits within the EU, comprising 1 of 3 admissions
annually. In general, the over-prescription and use of broad-spectrum
antibiotics are common practices that lead to the evolution and development of
resistance in infectious bacteria and will lead to loss of time and resources
in patient handling and adverse patient outcomes. It is estimated that 25 000
people die every year in Europe from antibiotic-resistant bacteria.
Conventional approaches have depended upon cultivation of bacteria with
subsequent testing for antibiotic sensitivity. Therefore, reliable and
time-effective microbiological diagnostics are essential for more effective
treatment of respiratory infections. In this project, we apply state-of-the-art
proteomics techniques for identifications of pathogens and antibiotic
resistance from clinical samples, without prior cultivation.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Material and
methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Nasal and nasopharyngeal swabs samples were collected, in commercial Amies
medium supplemented with 5x STGG, as a part of the EU-TAILORED-Treatment
project </span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>(</span><span
lang=DE-CH><a href="http://www.tailored-treatment.eu/"><span lang=EN-US
style='font-size:8.0pt;mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
mso-fareast-language:PT'>www.tailored-treatment.eu/</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>). Samples
were stored at -80°C until analyses. Different protocols for removal of human
cells and mucus were tested, including non-ionic detergents, <i>i.e.</i>,
Igepal, Saponin, Urea-Chaps, as well as cytolysis. Samples were concentrated
and analyzed by ‘proteotyping’, using a Lipid-based Protein Immobilization (LPI<sup>TM</sup>)
technology (WO2006068619), in which intact bacterial cells or cell fractions
are bound to a surface. Peptides were generated, using enzymatic digestion, and
then separated and analyzed, using liquid chromatography-tandem mass
spectrometry (LC-MS/MS). The mass spectra profiles were compared to a database
of reference peptide sequences, consisting of all complete genomes of the NCBI
Reference Sequence (RefSeq) Database. Results were confirmed by standard
microbiology, including cultivation of bacteria in selective media, MALDI-TOF
MS analyses and qPCR.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>
Proteotyping applied to clinical samples demonstrated that the number of viable
bacteria and detected proteins determined were ten-times higher when nasal and
nasopharyngeal swabs were stored in Amies media supplemented with STGG 5X media
compared to Amies media without STGG, after 1 and 2 months of storage at -70C.
Among the different protocols tested to remove human biomaterial, all
treatments proved effective to varying degrees, although the Igepal treatment
was able to retain the highest number of discriminatory peptides. Using
proteotyping, we were able to identify the pathogenic bacteria directly within
clinical samples (nasopharyngeal and nasal swabs) that had been identified to
be positive for respiratory infectious bacteria by standard methodologies at
clinical bacteriology laboratories at Sahlgrenska University Hospital (Sweden)
or at the University Medical Centre Utrecht (Netherlands).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Proteotyping
of infectious bacteria, using tandem LC-MS/MS enabled the differentiation and
identification of infectious bacteria in clinical samples from LRTIs. It has
high levels of resolution and highly reproducible detection of protein
biomarkers. Proteotyping identified biomarkers for species- and
sub-species-level strain discrimination and antibiotic resistance, all from
single MS analyses.</span><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO63 - Comparison Of
MLST, rMLST And wgMLST As Global Typing Schemes For <i style='mso-bidi-font-style:
normal'>S. Enterica</i><o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Madison Pearce<sup>1</sup>; Martin Maiden<sup>2</sup>; Kathie Grant<sup>3</sup>;
James Bray<sup>2</sup>; Keith Jolley<sup>2</sup>; Claire Jenkins<sup>3</sup>;
Tim Dallman<sup>3</sup>; Philip Ashton<sup>3</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University of Oxford/Public Health England/ NIHR HPRU,<sup>2</sup>
University of Oxford,<sup>3</sup> Public Health England<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><i><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Salmonella
enterica </span></i><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>(<i>S. enterica</i>) is a zoonotic, foodborne pathogen responsible for
millions of cases of acute gastrointestinal disease worldwide, each year.
MultiLocus Sequence Typing (MLST) compares sequence similarity across 7
housekeeping genes; ribosomal MLST (rMLST) compares sequence similarity across
a maximum of 53 ribosomal genes, whilst whole genome MLST (wgMLST) compares a
dataset against an entire reference genome. These varying levels of gene
coverage were compared and the utility of each approach, as a global typing
scheme for <i>S. enterica,</i> was assessed.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials |
Methods</span></b><b style='mso-bidi-font-weight:normal'><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Pairwise
comparisons were used to analyse various sets of isolates, from pubMLST’s
BIGSdb, at all three levels of genetic discrimination and to generate trees and
nets using the built-in Genome Comparator tool. These datasets were chosen for
depth and variety: an example of every serovar of <i>S. enterica</i> found
within BIGSdb was used, to ensure that serovars formed distinct genetic
lineages; Enteritidis, Gallinarum and Pullorum isolates were used to analyse
closely related serovars; Newport isolates were used to examine antigenically
identical isolates; and isolates from the phylogentic grouping lineage 3 were
used to analyse a genetically complex group.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Comparisons showed
that in general MLST, rMLST and wgMLST all distinguished serovars and
previously identified phylogenetic groupings with high congruence. The wgMLST
analyses took 24 – 96 hours to perform (compared to 3 – 20 minutes for rMLST)
but showed the highest level of resolution. The neighbornets illustrated that
possible genetic relationships decreased as number of genes increased. Analyses
of serovars within subspecies one, indicated a relatively high level of
recombination and horizontal gene transfer.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> rMLST for <i>Salmonella</i>
<i>enterica</i> provides greater discrimination and resolution between lineages
than MLST, while taking significantly less time to perform than wgMLST.
Furthermore, these results show high congruence between the three
methodologies, suggesting high compatibility and scalability for future
analyses.</span><span style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO64 - On The Road Of
Transition – From PFGE To NGS Based Strain Typing<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Elke Brockmann<sup>1</sup>; Jacob Bælum<sup>1</sup>; Mads Bennedsen<sup>1</sup>;
Annette Møller<sup>1</sup>; Birgitte Stuer-Lauridsen<sup>1</sup>; Martin
Abel-Kilstrup<sup>1</sup>; Anette Wind<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Chr. Hansen A/S<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Background:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> Restriction fragment profiling with rare
cutting restriction enzymes, commonly called PFGE after the fragment separation
methodology Pulsed Field Gel Electrophoresis, has for many years been
considered the “gold standard” of strain typing. Multi Locus Sequence Typing
(MLST) was in 1998 suggested as an alternative strain typing method based on
partial sequences of around seven house keeping genes. In times of Sanger
sequencing MLST has been a laborious and expensive method, not offering obvious
timely or monetary advantages compared to PFGE. This led to the situation that
laboratories, which had built a substantial database of PFGE-fingerprints,
would not find it easy to switch to sequence typing. In general it is observed
that PFGE has a higher discrimination power, whereas MLST is much less affected
by experimental variation and allows for studying the evolutionary relatedness
of isolates within a species or a species group. In addition MLST is fully
compatible with next generation sequencing (NGS) of whole genomes in the way
that the data required for determining the sequence type can be readily
extracted from a NG whole genome (WG) sequence. In fact the cost of a NG WG
sequence is already below that of traditional MLST. For a laboratory planning
to switch from PFGE to sequence based typing the interesting questions, which
is addressed here, is, if published MLST schemes provide a resolution power
comparable to PFGE or to which extent it will be necessary to increase the
number of genes used. MLST can also be seen as a step on the way to whole
genome strain typing, where the sequence type can be based on the core genome
alleles or the SNP (single nucleotide polymorphism) profile. The latter two
require however still a lot of manual reevaluation and are at the moment not
readily available for fast automated strain typing.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Materials &amp; Methods: </span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Illumina NGS
sequencing was performed at BGI (Shenzen, China). MLST typing was accomplished
with the help of CLC (Århus, Denmark). PFGE typing was performed with enzyme SmaI
for both <i>Lactococcus lactis</i> and <i>Streptococcus thermophilus</i>. The
comparison of PFGE and sequence typing was done in BioNumerics (Applied Math).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> As previously found, our results confirm the
lower resolution power of published MLST schemes for <i>Lactococcus lactis</i>
and <i>Streptococcus thermophilus</i> compared to PFGE typing. The combination
of two schemes however increases the resolution power to almost PFGE level.
Congruence analysis between the two typing approaches shows that they are not
fully congruent. Strain differentiation can therefore be further enhanced by
combining the two typing methods.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusion:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> MLST of NG whole genome sequences offers fast,
robust and easily interpretable strain typing, which can be adjusted to the
needed level of differentiation by adjusting the number of genes used.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO65 - Whole Genome
Sequencing Of Fecal Samples As A Tool For The Diagnosis And Genetic
Characterization Of Norovirus<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Herjan Bavelaar<sup>1</sup>; Janette Rahamat-Langendoen1, Hubert G.M.
Niesters2, Jan Zoll1, Willem J.G. Melchers<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Department of Medical Microbiology, Radboud University
Medical Center, Nijmegen, The Netherlands,<sup>2</sup> Department of Medical
Microbiology, Division of Clinical Virology, University Medical Center
Groningen, Groningen, The Netherlands<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Background: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Norovirus is one of the most frequent causes of
gastroenteritis, causing yearly epidemics and hospital outbreaks resulting in a
high burden on health care. Advances in molecular diagnostic techniques allow
rapid identification of infected patients. Besides, characterization of
norovirus by sequence analysis provides useful information for understanding
transmission during hospital outbreaks. Next generation sequencing techniques
(NGS) are increasingly being used in outbreak management and virus discovery.
For norovirus, the use of NGS focused on the most prevalent genotypes, most
notably genotype GII.4. A prerequisite for incorporating NGS into routine
diagnostics is the ability of these techniques to also detect and characterize
novel (recombinant) norovirus genotypes. We describe the detection of different
genotypes of norovirus directly from clinical samples using multiplexed whole
genome sequencing.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Materials and methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Whole genome sequencing was performed on fecal
samples from 10 patients with gastro-enteritis caused by different genotypes of
norovirus as confirmed by RT-PCR and Sanger sequencing. The cDNA fractions
obtained after rRNA-depletion and reverse transcription were used for
whole-transcriptome amplification for the detection of RNA viruses, using the
Qiagen Repli-G WTA single cell kit. Approximately 100 ng DNA per sample was
used for sequencing on an IonTorrent PGM system with an Ion 318 sequencing
chip. Assembly and determination of sequence coverage was done with Bowtie2
using the reference sequence as template. A phylogenetic tree was constructed
used a K-mer based Neighbour Joining method, using K-mer length of 15 and a
Jukes-Cantor nucleotide distance matrix.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Sufficient amounts of RNA were retrieved from
all clinical samples to perform whole-transcriptome sequencing for the
detection of RNA-viruses. Complete genomic norovirus sequences were obtained
from all clinical samples, permitting accurate genotyping by phylogenetic
analysis. In addition, a complete coxsackie B1 virus genome was retrieved.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusion: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Detailed information on viral content can be
obtained from fecal samples in a single-step approach, supporting clinical and
epidemiological purposes. Next-generation sequencing performed directly on
clinical samples can become a powerful tool in patient care and infection
control.<o:p></o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO66 - Plasmablasts
After Antigen Challenge - A Diagnostic Tool For Assessing Response To Influenza
Vaccination In Patients With Chronic Lymphocytic Leukemia<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Ewelina Grywalska<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Medical University of Lublin<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Introduction:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Chronic
lymphocytic leukemia (CLL) leads to significant immune system dysfunction. The
predominant clinical presentation in 50% of patients involves recurrent, often
severe, infections. Infections are also the most common (60-80%) cause of
deaths in CLL patients. The scope of infections varies with the clinical stage
of the disease. Treatment-naïve patients typically present with respiratory
tract infections caused by influenza. Immune system disturbances in CLL are still
not well defined. Clinical data indicates that despite normal serum
immunoglobulin (Ig) level, treatment-naïve patients may not respond to
influenza vaccination. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Aim of the study:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> The aim of
the study was to investigate changes in B-cell subpopulations in CLL patients,
including plasmablasts, in peripheral blood by flow cytometry after influenza
vaccination and to evaluate if plasmablasts may serve as a diagnostic tool for
assessing response to vaccination.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Material and
methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Forty treatment-naïve CLL patients and twenty healthy volunteers were
immunized with influenza vaccine. Specific antibody levels and frequencies of
plasmablasts were measured before vaccination and on day 30 by ELISA assay, and
day 7 by flow cytometry after vaccination, respectively. Both groups were also
evaluated for the levels of IgG and IgG subclasses, and the frequencies of
selected peripheral blood lymphocyte subpopulations before and 30 days after
immunization.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Of the forty
CLL patients studied, 100% lacked detectable changes in the serum level of
specific anti-influenzae IgG antibodies before and after vaccination (mean:
122.41±41.94 mU/ml vs. mean: 128.37±52.13 mU/ml, respectively; p=0.24). In none
of patients an increase of the percentages and absolute counts of plasmablasts
was noted. In the control group, an increase in circulating plasmablasts on day
7 post immunization corresponded with the appearance of specific antibody
levels on day 30 post immunization (r=0.823, p=0.000001) and was statistically
significantly higher than before a dose of influenza vaccine (before
vaccination: 20.12±14.93%, 0.46±0.36x10^3/mm^3; after vaccination:
46.81±26.87%, 1.15±0.77x10^3/mm^3; p=0.01). In contrast, CLL patients failed to
increase plasmablasts significantly in peripheral blood after antigen
challenge.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Our
findings indicate that treatment-naïve CLL patients have a block in terminal
B-cell differentiation and that flow cytometry-based assessment of plasmablasts
in peripheral blood after vaccination serves as a surrogate diagnostic marker
for assessing in vivo antibody response in patients with CLL.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<i style='mso-bidi-font-style:normal'><span style='font-size:10.0pt;mso-bidi-font-size:
11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:Cambria;
mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></i>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><i style='mso-bidi-font-style:normal'><span style='mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></i></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO67 - Discrimination
Of Vancomycin-resistant <i style='mso-bidi-font-style:normal'>Enterococcus
faecium</i> Clones By MLST (eBURST/BAPS) And Spectroscopic Techniques (MALDI-TOF
MS And FTIR-ATR)<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Ana R. Freitas<sup>1</sup>; Clara Sousa<sup>2</sup>; Jan Bavlovič<sup>3</sup>;
Liliana Silva<sup>4</sup>; Carla Novais<sup>1</sup>; Teresa M. Coque<sup>5</sup>;
João Lopes<sup>6</sup>; Luísa Peixe<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>UCIBIO/REQUIMTE. Departamento de Ciências Biológicas.
Laboratório de Microbiologia. Faculdade de Farmácia. Universidade do Porto.,<sup>2</sup>
CEB-Centro de Engenharia Biológica, Universidade do Minho.,<sup>3</sup> Faculty
of Pharmacy in Hradec Králové, Charles University, Prague and Faculty of
Military Health Sciences, University of Defence, Brno, Czech Republic,<sup>4</sup>
UCIBIO/REQUIMTE. Departamento de Ciências Biológicas. Laboratório de
Microbiologia. Faculdade de Farmácia. Universidade do Porto,<sup>5</sup> ESALD,
Instituto Politécnico de Castelo Branco,<sup>6</sup> Servicio de Microbiologia.
Hospital Ramón y Cajal. Instituto Ramón y Cajal de Investigación Sanitaria
(IRYCIS),<sup>7</sup> iMed, Departamento de Farmácia Galénica e Tecnologia
Farmacêutica, Faculdade de Farmácia, Universidade de Lisboa<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> The highly
recombinogenic population structure of <i>Enterococcus faecium</i> (Efm) has
been recently updated. Former clonal complexes (CC) delineated by eBURST were
split into smaller subgroups by the Bayesian Analysis of Population Structure
(BAPS) software, better reflecting their evolutionary history. Still, the
application of low-cost high throughput techniques is needed to identify
high-risk clones in a fast routine way enabling timely and appropriate
infection control measures. Our aim was to exploit the potential of
Matrix-assisted laser desorption/ionization time-of-flight mass spectrometry
(MALDI-TOF MS) and Fourier transform infrared spectroscopy with attenuated
total reflectance (FTIR-ATR) techniques in the discrimination of
vancomycin-resistant <i>Enterococcus faecium</i> (VREfm) clones associated with
human infections.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials and
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Ninety VREfm isolates obtained from hospitals, healthy humans and swine in
16 countries (1995-2012) were analyzed. They corresponded to 9 sequence types
(STs) clustering into 5 BAPS subgroups: BAPS 3.3a1 (18 ST18, 12 ST132), 3.3a2
(10 ST16, 9 ST17) and 2.1a (4 ST78, 12 ST117) greatly linked to hospital
outbreaks (former CC17); and BAPS 2.1b (6 ST5, 5 ST185) and 3.2 (14 ST6) mainly
linked to healthy pigs/humans and rarely to human infections (former CC5).
Bacterial cells grown in Mueller-Hinton agar (18h/37ºC) were used for FTIR-ATR
(4000-400cm<sup>-1</sup>, resolution of 4cm<sup>-1</sup>/32 spectra
co-additions) and to prepare bacterial extracts (according Bruker
recommendations) for MALDI-TOF MS (linear positive mode from 2000-20000m/z and
alpha-cyano-4-hydroxycinnamic acid matrix) experiments. Infrared (IR) spectra
were processed with standard normal variate, Savitzky-Golay filter (9,2,2)
prior to analysis. Both sets of spectra after application of mean centring were
modelled by partial least square discriminant analysis (PLSDA) with Matlab
version 8.3 (MathWorks, Natick, MA, USA) and PLS Toolbox version 7.5
(Eigenvector Research Inc., Wenatchee, WA, USA).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Three main
clusters were observed in the PLSDA model obtained with the mass spectra (first
two latent variables explaining 63.4% of spectral variability): i) BAPS 3.3a1;
ii) BAPS 3.3a2 and 2.1a (ST17/ST78 lineages); and iii) BAPS 2.1b and 3.2. BAPS
3.3a2 and 2.1a as well as “CC5” BAPS subgroups include single locus variants
(SLVs) as ST17 and ST78, and ST5 and ST6, respectively, andwere not separated
by MALDI-TOF MS. The PLSDA model (first two latent variables, 75.0% of the
spectral variability) obtained with IR spectra (1500–900 cm<sup>-1 </sup>enclosing
polysaccharide molecules vibrations) exhibited 5 distinct clusters
corresponding to the 5 BAPS subgroups. Most STs included a high diversity of
isolates as defined by PFGE for ST18 (16 PFGE-types), ST132 (9 PFGE-types),
ST16 (7 PFGE-types), ST17(9 PFGE-types), ST78 (3 PFGE-types) and ST117 (4
PFGE-types). ST5/ST6/ST185 corresponded to one PFGE-type.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>MALDI-TOF MS
seems to be insufficient to discriminate the main Efm clones. However, FTIR-ATR
perfectly corroborates the clustering of main Efm clones as obtained using
BAPS.The Efm strains analyzed seem more divergent in their carbohydrates
content (evaluated by FTIR-ATR) than in the ribosomal peptides and/or proteins
(evaluated by MALDI-TOF MS). The fact that SLVs from different BAPS were
discriminated by FTIR-ATR highlights the important contribution of carbohydrates
for niche adaption.</span><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO68 - Development Of A
Conventional Multiplex PCR For Identification Of <i style='mso-bidi-font-style:
normal'>Escherichia coli</i>, <i style='mso-bidi-font-style:normal'>Escherichia
fergusonii</i>, And <i style='mso-bidi-font-style:normal'>Escherichia albertii</i><o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Garcia Toledo, Lisley<sup>1</sup>; Gladney, L.M<sup>2</sup>; Fasulo, D.<sup>3</sup>;
Strockbine, N<sup>4</sup>; Carleton, H<sup>4</sup>; Lindsey, R. L.<sup>4</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Oak Ridge Institute for Science and Education,<sup>2</sup>Centers
for Disease Control and Prevention,<sup>3</sup> IHRC, Inc,<sup>4</sup> Centers
for Disease Control and Prevention,<sup>5</sup> Pattern Genomics, LLC,<sup>6</sup>
Enteric Diseases Laboratory Branch, Centers for Disease Control and Prevention<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Escherichia
coli</span></i><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>, <i>Escherichia albertii</i>, and<i> Escherichia fergusonii</i> are
closely related organisms with the potential to cause sickness in humans, such
as urinary tract infections and diarrheal illnesses. Identification of these
species is dependent on phenotypic testing, sequencing, or single PCR methods
that target only one species. To facilitate the rapid identification of these
potential pathogens, we developed a conventional multiplex PCR assay targeting
conserved, species-specific genes.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials |
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Using the Daydreamer<sup>TM</sup> (Pattern Genomics, USA) software platform,
we concurrently analyzed the whole genome sequence for 342 genomes (305 <i>E.coli,
</i>22 <i>E. albertii</i>, 12 <i>E. fergusonii</i> and 3 other <i>Enterobacteriaceae</i>)that
had been generated in-house or downloaded from NCBI. First, we identified
genomic regions that are conserved in, and unique to, each species. We removed
sequences that were not specific due to any blast hit to non-target organisms
on NCBI and then designed PCR primers for the remaining sequences. Next, the
target sequences were annotated using BLAST. For all three species, we selected
genes highly conserved at the nucleotide level and primers that had 100%
accuracy and 100% coverage to our training dataset as reported by the
Daydreamer<sup>TM</sup> <i>in silico </i>PCR tool. We screened the primers in a
conventional multiplex PCR assay format with a set of 10 reference strains and
selected those that yielded the most efficient amplification. Using a
collection of 30 additional <i>Escherichia </i>strains, we assessed the ability
of the selected primers to amplify their intended target sequences.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>We
identified eight genes that appeared to be conserved in, and unique to, the
species of interest: a putative transporter gene and intracellular growth
attenuator gene (<i>igaA</i>) for <i>E. coli</i>; three hypothetical protein
genes for <i>E. albertii</i>; and a hypothetical protein and two
transcriptional regulator genes for <i>E. fergusonii</i>. Findings from our
screening assay showed there were minimal differences in the performance of the
primers identified. We therefore selected primers that had similar annealing
conditions and targeting the intracellular growth attenuator protein (<i>igaA</i>)
for <i>E. coli, </i>a hypothetical protein for <i>E. albertii</i>, and a
transcriptional regulator for <i>E. fergusonii</i>. The sensitivity and
specificity of the conventional multiplex PCR assay was 100% and 100%,
respectively.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>We report
the development of a multiplex PCR assay that can quickly and accurately
distinguish between <i>E. coli</i>, <i>E. albertii, </i>and <i>E. fergusonii</i>
in a single PCR reaction. This approach should help reference and diagnostic
laboratories identify these closely related and easily confused species, thus
improving laboratory surveillance.<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO69 - A Comparison Of
Diversilab rep-PCR And cgMLST For Molecular Typing Of Carbapenem-resistant <i
style='mso-bidi-font-style:normal'>Acinetobacter baumannii</i> Isolates From
Greece And Spain<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Paul G Higgins<sup>1</sup>; Stefanie Gerson<sup>1</sup>; Jennifer Nowak<sup>1</sup>;
Esther Zander<sup>1</sup>; Harald Seifert<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University of Cologne<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The
incidence of carbapenem-resistant <i>Acinetobacter baumannii</i> (CRAB) is
increasing, especially in Mediterranean countries. Multiple studies have shown
that the most common carbapenem-resistance mechanism is OXA-23 and the most
prevalent clonal lineage is International Clone 2 (IC2). In this study, we have
characterised CRAB isolated from multiple cities in Greece and Spain and
compared clustering by rep-PCR (DiversiLab) to an<i> ad hoc</i> cgMLST scheme
(SeqSphere+) for molecular typing.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials |
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Seventy-six isolates were collected from Greece (n=44) and Spain (n=32).
Carbapenem-resistance was investigated by E-test and PCR-based methods.
Isolates were typed by rep-PCR (DiversiLab) and clusters were defined as having
≥ 98% similarity. Rep-PCR fingerprints were compared to our library of IC
lineages to determine their clonal background. WGS was performed using Nextera
XT chemistry and Illumina MiSeq paired-end read technology. An <i>ad hoc</i>
cgMLST scheme was set up using SeqSphere+ with <i>A. baumannii</i> ACICU as
reference strain to define the core genome. Genomes were assembled using
Velvet. Carbapenem-resistance genes were identified using ResFinder.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Seventy five
isolates were identified as CRAB. OXA-23 was the most common resistance
mechanism. Rep-PCR divided isolates into seven clusters. The majority of
isolates clustered with IC2 control strains (Table 1). Genomes were assembled
with a median of 191 contigs and median coverage of 47-fold. cgMLST further
subdivided DiversiLab clusters into smaller clusters (Figure 1). Not all
DiversiLab clusters correlated well with cgMLST (circled in Figure 1).
DiversiLab cluster E isolates from Sevilla were subdivided into two cluster
types by cgMLST which matched their carbapenem-resistance mechanism (OXA-58 and
OXA-23). Cluster C and cluster J isolates from Athens were identical by cgMLST,
as were A Coruña cluster C and cluster D. In the majority of cases, isolates
showing ≤ 10 allelic differences were from the same city. The one exception was
isolate R147 (Athens) with 3 allelic differences to isolates from Larissa. IC1
and unclustered strains were clearly delineated from the main group of IC2
isolates, with ≥1684 allelic differences.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The cgMLST<i>
ad hoc</i> scheme allowed a more detailed delineation than DiversiLab
clustering. Isolates that appeared to be closely related by rep-PCR were found
to contain upwards of several hundred allelic differences. However, clonal
lineages were clearly separated using this scheme. The use of a high-resolution
typing scheme will greatly aid in our understanding of <i>Acinetobacter
baumannii</i> populations.<o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<table class=MsoNormalTable border=0 cellpadding=0 style='mso-cellspacing:1.5pt;
 mso-yfti-tbllook:1184;mso-padding-alt:0cm 5.4pt 0cm 5.4pt'>
 <tr style='mso-yfti-irow:0;mso-yfti-firstrow:yes'>
  <td colspan=4 style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Table 1. DiversiLab
  clustering, carbapenem-resistance resistance mechanism and city of isolation<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:1'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>Cluster<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>City (number of isolates)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Acquired
  carbapenemases (number of isolates)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>International clone<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:2'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>A<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>Valencia (2)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>OXA-40 (2)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>Unclustered<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:3'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>C<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>A Coruna (1), Athens
  (22), Ciudad Real (1), Ipirus (1), Larissa (9), &nbsp; Sevilla (4), Valencia
  (3)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>OXA-40 (3), OXA-58 (1),
  OXA-23/58 (2), OXA-23 (35)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>IC2<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:4'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>D<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>A Coruña (2)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>OXA-23 (2)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>IC2<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:5'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>E<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>Ciudad Real (2), Sevilla
  (14) Valencia (1)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>OXA-40 (1), OXA-58 (8),
  OXA-23 (7), *no OXA (1)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>IC2<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:6'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>G<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>Athens (6)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>OXA-23 (6)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>IC1<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:7'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>H<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>Athens (2)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>OXA-23 (2)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>IC1<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:8'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>J<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>Athens (2)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>OXA-23 (2)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>IC2<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:9'>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>nd<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>Athens (2), Ciudad Real
  (1), Sevilla (1)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>OXA-23 (3), OXA-58 (1)<o:p></o:p></span></p>
  </td>
  <td style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>nd<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:10;mso-yfti-lastrow:yes'>
  <td colspan=4 style='padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto'><span
  style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
  Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>nd, not determined;
  *no OXA, the isolate was carbapenem-susceptible<o:p></o:p></span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal><span lang=DE-CH style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Imagem_x0020_50" o:spid="_x0000_i1027" type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/35imagem_2560108707_1.jpg"
 style='width:258pt;height:304pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S5_files/image007.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/35imagem_2560108707_1.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=260 height=306
src="public/abstracts/Webpages/PO_S5_files/image008.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/35imagem_2560108707_1.jpg"
v:shapes="Imagem_x0020_50"><![endif]></span><span lang=DE-CH style='font-size:
8.0pt;mso-bidi-font-size:11.0pt;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO70 - Multilocus
Sequencing Typing As A Tool To Investigate Childhood <i style='mso-bidi-font-style:
normal'>Haemophilus influenzae</i> Invasive Disease In Portugal<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Célia Bettencourt<sup>1</sup>; Vitor Borges<sup>1</sup>; José Gonçalo
Marques<sup>2</sup>; Florbela Cunha<sup>3</sup>; Maria Paula Bajanca-Lavado<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>Instituto Nacional de Saúde Dr. Ricardo Jorge,<sup>2</sup>
Hospital de Santa Maria, Centro Hospitalar Lisboa Norte, EPE,<sup>3</sup>
Hospital de Vila Franca de Xira<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: <i>Haemophilus
influenzae</i> is an important cause of serious childhood invasive disease
despite the use of the vaccine against serotype b strains (Hib). Six capsular
types, a-f, have been identified to date, although most of strains are
non-capsulated (NC).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Multilocus
Sequencing Typing (MLST) is a powerful method that allows a precise and
unambiguous characterization of <i>H. influenzae</i> genotypes.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>A partnership
between theNational Institute of Health and the Society for Paediatric
Infectious Diseases aimed to characterize invasive childhood infection in
Portugal.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The objective of
this study was to genotype isolates by MLST to follow the epidemiology of
disease.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Material /
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>This study was conducted between 1 January 2010 and 31 December 2015.
During this period 41 <i>H. influenzae</i>strains were analysed, mostly
isolated from blood (88%). Antibiotic resistance was assessed by the microdilution
assay and </span><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT'>β</span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>-lactamase production was determined with nitrocefin. Capsular status was
characterized by polymerase chain reaction using primers and conditions
described in the literature. MLST was performed by amplifyingand sequencing
internal fragments of the 7 housekeeping genes (<i>adk, atpG, frdB, fucK, mdh,
pgi </i>and </span><i><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
mso-fareast-language:PT'>recA</span></i><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-fareast-language:PT'>). Sequences were analyzed and
submitted to the MLST website </span><span lang=DE-CH><a
href="http://pubmlst.org/hinfluenzae/"><span lang=EN-US style='font-size:8.0pt;
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>http://pubmlst.org/hinfluenzae/</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> for
assignment of the sequence type (ST). To display the allelic distances between
the obtained STs, we applied the goeBURST algorithm implemented in the PHYLOViZ
platform.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Antimicrobial
susceptibility testing showed that most strains were susceptible to all
beta-lactams studied, with only three strains being ampicillin resistant due to
beta-lactamase production.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Most of invasive
disease was due to the presence of NC strains (27/41; 66%), while 14 isolates
(34%) were capsulated and characterized as follow: two serotype a (5%), 10 b
(24%) and two f (5%).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>As expected, MLST
typing revealed high genetic variability among 27 NC isolates, which had 24
(89%) different sequence types (STs), with four new STs represented by
previously unidentified allele combinations. In opposition, capsulated isolates
were very clonal: all 10 Hib were assigned to CC6 (eight strains ST6, one ST
1149, one ST 190), the two Hia strains were assigned to CC 23 (ST 23) and the
two Hif belonged to CC124 (ST 124 and ST 1188) (Figure 1).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Our data
indicate that invasive disease among Portuguese children is now due to highly
genetically diverse, fully susceptible NC strains, suggesting that no
particular virulent clone is responsible for epidemiological change of disease,
after vaccine implementation in the year 2000. Nevertheless, we are concerned
about Hib disease (24% of the isolates) despite the higher vaccine coverage.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>MLST typing
continues to show a high genetic diversity among NC strains and clonal
relationships among capsulated isolates.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>In conclusion, in
order to monitor the evolving dynamics of this pathogen and the epidemiology of
invasive disease, ongoing surveillance is needed to monitor the true magnitude
of this problem.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='mso-ansi-language:
EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape id="Imagem_x0020_51"
 o:spid="_x0000_i1026" type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/186imagem_2560108707_0.jpg"
 style='width:335pt;height:309pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S5_files/image009.png" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/186imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=337 height=311
src="public/abstracts/Webpages/PO_S5_files/image010.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/186imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_51"><![endif]></span><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO71 - Cost Effective
Strategy For Typing <i style='mso-bidi-font-style:normal'>Brucella abortus</i>
In An Endemic Scenario<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Giuliano Garofolo<sup>1</sup>; Ilenia Platone<sup>1</sup>; Katiuscia Zilli<sup>1</sup>;
Anna Abass<sup>1</sup>; Elisabetta Di Giannatale<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>Istituto Zooprofilattico Sperimentale dell´Abruzzo e
del Molise<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Brucella</span></i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> is a highly
clonal microorganism presenting elevated levels of nucleotide identity. For
this reason the VNTRs and SNPs are ideal for tracing its molecular evolution. </span><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>Recently
Garofolo et al. (2013) have shown that <i>Brucella abortus</i> in Italy is
divided in three major clades. </span><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Although the VNTRs are extremely valid for
typing <i>B. abortus</i>, they suffer of a high rate of homoplasy. In a
subsequent study we performed WGS SNPs analysis for reconstruct the <i>B.
abortus</i> phylogeny identifying and characterizing the three genetic clades
in a global context (2). SNPs tested on a large number of samples may be
prohibitive, in terms of costs and time. To date one cost-effective technique
is the Mismatch Amplification Mutation Assay, coupled with melt analysis
(Melt-MAMA). Here we present an analytical and epidemiological evaluation of
Melt-MAMA assays for the identification of Italian <i>B. abortus</i> clades.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Material and
Methods : </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Melt-MAMA Design<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>To identify the
three <i>B. abortus</i> clades we searched for six SNPs from the six specific
branches leading to the Italian strains. The assays satisfied the following
five conditions: (i) Two forward primer with in position 3´ corresponded to the
allelic status for the SNP to be searched; (ii) the presence of a common
reverse primer; (iii) an additional mismatch in the forward primer in in
anti-penultimate and penutimate position alternately; (iv) a GC-clamp in
position 5´ for one of the forward primer; (v) amplicon of reduced size (60 bp
-120 bp). <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Optimization of
MeltMAMA<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>To determine the
optimal condition, for each test, we tested the MeltMAMA in the following
manner: use of the pair of forward primer in three different reactions with
reciprocal concentration 1:1, 4:1 and 1:4. The optimum ratio was chosen and
adjusted with respect to the parameters of amplification. Assays were
considered valid when revealed the SNPs under identical amplification. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>B. abortus
isolates<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>A total of 200 <i>B.
abourtus</i> from the Italian epidemiological context were analyzed. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Six out of
13 assays developed equal to 46% were efficient, allowing the identification of
alleles requested. The optimization of the amplification parameters has allowed
to detect SNPs calls through the analysis of melting temperatures. According to
the assays, the 200 <i>B. abortus</i> were successful typed and demonstrated a
specific geographical spread (Fig.1).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: Using SNPs
to test phylogenetic hypotheses according to a hierarchical approach can reduce
the analysis of hypervariable markers such as VNTRs. In a scenario of
eradication of the disease through the policy of testing and slaughtering of
positive animals it is necessary to have tools of molecular epidemiology for
tracing outbreaks to monitor the phenomena of reintroduction of the disease in
free-areas. The MeltMAMA have proven effective in genotyping isolates using
individual PCR. This study is a step forward in the understanding of new
strategies for typing <i>Brucella</i> as a tool for the study of molecular
epidemiology of brucellosis in endemic regions.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=PT
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:PT;mso-fareast-language:PT'>References: </span></b><span
lang=PT style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:PT;mso-fareast-language:
PT'>1-G Garofolo et al. 2013. </span><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Investigating genetic diversity of <i>Brucella
abortus</i> and <i>Brucella melitensis</i> in Italy with MLVA-16. </span><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>Infection,
Genetics and Evolution 19, 59-70.<br>
2-G. Garofolo et al. (2014). </span><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Insights into the genetic lineages and current
spread of <i>B. abortus</i> and <i>B. melitensis</i> in Italy. </span><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>Proceedings of
Brucellosis 2014 International Research Conference. Berlin 9-12 September.</span><span
lang=PT style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:PT;mso-fareast-language:
PT'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=PT style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-ansi-language:PT;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Imagem_x0020_52" o:spid="_x0000_i1025" type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/201imagem_2560108707_0.jpg"
 style='width:292pt;height:412pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S5_files/image011.png" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/201imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=294 height=414
src="public/abstracts/Webpages/PO_S5_files/image012.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/201imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_52"><![endif]></span><span lang=PT style='font-size:
8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:PT;mso-fareast-language:PT'><o:p></o:p></span></p>

<span lang=PT style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:
115%;font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:
minor-latin;mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:PT;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO73 - Mixed <i
style='mso-bidi-font-style:normal'>Mycobacterium tuberculosis</i> Infections In
Switzerland Identified With Whole Genome Sequencing<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>David Stucki<sup>1</sup>; Marie Ballif<sup>2</sup>; Hans-Jakob Furrer<sup>3</sup>;
Kathrin Zürcher<sup>2</sup>; Matthias Egger<sup>2</sup>; Sébastien Gagneux<sup>1</sup>;
Lukas Fenner<sup>4</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Swiss Tropical and Public Health Institute, University
of Basel,<sup>2</sup> Institute of Social and Preventive Medicine, University
of Bern,<sup>3</sup> Clinic for Infectious Diseases, Bern University Hospital,
and University of Bern,<sup>4</sup> Institute of Social and Preventive
Medicine, University of Bern, and Swiss Tropical and Public Health Institute,
University of Basel</span></i><span lang=EN-GB style='mso-bidi-font-family:
Arial;color:black;mso-themecolor:text1;mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Infections with
multiple <i>Mycobacterium tuberculosis</i> genotypes are relevant for
tuberculosis diagnosis and treatment,&nbsp;as they can bias drug resistance
testing and render treatment of tuberculosis (TB) ineffective. In high
TB-incidence settings, their prevalence is up to 20%, however, little is known about
the prevalence of mixed <i>M. tuberculosis</i> infections in low-incidence
settings. Previous studies have used mixed MIRU-VNTR genotyping alleles as
indicator for mixed infections, but the sensitivity and specificity of using
MIRU-VNTR to detect mixed infections is debated. With the allele frequency data
obtained using whole genome sequencing, the presence of multiple genotypes can
be studied in detail. In this study, we used sequencing data of 520 <i>M.
tuberculosis</i> isolates from a nationwide study in Switzerland to identify
the presence of mixed infections.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> We
generated Illumina whole genome sequencing data of 520 single patient <i>M.
tuberculosis</i> isolates from 2000-2008 from HIV-coinfected and HIV-negative
TB patients. We analyzed sequencing reads at 177 genomic positions known to be
markers for phylogenetic lineages and sublineages, for the presence of minority
variants.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> We
identified at least 2/520 (0.4%) patient isolates which showed clear evidence
of a mixed population of two <i>M. tuberculosis</i> genotypes. Both isolates
consisted of two strains of the L4.5/LAM sublineage of <i>M. tuberculosis</i>,
however, with different proportions of the minor variant (29% and 16%).
Interestingly, one patient was born in South America, where the L4.5/LAM
sublineage is known to be highly prevalent. The second patient was born in
Switzerland. Both patients were HIV-negative. In the isolate from the
Swiss-born patient, a mutation in <i>rrs</i> (C491T), which is known to confer
streptomycin resistance, was found in the same proportion as the majority
variant. Neither of the two isolates had mixed MIRU-VNTR alleles. We did not
observe any mixed genotypes in isolates with multiple MIRU-VNTR alleles.<o:p></o:p></span></p>

<p class=MsoNormal><b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Conclusion:</span></b><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> We show that
infections with multiple <i>M. tuberculosis</i> strains do occur in low TB-incidence
settings such as Switzerland, but are infrequent compared to high-incidence
settings. The concurrent presence of a drug resistant and a susceptible
genotype in one isolate emphasizes the need to take mixed infections into
account for diagnosic tests and treatment of TB. The different birth origin of
the patients indicates that mixed infections do not only occur in particular
risk groups such as immigrants or HIV-positive TB patients. Whole genome
sequencing data analysis provided a clear advantage to identify mixed
infections, however, needs further development, especially for mixed infections
of the same phylogenetic sublineage, which were not identified in this study.</span><span
lang=EN-GB style='mso-bidi-font-family:Arial;color:black;mso-themecolor:text1;
mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=DE-CH><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
