<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns:mv="http://macVmlSchemaUri" xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta name=Title content="">
<meta name=Keywords content="">
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List href="PO_S7_files/filelist.xml">
<link rel=Edit-Time-Data href="PO_S7_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Bruno Gonçalves</o:Author>
  <o:LastAuthor>Bruno Gonçalves</o:LastAuthor>
  <o:Revision>1</o:Revision>
  <o:TotalTime>1</o:TotalTime>
  <o:Created>2016-03-08T21:53:00Z</o:Created>
  <o:LastSaved>2016-03-08T21:54:00Z</o:LastSaved>
  <o:Pages>22</o:Pages>
  <o:Words>9906</o:Words>
  <o:Characters>56467</o:Characters>
  <o:Company>IMM</o:Company>
  <o:Lines>470</o:Lines>
  <o:Paragraphs>132</o:Paragraphs>
  <o:CharactersWithSpaces>66241</o:CharactersWithSpaces>
  <o:Version>14.0</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="PO_S7_files/themedata.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>JA</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
   <w:UseFELayout/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="276">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Verdana;
	panose-1:2 11 6 4 3 5 4 4 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-1593833729 1073750107 16 0 415 0;}
@font-face
	{font-family:Verdana;
	panose-1:2 11 6 4 3 5 4 4 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-1593833729 1073750107 16 0 415 0;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-520092929 1073786111 9 0 415 0;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1073743103 0 0 415 0;}
@font-face
	{font-family:Tahoma;
	panose-1:2 11 6 4 3 5 4 4 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-520082689 -1073717157 41 0 66047 0;}
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;
	mso-fareast-language:EN-US;}
h1
	{mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"Heading 1 Char";
	mso-style-next:Normal;
	margin-top:24.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:1;
	font-size:14.0pt;
	font-family:Calibri;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#365F91;
	mso-themecolor:accent1;
	mso-themeshade:191;
	mso-font-kerning:0pt;
	mso-ansi-language:DE-CH;
	mso-fareast-language:EN-US;
	font-weight:bold;}
h2
	{mso-style-noshow:yes;
	mso-style-priority:9;
	mso-style-qformat:yes;
	mso-style-link:"Heading 2 Char";
	mso-style-next:Normal;
	margin-top:2.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:2;
	font-size:13.0pt;
	font-family:Calibri;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#365F91;
	mso-themecolor:accent1;
	mso-themeshade:191;
	mso-ansi-language:DE-CH;
	mso-fareast-language:EN-US;
	font-weight:normal;}
h5
	{mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"Heading 5 Char";
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	mso-outline-level:5;
	font-size:10.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	mso-ansi-language:DE-CH;
	mso-fareast-language:DE-CH;
	font-weight:bold;}
p.MsoToc1, li.MsoToc1, div.MsoToc1
	{mso-style-update:auto;
	mso-style-priority:39;
	mso-style-next:Normal;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:5.0pt;
	margin-left:0cm;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;
	mso-fareast-language:EN-US;}
p.MsoCommentText, li.MsoCommentText, div.MsoCommentText
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Comment Text Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;
	mso-fareast-language:EN-US;}
p.MsoHeader, li.MsoHeader, div.MsoHeader
	{mso-style-priority:99;
	mso-style-link:"Header Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	tab-stops:center 216.0pt right 432.0pt;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;
	mso-fareast-language:EN-US;}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{mso-style-priority:99;
	mso-style-link:"Footer Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	tab-stops:center 216.0pt right 432.0pt;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;
	mso-fareast-language:EN-US;}
span.MsoCommentReference
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-ansi-font-size:8.0pt;
	mso-bidi-font-size:8.0pt;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	mso-themecolor:hyperlink;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:purple;
	mso-themecolor:followedhyperlink;
	text-decoration:underline;
	text-underline:single;}
p.MsoDocumentMap, li.MsoDocumentMap, div.MsoDocumentMap
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Document Map Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-ansi-language:DE-CH;
	mso-fareast-language:EN-US;}
p
	{mso-style-priority:99;
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	mso-ansi-language:EN-GB;
	mso-fareast-language:EN-GB;}
p.MsoCommentSubject, li.MsoCommentSubject, div.MsoCommentSubject
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"Comment Text";
	mso-style-link:"Comment Subject Char";
	mso-style-next:"Comment Text";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;
	mso-fareast-language:EN-US;
	font-weight:bold;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Balloon Text Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:8.0pt;
	font-family:Tahoma;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-ansi-language:DE-CH;
	mso-fareast-language:EN-US;}
p.MsoNoSpacing, li.MsoNoSpacing, div.MsoNoSpacing
	{mso-style-priority:1;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:left;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;
	mso-fareast-language:EN-US;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	mso-add-space:auto;
	text-align:justify;
	text-justify:inter-ideograph;
	line-height:107%;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;
	mso-fareast-language:EN-US;}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	mso-add-space:auto;
	text-align:justify;
	text-justify:inter-ideograph;
	line-height:107%;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;
	mso-fareast-language:EN-US;}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	mso-add-space:auto;
	text-align:justify;
	text-justify:inter-ideograph;
	line-height:107%;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;
	mso-fareast-language:EN-US;}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	mso-add-space:auto;
	text-align:justify;
	text-justify:inter-ideograph;
	line-height:107%;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;
	mso-fareast-language:EN-US;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 1";
	mso-ansi-font-size:14.0pt;
	mso-bidi-font-size:14.0pt;
	font-family:Calibri;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#365F91;
	mso-themecolor:accent1;
	mso-themeshade:191;
	mso-ansi-language:DE-CH;
	font-weight:bold;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-noshow:yes;
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 2";
	mso-ansi-font-size:13.0pt;
	mso-bidi-font-size:13.0pt;
	font-family:Calibri;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#365F91;
	mso-themecolor:accent1;
	mso-themeshade:191;
	mso-ansi-language:DE-CH;}
span.Heading5Char
	{mso-style-name:"Heading 5 Char";
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 5";
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:10.0pt;
	font-family:"Times New Roman";
	mso-ascii-font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:"Times New Roman";
	mso-bidi-font-family:"Times New Roman";
	mso-ansi-language:DE-CH;
	mso-fareast-language:DE-CH;
	font-weight:bold;}
p.bodytext, li.bodytext, div.bodytext
	{mso-style-name:bodytext;
	mso-style-unhide:no;
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	mso-ansi-language:DE-CH;
	mso-fareast-language:DE-CH;}
span.CommentTextChar
	{mso-style-name:"Comment Text Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Comment Text";
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:10.0pt;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Arial;
	mso-ansi-language:DE-CH;}
span.CommentSubjectChar
	{mso-style-name:"Comment Subject Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-parent:"Comment Text Char";
	mso-style-link:"Comment Subject";
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:10.0pt;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Arial;
	mso-ansi-language:DE-CH;
	font-weight:bold;}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Balloon Text";
	mso-ansi-font-size:8.0pt;
	mso-bidi-font-size:8.0pt;
	font-family:Tahoma;
	mso-ascii-font-family:Tahoma;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Tahoma;
	mso-bidi-font-family:Tahoma;
	mso-ansi-language:DE-CH;}
span.st
	{mso-style-name:st;
	mso-style-unhide:no;}
p.western, li.western, div.western
	{mso-style-name:´´western´´;
	mso-style-unhide:no;
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	mso-ansi-language:PT;
	mso-fareast-language:PT;}
p.p1, li.p1, div.p1
	{mso-style-name:´´p1´´;
	mso-style-unhide:no;
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	mso-ansi-language:PT;
	mso-fareast-language:PT;}
span.HeaderChar
	{mso-style-name:"Header Char";
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:Header;
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Arial;
	mso-ansi-language:PT;}
span.FooterChar
	{mso-style-name:"Footer Char";
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:Footer;
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Arial;
	mso-ansi-language:PT;}
p.Affiliations, li.Affiliations, div.Affiliations
	{mso-style-name:Affiliations;
	mso-style-unhide:no;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:17.85pt;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	text-indent:-17.85pt;
	line-height:150%;
	mso-pagination:none;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Calibri;
	mso-fareast-font-family:Calibri;
	mso-bidi-font-family:"Times New Roman";
	mso-ansi-language:EN-GB;
	mso-fareast-language:EN-US;}
span.DocumentMapChar
	{mso-style-name:"Document Map Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Document Map";
	font-family:"Times New Roman";
	mso-ascii-font-family:"Times New Roman";
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:"Times New Roman";
	mso-bidi-font-family:"Times New Roman";
	mso-ansi-language:DE-CH;}
p.IMMEM, li.IMMEM, div.IMMEM
	{mso-style-name:IMMEM;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"Heading 1";
	mso-style-link:"IMMEM Caráter";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:16.0pt;
	margin-left:0cm;
	text-align:right;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:1;
	font-size:12.0pt;
	font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-ansi-language:EN-US;
	mso-fareast-language:EN-US;
	font-weight:bold;}
p.IMMEMl, li.IMMEMl, div.IMMEMl
	{mso-style-name:IMMEM_l;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:IMMEM;
	mso-style-link:"IMMEM_l Caráter";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:left;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:1;
	font-size:12.0pt;
	font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-ansi-language:EN-US;
	mso-fareast-language:EN-US;
	font-weight:bold;}
span.IMMEMCarter
	{mso-style-name:"IMMEM Caráter";
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:IMMEM;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Arial;
	mso-bidi-font-family:Arial;
	font-weight:bold;}
p.IMMEMr, li.IMMEMr, div.IMMEMr
	{mso-style-name:IMMEM_r;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:IMMEM;
	mso-style-link:"IMMEM_r Caráter";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:right;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:1;
	font-size:12.0pt;
	font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-ansi-language:EN-US;
	mso-fareast-language:EN-US;
	font-weight:bold;}
span.IMMEMlCarter
	{mso-style-name:"IMMEM_l Caráter";
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-parent:"IMMEM Caráter";
	mso-style-link:IMMEM_l;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Arial;
	mso-bidi-font-family:Arial;
	font-weight:bold;}
span.IMMEMrCarter
	{mso-style-name:"IMMEM_r Caráter";
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-parent:"IMMEM Caráter";
	mso-style-link:IMMEM_r;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Arial;
	mso-bidi-font-family:Arial;
	font-weight:bold;}
p.navegacao-link, li.navegacao-link, div.navegacao-link
	{mso-style-name:navegacao-link;
	mso-style-unhide:no;
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:7.5pt;
	font-family:Verdana;
	mso-fareast-font-family:"Times New Roman";
	mso-bidi-font-family:"Times New Roman";
	color:#999999;
	mso-ansi-language:PT;
	mso-fareast-language:PT;}
p.Abstract, li.Abstract, div.Abstract
	{mso-style-name:Abstract;
	mso-style-unhide:no;
	margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	line-height:150%;
	mso-pagination:none;
	font-size:12.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Calibri;
	mso-fareast-font-family:Calibri;
	mso-bidi-font-family:"Times New Roman";
	color:#006B8E;
	mso-ansi-language:EN-GB;
	mso-fareast-language:EN-US;}
p.IMMEMdatetime, li.IMMEMdatetime, div.IMMEMdatetime
	{mso-style-name:IMMEM_date_time;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"Heading 2";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:12.0pt;
	margin-left:0cm;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:2;
	font-size:12.0pt;
	font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	mso-ansi-language:EN-US;
	mso-fareast-language:PT;
	font-weight:bold;
	mso-bidi-font-weight:normal;
	font-style:italic;
	mso-bidi-font-style:normal;}
p.IMMEMindex, li.IMMEMindex, div.IMMEMindex
	{mso-style-name:IMMEM_index;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:IMMEM;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:16.0pt;
	margin-left:0cm;
	text-align:right;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:1;
	font-size:12.0pt;
	font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-ansi-language:EN-US;
	mso-fareast-language:EN-US;
	font-weight:bold;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"ＭＳ 明朝";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
@page WordSection1
	{size:595.0pt 842.0pt;
	margin:72.0pt 89.85pt 72.0pt 89.85pt;
	mso-header-margin:35.45pt;
	mso-footer-margin:35.45pt;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
@list l0
	{mso-list-id:325549294;
	mso-list-template-ids:452521116;}
@list l1
	{mso-list-id:394860103;
	mso-list-template-ids:-1813850462;}
@list l2
	{mso-list-id:460225044;
	mso-list-template-ids:1776606900;}
@list l3
	{mso-list-id:629283510;
	mso-list-template-ids:-579191296;}
@list l4
	{mso-list-id:1298030603;
	mso-list-template-ids:-1032794714;}
@list l5
	{mso-list-id:1425496888;
	mso-list-template-ids:1946585780;}
ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;}
table.MsoTableGrid
	{mso-style-name:"Table Grid";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-priority:59;
	mso-style-unhide:no;
	border:solid windowtext 1.0pt;
	mso-border-alt:solid windowtext .5pt;
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-border-insideh:.5pt solid windowtext;
	mso-border-insidev:.5pt solid windowtext;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-ansi-language:PT;}
</style>
<![endif]-->
</head>

<body bgcolor=white lang=EN-US link=blue vlink=purple style='tab-interval:36.0pt'>

<div class=WordSection1>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO11 - Transfer Of
Silenced VanA Gene Cluster By Broad Host-range Plasmid<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Audun Sivertsen<sup>1</sup>; Torunn Pedersen<sup>2</sup>; Kjersti Wik Larssen<sup>3</sup>;
Kåre Bergh<sup>3</sup>; Torunn Gresdal Rønning<sup>3</sup>; Andreas Radke<sup>3</sup>;
Kristin Hegstad<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University of Tromsø – The Arctic University of
Norway,<sup>2</sup> University hospital of North-Norway,<sup>3</sup> St. Olavs
Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Keywords: VRE, IS-elements, Outbreak, plasmid, HGT</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-font-kerning:18.0pt;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Objectives: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>The purpose of this study was to explain
vancomycin resistance development in a nosocomial outbreak of <i>vanA­</i>-containing,
vancomycin susceptible enterococci within two hospitals in Trøndelag, Norway.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-font-kerning:18.0pt;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Materials &amp; methods: </span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>48 <i>E. faecium </i>and
one <i>E. faecalis</i> linked to a nosocomial outbreak and shown to be <i>vanA-</i>containing
vancomycin susceptible or resistant enterococci (VSE/VRE) by <i>vanA</i> PCR
and phenotypic testing, were included in the study. 42 of the <i>E. faecium </i>isolates
were clonal by PFGE and six were of unique pulsotypes. Four of the clonal <i>vanA</i>-containing
VSEfm isolates and two VREfm isolated after vancomycin treatment of VSEfm
infections were whole-genome sequenced by Illumina MiSeq. The plasmid content
of all unique pulsotypes was analyzed with S1-nuclease PFGE and hybridizations.
Transfer of <i>vanA</i> was assessed by filter mating.The VSE (n=35) were
cultivated in BHI broth containing 8 mg/L vancomycin for <i>in vitro</i>
development of VREfm. The <i>vanA</i> cluster composition was determined by
contig gap closure with PCR and Sanger sequencing, and transcription profile
was assessed with RT-qPCR.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-font-kerning:18.0pt;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>The only genetic difference between the <i>vanA</i>
clusters of VSE and VRE was an IS<i>L3</i>-family insertion sequence element
found upstream of <i>vanHAX</i> in the VSEfm in all pulsotypes<i>.</i>
Furthermore, all VSEfm and VREfm isolates had insertions of IS<i>1216</i> in
the intergenic region between <i>vanX</i> and <i>vanY</i> as well as IS<i>1542</i>
between <i>orf2</i> and <i>vanR </i>as compared to the prototype <i>vanA</i>
gene cluster.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Growth of all the
VSEfm was observed after exposure to vancomycin or teicoplanin for 24 to 72
hours. All the resulting resistant mutants had lost IS<i>L3</i>. There was a
60-fold reduction of the essential <i>vanHAX</i> gene transcription due to the
IS<i>L3</i> insertion. <i>In vitro </i>produced resistant mutants expressed <i>vanHAX</i>
constitutively and not through induction during vancomycin exposure, as is the
case of wild-type VRE (BM4147). This could be linked a 7-fold reduction in the
expression of the <i>vanHAX</i> activator, coded by <i>vanRS,</i> due to an
insertion of IS<i>1542</i> in the <i>vanRS</i> promoter area.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The <i>vanA </i>gene
cluster was located on a transferrable plasmid of same replicon type and
similar size in both PFGE-related and unrelated isolates.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-font-kerning:18.0pt;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusions: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>IS<i>L3</i> insertion upstream of <i>vanHAX</i>
caused the VanA phenotype to be silenced. Exposure to vancomycin resulted in
resistant variants that had lost IS<i>L3</i>. IS<i>1542</i> insertion upstream
of <i>vanRS</i> resulted in a constitutive transcription of the VanR-activated <i>vanHAX</i>
and <i>vanRS</i> promoters. The <i>vanA </i>gene cluster was located on a
transferrable broad host-range plasmid also detected in outbreak isolates with
different PFGE types as well as an <i>E. faecalis</i>. Both genotypic and
phenotypic susceptibility testing is necessary to disclose these potentially
vancomycin resistant isolates.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO12 - High-level
Beta-lactam Resistance In A <i style='mso-bidi-font-style:normal'>Staphylococcus
aureus</i> Strain Carrying The MecC Gene Determinant<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Catarina Milheiriço<sup>1</sup>; Alexander Tomasz<sup>2</sup>; Hermínia de
Lencastre1,<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>Laboratory of Molecular Genetics, Instituto de Tecnologia
Química e Biológica António Xavier (ITQB), Universidade Nova de Lisboa,<sup>2</sup>
Laboratoty of Microbiology and Infectious Diseases, The Rockefeller University<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methicillin-resistant
<i>Staphylococcus aureus</i> (MRSA) is one of the most important multi-resistant
pathogens worldwide. Most MRSA carry the common genetic determinant <i>mecA </i>encoding
for the low affinity penicillin binding protein PBP2A. A new methicillin
resistant determinant – <i>mecC – </i>was recently identified in some MRSA
isolates: it shares only 69% identity with <i>mecA</i> from MRSA strain N315
and is integrated in a new SCC<i>mec</i> structure (SCC<i>mec</i> XI). Strains
carrying <i>mecC </i>have typically low MICs to oxacillin; however, no data are
available on the impact of extensive beta-lactam selection on the emergence of <i>mecC
</i>carrying strains with increased levels of resistance.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>We exposed
the prototype strain LGA251 carrying the <i>mecC </i>determinant to a
step-selection procedure with increasing concentrations of oxacillin in the
growth medium. Three LGA251 derivatives able to grow at elevated oxacillin
concentrations were isolated. Efforts to identify the mechanism of increased
resistance found no mutations in the determinants of PBP2 and PBP4 or in the <i>mecC
</i>regulatory genes. However, a point mutation was identified in the <i>mecC</i>
gene of the most resistant derivative strain of LGA251. Whole genome sequencing
(WGS) was used to identify determinants affected by this step-selection
procedure.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Eleven
different single nucleotide polymorphisms (SNPs) affecting eight different open
reading frames (<i>orfs</i>) were identified in the WGS data analysis. As
expected, accumulation of mutations occurred over time: the first derivative
strain (LGA251 (50) able to grow in the presence of 50 </span><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>μ</span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>g/ml of oxacillin
in the growth medium) – was the one with fewer mutations and these were
maintained in the more resistant derivative strains (LGA251 (100) and LGA251
(800)). In addition, there were two regions deleted during the selection
process, comprising ten different <i>orfs</i>. Compromised <i>orfs</i> were
scattered throughout the core genome of LGA251 affecting several metabolic
functions.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Detection of
mutations in genes without a direct link to the common mechanism of methicillin
resistance indicates that determinants located in the genetic background of
bacteria can have profound influence on beta-lactam resistance levels. Studies
are under way to identify <i>orfs </i>that play key roles in defining the
oxacillin resistance level of bacterial isolates recovered from the
step-selection process.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-font-style:italic'>Funding</span></b><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'> Project PTDC/BIA-MIC/3195/2012 and SFRH/BPD/63992/2009 (FCT, Portugal), US
Public Health Service Award R01-AI457838-15.</span><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO13 - Resistance To
Daptomycin In Vancomycin-resistant <i style='mso-bidi-font-style:normal'>Enterococcus
faecium</i> During Daptomycin Treatment<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Christine Elmeskov<sup>1</sup>; Mette Pinholt<sup>1</sup>; Peder Worning<sup>1</sup>;
Jesper Boye Nielsen<sup>1</sup>; Heidi Gumpert<sup>1</sup>; Dorte Frees<sup>2</sup>;
Henrik Westh<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Hvidovre University Hospital,<sup>2</sup> University
of Copenhagen<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-font-weight:bold'>Keywords<b>: </b></span><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>whole genome
sequencing, Enterococcus, VRE</span><span style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Daptomycin
is a cyclic lipopeptide used in the treatment of vancomycin-resistant <i>Enterococcus
faecium </i>(VREfm). However, the finding of daptomycin resistant VREfm has
challenged the treatment of nosocomial VREfm infections. VREfm can cause
urinary tract infections, intraabdominal infections and catheter related
infections usually in older or weaker patients.We report a case where a daptomycin
susceptible VREfm isolate became resistant during daptomycin and linezolid
therapy. The isolates were identified in a patient with gallstone-induced
pancreatitis complicated by pancreatic necrosis and pancreatic abscesses. The
patient was treated with a combination of daptomycin and linezolid for 40 days
prior to the identification of the daptomycin resistant VREfm.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>In the present
study we examined the genetic changes in a VREfm strain that became daptomycin
resistant during daptomycin treatment.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials |
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>The daptomycin susceptible isolate, V1164, was identified in drain fluid
from an intraabdominal abscess, and the resistant isolate, V1225, was
identified from the patient’s bloodstream. Daptomycin MIC was established on
both isolates using E-test. Whole genome sequencing of the two isolates was
performed to reveal their genetic background and to uncover potential mutations
causing daptomycin resistance. Sequencing was performed in an Illumina MiSeq
using Nextera XT library preparation Kit running 2x150bp paired end reaction.
Reads were assembled using Velvet and VelvetOptimizer. The reads were mapped to
a reference genome (AUS004_NC-017022) using Stampy and SNPs were called using
Samtools.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Genes previously
identified to be related to daptomycin resistance were investigated in CLC
Genomics by using BLAST. The BLASTed genes were cardiolipin synthase (<i>cls</i>),
cyclopropane fatty acid synthase (<i>cfa</i>) and a glycerolphosphoryl diester
phosphodiesterase (<i>gdpD</i>), all involved in phospholipid metabolism and
genes ecoding the LiaFSR regulatory system and the YycFGHIJ system, involved in
the cell envelope stress response and cell wall homeostasis, respectively. The
HD Domain protein believed to be involved in nucleic acid metabolism, signal
transduction and possibly other functions, and a hypothetical protein, encoding
a putative N-acetylmuramoyl-L-alanine amidase, were also BLASTed.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>V1164
exhibited a daptomycin MIC of 2 mg/L and V1225 exhibited a daptomycin MIC of 8
mg/L.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The two VRE
genomes were ST18 and one SNP apart compared to the reference genome. We
identified five mutations in <i>gdpD</i>, of which three were non-synonymous
mutations and two were synonymous mutations. We also identified a deletion in a
hypothetical protein domain of 34 bases, and a 16.000 bp deletion that
includedthe entire genetic region of the HD domain protein.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The
combination therapy with daptomycin and linezolid did not protect against
development of daptomycin resistance. The mutations identified in <i>gdpD</i>
in V1225 are likely to cause daptomycin resistance, as previous studies have
reported that mutations in <i>gdpD</i> are associated with daptomycin
resistance. The deletion of the HD domain is also interesting, as previous
studies only found amino acid changes in a daptomycin resistant strain. Other
mutations in genes previously reported to be associated with resistance, such
as <i>cls</i> or <i>cfa</i>, were not identified in this study.<o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO15 - Investigation Of
The Mechanism Involved In Tigecycline Resistance In</span></b><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'> <b style='mso-bidi-font-weight:
normal'><i style='mso-bidi-font-style:normal'>Enterococcus</i> Spp.<o:p></o:p></b></span></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Stefan Fiedler<sup>1</sup>; Jennifer Bender<sup>1</sup>; Carola Fleige<sup>1</sup>;
Uta Geringer<sup>1</sup>; Ingo Klare<sup>1</sup>; Guido Werner<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Robert Koch-Institute, Division 13 Nosocomial
Pathogens and Antibiotic Resistances<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Tigecycline
(TGC) represents one of the last-line therapeutics to combat multi-drug
resistant bacterial pathogens including VRE and MRSA. The National Reference
Centre for Staphylococci and Enterococci at the RKI received 79 TGC‑resistant <i>E.&nbsp;faecium</i>
and <i>E.&nbsp;faecalis</i> isolates in recent years. The precise mechanism of
how enterococci become resistant to TGC remains undetermined. As part of an
ongoing investigation this abstract is to describe examinations of efflux pumps
and their contributions to TGC resistance in clinical isolates of <i>Enterococcus</i>
spp.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>High and low
level TGC-resistant strains were analyzed with respect to genome and
transcriptome differences by means of whole genome sequencing, whole
transcriptome sequencing and RT-qPCR. Genes of interest were cloned and
expressed in <i>Listeria&nbsp;monocytogenes </i>for functional analyses
including determination of the minimum inhibitory concentration (MIC) to TGC.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>As analyzed
by microbroth dilution assays, varying levels of TGC MICs exist for the 27
strains investigated. Comparative genome analyses of three isogenic strains,
showing different levels of TGC resistance, revealed the MFS efflux pump TetL
and the ribosomal protection protein TetM as possible drug resistance proteins.
Subsequent RT-qPCR confirmed the up‑regulation of the respective genes compared
to the isogenic TGC‑sensitive strain. A correlation of gene copy number and
level of resistance could be inferred from further analyses. Eventually,
expression of both <i>tet</i>(L) and <i>tet</i>(M) in <i>L.&nbsp;monocytogenes</i>
unequivocally demonstrated the potential to increase TGC MICs upon acquisition
of the loci.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Our results
indicate that increased expression of two determinants, a tetL‑encoded MFS-pump
and a tetM-encoded ribosomal protection protein, is capable of conferring TGC
resistance in some enterococcal strains. However, as not all TGC-resistant
isolates investigated in this study were tested positive for <i>tet</i>(L),
alternative TGC resistance mechanisms are suspected and will be addressed in
future experiments.</span><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO16 - Characteristics
Of Publicly Available <i style='mso-bidi-font-style:normal'>Staphylococcus
aureus</i> Genomes<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Nir Gilad<sup>1</sup>; Michal Ziv-Ukelson<sup>2</sup>; Vered Chalifa-Caspi<sup>1</sup>;
Jacob Moran-Gilad<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Ben-Gurion University,<sup>2</sup> Ben Gurion
University<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Genome
depositions of important pathogens such as <i>S. aureus</i> are rapidly
accumulating in public domain databases such as NCBI and could prove useful for
development and evaluation of high-throughput bioinformatics tools as well as
cross-sectional studies of pathogenomics. However, concerns have been raised
regarding the quality of publicly deposited genomic data for many
microorganisms. We thus sought to determine the characteristics of publicly
available <i>S. aureus </i>genomes. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Available <i>S.
aureus</i> genome assemblies were downloaded from NCBI and checked for quality
using several software tools. The genomes were subsequently analysed using an
in house <i>S. aureus</i> pipeline at the Surveillance and Pathogenomics
Israeli Centre of Excellence (SPICE). Genus and species identification (ID) and
inferred antimicrobial susceptibility testing (AST) were determined by
performing BLAST analyses as well as <i>in silico</i> PCR analyses according to
published methods and using in house as well as public resistance gene
databases such as The Comprehensive Antibiotic Resistance Database (CARD),
consisting of over 3,000 genes in total. Additional characteristics such as SCC<i>mec</i>
type, PVL and USA300 lineage were determined based on published gene markers.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Overall, 4,262
genomes were included in the analysis. All genomes appeared to belong to the <i>Staphylococcus</i>
genus. The <i>nuc</i> gene marker for <i>S. aureus</i> was missing in 19
genomes (0.45%). Following additional analysis, 12 samples were identified as
species other than <i>S. aureus</i> (0.28%). The majority (3,903, 91.6%) of
available genomes were methicillin-resistant <i>S. aureus</i> (MRSA). Only 122
of those (3.1%) were depositions tagged as MRSA and 2 of those 122 samples
(1.6%) were <i>mecA</i> and SCC<i>mec</i> negative. SCC<i>mec</i> typing could
be determined in 96.3% and types I, II, III and IV accounted for 3.8%, 56.3%,
0.9% and 29.3% of typeable samples, respectively. The pantone valentine
leucocidin (PVL) gene was present in 661 samples (15.5%) and 712 samples
contained the USA300 lineage markers (16.7% in total, 18.2% of MRSA). Amongst
MRSA genome assemblies, major antimicrobial resistance determinants were found
in the following proportions: macrolides: <i>ermA</i> 60%, <i>ermC</i> 6%, <i>ermB</i>
1%; beta-lactamases: <i>blaZ</i> 53.2%; aminoglycoside modifying enzymes:
APH(3´)-IIIa 25.9%, AAC(6´)-Ie-APH(2´´)-Ia 13.6%, aad(6) 3.7%, ANT(6)-Ia 0.3%;
tetracyclines: <i>tetK</i> 8.8%; disinfectants <i>qacA</i> 1.6%;
chloramphenicol <i>cat</i> 0.1%.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Analysis of over
4,000 <i>S. aureus</i> genome assemblies available through the public domain
showed a very low rate of erroneous genus, species and MRSA classification,
suggesting publicly available <i>S. aureus</i> datasets could be safely used
for developing and validating Staph bioinformatics pipelines for clinical and
public health usage. Systematic analysis of important microbiological
characteristics of <i>S. aureus</i> revealed that publicly available datasets
are relatively enriched with samples of MRSA and PVL-positive samples compared
to natural epidemiology, most probably reflecting the medical and research
focus in this field. High throughput screening of <i>S. aureus</i> genomes for
inferring antimicrobial resistance was feasible and its application on publicly
available samples may be helpful for generating relevant cross-sectional data
on the global prevalence of resistance determinants.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO17 - Comparison Of
Extended-Spectrum Beta-Lactamase (ESBL)-producing <i style='mso-bidi-font-style:
normal'>Escherichia coli</i> Isolates From Hospitals, Ambulatory Settings And
The Community<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Michael Pietsch<sup>1</sup>; Christoph Eller<sup>2</sup>; Linda Falgenhauer<sup>3</sup>;
Rasmus Leistner<sup>4</sup>; Giuseppe Valenza<sup>5</sup>; Constanze Wendt<sup>6</sup>;
Guido Werner<sup>1</sup>; Yvonne Pfeifer<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Robert Koch Institute, FG 13 Nosocomial Pathogens and
Antibiotic Resistance,<sup>2</sup> Department of Laboratory Medicine,
University Hospital Halle,<sup>3</sup> Institute of Medical Microbiology,
Justus Liebig University Giessen and German Center for Infection Research
(DZIF), Partner site Giessen-Marburg-Langen, Campus Giessen,<sup>4</sup>
Charité Universitätsmedizin, Institut of Hygiene and Environmental Medicine,<sup>5</sup>
Bayerisches Landesamt für Gesundheit und Lebensmittelsicherheit (LGL),<sup>6</sup>
Labor Limbach<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> The increase of <i>Escherichia
coli</i> producing extended-spectrum beta-lactamases (ESBL) in hospitals and
their emergence as colonisers of healthy humans is of concern.<br>
Here we report the results of the molecular characterisation of ESBL-producing<i>
E.&nbsp;coli</i> isolates from German hospitals and medical practices. The
isolates were collected in different studies in the scope </span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>of the interdisciplinary
research project “RESET” (</span><span lang=DE-CH><a
href="http://www.reset-verbund.de"><span lang=EN-US style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-fareast-language:PT'>www.reset-verbund.de</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials/Methods</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> We analysed 527 <i>E.&nbsp;coli</i>
with ESBL phenotype from three studies: 128 nosocomial <i>E. coli </i>and 105
“ambulant” <i>E.&nbsp;coli </i>isolates from outpatient departments, 211 <i>E.&nbsp;coli</i>
from healthy participants (community study) and 83 community-acquired <i>E.&nbsp;coli</i>
from a case-control study. The presence of beta-lactamase genes was tested by
PCR and sequencing. Phylogenetic grouping and screening for the sequence type <i>E.
coli </i>ST131 was performed by PCR. Furthermore, we randomly selected 264
isolates (50%) for multilocus sequence typing (MLST) analyses.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> The ESBL-<i>E.&nbsp;coli</i>
from hospital patients and outpatients mainly harboured CTX-M-15 (ambulant 51%;
nosocomial 49%), CTX-M-1 (ambulant 26%; nosocomial 30%) and CTX-M-14 (ambulant
7%; nococomial 5%). <i>E.&nbsp;coli</i> from healthy persons (community study)
showed a similar high proportion of CTX-M-15 (46%) and CTX-M-1(24%) but <i>E.&nbsp;coli</i>
from the case-control study harboured more CTX-M-1 (42%) and less CTX-M-15
(29%). Sequence type ST131 was present in 40% of the ambulant and 32% of the
nosocomial <i>E. coli,</i> and the ST131 proportion among CTX-M-15 producing <i>E.
coli</i> was particularly high (92% ambulant and 86% nosocomial). Analyses of
ESBL-<i>E. coli</i> of the case-control-study and the community study revealed
a lower proportion of ST131 which was 26% and 12%, respectively. MLST analyses
revealed a high variety of STs (n=62) but several STs (including 2-12 isolates)
were present in all three study collections: ST410, ST405, ST10 and ST38.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>:</span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> We confirmed
CTX-M-15 and CTX-M-1 as the most frequent ESBL-types in <i>E.&nbsp;coli</i>
from humans. Furthermore, we found the worldwide successful clonal lineage <i>E.&nbsp;coli</i>-ST131
to be the dominant ST among isolates causing different infections (<i>E. coli</i>
from hospitalised patients and outpatients). Apart from ST131 we detected <i>E.
coli</i> ST410, ST405, ST10 and ST38 in all three different study populations.
These four STs were also detected in previous studies involving <i>E. coli </i>from
livestock, whole genome comparisons of isolates from different settings are
ongoing as part of this project. The great variety of <i>E. coli</i>-STs in our
analyses indicated the importance of horizontal transfer of ESBL genes, a
hypothesis which needs more detailed plasmid analyses in future.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO18 - Characterization
And Molecular Epidemiology Of Plasmid-mediated Quinolone Resistance <i
style='mso-bidi-font-style:normal'>Escherichia coli</i> Isolates From Algeria.<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Yanat Betitra<sup>1</sup>; Machuca Jesús<sup>2</sup>; Díaz de Alba Paula<sup>2</sup>;
Abdelaziz Touati<sup>1</sup>; Pascual Alvaro<sup>2</sup>; Rodriguez Martinez
Jose Manuel<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Laboratory of microbial ecology. University of Bejaia,<sup>2</sup>
Department of microbiology. Faculty of medicine. University of Seville<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-font-weight:bold'>Keywords:</span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> <i>E. coli</i>,
PMQR, PFGE</span><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Fluoroquinolone
resistance in <i>Escherichia coli</i>is an important issue all over the
world.The aim of this study is to evaluate the genetic relatedness of <i>Escherichia
coli</i> isolates coding for Plasmid-Mediated Quinolone Resistance mechanisms
(PMQR) and recovered from urine of outpatients in Algeria.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Ninety six
nalidixic acid resistant <i>E. coli</i> isolates were collected from four
different laboratories in Northern Algeria between 2010 and 2014.
&nbsp;Antibiotic susceptibility was determined according to the CLSI
guidelines. Genes coding for PMQR and beta-lactamases were identified by PCR
and their identities were confirmed by DNA sequencing. Quinolone
Resistance-Determining Region (QRDR) of <i>gyr</i>A and <i>par</i>C genes was
also analyzed. Clonal relatedness was studied by pulsed-field gel
electrophoresis (PFGE).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The rate of
PMQR genes in <i>E. coli</i> was 16,6% (16/96):&nbsp; 11 contained<i>
aac(6’)-Ib-cr</i> (one of&nbsp; which was also positive for <i>qnrB1</i>-like)
, 2 <i>qnrS1-</i>like and 3&nbsp; <i>oqx</i>A/B genes. Either, two mutations in
<i>gyr</i>A (Ser83→Leu&nbsp; and&nbsp; Asp87→Asn) or one or two substitutions in
<i>par</i>C (Ser80→Ile and Glu84→Val or Val 87→Ala) were the most common
associated mechanisms in those isolates with MICs of cipro</span><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>ﬂ</span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>oxacin
ranging from 8 to&nbsp; &gt;256 mg/L. PMQR was associated with ESBL and
plasmidic AmpC-type ß-lactamase in 75% and 6,25% respectively. The most
prevalent betalactamases were CTX-M-15 (n:11), CTX-M-3 (n:1) and one CMY-4
(n:1). Most of PMQR-positive <i>E. coli</i> isolates were clonally unrelated
with the exception of two small clonal groups comprising two and four isolates
and producing both AAC(6´)Ib-cr and CTX-M-15.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>A high
prevalence of PMQR in <i>E. coli</i> isolates has been detected in Northern
Algeria. These determinants were frequently associated to other resistance
determinants (ESBL and plasmidic AmpC). Most of the isolates were genetically
unrelated.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='mso-ansi-language:
EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shapetype id="_x0000_t75"
 coordsize="21600,21600" o:spt="75" o:preferrelative="t" path="m@4@5l@4@11@9@11@9@5xe"
 filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Imagem_x0020_19" o:spid="_x0000_i1026" type="#_x0000_t75"
 alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/69imagem_2560108707_0.jpg"
 style='width:326pt;height:178pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S7_files/image001.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/69imagem_2560108707_0.jpg"
  croptop="5415f" cropbottom="10534f"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=328 height=180
src="public/abstracts/Webpages/PO_S7_files/image002.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/69imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_19"><![endif]></span><span style='mso-ansi-language:
EN-US;mso-fareast-language:PT'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
<o:p></o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO19 - Molecular
Characterisation Of Carbapenem-resistant Gram-negative Bacteria From A
Bulgarian Hospital<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Yvonne Pfeifer<sup>1</sup>; Angelina Trifonova<sup>2</sup>; Michael Pietsch<sup>1</sup>;
Magdalena Brunner<sup>1</sup>; Iva Todorova<sup>2</sup>; Ivanka Gergova<sup>2</sup>;
Gottfried Wilharm<sup>1</sup>; Guido Werner<sup>1</sup>; Encho Savov<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Robert Koch Institute,<sup>2</sup> Military Medical
Academy<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Background: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Gram-negative bacteria with resistance to
carbapenems are an emerging threat in hospitals worldwide. From September 2014
to January 2015, 71 carbapenem-resistant isolates (50 <i>Acinetobacter </i>spp.,
14 <i>Proteus mirabilis</i>, five <i>Escherichia coli</i>, one <i>Providencia rettgeri</i>
and one <i>Pseudomonas aeruginosa</i>) were identified in a hospital in Sofia,
Bulgaria. The <i>Acinetobacter </i>spp. were isolated mainly from respiratory
specimen, the Enterobacteriaceae were mainly from urine samples. In January
2015, all isolates were sent to the Robert Koch Institute for molecular
characterisation.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Materials/Methods: </span></b><i><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Acinetobacter</span></i><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> species
identification was performed by PCR and sequencing of the <i>rpoB</i> gene.
Antimicrobial susceptibility testing for all 71 isolates was performed by
microbroth dilution and Etest. Beta-lactamase genes were identified by PCR and
sequencing. Bacterial strain typing was performed by enzymatic macrorestriction
and subsequent pulsed field gel electrophoresis (PFGE). Selected strains of <i>A.
baumannii</i> and <i>E. coli</i> were analysed by multilocus sequence typing
(MLST).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Species identification of the 50 <i>Acinetobacter</i>
isolates revealed the presence of 47 <i>Acinetobacter baumannii</i>, one <i>Acinetobacter
seifertii</i>, one <i>Acinetobacter radioresistens</i>, and one <i>Acinetobacter
pittii</i>. The <i>A. pittii</i> strain and 44 <i>A. baumannii</i> isolates
produced either carapenemases OXA-23 and/or OXA-72. These isolates were
resistant to imipenem, meropenem, amikacin and ciprofloxacin but remained
susceptible to colistin. The <i>A. radioresistens</i> strain harboured the <i>bla</i><sub>OXA-23
</sub>gene<sub> </sub>but was only ertapenem-resistant similar to three <i>A.
baumannii</i> and the <i>A. seifertii</i> strain without any carbapenemase
genes. PFGE typing of the 47 <i>A. baumannii</i> showed the presence of at
least seven different strains, and five of these were assigned to the
international clonal lineage 2 (IC 2). MLST analyses confirmed the dominance of
sequence type ST2 (Pasteur scheme) which included ST350, ST208, ST436 and ST437
(Oxford scheme), respectively. Further, eight <i>A. baumannii</i>-ST231 (IC 1)
and one <i>A. baumanni-</i>ST502(IC 3) were identified.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Analyses of the
Enterobacteriaceae isolates revealed VIM-2 presence in the <i>P. aeruginosa</i>
isolate and NDM-1 production in one <i>P. rettgeri</i>, one <i>P. mirabilis</i>
and in the five <i>E. coli</i> isolates, respectively. These NDM-1 <i>E. coli </i>co-harboured
beta-lactamases CMY-4 and CTX-M-15, showed identical macrorestriction patterns
(PFGE) and were assigned to sequence type ST101. Furthermore, 13 <i>P. mirabilis</i>
isolates from 13 patients were positive for VIM-1, CM-99 and SHV-12 and their
identical PFGE patterns indicated the clonal transfer.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusion: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Our study confirmed the presence of different
carbapenemase producing bacteria in Bulgarian hospitals. For OXA carbapenemase
producing <i>A. baumannii</i> the occurrence of different strains over the full
period of time may indicate a permanent presence in the hospital environment or
an import of distinct strains from outside and subsequent clonal transfer
within the hospital. Furthermore, the repeated occurrence of one VIM-1
producing <i>P. mirabilis</i> and one NDM-1 producing <i>E. coli</i>-ST101
clone is of concern since only few therapeutic options are available in case of
infections. Intensified hygiene measurements and surveillance are necessary to
control the further spread of these bacteria.</span><span style='font-size:
8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO21 - Emergence Of CTX-M-producing
Clinical Isolates Of <i style='mso-bidi-font-style:normal'>Klebsiella
pneumoniae</i> In The University Hospital Establishment Of Oran, Algeria.<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Zemmour Assia<sup>1</sup>; Dali-Yahia Radia<sup>2</sup>; Saidi Ouahrani
Nadjia<sup>1</sup>; Rahmani Bouabdallah<sup>1</sup>; Christian G. GISKE<sup>3</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Universty Of Sciences And Technology,<sup>2</sup>
University Hospital Establishment 1st, Novembre 1954,<sup>3</sup> Karolinska
University Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-font-weight:bold'>Keywords:<b> </b></span><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Klebsiella
pneumoniae</span></i><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>, ESBL, CTX-M–producing, SYBR Green PCR, 16S rRNA methylases, AmpC,
Algeria.</span><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-font-weight:
bold'>Klebsiella pneumoniae </span></i><i style='mso-bidi-font-style:normal'><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>(<span
style='mso-bidi-font-weight:bold;mso-bidi-font-style:italic'>KP</span>)</span></i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> is a
gram-negative bacterium with rapidly emerging resistance compared to most other
gram-negative species. Infections with <i><span style='mso-bidi-font-weight:
bold'>KP</span></i> are usually hospital-acquired and occur primarily in
patients with impaired host defenses.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials /
Methods:&nbsp;</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>The aim of study was to characterize the
molecular epidemiology of 198 Extended-Spectrum Beta-lactamase (ESBL)-producing
<i><span style='mso-bidi-font-weight:bold'>KP</span></i><b> </b>from the
University Hospital Establishment of Oran, Algeria. Species identification was
done with API®20E and ESBL-production was confirmed according to The Clinical
and Laboratory Standards Institute 2015 recommendations. Strains were derived
from blood (n=28), protected specimen brush (n=27), urine (n=45), wound secretions
(n=50), central venous catheters (n=27), cerebrospinal fluid (n=9), vaginal
discharge (n=4), abdominal, or dialysis fluid (n=7) and (n=1) liquid pleural.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>A SYBR Green PCR
assay was carried out for detection of the following genotypes CTX-M-1 group,
CTX-M-2 group, and CTX-M-9 group. To cover other potential CTX-M genotypes, a
set of CTX-M universal primers was also used. Strains with resistance to
amikacin were also tested for the presence of the 16S rRNA methylases <i>armA</i>,
<i>rmtA</i>, <i>rmtB</i>, <i>rmtC</i>, and <i>rmtD</i>. Strains which were
negative for CTX-M and/or cefoxitin resistant were tested with the microarray
Check MDR CT101 (Checkpoints®, The Netherlands).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:&nbsp;</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The
phenotypical tests and analyses with WHONET 5.6 software, almost strains (99%)
was resistant to Cefotaxim, 40 % of strains were resistant to ciprofloxacin and
only 6/198 were also resistant to cefoxitin (3 %), indicating that they could
have concomitant production of an acquired AmpC enzyme. Resistance to
ceftazidime was also very common (84 %). Gentamicin resistance was seen in 92.5
% of isolates whereas amikacin resistance was seen in 26/198 of isolates (13 %)<b>.</b>
Two strains were resistant to ertapenem and only one strain was resistant to
imipenem (See the table).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The molecular
analyses showed that 190 /198 strains (95.95 %) were positive with primers for
the CTX-M-1 group, 1/198 was positive with primers for the &nbsp;CTX-M-9 group,
and 7/198 strains (3.5%) were negative with CTX-M universal primers. For
strains that were resistant to amikacin we found 20/26 isolates (76.2%)
positive for <i>armA</i> and 6/26 isolates were negative for all 16S rRNA
methylases. Microarray analysis showed that 5/7 were negative for all ESBLs in
the assay, one isolate was positive for CTX-M-1 group, and one isolate had an
SHV-ESBL Among cefoxitin resistant isolates 5/6 isolates were CMY II, and one
isolate was negative.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion:&nbsp;</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The study
showed that the CTX-M is still the predominant genotype of ESBL, and that the
ESBL positive strains have high resistance levels. The strains are mainly
producing CTX-M group-1 and concurrent occurrence in some isolates of CTX-M
producing and AmpC (CMY II) or 16S rRNA methylases raises clinical concern and
may become a major therapeutic challenge in the future.<o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Imagem_x0020_53" o:spid="_x0000_i1025" type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/32imagem_2560108707_0.jpg"
 style='width:333pt;height:185pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S7_files/image003.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/32imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=187
src="public/abstracts/Webpages/PO_S7_files/image004.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/32imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_53"><![endif]></span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO22 - Epidemiology Of
Carbapenem-resistant <i style='mso-bidi-font-style:normal'>Pseudomonas putida</i>
Group Clinical Isolates Recovered From Screening Cultures<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Silke Peter<sup>1</sup>; Philipp Oberhettinger<sup>1</sup>; Ariane Kirste<sup>1</sup>;
Wichard Vogel<sup>2</sup>; Daniela Dörfel<sup>2</sup>; Daniela Bezdan<sup>3</sup>;
Stephan Ossowski<sup>3</sup>; Matthias Marschal<sup>1</sup>; Jan Liese<sup>1</sup>;
Matthias Willmann<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Institute of Medical Microbiology and Hygiene,
University of Tübingen,<sup>2</sup> Medical Center, Department of Hematology,
Oncology, Immunology, Rheumatology &amp; Pulmonology, University of Tübingen,<sup>3</sup>
Genomic and Epigenomic Variation in Disease Group, Centre for Genomic
Regulation (CRG), Barcelona, Spain Universitat Pompeu Fabra (UPF)<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><i><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Pseudomonas putida
</span></i><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>is a gram negative, rode-shaped, non-fermenting bacterium that is
ubiquitously encountered in the environment. <i>P. putida</i> is a rather rare
cause of human disease but is frequently found in water systems. Antimicrobial
resistance patterns of isolates reported varied between the different studies
and multidrug resistant <i>P. putida</i> isolates harbouring
metallo-beta-lactamases (MBLs) have been reported from all over the world. <i>P.
putida</i> was suggested to be a platform for the exchange of genetic
resistance elements between <i>P. putida</i> and <i>P. aeruginosa</i>. However,
the detailed mechanisms and frequency of antimicrobial resistance exchange and
their relevance within the hospital setting remain poorly understood.&nbsp;
&nbsp;<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>In total, 101 <i>P.
putida</i> group strains were recovered from rectal swabs, throat swabs or
stool samples taken within the scope of a weekly routine screening program for
a period of 28 month. Antimicrobial susceptibility testing, genomic
fingerprinting and whole genome sequencing (WGS) was performed.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The following
resistant rates were detected in the <i>P. putida</i> isolates: resistance to
piperacillin (63%), piperacillin-tazobactam (60%), meropenem (57%),
ciprofloxacin (45%) and gentamycin (14%). Meropenem non-susceptible strains
(n=64) were further subjected to molecular detection of the MBL genes <i>bla</i><sub>VIM</sub>
and <i>bla</i><sub>IMP</sub>. VIM was detected in 42 isolates (41.6%). The VIM
positive isolates displayed a similar antimicrobial susceptibility patterns
with the exception of gentamycin, which was susceptible in 28 of 42 (66.7%)
isolates, suggesting the presence of at least two different VIM positive <i>P.
putida</i> group clusters. Genomic fingerprinting of the isolates applying the
DiversiLab Pseudomonas Kit (bioMérieux, France) revealed 15 different clusters.
The cluster 8 and cluster 15 comprised 10 isolates each, followed by cluster 1
containing 3 isolates.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>During the study
period five <i>P. aeruginosa</i> VIM positive isolate were recovered from the
same patient cohort. WGS was performed for all VIM positive <i>P. putida</i>
group and <i>P. aeruginosa</i> isolates. The phylogenetic analysis and the
comparison of the resistance gene content are still on-going at present.</span><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<i style='mso-bidi-font-style:normal'><span style='font-size:10.0pt;mso-bidi-font-size:
11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:Cambria;
mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></i>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><i style='mso-bidi-font-style:normal'><span style='mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></i></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO23 - First Report Of
Macrolide Resistance Among <i style='mso-bidi-font-style:normal'>Shigella</i>
Isolates In Israel Caused By The mphA And Erm(b) Genes<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Analia V. Ezernitchi<sup>1</sup>; Elizabeth Sirotkin<sup>1</sup>; Vered
Agmon<sup>1</sup>; Lea Valinsky<sup>1</sup>; Assaf Rokney<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Ministry of Health Israel<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><i><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Shigella</span></i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> species
remain an infection agent of public concern, affecting mostly children, with
high incidence in developing areas as well as in industrialized countries.
Although azithromycin is recommended for treatment of shigellosis, there are
currently no CLSI susceptibility breakpoints. Recently, cases of <i>Shigella
flexneri</i> with decreased susceptibility to azithromycin (DSA) have been
reported globally. We aimed to assess the prevalence of macrolide resistance genes
among <i>Shigella</i> <i>flexneri</i> in Israel.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>All <i>Shigella
flexneri</i> isolates (162), received at the National Reference Center during
2014, were serotyped, and screened by PCR for the presence of macrolide
resistance genes <i>erm(B)</i> and <i>mphA</i> that are commonly
plasmid-encoded. Azithromycin minimum inhibitory concentration (MIC) values
were assessed by the Broth Microdilution (BMD) method. Antimicrobial
susceptibility to additional drugs (ampicillin, trimethoprim-sulfamethoxazole,
ceftriaxone, tetracycline, nalidixic acid and chloroamphenicol) was tested by
the disk diffusion method according to the CLSI guidelines. Clonal relatedness
was assessed by Pulsed-Field Gel Electrophoresis (PFGE).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The <i>mph</i>A
gene was detected in 11 isolates (7%), 4 of which harbored <i>erm</i>(B) as
well. The presence of <i>mph</i>A or <i>erm</i>(B) correlated with high
azithromycin MIC values (&gt;256 </span><span lang=DE-CH style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-fareast-language:PT'>μ</span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>g/ml). &nbsp;Among
the 11 isolates, 3 were MDR and 2 were XDR, and all were sensitive to
ciprofloxacin. The strains displaying high azithromycin MIC values included
different <i>Sh</i>. <i>flexneri</i> serotypes (1b, 1a, 3a, and 6) and
different PFGE patterns, indicating different lineages. Among <i>Sh. flexneri</i>,
34 isolates (20%) were resistant to nalidixic acid.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Tracing
antimicrobial resistance trends has direct impact on the outcome of
shigellosis. The emergence of DSA <i>Shigella</i> reinforces the necessity of
the establishment of clinical breakpoints that would warrant routine testing,
reporting and surveillance for this drug of choice.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO24 - CTX-M-55
Extended-spectrum Beta-lactamase Among <i style='mso-bidi-font-style:normal'>Enterobacteriaceae</i>
From The Korean Antimicrobial Resistance Monitoring System, 2008 To 20<sup>1</sup>5<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Jin Seok Kim, Soo Jin Kim, Jungsun Park, Gyung Tae Chung and Junyoung Kim,<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Korea National Institute of Health, Division of
Enteric Diseases<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Keywords: ESBL, CTX-M-55, IS<i>Ecp1</i>, IncI1, IncA/C</span><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The
emergence and rapid dissemination of CTX-M-type extended-spectrum
beta-lactamases (ESBLs) in <i>Enterobacteriaceae</i> are a serious global
concern for the public health. Recently, CTX-M-55, a variant of the most common
CTX-M-15, has been increasingly reported in several countries. This study
characterized the CTX-M-55 beta-lactamase in <i>Enterobacteriaceae</i> from
clinical isolatesin Republic of Korea.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials and
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Among cefotaxime-resistant strains collected, the <i>bla</i><sub>CTX-M-55</sub>
gene and their genetic environment were determined by using PCR and DNA
sequencing. PCR-based replicon typing (PBRT) of plasmids encoding CTX-M-55 was
also carried out. Two IncI1 plasmids (pSEN101134 and pSal111105) were sequenced
using an Ion Torrent PGM sequencer and the initial reads were de novo assembled
using Newbler and CLC Genomics Workbench.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>From the
national surveillance of antimicrobial resistance among <i>Enterobacteriaceae</i>,
we screened five CTX-M-55-producing <i>Salmonella</i> and one <i>Shigella </i>strains.
By PBRT, the <i>bla</i><sub>CTX-M-55</sub> genes were placed on the IncI1 (<i>n</i>=2),
IncA/C (<i>n</i>=3) and IncB/O (<i>n</i>=1) plasmids. IncI1 plasmids possessed
a known IS<i>Ecp1</i>-<i>bla</i><sub>CTX-M-55</sub>-<i>orf477</i> transposable
unit, whereas the <i>bla</i><sub>CTX-M-55</sub> genes in IncA/C and IncB/O
plasmids were preceded by IS<i>26</i> or IS<i>5 </i>sequences, respectively.
Plasmid pSEN111105 is 86179 bp in size with a GC content of 49.7% and contains
114 putative ORFs, and pSEN101134 has quite similar structure (86194 bp; 117
ORFs, &gt;99.9% nucleotide identities). When compared with other sequenced
IncI1 plasmids encoding <i>bla</i><sub>CTX-M-55</sub> identified from <i>Shigella
sonnei</i> (p1081-CTXM) and <i>Klebsiella pneumoniae</i> (p628-CTXM), they were
closely related to each other with the exceptions of IS elements in the IncI1
shufflon.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>With regard
to various plasmid replicon types and the genetic environment surrounding the <i>bla</i><sub>CTX-M-55</sub>
gene, CTX-M-55 ESBL has emerged independently and disseminated in Korea. In
addition, these results indicate that the IncI1 plasmid may be used as a
successful genetic vehicle, favoring the spread of various <i>bla</i><sub>CTX-M</sub>
genes among different members of <i>Enterobacteriaceae</i>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>&nbsp;<o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO25 - Active Screening
And Genotyping Successfully Controls Outbreaks Of Multi-drug-resistant <i
style='mso-bidi-font-style:normal'>Enterobacteriaceae</i><o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Anneke van der Zee<sup>1</sup>; Arjan Burggraaf<sup>1</sup>; Lieuwe Roorda<sup>1</sup>;
Fer Vlaspolder<sup>2</sup>; Jacobus M Ossewaarde<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Maasstad Hospital,<sup>2</sup> Maastad Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Abstract: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Nosocomial
infections pose an increasing threat to vulnerable patients by the rapid
emergence and dissemination of multi-drug-resistant Enterobacteriaceae.
Enterobacteriaceae can exhibit resistance to multiple antibiotics such as
extended- spectrum beta-lactamases (ESBL), including carbapenemases,
cephalosporins, quinolones and aminoglycosides. Treatment options for infected
patients can therefore be severely limited.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>In our
hospital we implemented a modular screening protocol for detection of relevant
resistance genes, ie: KPC / VIM / IMP / NDM / OXA-48, ESBL (CTX-M), and
AmpC.Rectal swabs of critical patients (intensive care, neonatal unit, burn
wound unit), patients at risk or hospitalized for &gt;7 days were cultured
overnight in different selective broths containing: ertapenem, cefotaxime,
ceftazidime, gentamicine/ciprofloxacine, tobramycine/ ciprofloxacine,
ceftazidime/ciprofloxacine. Fully automated screening PCRs were carried out,
and if positive, samples were subcultured on selective or chromogenic agars for
isolate identification, antibiotic susceptibility testing, and PCR
confirmation.Multi-drug-resistant isolates are defined by the presence of
gentamicin/tobramycin andciprofloxacin, or ESBL, AmpC, or carbapenemase
positive.Possible transmission or source tracing was investigated by genotyping
of MDR strains with amplified fragment length polymorphism (AFLP). Genotype
banding patterns were analysed using Bionumerics.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Five
outbreaks of ESBL positive <i>Klebsiella pneumoniae</i> and <i>Escherichia coli</i>
were identified by clusters of identical genotypes within a timespan of 7
months. The largest outbreak included 5 neonates who were colonized with ESBK <i>K.pneumoniae</i>.
Two clusters of OXA-48 producing <i>Enterobacter cloacae</i> were
identified.Appropriate measurements and precautions were taken for containment
of the outbreaks.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Active
screening for, and genotyping of MDR Enterobacteriaceae has proven to be an
effective method for reducing the frequency of nosocomial infections. It provides
increased safety for patients, many clusters will be detected in an early phase
still without clinical infections. Furthermore, the method is patient-friendly
as usually only one swab is required for screening.Thus, routine identification
and monitoring of clusters of MDR strains can limit further dissemination, and
keep our hospitals safe.<o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO26 - Genetic
Diversity And Molecular Analysis Of Quinolone-resistant <i style='mso-bidi-font-style:
normal'>Shigella sonnei</i> Isolates From Humans In Republic Of Korea, 2008-2014<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Junyoung Kim<sup>1</sup>; Soo Jin Kim, Jin Seok Kim, Nan-Ok Kim, Sahyun
Hong and Gyung Tae Chung<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Korea National Institute of Health, Division of
Enteric Diseases<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Keywords: <i>Shigella sonnei, </i>Quinolone, QRDR, PFGE, MLVA</span><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Shigella
sonnei</span></i><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>, a causative agent of bacterial dysentery, has become prevalent in the
developing countries as well as developed countries. However, the increase of
drug-resistant <i>S. sonnei</i> is becoming more difficult to treat these
infections, posing a serious public health problem. We characterized quinolone
resistance determinants and genetic relatedness of <i>S. sonnei </i>isolates in
Republic of Korea.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials and
Results: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>PCR-based sequence analysis was carried out for screening the determinants
of quinolone resistance as follows; for the quinolone resistance-determining
regions (QRDRs) in DNA topoisomerase II and IV, and for the presence of
plasmid-mediated quinolone resistance (PMQR) genes.Quinolone-resistant isolates
were typed by pulsed-filed gel electrophoresis (PFGE) and multiple-locus
variable number tandem repeat analysis (MLVA).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Between 2008
and 2014, we collected 157 quinolone-resistant <i>S. sonnei</i> isolates from
the stool samples of patients with diarrhea including 31 domestic and 126
travel-related cases. All quinolone resistant isolates contain one or more
mutations in QRDRs in GyrA (Ser83Leu and Asp87Gly) and ParC (Ser80Ile and
Gln91Arg). The <i>aac(6’)-Ib-cr </i>gene was identified in five isolates of <i>S.
sonnei</i>, whereas none of PMQR genes was detected. Based on the genetic
relatedness among resistant isolates using PFGE and MLVA, domestic clones were
clearly distinct from isolates linked to foreign travel that were further
discriminatedfrom each country of location.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>These
results indicate that with an increase of international travel to shigellosis
endemic area, importation and subsequent regional spread of drug-resistant <i>S.
sonnei </i>has increased in Korea. Therefore, continued monitoring is necessary
for preventing the spread of drug-resistance <i>S. sonnei</i> strains.<o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO27 - Antimicrobial
Resistance And Virulence Profile In Community And Hospital-acquired Urinary
Tract Infections Caused By <i style='mso-bidi-font-style:normal'>Klebsiella
pneumoniae</i><o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>C. Caneiras<sup>1</sup>; L. Lito<sup>2</sup>; J. Melo-Cristino<sup>3</sup>;
A. Duarte<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>1iMed.UL - Research Institute for Medicines and
Pharmaceutical Sciences, Faculty of Pharmacy, University of Lisbon,<sup>2</sup>
Laboratory of Microbiology, Centro Hospitalar Lisboa Norte,<sup>3</sup>
Laboratory of Microbiology, Centro Hospitalar Lisboa Norte,<sup>4</sup>
Institute of Microbiology, Institute of Molecular Medicine, Faculty of
Medicine, University of Lisbon<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Urinary
tract infections (UTIs) are the most common bacterial infection. Emergence of
antimicrobial resistance and production of extended spectrum </span><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>β</span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>-lactamases
(ESBLs) are responsible for empirical therapy failures with significative
medical and finantial impact. The aim of this study was to evaluate the
relationship between the resistance and virulence determinants in <i>Klebsiella
pneumoniae</i> clinical isolates recovered in hospital and community environment.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials |
Methods</span></b><b style='mso-bidi-font-weight:normal'><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> This study
included 50 UTI clinical isolates of <i>K.pneumoniae</i> collected in 10
community laboratories. Susceptibilities to antimicrobial agents were
determined by disk diffusion: imipenem, ciprofloxacin, gentamicin, cefoxitine,
cefotaxime, ceftazidime, fosfomycin, levofloxacin, nitrofurantoin,
trimethoprim/sulfamethoxazole and amoxicillin/clavulanic acid.The results were
interpreted according to CLSI and EUCAST guidelines The ESBLs were identified
for<i> bla</i><sub>-CTX-M, </sub><i>bla</i><sub>-TEM</sub> and <i>bla</i><sub>-KPC</sub>
gene and screened for gene markers of 8 virulence factors (VF) genes: <i>K2A,
fimH, mrkD<sub>V1</sub>, mrkD<sub>V2-4</sub>, khe, rmpA, magA, </i>and <i>iucC</i>
by PCR amplification. The virulence results were compared with 31 urinary
clinical isolates from one tertiary care hospital. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Six</span><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>β</span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>-lactamases
(12%) were identified: TEM-156 (n=2), SHV-11 (n=1) and the ESBL TEM-24 (n=1),
SHV-33 (n=1) and CTX-M-15 (n=1). Significative antimicrobial resistance (22%)
was found to nitrofurantoin and trimethoprim/sulfamethoxazole. In
community-acquired UTI isolates the absence of VF (16% <i>Vs</i> 0%,
p&lt;0,005) and the presence of type 3 fimbrial adhesins - variant 2-4 (44% <i>Vs</i>
0%, p&lt;0,0001) was significant compared with hospital isolates characterized
by haemolysin (84% <i>Vs</i> 46%, p=0,0009), fimbrial adhesins type 1 (97% <i>Vs</i>
40%, p&lt;0,0001), fimbrial adhesins type 3 – variant 1 (94% <i>Vs</i> 20%,
p&lt;0,0001) and K2 serotype (29% Vs 6%, p=0,0081). Community (C-VP) and
hospital-virulence profiles (H-VP) were numbered according the predominance.18
different C-VP were identified. 10% of the isolates presented the C-VP1 (<i>khe</i>),
the C-VP2 (<i>khe</i>, <i>mrkD<sub>V2-4</sub>) </i>and C-VP3 (<i>fimH, mrkD<sub>V2-4</sub></i>).
8% had the C-VP4(<i>fimH, iucC, mrkD<sub>V2-4</sub></i>). None of this C-VP was
detected in hospital isolates witch presented 9 H-VP. The most frequent
profiles were H-VP1 (<i>fimH, khe, mrkD<sub>V1</sub>) </i>in 41% and H-VP2<i>
(K2, fimH, khe, mrkD<sub>V1</sub></i>) in 22%of the isolates, not identified in
community isolates. The profile of community </span><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>β</span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>-lactamases
producers was also studied: TEM-156 (<i>mrkD<sub>V1</sub> </i>and<i> khe, mrkD<sub>V1</sub>)</i>,
SHV-11 (<i>mrkD<sub>V2-4</sub></i>) TEM-24 (absence of VF), SHV-33 (<i>fimH,
mrkD<sub>V2-4</sub></i>) and CTX-M-15 (<i>khe). </i>Different profiles were identified
for hospital </span><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT'>β</span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>-lactamases: TEM-24 (H-VP1 and <i>fimH, khe</i>) and for CTX-M-15 ESBL
alone (H-VP1 and <i>fimH, mrkD<sub>V1</sub>)</i> or in co-expression with KPC-3
carbapenemase:<i> H-PV 4 (fimH, khe, mrkD<sub>V1</sub>, iucC) and H-PV6 (K2,
fimH, mrkD<sub>V1</sub>)</i> and H-PV9 (<i>K2, fimH, khe, mrkD<sub>V1</sub>,
iucC). </i><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Our results
indicate that the hospital or community source significantly impacts on the
virulence of <i>K. pneumoniae</i> isolates, with abilities to simultaneously
express several virulence factors, organized in specific and successful
virulence patterns. The presence of ESBL enzymes itself does not suggest an
association to virulence but he convergence of virulence and resistance genes
is more significant at hospital-adquired UTI and can potentially lead to the
emergence of untreatable <i>K. pneumoniae</i> infections.</span><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO28 - Increasing
Cotrimoxazole Susceptibility Associated With Predominance Of International
Clone 2 <i style='mso-bidi-font-style:normal'>Acinetobacter baumannii</i>
Clinical Isolates: A Nationwide<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Spyros Pournaras<sup>1</sup>; Konstantina Dafopoulo<sup>1</sup>; Olympia
Zarkotou<sup>2</sup>; Aggeliki Poulou<sup>3</sup>; Stamatina Levidiotou<sup>4</sup>;
Achilleas Gikas<sup>5</sup>; Athanasios Tsakris1, Greek Study Group on
Acinetobacter Antimicrobial Resistance<o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Medical School, University of Athens,<sup>2</sup>
Tzaneio Hospital,<sup>3</sup> Serres Hospital,<sup>4</sup> Ioannina University
Hospital,<sup>5</sup> Heraklion University Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT'>Keywords: Clonal complex; molecular epidemiology; carbapenem-hydrolyzing
oxacillinase</span><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: <i>Acinetobacter</i>
<i>baumannii</i> clinical isolates are very common in Greek hospitals and
currently exhibit almost universal resistance to carbapenems and most
antimicrobials except colistin. Lately, cotrimoxazole
(trimethoprim/sulfamethoxazole, SXT)-susceptible isolates are increasingly
recovered. We report herein that the enhanced SXT susceptibility is associated
with the predominance of international clone (IC) 2 <i>A. baumannii </i>isolates
in multiple hospitals from all over Greece.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: A
collection of 142 single-patient carbapenem-resistant<i> A. baumannii </i>invasive
isolates consecutively recovered during 2015 from nine hospitals located
throughout Greece were tested. Antimicrobial susceptibility testing was
performed by automated systems and/or Etest. The 3LST multiplex PCR and the
single-locus <i>bla</i><sub>OXA-51-like</sub>sequencing for clonal assignment
and PCR/sequencing for carbapenemase-encoding genes were performed. PCR mapping
for the possible presence of IS<i>Aba</i>1 upstream of <i>bla</i><sub>OXA-23</sub>
gene was also undergone.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: Of the 142
isolates tested, 23 (16.2%) belonged to IC1, carrying <i>bla</i><sub>OXA-69</sub>,
and 119 (83.8%) to IC2, carrying <i>bla</i><sub>OXA-66</sub>. In the 9
participating hospitals overall, 34.5% of isolates showed SXT susceptibility;
all 23 IC1 isolates were SXT-resistant, while 49 of the IC2 isolates (41.2%)
were SXT-susceptible. Among the other antibiotics tested, isolates belonging to
IC1 exhibited considerably higher activity to tetracycline (35.3%), minocycline
(100%), tobramycin (64.7%) and ampicillin/sulbactam (18.2%) compared with IC2
isolates (1.7%, 21.1%, 12.5% and 4.3%, respectively), while 8.8% of the
isolates overall exhibited resistance to colistin, without considerable
differences between lineages. Within IC2, SXT-susceptible and SXT-resistant
isolates had similar antibiotic resistance phenotypes. All isolates carried the
gene <i>bla</i><sub>OXA-51-like</sub>, 140/142 isolates (98.6%) carried <i>bla</i><sub>OXA-23-like
</sub>and 2/142 (1.4%) had both <i>bla</i><sub>OXA-23-like </sub>and<sub> </sub><i>bla</i><sub>OXA-58-like
</sub>genes<sub>. </sub>The IS<i>Aba1</i><sub> </sub>insertion was found
upstream of <i>bla</i><sub>OXA-23-like </sub>in all isolates.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: Almost all
<i>A. baumannii</i> isolates tested produced OXA-23 carbapenemase; the majority
of isolates belonged to IC2 and fewer to IC1. Of the limited antimicrobials
that retained susceptibility, SXT exhibited a considerable degree of activity,
which was observed exclusively among IC2 isolates. Given the ongoing expansion
of IC2 <i>A. baumannii</i> isolates, this finding may indicate a potential for
further increase in SXT activity. It should be noted that IC2 isolates
exhibited higher resistance rates to several antimicrobials, except SXT, and
this might have contributed to its predominance in Greek hospitals. The
clinical efficacy of SXT against <i>A. baumannii</i> infections warrants active
investigation.<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO29 - Faecal Carriage
Of ESBL Producing Bacteria Among Children In Dar Es Salaam, Tanzania<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Marit Gjerde Tellevik<sup>1</sup>; Sabrina J. Moyo<sup>2</sup>; Samuel Y.
Maselle<sup>3</sup>; Bjørn Blomberg<sup>1</sup>; Nina Langeland<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Haukeland University Hospital,<sup>2</sup> University
of Bergen,<sup>3</sup> Muhimbili University of Health and Allied Sciences<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Keywords: ESBL, Tanzania, faecal carriage</span><i style='mso-bidi-font-style:
normal'><span style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></i></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></i></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Background:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> Faecal carriage of Extended-spectrum
beta-lactamase (ESBL)-producing bacteria is a potential risk for transmission
and infection, and of particularly concern in healthcare settings, especially
in developing countries where infection control is limited. Infections caused
by resistant organisms pose an important challenge for treatment of both common
and life-threatening infections. Little is known about faecal carriage of
antibiotic resistance in Tanzania. This study aimed to investigate the
prevalence, etiology and risk factors of faecal carriage of ESBL-type
antibiotic resistance among young children in Tanzania.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Materials/Methods:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> This project was part of a larger study
assessing causes of diarrhoea among children. Study population was children in
Dar es Salaam &lt; 2 years of age, recruited in the period August 2010 to July
2011. Cases were children admitted due to diarrhoea at three study hospitals.
Controls were children with no history of diarrhoea for one month prior to the
study enrollment. Stool specimens were collected on inclusion. Samples were
screened for ESBL-producing bacteria using ChromID ESBL agar (ESBL<sub>A</sub>
and ESBL<sub>M</sub>) and ChromID CARBA SMART (ESBL<sub>CARBA</sub>)
(BioMérieux, Marcy l’Etoile, France). Isolates were identified with MALDI-TOF
MS using the Microflex LT instrument and MALDI Biotyper 3.1 software (Bruker
Daltonics, Bremen, Germany). Confirmation of ESBL phenotype was performed using
the BBL Sensi-Disc ESBL Confirmatory Test Disks (Becton Dickinson, Sparks, MD,
USA). Molecular methods will be applied for further characterization.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> 12% (30/250) of the controls and 48.4%
(120/248) of the cases were ESBL screening-positive. The prevalence of
ESBL-producing isolates was significantly higher in children aged &gt;12 months
compared to children &lt;12 months, 39.2% vs. 18.1% (P &lt; 0.001; OR = 2.91;
95% CI: 1.91 – 4.44). HIV-positive children had higher risk of ESBL carriage
compared to HIV-negative children, 89.3% vs. 17.1% (P &lt; 0.001; OR = 40.43;
95% CI: 11.78 – 138.72). Use of antibiotics was a risk factor for faecal
carriage of ESBL in the total study population (P &lt; 0.001; OR = 2.65; 95%
CI: 1.78 – 3.93), but when analyzing cases and controls separately, use of
antibiotics was not associated with ESBL carriage. The majority of isolates
were identified as <i>Klebsiella. pneumoniae</i> or <i>Escherichia coli</i>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusion:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> We report high prevalence of faecal carriage of
ESBL-positive bacteria on hospital admission among young children in Tanzania.
These children may pose a risk of transmission in healthcare settings. Age &gt;
12 months and HIV infection increased the risk of carrying ESBL in this study
population.Information on use of antibiotics was only collected for the last 2
weeks prior to study enrollment.Colonization with ESBL-producing bacteria can
last for several weeks. Hence, the collected data on antibiotic use are
possibly not comprehensive enough to evaluate the impact of prior use of
antibiotics in this study. ESBL<sub>CARBA</sub> phenotype was not prevalent in
this study.<o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO30 - Establishing A
Closed Reference Sequence Of A Vancomycin-resistance Plasmid From <i
style='mso-bidi-font-style:normal'>Enterococcus faecium</i><o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Heidi Gumpert<sup>1</sup>; Mette Pinholt<sup>1</sup>; Peder Worning<sup>1</sup>;
Jesper Boye Nielsen<sup>1</sup>; Henrik Westh<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Hvidovre University Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The increase
in hospital-associated vancomycin-resistant <i>Enterococcus faecium </i>(VREfm)
isolates is a worrisome trend. Indeed, in Copenhagen a ten-fold increase in the
number of <i>vanA </i>VREfm isolates was observed between 2012 and 2014. All
isolates are whole genome sequenced and genomic epidemiology reveals the dissemination
of the<i> vanA</i> Tn<i>1546 </i>transposon amongst a polyclonal VREfm
population (see M. Pinholt). However, the plasmid that is believed to have
disseminated the Tn<i>1546 </i>transposon has proven to be difficult to
assemble. Performing a hybrid assembly using high-quality Illumina reads
coupled with long PacBio reads can overcome the difficulty in assembling <i>E.
faecium </i>isolates due to their propensity to acquire mobile genetic
elements, resulting in high-quality assemblies including closed plasmids. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>One VREfm
isolate belonging to the largest clonal group (ST80) of <i>E. faecium</i>
isolates from Copenhagen between 2012 and 2014 was selected for whole genome
sequencing on Illumina MiSeq and Pacific Biosciences PacBio sequencing
platforms.&nbsp; Illumina sequencing produced 150 base-pair (bp) paired-end
(PE) reads and PacBio sequencing produced an average sub-read length of 5.8
kbp. Coverage of 30X and 441X was obtained from Illumina and PacBio sequencing,
respectively, based on an&nbsp;<i>E. faecium </i>genome size of 3.2MB. A hybrid
assembly using both sequencing platforms was performed using SPAdes, and
annotated using RAST. Plasmid replicons and resistance genes were identified
via PlasmidFinder and ResFinder online tools, respectively.&nbsp; Closed
plasmids were identified by overlapping contig ends and enumerating aligned PE
reads that bridged the plasmid contig ends. Specifically, PE reads were aligned
using Bowtie2, while bridging reads were determined using a developed BioPython
script with the PySam library. Additionally, BLAST and the NCBI database was
used to find plasmids sharing homology.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Incorporating
the PacBio reads into the assembly reduced the number of contigs from 203 to 34
and increased the N50 nearly 10-fold from 45,373 to 424,944, when compared to
assembly using PE reads alone.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>A total of 5
plasmid replicons were detected in the VREfm assembly using PlasmidFinder, with
only a rep2 replicon not on a closed plasmid. A further 4 closed plasmids were
identified based on difference in coverage compared to the chromosome,
identification of closed circular sequences, replication and mobilization
genes, and homology to previously sequenced plasmids.&nbsp; Plasmid sizes range
from 4.5kbp to 183kbp.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>A closed 32kbp
plasmid bearing a rep17 replicon was found to harbour the Tn<i>1546 vanA </i>transposon,
in addition to genes involved in macrolide (<i>ermB), </i>and aminoglycoside (<i>aph-3,
sat4, ant-6</i>) resistance. This plasmid is highly similar to the multi-drug
resistance pS177 plasmid, except that it does not contain the Axe-Txe toxin-antitoxin
system. Interestingly, a 61kbp plasmid was found to be highly similar to the Tn<i>1546
</i>harbouring pZB18 plasmid, except that Tn<i>1546 </i>is not harboured on
this plasmid.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The
inclusion of PacBio sequencing reads greatly improved the quality of the
assembly of the VREfm isolate resulting in closed plasmid sequences for 8/9
identified plasmids. The ability to reduce the vancomycin-resistance plasmid
from 11 contigs into one closed plasmid will prove to be a valuable reference
for further study of the dissemination of this resistance plasmid amongst
polyclonal isolates from the Copenhagen region.</span><i style='mso-bidi-font-style:
normal'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p></o:p></span></i></p>

<i style='mso-bidi-font-style:normal'><span style='font-size:10.0pt;mso-bidi-font-size:
11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:Cambria;
mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></i>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><i style='mso-bidi-font-style:normal'><span style='mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></i></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO31 - Ice Carrying
Antimicrobial Resistance Genes Among Pig Isolates Of <i style='mso-bidi-font-style:
normal'>Streptococcus suis</i> In Poland<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Ewa Sadowy<sup>1</sup>; Agnieszka Bojarska<sup>1</sup>; Zygmunt Pejsak<sup>2</sup>;
Waleria Hryniewicz<sup>3</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Department of Molecular Microbiology, National
Medicines Institute,<sup>2</sup> Department of Swine Diseases, National
Veterinary Research Institute,<sup>3</sup> Department of Epidemiology and
Clinical Microbiology, National Medicines Institute<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Streptococcus
suis </span></i><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>represents an important factor of morbidity and mortality in pigs, and an
emerging zoonosis in humans. The aim of this work was to perform molecular and
microbiological analyses of <i>S. suis</i> isolated from pigs in Poland and to
characterize integrative and conjugative elements (ICEs) involved in
acquisition of antimicrobial resistance.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials &amp;
methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>The studied group comprised 96 <i>S. suis </i>isolates (42 from brain and
54 from lungs), collected during 2003-2012. Antimicrobial susceptibility of
isolates was evaluated by microdilution method, their clonality by multilocus
sequence typing (MLST) and serotypes were determined using available PCR-based
schemes. ICEs in selected six isolates were analyzed by the whole genome sequencing
(WGS).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Among 96 <i>S.
suis</i> isolates, 14 serotypes were observed, with most common serotype 2 (29
isolates); 6 isolates could not be serotyped using the current PCR-based
serotyping schemes. MLST revealed 52 different sequence types (STs), with 39 novel
STs; the most common ST1 (13 isolates) was characteristic for isolates from
brain. Seven isolates (7.6%) showed intermediate penicillin nonsusceptibility
(MIC, 0.25-2.0 mg/L); 54%, 62%, 35%, 43%, 84% of isolates were resistant to
erythromycin, lincomycin, tiamulin, streptomycin, tetracycline, respectively;
one isolate was resistant to gentamicin and full susceptibility to florfenicol,
linezolid, and vancomycin was observed. Major determinant of tetracycline
resistance was <i>tet(O)</i> (55 isolates), followed by <i>tet(M)</i> and <i>tet(W)</i>
(17 and 11 isolates, respectively; <i>tetS, tetQ, tetT, tetL, tetK, tet40 </i>were
not found. Macrolide-resistance was mediated by <i>erm(B)</i> and <i>mef(A)</i>
genes (61 and 5 isolates, respectively). All streptomycin-resistant isolates
carried <i>aad(E)</i>. Among six isolates analyzed by WGS, the acquisition of
resistance determinants was due to the presence of ~50 - ~70 kb ICEs in various
genomic localizations and showing different structures, especially in their
central part, containing “cargo” antimicrobial resistance genes. Genomic
localization and structures of these ICEs, and their comparisons with other
streptococcal MGEs will be presented.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Pig isolates
of <i>S. suis </i>are remarkably diverse in the respect of their STs, serotypes
and antimicrobial resistance profiles. ICEs play a key role in the acquisition
of resistance determinants under the selective pressure of high use of
antimicrobials in pig farms, leading to formation of a significant reservoir of
resistance genes.</span><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO32 - Investigations
On Bacterial Fitness Of <i style='mso-bidi-font-style:normal'>Escherichia coli</i>
With Plasmid</span></b><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'> <b style='mso-bidi-font-weight:normal'>Mediated Beta-lactam Resistance<o:p></o:p></b></span></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Michael Pietsch<sup>1</sup>; Yvonne Pfeifer<sup>1</sup>; Guido Werner<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Robert Koch Institute, FG 13 Nosocomial Pathogens and
Antibiotic Resistance<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>An important
factor for the dissemination of antibiotic resistance are the fitness costs.
Conjugative plasmids that carry antibiotic resistance genes are beneficial for
the bacterial host, but may cause fitness costs in absence of antibiotic
selective pressure. Successful plasmids, however, often show low fitness costs
or even beneficial fitness effects caused by better adaption to the host. In
this study we investigated the adaption mechanisms of different conjugative
plasmids <i>in vitro</i> onto an isogenic <i>Escherichia coli</i> strain.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Material/Methods:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Plasmids
carrying different genes encoding carbapenemases, AmpC or ESBL were transferred
by broth mate conjugation experiments into an isogenic strain, <i>E. coli</i>
K12J53. Using S1 nuclease-restriction followed by pulsed field gel
electrophoresis (PFGE), transconjugants harbouring only one plasmid were
selected and plasmid sizes and replicon types were determined. Furthermore, the
antimicrobial susceptibilities were tested. To characterise fitness loss or
gain through plasmid acquisition, growth comparisons of plasmid-carrying <i>E.
coli</i> K12J53 vs. plasmid-free <i>E. coli</i> K12J53 were conducted including
parallel Bioscreen® assays in rich medium (lysogeny broth (LB)), and artificial
urine medium (AUM). Six transconjugants (carrying: <i>bla</i><sub>CMY-2 </sub>(n=2),
<i>bla</i><sub>CTX-M-15</sub>, <i>bla</i><sub>CTX-M-14</sub>, <i>bla</i><sub>VIM-1</sub>,
<i>bla</i><sub>OXA-48</sub>) were selected for long-term transfers (4000
generations) in LB and checked weekly for possible changes in the relative
fitness and antimicrobial susceptibility.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: For the
growth comparison experiments we selected 14 transconjugants carrying one of
the following resistance genes on a plasmid: <i>bla</i><sub>SHV-12</sub>, <i>bla</i><sub>CTX-M-1</sub>,
<i>bla</i><sub>CTX-M-14,</sub> <i>bla</i><sub>CTX-M-15 </sub>(n=2), <i>bla</i><sub>CMY-2
</sub>(n=3), <i>bla</i><sub>KPC-2</sub>, <i>bla</i><sub>VIM-1 </sub>(n=2), <i>bla</i><sub>NDM</sub>,
<i>bla</i><sub>OXA-48</sub> (n=2). The plasmid sizes varied between 60 -
260&nbsp;kb and plasmid incompatibility groups IncF, IncA/C, IncL/M, IncN, IncR
and IncI1 could be identified. Investigation of relative fitness in rich medium
showed no significant fitness loss in the majority of the isolates. Analyses
revealed a fitness loss for three isolates (isolate 1: <i>bla</i><sub>OXA-48</sub>,
IncL/M, 60&nbsp;kb; isolate 2: <i>bla</i><sub>CMY-2</sub> + <i>bla</i><sub>TEM-1</sub>,
IncA/C, 150&nbsp;kb; isolate 3: <i>bla</i><sub>VIM-1</sub>, IncN/R,
100&nbsp;kb), and for one isolate a fitness gain (isolate 4: <i>bla</i><sub>CMY-2</sub>
+ <i>bla</i><sub>TEM-1</sub>, IncA/C, 150&nbsp;kb) comparing to the
plasmid-free <i>E. coli</i> K12J53. Comparative results were observed in AUM:
Most of the isolates showed no significant change in growth rate and the above
named three isolates indicated a fitness loss. However, no fitness gain could
be observed.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: A spectrum
of <i>E.&nbsp;coli</i> with various plasmid mediated beta-lactam resistance
genes was created to investigate genetic adaptive changes to acquired plasmids.
The results of our growth experiments are in accordance with previous studies,
reporting the fitness cost of resistance caused by the acquisition of
conjugative plasmids as highly variable. This may result in fitter plasmid
harbouring hosts than their plasmid-free ancestors (isolate 4). Long-term
growth experiments and competitive growth experiments are initiated at the
moment to evaluate the observed fitness costs and investigate compensation of
initial fitness costs over time in detail.</span><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO33 - A GI Identified
In Clinical And Environmental <i style='mso-bidi-font-style:normal'>Mycobacterium
avium</i> subsp.<i style='mso-bidi-font-style:normal'> hominissuis</i> Isolates
From Germany<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Andrea Sanchini<sup>1</sup>; Torsten Semmler<sup>1</sup>; Lei Mao<sup>1</sup>;
Astrid Lewin<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Robert Koch Institute<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Among more
than 150 species belonging to the group of non-tuberculous mycobacteria, <i>Mycobacterium
avium</i> subsp. <i>hominissuis</i> (MAH) belongs to the clinically relevant
species. MAH is an opportunistic pathogen and also widespread in the
environment. Genomic islands (GI)s contribute to the accessory genome of
microorganisms. GIs influence virulence, drug-resistance or fitness and trigger
bacterial evolution. Research on GIs has only been started in MAH. We
previously identified a GI in four MAH genomes. In extension to our previous
study, here we further characterized this GI GIin a collection of MAH isolates
from Germany. We aimed to explore the MAH genomic diversity and its potential
impact on pathogenicity.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>We
investigated the whole genome sequencing data of 41 MAH isolates from clinical
(n=20) and environmental sources (n=21), including the two reference strains <i>M.
avium</i> 104 and MAH TH135. We identified the GI in 39/41 isolates (95.1 %).
All GIs integrated in the same genomic locus that we termed as insertion
hotspot. We found high diversity in the genetic structure of the GI: a total of
eight highly divergent types of GI have been identified, designated A-H (sizes
6.2 - 73.3 kb). In the insertion hotspot, these GIs arranged in single GI
(23/41, 56.1 %), combination of two different GIs (14/41, 34.1 %) or three
different GIs (2/41, 4.9 %). Remarkably two GI types shared more than 80% DNA
sequence identity with the highly pathogenic <i>M. canettii</i>, responsible
for Tuberculosis. We identified 253 different genes in all GIs, among which the
previously documented virulence-related genes <i>mmpL</i> and <i>mce</i>. We
performed homology protein modeling on 47 proteins of the GIs annotated as
hypothetical protein, in order to decipher their putative function. Five
proteins were predicted to have a function in drug resistance.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Our study
expands the knowledge on MAH genome plasticity. The diversity of the GIs in the
insertion hotspot and the sequence similarities with other mycobacteria
suggests cross-species transfer, involving also highly pathogenic species. The
presence of virulence-related genes on the GIs and their shuffling may generate
new species that can cause new outbreaks of public health relevance.</span><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO34 - Magnitude Of
Gene Mutations Conferring Drug Resistant In <i style='mso-bidi-font-style:normal'>Mycobacterium
tuberculosis</i> Strains In Southwest Ethiopia<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Mulualem Tadesse<sup>1</sup>; Dossegnaw Aragaw<sup>1</sup>; Gemeda Abebe<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Jimma University</span></i><i style='mso-bidi-font-style:
normal'><span lang=EN-GB style='mso-bidi-font-family:Arial;color:black;
mso-themecolor:text1;mso-ansi-language:EN-GB'><o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT;mso-bidi-font-weight:bold'>Keywords:&nbsp;</span><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>drug resistance, gene mutation, <i>Mycobacterium
tuberculosis</i></span><span lang=EN-GB style='mso-bidi-font-family:Arial;
color:black;mso-themecolor:text1;mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-bidi-font-family:Arial;
color:black;mso-themecolor:text1;mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>:&nbsp;The
nature and frequency of mutations in rifampicin (RIF) and isoniazid (INH)
resistance <i>M. tuberculosis</i> isolates vary considerably according to the
geographic locations. However, information regarding specific mutational
patterns in Ethiopia remains limited. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: Mutations
associated with RIF and INH resistance were studied by GenoType MTBDRplus line
probe assay in 112 <i>M. tuberculosis</i> isolates. Culture (MGIT960) and
identification tests were performed at Mycobacteriology Research Center of
Jimma University, Ethiopia. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: Mutations
conferring resistance to INH, RIF and MDR were detected in 36.6% (41/112),
30.4% (34/112) and 27.7% (31/112) of <i>M. tuberculosis</i> isolates
respectively.The retreatment category ‘treatment failure’ is associated with a
high rate of mutations associated with drug resistance (p-value &lt;0.05).
Among 34 rifampicin resistant isolates, 82.4% (28/34) had <i>rpoB</i> gene
mutations at codon 531, 2.9% (1/34) at codon 526 and 5 had mutations only at
wild type probes. The later isolates were depicted as unknown. Of 41 INH
resistant strains, 87.8% (36/41) had mutations in the <i>katG</i> gene at
Ser315Thr1 and 9.8% (4/41) of strains had mutation in the <i>inhA</i> gene at
C15T. One INH resistant strain had mutation only at <i>KatG</i> wild type
probe. Mutations in <i>inhA </i>promoter region were strongly associated with
INH monoresistance. Monoresistance to INH (10 isolates) was frequently observed
as compared to RIF monoresistance (3 isolates). <o:p></o:p></span></p>

<p class=MsoNormal><b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Conclusions</span></b><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: High rate of
drug resistance, including MDR, was commonly observed among failure cases. The
most frequent gene mutations associated with the resistance to INH and RIF were
observed in the codon 315 of the <i>katG</i> gene and codon 531 of the <i>rpoB</i>
gene, respectively. Further studies on mutations in different geographic
regions using DNA sequencing techniques are warranted to improve the kit by
including more specific mutation probes in the kits.</span><span lang=EN-GB
style='mso-bidi-font-family:Arial;color:black;mso-themecolor:text1;mso-ansi-language:
EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=DE-CH><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
