<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta name=Title content="">
<meta name=Keywords content="">
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List href="PO_S4_files/filelist.xml">
<!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Bruno Gonçalves</o:Author>
  <o:LastAuthor>Bruno Gonçalves</o:LastAuthor>
  <o:Revision>1</o:Revision>
  <o:TotalTime>0</o:TotalTime>
  <o:Created>2016-03-08T21:51:00Z</o:Created>
  <o:LastSaved>2016-03-08T21:51:00Z</o:LastSaved>
  <o:Pages>5</o:Pages>
  <o:Words>2716</o:Words>
  <o:Characters>15487</o:Characters>
  <o:Company>IMM</o:Company>
  <o:Lines>129</o:Lines>
  <o:Paragraphs>36</o:Paragraphs>
  <o:CharactersWithSpaces>18167</o:CharactersWithSpaces>
  <o:Version>14.0</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="PO_S4_files/themedata.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:TrackMoves/>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>JA</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
   <w:UseFELayout/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="276">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1073743103 0 0 415 0;}
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"ＭＳ 明朝";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
@page WordSection1
	{size:595.0pt 842.0pt;
	margin:72.0pt 89.85pt 72.0pt 89.85pt;
	mso-header-margin:35.45pt;
	mso-footer-margin:35.45pt;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;}
</style>
<![endif]-->
</head>

<body bgcolor=white lang=EN-US style='tab-interval:36.0pt'>

<div class=WordSection1>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO46 - Clonal Dynamics
Of Diverse <i style='mso-bidi-font-style:normal'>Salmonella enterica</i>
Serotypes From Traded Food Products In Portugal<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Joana Campos<sup>1</sup>; Joana Mourão<sup>1</sup>; Leonor Silveira<sup>2</sup>;
Jorge Machado<sup>2</sup>; Carla Novais<sup>1</sup>; Luísa Peixe<sup>1</sup>;
Patrícia Antunes<sup>3</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>UCIBIO/REQUIMTE. Laboratório de Microbiologia,
Faculdade de Farmácia, Universidade do Porto,<sup>2</sup> Departamento de
Doenças Infeciosas, Instituto Nacional de Saúde Dr. Ricardo Jorge, Lisboa,<sup>3</sup>
UCIBIO/REQUIMTE. Laboratório de Microbiologia, Faculdade de Farmácia,
Universidade do Porto,<sup>4</sup> Faculdade de Ciências da Nutrição e
Alimentação, Universidade do Porto<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-ansi-language:PT;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-font-weight:bold'>Keywords:<b> </b></span><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Salmonella</span></i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> clones,
antimicrobial resistance, metals.</span><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>
Non-typhoidal <i>Salmonella</i> infections represents a global public health
burden, being the identification and tracking of emerging serotypes/clones in
the food chain crucial to contain their spread. In this study we assessed the
serotype/clonal distribution trends and their association with antimicrobial
resistance in <i>Salmonella</i> isolates from foodstuffs traded in Portugal.
Results were also compared with previous data from human clinical isolates
(2009-2015).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials/Methods:
</span></b><i><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Salmonella </span></i><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>isolates (n=250; 13 serotypes), resulting from
food safety and quality control, from diverse national/imported food sources
(pork/Pk-191, poultry/Py-42, clam-10, beef/Bf-6, cooked food/Cf-1) recovered in
different regions of Portugal (2014-2015) were studied. In these isolates,
sulfametoxazole resistance (<i>sul1</i>/<i>sul2</i>/<i>sul3</i>) and other
genes linked with class 1 integrons (<i>intI1</i>/antibiotic resistance gene
cassettes) were screened by PCR. Then, in representative isolates detection of
other antibiotic resistance, as well as metals (copper-<i>pcoD</i>,
copper/silver-<i>silA</i>) tolerance genes by PCR, susceptibility tests to 12
antibiotics (ampicillin-A; chloramphenicol-C; ciprofloxacin; gentamicin-G;
kanamycin; meropenem; nalidixic acid-Na; pefloxacin-P; streptomycin-S;
sulfametoxazole-Su; tetracycline-T; trimethoprim-Tr) and beta-lactamase
production by disk diffusion (amoxicillin-clavulanic acid; cefepime;
ceftazidime-Cz; cefotaxime-Cx; Cefoxitin-Fx) (CLSI/EUCAST) and clonal
relatedness (PFGE/MLST) were performed.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: In <i>S</i>.
4,[5],12:i:- we detected the two major clones circulating in Europe, with the
predominance of multidrug-resistant (MDR) “European clone” (86%-85/99;
Pk+Py+Bf+Cf; <i>sul2</i>±<i>intl1</i>/<i>pcoD+silA</i>; mostly ASSuT) with
PFGE-types similar to those emerging in clinical sources. The frequency of the
“Spanish clone” (<i>intl1</i>+<i>sul1</i>+<i>sul2</i>+<i>sul3</i>/<i>silA</i>;
ACSSuTTr)<i> </i>is decreasing (5%-5/99; Pk+Py+Bf) comparing to previous years,
as occurring in human cases. In <i>S. </i>Typhimurium, &quot;DT104&quot;
(32%-20/62; Pk+Py; <i>intl1</i>+5´CS-<i>aadA2</i>+5´CS-<i>bla</i><sub>PSE-1</sub>+<i>sul1</i>)
and “OXA-30-producing” (15%-9/62; Pk; <i>intl1</i>+5´CS-<i>bla</i><sub>OXA-30</sub>+<i>sul1</i>)
clones were the most frequent and presented identical ACSSuT and PFGE profiles
to those described in humans. “<i>S</i>. Typhimurium European clone” (8%-5/62;
Pk; <i>sul2</i>/<i>pcoD+silA</i>), which is emerging in clinical sources,
presented similar MDR (mostly ASSuT) and related PFGE-types to <i>S</i>.4,[5],12:i:-
“European clone”. In <i>S. </i>Rissen stands out the maintenance of the MDR
clone frequently associated with class 1 integrons (36%-11/31; Pk;<i>
intl1+sul1</i>±<i>sul3</i>/<i>pcoD+silA</i>). Noteworthy, a
MDR-ACzCxFx(K)PNaSuT <i>S. </i>Heidelberg clone/ST15 (80%-4/5; imported Py
gizzards) CMY-2-producing was detected.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>In the
analysed <i>Salmonella</i> collection we detected in different food sources the
persistence of serotypes/clones frequently associated with human infections, contributing
for the high rates of MDR. Noteworthy is the expansion of MDR <i>S. </i>4,[5],12:i:-
“European clone” and <i>S</i>. Rissen clone from pork foods enriched with
copper/silver tolerance genes and the detection of MDR S. Heidelberg-CMY-2
producing clone from imported poultry. The acquisition of antimicrobial
resistance genes (antibiotics/metals) seems to have a relevant role for the
success of these clones. However, understanding all the adaptive factors
promoting their survival/persistence is crucial for implementing effective
strategies in food safety at global level. <o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO47 - Antibiotic
Resistance And Molecular Characterization Of <i style='mso-bidi-font-style:
normal'>Staphylococcus pseudintermedius</i> Strains Isolated From Companion
Animals In Russia<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>A. Balbutskaya<sup>1</sup>; O. Dmitrenko<sup>2</sup>; V. Skvortsov<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Belgorod department of «All-Russian scientific
research institute of experimental veterinary medicine named after
Ya.R.Kovalenko»,<sup>2</sup> Federal State Institution «N.F. Gamaleya Federal
Scientific Center of Epidemiology and Microbiology» Ministry of</span></i><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'> Health RF<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>
Coagulase-positive <i>Staphylococcus pseudintermedius </i>has emerged as a
cause of wide range infections in many kinds of animal species, predominantly
in dogs and cats. There is evidence of transmission of this pathogen between
companion animals and their owners. <i>S. pseudintermedius </i>has been also
identi</span><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT'>ﬁ</span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>ed as a causative agent of severe bacterial infections in humans.
Acquisition of staphylococcal cassette chromosome (SCC) <i>mec</i> elements and
various resistance genes by several clonal lineages of <i>S. pseudintermedius </i>and
rapid spread of multidrug resistant methicillin-resistant (MDR MRSP) clones in
many countries leaded to treatment challenges to veterinary medicine and public
health issue. This study was performed to evaluate prevalence of MRSP and MDRSP
among companion animals in the Central region of Russia and to characterize
colonizing and infecting strains. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials/Methods:
</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Oral, nasal and skin swabs samples were taken from 53 healthy dogs. 87 dogs
and 29 cats with various pyoinflammatory diseases were also sampled. A total of
30 <i>S. pseudintermedius </i>isolates were obtained from healthy dogs and 57
isolates from diseased animals were collected. The identity of the isolates was
confirmed by multiplex PCR as described Sasaki T. et al. (2010) and
amplification and partial sequencing of <i>kat</i> gene with primers designed
by BlaiottaG. etal. (2010). Susceptibility to 8 antimicrobials was determined
by disc testing. The molecular characteristic of the isolates was carried out
by PCRs to determine <i>mecA</i> gene, SCC<i>mec</i> types and sequencing <i>spa</i>
according Kondo Y. et al. (2007) and Moodley A. et al. (2009), respectively.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Frequency of
resistance to antimicrobials was as follow (healthy/diseased animals (%)):
clindamycin (0/38,6), erythromycin (0/38,6), doxycycline (10/29,8),
chloramphenicol (0/21), gentamycin (0/21), oxacillin (0/8,8), ciprofloxacin,
(0/3,5) and fusidic acid (0/0). Overall, 23% of <i>S. pseudintermedius </i>isolates
were MDR. The molecular characterization has shown that 5 isolates were <i>mecA</i>
positive. A total of 8 different <i>spa</i> types were found in 14 <i>S.
pseudintermedius </i>isolated from diseased dogs and 1 isolate from healthy
dog. 4 MRSP isolates belonged to SCC<i>mec</i> type IV and presented the
additional ccr1 gene complex, 2 of them had <i>spa</i> type t03 and t05. The
remaining isolate belonged to SCC<i>mec</i> type III and had <i>spa</i> type
t02. The combination of SCC<i>mec</i> type III and <i>spa</i> t02 is the most
common for European epidemic MRSP clone ST71. Methicillin-susceptible <i>S.
pseudintermedius </i>isolates were identified harboring the most frequent <i>spa</i>
type t71 (n=7) and single <i>spa</i> types t02, t05, t09, t15, t17.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>This study
shows the high prevalence (57%) of <i>S. pseudintermedius</i> carriage in
healthy dogs in Russia. The pathogen caused disease in 10% of examined cats and
63% of dogs. MRSP was detected from </span><span lang=DE-CH style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-fareast-language:PT'>ﬁ</span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>ve (5,7%) diseased
dogs. All of MRSP were MDR. To our knowledge, this is the first report of
canine infections caused by MRSP in Russia, suggesting that MRSP ST71 is
involved. <i>Spa </i>gene was not amplified in 83% of <i>S. pseudintermedius</i>,
suggesting rearrangements in the <i>spa</i> or lacking of <i>spa</i> in this
isolates. Additional studies are needed to modify <i>spa</i>-typing protocol to
amplify non-typeable samples of our population. Molecular typing is required to
determinate circulation of epidemic clones in different geographic areas.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO48 - Isolation Of <i
style='mso-bidi-font-style:normal'>Arcobacter</i> spp. </span></b><b
style='mso-bidi-font-weight:normal'><span lang=PT style='mso-ansi-language:
PT;mso-fareast-language:PT'>Quinolone Resistant From Fresh Vegetables<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Isidro Favian Bayas Morejón<sup>1</sup>; Ana González Pellicer<sup>1</sup>;
María Antonia Ferrús Pérez<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>Universitat Politècnica de València,<sup>2</sup>
mferrus@btc.upv.es<o:p></o:p></span></i></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span lang=PT
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:PT;
mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Arcobacter </span></i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>is
considered an emergent foodborne and waterborne enteropathogen. At the present,
the genus <i>Arcobacter </i>includes 23 species, some of them isolated from
food. The species <i>A. butzleri</i>, <i>A. skirrowii</i>, <i>A. cryaerophilus</i>
and <i>A. cibarius</i> have been directly associated with animal and human
infections. Besides, <i>A. butzleri</i> and <i>A. cryaerophilus</i> are the
most common species detected in food, and the number of resistant strains to
quinolones is increasing, which represents an enormous challenge for public
health. Nowadays the consumption of vegetables has grown because they are
considered healthy food but the presence of <i>Arcobacter</i> in them is little
known. All these factors motivated us to study the presence of <i>Arcobacter</i>
in vegetables and the susceptibility to quinolones of the obtained isolates.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials/Methods:
</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>One-hundred vegetable samples were studied: 41 lettuces, 34 chard, 21
spinach and 4 cabbage. Selective enrichment of the samples was carried out
using<i>Arcobacter</i> Broth supplement with
Cephoperazone-AmphotericinB-Teicoplanin. After 48h incubation at 37ºC under
microaerophilic conditions, 100µl of the enrichment broth were placed onto the
surface of <i>Arcobacter</i> agar plates supplemented with 5% sheep blood using
a membrane filter. The plates were incubated at 30ºC in aerobic conditions,
finally the filters were removed and the plates were incubated at 37ºC under
microaerophilic conditions. Presumptive arcobacters were identified by specific
genus PCR that amplifies a 331-bp fragment of 23S rRNA gene. Species
identification was performed using the 16S rDNA-RFLP method described by
Figueras et al. (2012). <i>Arcobacter</i> susceptibility to ciprofloxacin and
levofloxacin were determined with the disc diffusion and E-test strips methods.
The diameter of the inhibition zones in discs and the MICs were determined.
Additionally, to clarify the mechanism of quinolone resistance, we analysed the
region QRDR of <i>gyrA</i> gene because mutations in the <i>gyrA </i>gene of
Gram-negative bacteria results in resistance to fluoroquinolones. A 344-bp
fragment of this region was amplified, purified and then sequenced by
IBMCP-UPV.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Arcobacter </span></i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>spp. were
isolated in 17 samples over the 100 vegetables samples. They were most detected
in spinaches (8 of 21 samples) followed by chard (4 of 34), lettuces (4 of 41),
and cabbages (1 of 4). A total of 25 isolates were obtained, 13 from spinach, 6
chard, 5 lettuces and 1 cabbage. By PCR, 17 of the isolates were identified as <i>A.
butzleri and </i>8 as <i>A. cryaerophilus. </i>All the <i>A. cryaerophilus</i>
isolates and 15 <i>A. butzleri</i> isolates were susceptible to quinolones.
However, two <i>A. butzleri</i> isolates showed antimicrobial resistance by
disc diffusion method and E-test strips. The sequencing of the PCR product
revealed the presence of a mutation in position 254 of <i>gyrA</i> gene (C-T
transition) in the resistant isolates, while the sensitive isolates and the
reference strain &nbsp;<i>A. butzleri</i>DSM 8739 showed no mutation.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The results
of this study indicate high levels of <i>Arcobacter</i> contamination in
vegetables. In addition, two of the isolates of <i>A. butzleri</i> were
resistant to quinolones. This could pose a risk to public health because they
are generally eaten raw.</span><b style='mso-bidi-font-weight:normal'><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></b></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO49 - Study Of Population
Diversity Of <i style='mso-bidi-font-style:normal'>Toxoplasma gondii</i> Human
Strains In Portugal<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Anabela Vilares<sup>1</sup>; Idalina Ferreira<sup>1</sup>; Susana Martins<sup>1</sup>;
Maria João Gargaté<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>National Institute of Health Dr Ricardo Jorge,
Department of Infectious Diseases, Reference Unit for Parasitic and Fungal
Infections<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Toxoplasma
gondii</span></i><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'> is an obligate intracellular protozoan parasite, which is responsible for
toxoplasmosis in different species, including humans. It has been estimated
that one third of the world population has been infected. In most human adults
it does not cause serious illness, however, blindness and mental retardation
can be caused in congenitally infected children and severe diseases or death in
those with compromised immunity.<i>T. gondii </i>have a distinct clonal
population structure composed of type I, II and III lineages in North America
and Europe. But more recent studies demonstrated high diversity, recombinant
and atypic strains (more virulent) in South America. The genotype <i>T. gondii</i>
scenario is lacking in Portugal, except for three limited evaluations, not even
the classical typing studies have been performed before in humans. In order to
achieve this knowledge we studied the genetic diversity of <i>T. gondii</i>
isolated from human products.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>We studied
47 strains from congenital toxoplasmosis and one from a positive HIV with
cerebral toxoplasmosis. Most isolates were from the central region of Portugal;
however 25% were from north, 6.3% from south and 2% from Azores islands. All
samples belong to the strain collection (since 1994 until 2013) of the National
Reference Laboratory of Parasitic and Fungal Infections, Portuguese National
Institute of Health (NIH). We performed 5 microsatellites (B17, B18, TgM-A, W35
and Tub2) multiplex PCR and a combination of Sanger sequencing and Next
Generation Sequencing (NGS) of several loci likely responsible for virulence
(Sag2, CB21-4, PK1, L363, Sag1, Gra6 and Sag3).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>We
successfully sequenced the two ends of Sag2 (I, II, III) gene and this allowed
the differentiation among the three “classical” <i>T. gondii</i> strain
types.Thirty-five (73%) strains were type II and thirteen (27%) were type I.
PCR multiplex microsatellites analysis and NGS were applied to all samples and
were highly efficient. Microsatellites analysis identified ten (21%)
recombinant strains and NGS approach allowed the identification of new and
unexpected recombinant strains, also increased the extension of the diversity
in <i>T. gondii </i>strains isolated in Portugal.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>This work
showed the first evidence of recombinant strains circulating in Portugal isolated
from humans and also the first report of NGS in <i>T. gondii</i> strains
isolated in Portugal. The combination of data and the high resolution genetic
characterization, enhance our understanding of the molecular epidemiology and
genetic variation of <i>T. gondii</i>, which will be ultimately beneficial for
the control of <i>T. gondii&nbsp;</i>transmission.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO50 - Identification
Of Potential Amplicon Sequencing Targets For Identification And Strain-level
Characterization Of Shiga Toxin-producing <i style='mso-bidi-font-style:normal'>Escherichia
coli</i><o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Lori Michelle Gladney<sup>1</sup>; Daniel Fasulo<sup>2</sup>; Alexandra
Mercante<sup>1</sup>; Lisley Garcia-Toledo<sup>1</sup>; Rebecca Lindsey<sup>1</sup>;
Eija Trees<sup>1</sup>; Nancy Strockbine<sup>1</sup>; Efrain Ribot<sup>1</sup>;
Heather Carleton<sup>1</sup>; John Besser<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Centers for Disease Control and Prevention,<sup>2</sup>
Pattern Genomics</span></i><span lang=EN-GB style='mso-bidi-font-family:Arial;
color:black;mso-themecolor:text1;mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-bidi-font-family:Arial;color:black;mso-themecolor:text1;mso-ansi-language:
EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Shiga
toxin-producing <i>Escherichia coli</i> (STEC) is an important foodborne
pathogen. STEC infections have been diagnosed by culture and monitored by
public health agencies using a variety of culture-based subtyping methods.
Recently, culture-independent diagnostic tests (CIDTs) have been introduced in
many clinical labs. CIDTs are rapid and cost-effective, but do not result in an
isolate. To maintain public health surveillance activities, we are developing a
PCR-based amplicon sequencing assay that may be used directly with complex samples
such as stool. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>A total of 342
genomes (104 STEC, 201 other <i>E. coli</i>, and 37 other <i>Enterobacteriaceae</i>)
were screened to identify PCR targets that are conserved in, and unique to
O157:H7 and present in single copy using Daydreamer<sup>TM</sup> software
(Pattern Genomics).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Next, the O157:H7
Shiga toxin (<i>stx</i>) phage genome was explored to identify heterogeneous
regions flanked by stable loci that may serve as potential amplifiable
subtyping markers. A specific region of the phages was extracted from the
antiterminator (N) to the phage lysis gene (R). Sequence alignment of the
phages was performed using Mega5. Then, the alignments were trimmed to
conserved ends (repressor CI to lysis gene S) and data were concatenated for
strains that contain multiple phages. </span><span lang=DE-CH style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-fareast-language:PT'>Phylogenetic trees were created using
Mega5 and kSNP3.0. Strain differences were calculated using MuMmer 3.23.
Lastly, the whole genome phylogeny of the host strains was compared to the
phage genome phylogeny to see if their topologies were concordant.</span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>We identified a
single copy PCR target that is specific to O157:H7. The function is unknown;
however, the target resides in an iron-sulfur cluster. We designed primers for
three sets of amplicons that range from 67-91 bp. We performed <i>in</i> <i>silico</i>
PCR on 854 genomes (including 145 O157) to access the overall successfulness of
the design. The coverage of our primers was 97% while the accuracy was 100%.
Four strains failed to amplify because they did not contain the target region.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>We observed that
the phylogeny based on the Shiga toxin phage regions (12-16 kb) in the O157:H7
strains does reflect the topology of the whole genome phylogeny and contains
reasonable differences among strains (2-310 SNPs). There are also conserved
regions to place primers which flank heterogeneous regions and at reasonable
amplicon lengths. Primers can be designed so that amplicons overlap, are linked
to <i>stx</i>, and may be sequenced using next generation sequencing. In
addition, where there are multiple copy phages in a single host strain, they
appear different enough that they would be unlikely to collapse with assembly.<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Our data suggest that targets are present in the O157 phage genome that can
be directly linked to <i>stx</i> and contain conserved and heterogeneous
regions that can be exploited in the development of a culture-independent
subtyping assay. We also report a single copy O157:H7 marker that can be used
to infer mixed STEC infections and separate normal flora background through a
statistical association of O157 copy numbers with virulence and antimicrobial
resistance markers. Next, we plan to evaluate the heterogeneity in adjacent
regions to <i>stx</i> which may be used to differentiate strains. The region
with genetic differences that reflect epidemiological associations and the least
number of amplicons will be selected. Lastly, we plan to evaluate the
specificity of the conserved phage regions to all STEC and the lambdoid Shiga
toxin phages.</span><span lang=EN-GB style='mso-bidi-font-family:Arial;
color:black;mso-themecolor:text1;mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=DE-CH><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
