<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns:mv="http://macVmlSchemaUri" xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta name=Title content="">
<meta name=Keywords content="">
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List href="PO_S8_files/filelist.xml">
<link rel=Edit-Time-Data href="PO_S8_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Bruno Gonçalves</o:Author>
  <o:LastAuthor>Bruno Gonçalves</o:LastAuthor>
  <o:Revision>1</o:Revision>
  <o:TotalTime>1</o:TotalTime>
  <o:Created>2016-03-08T21:54:00Z</o:Created>
  <o:LastSaved>2016-03-08T21:55:00Z</o:LastSaved>
  <o:Pages>35</o:Pages>
  <o:Words>17425</o:Words>
  <o:Characters>99329</o:Characters>
  <o:Company>IMM</o:Company>
  <o:Lines>827</o:Lines>
  <o:Paragraphs>233</o:Paragraphs>
  <o:CharactersWithSpaces>116521</o:CharactersWithSpaces>
  <o:Version>14.0</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="PO_S8_files/themedata.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>JA</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
   <w:UseFELayout/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="276">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
@font-face
	{font-family:Arial;
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073711037 9 0 511 0;}
@font-face
	{font-family:"ＭＳ ゴシック";
	mso-font-charset:78;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1791491579 18 0 131231 0;}
@font-face
	{font-family:Verdana;
	panose-1:2 11 6 4 3 5 4 4 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-1593833729 1073750107 16 0 415 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1107305727 0 0 415 0;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-520092929 1073786111 9 0 415 0;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1073743103 0 0 415 0;}
@font-face
	{font-family:Tahoma;
	panose-1:2 11 6 4 3 5 4 4 2 4;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-520082689 -1073717157 41 0 66047 0;}
@font-face
	{font-family:MingLiU;
	mso-font-alt:細明體;
	mso-font-charset:136;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:-1610611969 684719354 22 0 1048577 0;}
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;
	mso-fareast-language:EN-US;}
h1
	{mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"Heading 1 Char";
	mso-style-next:Normal;
	margin-top:24.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:1;
	font-size:14.0pt;
	font-family:Calibri;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#365F91;
	mso-themecolor:accent1;
	mso-themeshade:191;
	mso-font-kerning:0pt;
	mso-ansi-language:DE-CH;
	mso-fareast-language:EN-US;
	font-weight:bold;}
h2
	{mso-style-noshow:yes;
	mso-style-priority:9;
	mso-style-qformat:yes;
	mso-style-link:"Heading 2 Char";
	mso-style-next:Normal;
	margin-top:2.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:2;
	font-size:13.0pt;
	font-family:Calibri;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#365F91;
	mso-themecolor:accent1;
	mso-themeshade:191;
	mso-ansi-language:DE-CH;
	mso-fareast-language:EN-US;
	font-weight:normal;}
h5
	{mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"Heading 5 Char";
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	mso-outline-level:5;
	font-size:10.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	mso-ansi-language:DE-CH;
	mso-fareast-language:DE-CH;
	font-weight:bold;}
p.MsoToc1, li.MsoToc1, div.MsoToc1
	{mso-style-update:auto;
	mso-style-priority:39;
	mso-style-next:Normal;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:5.0pt;
	margin-left:0cm;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;
	mso-fareast-language:EN-US;}
p.MsoCommentText, li.MsoCommentText, div.MsoCommentText
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Comment Text Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;
	mso-fareast-language:EN-US;}
p.MsoHeader, li.MsoHeader, div.MsoHeader
	{mso-style-priority:99;
	mso-style-link:"Header Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	tab-stops:center 216.0pt right 432.0pt;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;
	mso-fareast-language:EN-US;}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{mso-style-priority:99;
	mso-style-link:"Footer Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	tab-stops:center 216.0pt right 432.0pt;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;
	mso-fareast-language:EN-US;}
span.MsoCommentReference
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-ansi-font-size:8.0pt;
	mso-bidi-font-size:8.0pt;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	mso-themecolor:hyperlink;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:purple;
	mso-themecolor:followedhyperlink;
	text-decoration:underline;
	text-underline:single;}
p.MsoDocumentMap, li.MsoDocumentMap, div.MsoDocumentMap
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Document Map Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-ansi-language:DE-CH;
	mso-fareast-language:EN-US;}
p
	{mso-style-priority:99;
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	mso-ansi-language:EN-GB;
	mso-fareast-language:EN-GB;}
p.MsoCommentSubject, li.MsoCommentSubject, div.MsoCommentSubject
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"Comment Text";
	mso-style-link:"Comment Subject Char";
	mso-style-next:"Comment Text";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:DE-CH;
	mso-fareast-language:EN-US;
	font-weight:bold;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Balloon Text Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:8.0pt;
	font-family:Tahoma;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-ansi-language:DE-CH;
	mso-fareast-language:EN-US;}
p.MsoNoSpacing, li.MsoNoSpacing, div.MsoNoSpacing
	{mso-style-priority:1;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:left;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;
	mso-fareast-language:EN-US;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	mso-add-space:auto;
	text-align:justify;
	text-justify:inter-ideograph;
	line-height:107%;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;
	mso-fareast-language:EN-US;}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	mso-add-space:auto;
	text-align:justify;
	text-justify:inter-ideograph;
	line-height:107%;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;
	mso-fareast-language:EN-US;}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	mso-add-space:auto;
	text-align:justify;
	text-justify:inter-ideograph;
	line-height:107%;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;
	mso-fareast-language:EN-US;}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	mso-add-space:auto;
	text-align:justify;
	text-justify:inter-ideograph;
	line-height:107%;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-ansi-language:PT;
	mso-fareast-language:EN-US;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 1";
	mso-ansi-font-size:14.0pt;
	mso-bidi-font-size:14.0pt;
	font-family:Calibri;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#365F91;
	mso-themecolor:accent1;
	mso-themeshade:191;
	mso-ansi-language:DE-CH;
	font-weight:bold;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-noshow:yes;
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 2";
	mso-ansi-font-size:13.0pt;
	mso-bidi-font-size:13.0pt;
	font-family:Calibri;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#365F91;
	mso-themecolor:accent1;
	mso-themeshade:191;
	mso-ansi-language:DE-CH;}
span.Heading5Char
	{mso-style-name:"Heading 5 Char";
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 5";
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:10.0pt;
	font-family:"Times New Roman";
	mso-ascii-font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:"Times New Roman";
	mso-bidi-font-family:"Times New Roman";
	mso-ansi-language:DE-CH;
	mso-fareast-language:DE-CH;
	font-weight:bold;}
p.bodytext, li.bodytext, div.bodytext
	{mso-style-name:bodytext;
	mso-style-unhide:no;
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	mso-ansi-language:DE-CH;
	mso-fareast-language:DE-CH;}
span.CommentTextChar
	{mso-style-name:"Comment Text Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Comment Text";
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:10.0pt;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Arial;
	mso-ansi-language:DE-CH;}
span.CommentSubjectChar
	{mso-style-name:"Comment Subject Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-parent:"Comment Text Char";
	mso-style-link:"Comment Subject";
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:10.0pt;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Arial;
	mso-ansi-language:DE-CH;
	font-weight:bold;}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Balloon Text";
	mso-ansi-font-size:8.0pt;
	mso-bidi-font-size:8.0pt;
	font-family:Tahoma;
	mso-ascii-font-family:Tahoma;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Tahoma;
	mso-bidi-font-family:Tahoma;
	mso-ansi-language:DE-CH;}
span.st
	{mso-style-name:st;
	mso-style-unhide:no;}
p.western, li.western, div.western
	{mso-style-name:´´western´´;
	mso-style-unhide:no;
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	mso-ansi-language:PT;
	mso-fareast-language:PT;}
p.p1, li.p1, div.p1
	{mso-style-name:´´p1´´;
	mso-style-unhide:no;
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	mso-ansi-language:PT;
	mso-fareast-language:PT;}
span.HeaderChar
	{mso-style-name:"Header Char";
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:Header;
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Arial;
	mso-ansi-language:PT;}
span.FooterChar
	{mso-style-name:"Footer Char";
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:Footer;
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Arial;
	mso-ansi-language:PT;}
p.Affiliations, li.Affiliations, div.Affiliations
	{mso-style-name:Affiliations;
	mso-style-unhide:no;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:17.85pt;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	text-indent:-17.85pt;
	line-height:150%;
	mso-pagination:none;
	font-size:10.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Calibri;
	mso-fareast-font-family:Calibri;
	mso-bidi-font-family:"Times New Roman";
	mso-ansi-language:EN-GB;
	mso-fareast-language:EN-US;}
span.DocumentMapChar
	{mso-style-name:"Document Map Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Document Map";
	font-family:"Times New Roman";
	mso-ascii-font-family:"Times New Roman";
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:"Times New Roman";
	mso-bidi-font-family:"Times New Roman";
	mso-ansi-language:DE-CH;}
p.IMMEM, li.IMMEM, div.IMMEM
	{mso-style-name:IMMEM;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"Heading 1";
	mso-style-link:"IMMEM Caráter";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:16.0pt;
	margin-left:0cm;
	text-align:right;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:1;
	font-size:12.0pt;
	font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-ansi-language:EN-US;
	mso-fareast-language:EN-US;
	font-weight:bold;}
p.IMMEMl, li.IMMEMl, div.IMMEMl
	{mso-style-name:IMMEM_l;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:IMMEM;
	mso-style-link:"IMMEM_l Caráter";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:left;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:1;
	font-size:12.0pt;
	font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-ansi-language:EN-US;
	mso-fareast-language:EN-US;
	font-weight:bold;}
span.IMMEMCarter
	{mso-style-name:"IMMEM Caráter";
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:IMMEM;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Arial;
	mso-bidi-font-family:Arial;
	font-weight:bold;}
p.IMMEMr, li.IMMEMr, div.IMMEMr
	{mso-style-name:IMMEM_r;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:IMMEM;
	mso-style-link:"IMMEM_r Caráter";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:right;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:1;
	font-size:12.0pt;
	font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-ansi-language:EN-US;
	mso-fareast-language:EN-US;
	font-weight:bold;}
span.IMMEMlCarter
	{mso-style-name:"IMMEM_l Caráter";
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-parent:"IMMEM Caráter";
	mso-style-link:IMMEM_l;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Arial;
	mso-bidi-font-family:Arial;
	font-weight:bold;}
span.IMMEMrCarter
	{mso-style-name:"IMMEM_r Caráter";
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-parent:"IMMEM Caráter";
	mso-style-link:IMMEM_r;
	font-family:Arial;
	mso-ascii-font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Arial;
	mso-bidi-font-family:Arial;
	font-weight:bold;}
p.navegacao-link, li.navegacao-link, div.navegacao-link
	{mso-style-name:navegacao-link;
	mso-style-unhide:no;
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan;
	font-size:7.5pt;
	font-family:Verdana;
	mso-fareast-font-family:"Times New Roman";
	mso-bidi-font-family:"Times New Roman";
	color:#999999;
	mso-ansi-language:PT;
	mso-fareast-language:PT;}
p.Abstract, li.Abstract, div.Abstract
	{mso-style-name:Abstract;
	mso-style-unhide:no;
	margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	line-height:150%;
	mso-pagination:none;
	font-size:12.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:Calibri;
	mso-fareast-font-family:Calibri;
	mso-bidi-font-family:"Times New Roman";
	color:#006B8E;
	mso-ansi-language:EN-GB;
	mso-fareast-language:EN-US;}
p.IMMEMdatetime, li.IMMEMdatetime, div.IMMEMdatetime
	{mso-style-name:IMMEM_date_time;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"Heading 2";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:12.0pt;
	margin-left:0cm;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:2;
	font-size:12.0pt;
	font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	mso-ansi-language:EN-US;
	mso-fareast-language:PT;
	font-weight:bold;
	mso-bidi-font-weight:normal;
	font-style:italic;
	mso-bidi-font-style:normal;}
p.IMMEMindex, li.IMMEMindex, div.IMMEMindex
	{mso-style-name:IMMEM_index;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:IMMEM;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:16.0pt;
	margin-left:0cm;
	text-align:right;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:1;
	font-size:12.0pt;
	font-family:Arial;
	mso-fareast-font-family:"ＭＳ ゴシック";
	mso-fareast-theme-font:major-fareast;
	mso-ansi-language:EN-US;
	mso-fareast-language:EN-US;
	font-weight:bold;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"ＭＳ 明朝";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
@page WordSection1
	{size:420.0pt 595.0pt;
	margin:42.55pt 1.0cm 1.0cm 1.0cm;
	mso-header-margin:14.2pt;
	mso-footer-margin:14.2pt;
	mso-gutter-margin:14.2pt;
	mso-title-page:yes;
	mso-header:url(":PO_S8_files:header.htm") h1;
	mso-even-footer:url(":PO_S8_files:header.htm") ef1;
	mso-first-header:url(":PO_S8_files:header.htm") fh1;
	mso-first-footer:url(":PO_S8_files:header.htm") ff1;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
@page WordSection2
	{size:595.0pt 842.0pt;
	margin:72.0pt 89.85pt 72.0pt 89.85pt;
	mso-header-margin:35.45pt;
	mso-footer-margin:35.45pt;
	mso-header:url(":PO_S8_files:header.htm") h1;
	mso-even-footer:url(":PO_S8_files:header.htm") ef1;
	mso-first-header:url(":PO_S8_files:header.htm") fh1;
	mso-first-footer:url(":PO_S8_files:header.htm") ff1;
	mso-paper-source:0;}
div.WordSection2
	{page:WordSection2;}
 /* List Definitions */
@list l0
	{mso-list-id:325549294;
	mso-list-template-ids:452521116;}
@list l1
	{mso-list-id:394860103;
	mso-list-template-ids:-1813850462;}
@list l2
	{mso-list-id:460225044;
	mso-list-template-ids:1776606900;}
@list l3
	{mso-list-id:629283510;
	mso-list-template-ids:-579191296;}
@list l4
	{mso-list-id:1298030603;
	mso-list-template-ids:-1032794714;}
@list l5
	{mso-list-id:1425496888;
	mso-list-template-ids:1946585780;}
ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;}
table.MsoTableGrid
	{mso-style-name:"Table Grid";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-priority:59;
	mso-style-unhide:no;
	border:solid windowtext 1.0pt;
	mso-border-alt:solid windowtext .5pt;
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-border-insideh:.5pt solid windowtext;
	mso-border-insidev:.5pt solid windowtext;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:Cambria;
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Cambria;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:minor-latin;
	mso-ansi-language:PT;}
</style>
<![endif]-->
</head>

<body bgcolor=white lang=EN-US link=blue vlink=purple style='tab-interval:36.0pt'>

<div class=WordSection1>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO35 - Transforming
Public Health Microbiology In The United States With Whole Genome Sequencing
(WGS) - PulseNET And Beyond<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Peter Gerner-Smidt for PulseNet USA<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Centers for Disease Control and Prevention, Atlanta,
United States<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Multiple
different phenotypic and molecular methods and technologies are used in public
health laboratories (PHLs) to characterize and subtype foodborne bacterial
pathogens. This traditional strain characterization is labor and resource
intensive and has a turn-around-time (TAT) of up to several months. Most of
this information may be extracted from the genome sequence of any organism.
With the introduction of next generation sequencing technologies and advances
in bioinformatics it is now possible to characterize foodborne pathogens in a
single workflow with a TAT of ≤ four working days in a cost-efficient manner.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> The Enteric
Diseases Laboratory Branch at CDC is working with partners in federal agencies,
PHLs in the states and internationally to build applications to extract
information for strain characterization and subtyping using the gene by gene
approach (MLST) from the genome sequences. Average nucleotide identity (ANI) is
used for species identification, tools for serotyping, virulence, plasmid, and
antimicrobial resistance characterization developed by others are used, as are
established schemes for 7-gene MLST and core genome (cg)MLST. Whole genome
(wg)MLST schemes are developed as needed. The PulseNet infrastructure,
analytical platform and databases are used because of their comprehensiveness
and versatility, and because the PHLs are familiar with them after working with
them for 20 years.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> WGS has
successfully been used for national surveillance of listeriosis and in outbreak
investigations of all enteric foodborne bacterial pathogens since 2013. WGS
adds phylogenetic relevance, increased resolution and precision compared to
current methods thereby enhancing case definition and source tracking in
outbreak investigations. By the end of 2015, a WGS database and application
will be implemented for routine identification, lineage determination, 7-gene
MLST and subtyping by cg- and wgMLST of&nbsp;<i>Listeria monocytogenes</i>&nbsp;at
CDC and in states with sequencing capacity. At the beginning of 2016, similar
applications for identification of&nbsp;<i>Campylobacteraceae,</i>&nbsp;7-gene
MLST and subtyping of&nbsp;<i>C. jejuni/coli,&nbsp;</i>identification,
serotyping, virulence and antimicrobial resistance characterization of<i>&nbsp;</i>&nbsp;Shiga
toxin-producing&nbsp;<i>E. coli&nbsp;</i>(STEC) will be undergoing external
validation;&nbsp;a pilot-application for the identification, serotyping,
plasmid profiling, antimicrobial resistance characterization and MLST of&nbsp;<i>Salmonella</i>&nbsp;and
identification, toxin-gene subtyping and MLST for <i>Clostridium botulinum</i>&nbsp;
will be undergoing internal validation; databases for&nbsp;<i>Vibrio&nbsp;</i>spp.,
other diarrhoeagenic&nbsp;<i>E. coli&nbsp;</i>(enterotoxigenic&nbsp;<i>E.
coli&nbsp;</i>[ETEC], enteropathogenic&nbsp;<i>E. coli&nbsp;</i>[EPEC],
enteroinvasive&nbsp;<i>E. coli [</i>EIEC]&nbsp;including<i>&nbsp;Shigella</i>),&nbsp;<i>Yersinia</i>&nbsp;and&nbsp;<i>Cronobacter&nbsp;</i>to
follow within two years.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion:&nbsp;</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>WGS will
gradually replace and enhance current laboratory methods for foodborne
pathogens in the United States within the next three- five years.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO36 - Oxa-48-producing
<i style='mso-bidi-font-style:normal'>Enterobacteriaceae</i> In Poland,
2013-2015<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Izdebski Radosław<sup>1</sup>; Baraniak Anna<sup>1</sup>; Urbanowicz Paweł<sup>1</sup>;
Fiett Janusz<sup>1</sup>; Żabicka Dorota<sup>1</sup>; Bojarska Katarzyna<sup>1</sup>;
Literacka Elżbieta<sup>1</sup>; Hryniewicz Waleria<sup>1</sup>; Gniadkowski
Marek<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>National Medicines Institute<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Carbapenemase-producing
<i>Enterobacteriacae</i> (CPE) create a major threat for public health. Three
main types of carbapenemases are observed in CPE, namely the KPC-, MBL- and
OXA-48-like enzymes. OXA-48 was identified first in 2001 in Turkey, and with
years most of the east and south Mediterranean region has become endemic for
its producers. Another large reservoir is India where a related enzyme,
OXA-181, has been routinely identified. Extensive travelling and migrations are
at the origin of the increasing presence of organisms with OXA-48-like enzymes
in Europe, especially in France, Germany or Spain. </span><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>The dissemination of OXA-48
producers is attributed to high mobility of <i>bla</i><sub>OXA-48</sub>-like
genes due to their location in Tn<i>1999</i>-like transposons and pOXA-48-like
plasmids but also to spread of some bacterial clone variants, like <i>Klebsiella
pneumoniae</i> ST11 or ST395, or <i>Escherichia coli</i> ST38. So far Poland
has recorded a relatively small number of OXA-48 clinical cases (n=24 by
November 2015). The aim of this study was to analyze all but two cases
identified by the mid-2015.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>Materials and Methods: </span></b><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>Twenty
non-duplicate <i>Enterobacteriacae</i> isolates (11 <i>E. coli</i>, 8 <i>K.
pneumoniae</i> and 1 <i>Enterobacter aerogenes</i>) from 18 patients in 8
cities were analyzed. The species were identified by Vitek2. OXA-48detection
was done by Carba NP test, temocillin disc, and by PCR and sequencing of <i>bla</i><sub>OXA-48
</sub>genes. Location of the genes in Tn<i>1999</i>-like elements was checked
by PCR mapping. Identification of <i>bla</i><sub>OXA-48</sub>-carrying plasmids
was performed by the S1 analysis, followed by hybridization with <i>bla</i><sub>OXA-48</sub>.
The isolates were subjected to mating; plasmid DNAs of transconjugants was used
in the PBRT analysis. Chromosomal localization of <i>bla</i><sub>OXA-48 </sub>genes
in some isolates was confirmed by the I-CeuI analysis.Other β-lactamase genes
were identified by PCR and sequencing. <i>E. coli</i> &amp; <i>K. pneumoniae</i>
isolates were typed by PFGE and MLST.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>Results: </span></b><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>Of the 18
cases, 14 have been associated with arrivals from different countries. Eight
PFGE patterns were distinguished among each <i>E. coli</i> and <i>K. pneumoniae</i>
isolates, classified then by MLST into 4 and 5 STs, respectively. The most
prevalent were <i>E. coli</i>ST38 (n=6), <i>E. coli</i> ST410 (n=3) and <i>K.
pneumoniae</i> ST395 (n=3). All of the isolates carried the <i>bla</i><sub>OXA-48</sub>
gene which resided in the Tn<i>1999.1</i> (n=8), Tn<i>1999.2 (</i>n=10) or ΔTn<i>1999</i>
(n=2) transposon variants. The S1 analysis showed high diversity of plasmid
profiles (1-6 plasmids per isolate), including <i>bla</i><sub>OXA-48</sub>-carrying
plasmids of ~50kb–~260kbin 12 isolates; of which 11 isolates (6 <i>K.
pneumoniae</i>, 4 <i>E. coli</i> and 1 <i>E. aerogenes</i>) produced
transconjugants with high efficiency (10<sup>-3</sup>-10<sup>-5</sup> per donor
cell). The PBRT revealed replicons L (n=7; plasmids of~60kb), M (n=1; ~80kb)
and X3 (n=3; ~50kb). The only <i>bla</i><sub>OXA-48</sub> plasmid of ~260kb was
non-transferable either by conjugation or electroporation. The remaining 8
isolates had <i>bla</i><sub>OXA-48</sub> inserted into the chromosome. In all <i>E.
coli</i> ST38 isolates <i>bla</i><sub>OXA-48</sub> was located in the
chromosome. Fifteen isolates had ESBLs, including CTX-M-15 (n=9), -24 (n=4), -3
(n=1) and -55 (n=1); three isolates produced also the CMY-2 cephalosporinase.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>Conclusions: </span></b><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>This is the
first broad analysis of OXA-48 CPE in Poland, encompassing most of the cases
recorded so far. It showed high diversity of OXA-48 producers, attributed to
multiple independent imports. Only the 3 <i>E. coli</i> ST410 isolates
represented a small outbreak in one of the hospitals.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO37 - Report Of The
First Worldwide Diffusion Of A Multiresistant Coagulase Negative Staphylococci
And Comparative Genomics With Other <i style='mso-bidi-font-style:normal'>S.
capitis</i><o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>M Butin<sup>1</sup>; P Martins Simões<sup>2</sup>; L Hoden<sup>3</sup>; Y
Dumont<sup>3</sup>; H Lemriss<sup>4</sup>; A Ibrahimi<sup>4</sup>; J-C Picaud<sup>1</sup>;
A. Kearns<sup>5</sup>; M A Deighton<sup>6</sup>; O Denis<sup>7</sup>; J-P
Rasigade<sup>2</sup>; F Laurent<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Neonatal Intensive Care Unit, Hospices Civils de Lyon; <sup>2</sup>CIRI -
International Centre for Research in Infectious diseases, INSERM U1111,
University of Lyon; <sup>3</sup>Department of Clinical Microbiology, Hospices
Civils de Lyon; <sup>4</sup>Medical Biotechnology laboratory. (MedBiotech),
Medical and Pharmacy School, University Mohammed V Souissi; <sup>5</sup>Public
Health England, staphylococcus Reference Service, Colindale; <sup>6</sup>RMIT
University, Bundoora; <sup>7</sup>Erasme Hospital, Université Libre de
Bruxelles, Laboratoire de Référence MRSA - Staphylocoques, Department of
Microbiology<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Cases and
series of neonatal sepsis involving methicillin-resistant <i>Staphylococcus
capitis</i> (MRSC) have been previously reported from different countries, and
were initially considered as unrelated epidemic bursts. However, we recently
found that a single multidrug-resistant clone of <i>SC</i>, named NRCS-A and
defined by a specific pulsed-field gel electrophoresis (PFGE) banding pattern
was involved in neonatal sepsis in several neonatal ICUs (NICUS) from France,
Belgium, the UK and Australia. All NRCS-A isolates exhibited a decreased
susceptibility to all antimicrobial agents frequently used in NICUs. This led
us to suspect that the initial report of NRCS-A in four distant countries was
only the tip of the iceberg. Here, we investigated the diffusion of this clone
at a larger geographical scale and we analyzed the WGS of 4 NRCS-A strains from
distinct countries to determine the clone´s genomic characteristics. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>Materials and Methods: </span></b><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>In
collaboration with laboratories of hospitals worldwide, clinical MRSC isolates
from blood cultures of neonates have been collected and subjected to PFGE
typing using <i>Sma</i>I restriction enzyme.<b> </b>Four of these SC<i> </i>strains,
isolated in France, Belgium, Australia and United Kingdom (n=1 each), were
whole genome sequenced. A closed reference genome and methylome was obtained
(SMRT-BacBio) for the prototype strain CR01. NRCS-A genomes were searched for
virulence factors, resistance genes, mobile genetic elements (MGEs) and clone
specific genes. Six publicly available WGS of non-NRCS-A strains were used for
comparison.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>Results: </span></b><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>The NRCS-A PFGE
profile was identified in 154 <i>SC</i> isolates collected between 1994 and
2015 from 17 distant countries (Australia, Belgium, Brazil, Canada, Czech
Republic, Denmark, Finland, France, Germany, Netherlands, New Zealand, Norway,
South Korea, Switzerland, Taiwan, UK and USA).<b> </b>The WGS analysis of 4 of
these NRCS-A strains showed that antimicrobial resistance genes were found in
MGEs and that all <i>SC</i> genomes (NRCS-A and non-NRCS-A) shared virulence
factors including phenol soluble modulins (PSMs), hemolysins, cell wall
associated fibronectin, fibrinogen binding proteins and the <i>icaRADBC</i>
operon. Of importance, when comparing <i>S. capitis</i> genomes, we found
exclusively within the NRCS-A clone a gene coding for nisin resistance (<i>nsr</i>),
a “broad-spectrum” bacteriocin, that could play a pivotal role and give a
selective advantage to NRCS-A clone in the initial step of gut colonization in
neonates.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>Conclusions: </span></b><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>Our results
highlight the endemic distribution of the clone <i>S.&nbsp;capitis </i>NRCS-A
in NICUs worldwide, that constitutes the first report of a single coagulase-negative
staphylococci clone dissemination on such a large geographical scale.
Furthermore, the genomic characteristics of the NRCS-A clone emphasize the
contribution of MGEs to the emergence of its multidrug-resistance profile.
Comparative genomic analysis with non-NRCS-A genomes revealed differences in
the core genome, MGEs content but not in virulence factors profiles. The
identification of NRCS-A lineage specific genes suggests that it might be well
adapted to the NICU environment both by its high adaptation to antibiotic
pressure but also by specific survival advantages, contributing to a better
fitness in the very early steps of colonization. These observations raise
important questions regarding pathogen reservoirs and ways of transmission
within and between NICUs of different hospitals.</span><span style='font-size:
8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p></o:p></span></p>

<span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO39 - Monitoring
Clonal Diversity Of Extended-spectrum Cephalosporin-resistant <i
style='mso-bidi-font-style:normal'>Enterobacteriaceae</i> Colonizing The Human
Gut 3 Months After Traveling To India<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>João Pires<sup>1</sup>; Esther Kuenzli<sup>2</sup>; Sara Kasraian<sup>1</sup>;
Regula Tinguely<sup>1</sup>; Hansjakob Furrer<sup>3</sup>; Markus Hilty<sup>1</sup>;
Christoph Hatz<sup>2</sup>; Andrea Endimiani<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Institute for Infectious Diseases, University of Bern,<sup>2</sup>
Swiss Tropical and Public Health Institute, Basel,<sup>3</sup> Epidemiology,
Biostatistics and Prevention Institute, University of Zurich,<sup>4</sup>
Department of Infectious Diseases, Bern University Hospital, University of Bern<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>Background:</span></b><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'> The role of
international travel is being increasingly recognized as a very important
factor in the spread of extended-spectrum cephalosporin-resistant <i>Enterobacteriaceae
</i>(ESC-R-Ent). In particular, colonization by ESC-R-Ent in the human gut is
frequently reported when people return from high prevalence areas. This
phenomenon increases the risk of importation of new epidemic clones which can
spread in new geographical regions. Silent carriage of these pathogens in the
gut is of extreme importance as being colonized by ESC-R-Ent represents a risk
factor for developing future extra-intestinal infections. However, factors
modulating the acquisition and persistence of ESC-R-Ent in the gut are unknown.
The aim of this work was to monitor this condition in volunteers travelling to
a high ESC-R-Ent prevalence geographic area.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>Methods:</span></b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'> Thirty-six Swiss volunteers
travelling to India underwent screening before and after their trip. For a
subset of 17 volunteers, a screening after 3 months was also performed. An
aliquot of native stools were enriched in LB broth with a cefuroxime disc.
Enrichments were plated in BLSE, ChromID ESBL, and a modified Supercarba plate
containing imipenem. Species identification was obtained using the MALDI-TOF.
At least 5 <i>Enterobacteriaceae</i> colonies were analyzed. Double-disk
synergism test was performed and clonal relatedness in <i>E. coli </i>was
assessed using rep-PCR, Phylogenetic group (PhG) analysis, and MLST. The Check
Points CT103 XL microarray and PCR/DNA sequencing were implemented to detect
and characterize ESBL, plasmid-mediated AmpC, and carbapenemase-producing <i>Enterobacteriaceae</i>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>Results: </span></b><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>For the 36
volunteers, 5 of them were colonized before travelling and 29 after travelling.
In the subgroup of 17 volunteers, only 2 were found colonized before travelling
while all of them were colonized after their trip; 4 of them still remained
colonized three months later. Only one volunteer was colonized at the three
time points. For this subgroup, we detected 57 different isolates (56 <i>E.
coli </i>and one <i>K. pneumoniae</i>) of which 50 were ESBL and 7
plasmid-mediated AmpC producers. A low β-lactamase diversity was identified,
with isolates mostly CTX-M-types, mainly group 1 (n=47, including CTX-M-15),
and to a lesser extent DHA (n=5), CTX-M-group 4 (n=2), CMY-2 (n=2), and VEB
(n=1). Surprisingly, rep-PCR revealed that the clonal <i>E. coli</i> diversity
found in each volunteer can vary from 1 to 5 clones when returning to
Switzerland. These clones were mostly from PhG A (60.7%) and D (21.7%) and
diverse non-epidemic STs. Remarkably, when multiple clones were colonizing the
gut, they can also produce different β-lactamase types. Interestingly, for the
volunteers still colonized after 3 months, only one clone was persisting in the
gut.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>Conclusion: </span></b><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>Our data
indicate that the initial stages of colonization when traveling to high
prevalence areas involve a polyclonal acquisition of ESC-R-Ent. This might be
linked to non-human sources (e.g., food-chain, environment), since the isolates
belong to a multiplicity of genetic backgrounds that are usually linked with
these settings. Moreover, colonization of the gastrointestinal tract by
ESC-R-Ent also decreases over time but when persistent, only one clone is then
identified. A further characterization of these isolates will elucidate the
bacterial features that might be associated with enhanced colonization
capabilities and persistence in the gut.</span><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
<o:p></o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO40 - Ongoing
Evolution In The Hypervirulent CC17 Clone Of <i style='mso-bidi-font-style:
normal'>Streptococcus agalactiae<o:p></o:p></i></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Elisabete Martins<sup>1</sup>; Cristiano Rousado<sup>1</sup>; José
Melo-Cristino<sup>1</sup>; Mário Ramirez<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>Instituto de Medicina Molecular, Faculdade de
Medicina, Universidade de Lisboa<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>Background: </span></b><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>Group B
streptococci (GBS) is a leading cause of neonatal invasive disease (NID) in
most countries. CC17 is recognized as a worldwide disseminated hypervirulent
clone due to its increased ability to cause neonatal infections, particularly
late-onset disease (LOD) and meningitis. This clone has been shown to be highly
stable and to have persisted for decades as a leading cause of NID.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>Material/methods: </span></b><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>We have
analysed a collection of 97 isolates representing CC17 recovered from NID in newborns
in Portugal (2005-2014). These had been previously characterized by serotyping,
multilocus sequence typing, presence of genes encoding alpha-like proteins and
resistance to macrolides and tetracycline, and evaluated for susceptibility to
chloramphenicol, clindamycin, erythromycin, levofloxacin, penicillin,
tetracycline and vancomycin. High-level resistance to gentamicin and
streptomycin was tested by disk diffusion according to the CLSI guidelines for
enterococci. Presence of the pilus islands (PI) 1, 2a and 2b was tested by PCR.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>Results:</span></b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'> The collection was mostly composed
of representatives of ST17 and its SLVs, carrying the surface protein encoding
gene <i>rib</i> and the combination of PI-1+PI-2b. Five new STs were identified
among the 15 SLVs of ST17, indicating an ongoing diversification within CC17.
Additionally, five isolates carried only PI-2b (Figure 1). The lack of PI-1 had
not been previously observed in CC17 isolates from Portugal.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>All isolates were susceptible to
penicillin, vancomycin and levofloxacin, and one isolate was resistant to
chloramphenicol. Most isolates (n=87) were resistant to tetracycline and
carried the <i>tetM</i> gene (n=83). One isolate presented high-level
resistance to gentamicin andcarried the <i>aac(6’)-Ie-aph(2’’)-Ia </i>gene,
encoding a bifunctional aminoglycoside-modifying enzyme, but no other
resistance determinant. Macrolide resistance was low (n=7), in agreement with
previous reports, as this clone is frequently susceptible. The isolates
carrying exclusively PI-2b also carried the <i>ermB </i>gene and presented the
cMLS<sub>B</sub> phenotype. Most carried the unusual tetO gene (n=3) and
presented high-level resistance to aminoglycosides (n=4), harbouring the <i>aph(3)-IIIa</i>
and <i>aadE</i> genetic determinants, both encoding streptomycin-modifying
enzymes.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>Conclusions: </span></b><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>While the
dominance of serotype III/CC17 among neonatal invasive disease has been
described worldwide, our data suggest that diversification within this very
stable genetic lineage is ongoing, as illustrated by the possible emergence of
a highly resistant sublineage within CC17. Antimicrobial resistance has a
significant impact in clinical practice, including preventive strategies such
as intrapartum antibiotic prophylaxis but also in the treatment of GBS
infections. The increasing burden of GBS disease and dynamic nature of this
pathogen substantiate the need for continued monitoring and detailed
characterization of the genetic lineages circulating at both regional and
global contexts.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shapetype id="_x0000_t75"
 coordsize="21600,21600" o:spt="75" o:preferrelative="t" path="m@4@5l@4@11@9@11@9@5xe"
 filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Imagem_x0020_54" o:spid="_x0000_i1030" type="#_x0000_t75"
 alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/139imagem_2560108707_0.jpg"
 style='width:333pt;height:251pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S8_files/image001.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/139imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img width=335 height=253
src="public/abstracts/Webpages/PO_S8_files/image002.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/139imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_54"><![endif]></span><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO41 - Integrated
Analysis Pipelines For Whole Genome MLST And Whole Genome SNP In Bionumerics®
7.6 Applied To Publicly Available <i style='mso-bidi-font-style:normal'>Listeria
monocytogenes </i>Isolates<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Katleen Vranckx<sup>1</sup>; Katrien De Bruyne<sup>1</sup>; Bruno Pot<sup>1</sup>;
Hannes Pouseele<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Applied Maths NV<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>Background: </span></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Listeria
monocytogenes </span></i><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>(Lmo) is a ubiquitous organism in the
environment and a rare cause of human disease. Though listeriosis occurs
infrequently, it is characterized by a high case-fatality rate which can exceed
30%. Following a considerable costs reduction, complete Lmo genome sequencing
dramatically increased the number of publically available genomes on the
Sequence Read Archive (SRA) of NCBI. A key challenge is to rapidly compute and
interpret the relevant information from this vast amount of data. Rapid and
automated processing of whole genome sequencing (WGS) data ensures a reliable
and easy to follow workflow in routine molecular surveillance, reducing the
time needed to detect and contain an outbreak. Moreover, WGS has the potential
to provide information on traditional typing technologies such as MLST, MLVA,
virulence and resistance determination.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials and
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>In this study, we compare two subsequent pipelines for high resolution
WGS-based molecular typing.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>First, whole
genome multilocus sequence typing (wgMLST) is applied to WGS data from around
6000 isolates available on NCBI, with the purpose to detect clusters of highly
related strains. A wgMLST scheme was created from a limited set of reference
sequences, from which all coding regions were extracted and used to create a
set of discernible loci. An assembly-free and a BLAST-based allele calling
algorithm were used to independently determine locus presence and allelic
variants.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>A cluster defined by
wgMLST can then be further characterized by whole genome single-nucleotide
polymorphism analysis (wgSNP). As a working example we here focus on clusters
containing isolates originating from avocado or guacamole. SNP variants are
detected by mapping the WGS reads to a reference chosen from within the cluster
to maximize the resolution.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Both analysis
pipelines run on the BioNumerics® Calculation Engine, which is fully integrated
with the BioNumerics® 7.6 software.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>BioNumerics®
7.6 software offers a powerful platform where both wgMLST and wgSNP analysis
can be performed at high-throughput rates. Using this platform, WGS analysis
results can be validated against traditional data such as MLST or PFGE, rapidly
providing a robust, portable and high resolution picture of molecular typing
data. We demonstrate that wgMLST is suitable for the analysis of very large
(and growing) datasets, making it a suitable technique for outbreak
surveillance. The added resolution of wgSNP against an internal reference sequence
increases the confidence in the detected clusters and supports epidemiologist
in their source tracking efforts.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The
combination of two complementary approaches, wgMLST and wgSNP, on a virtually
unlimited number of samples, managed by a single software platform that also
stores historical data as well as metadata, opens many perspectives for food
safety and public health monitoring programs. The BioNumerics® solution
combines the power of a cluster or cloud implementation with the ease of use of
a local database and management software. The possibility to extract
‘historical’ typing data, resistance and virulence data from wgMLST schemes,
moreover reduces the total analysis time and cost, and may lead to more
efficient outbreak detection.</span><span style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:10.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO42 - From Typened To
Regiotype To Eurotype: Moving Towards A Comprehensive Surveillance Strategy For
Emerging Viruses<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>R. Poelman<sup>1</sup>; D. Luijt<sup>2</sup>; R. van Rhee-Luderer<sup>3</sup>;
W. Niessen<sup>4</sup>; M. van Genne<sup>1</sup>; C. Van Leer-Buter<sup>1</sup>;
B. Niesters<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University Medical Center Groningen,<sup>2</sup>
Certe, Center for infectious Diseases,<sup>3</sup> Izore, Center for infectious
Diseases,<sup>4</sup> Regional Public Health service (GGD)<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Patients
with respiratory illness are routinely screened for viral infections. This
screening was expanded by routinely sequencing enteroviruses and rhinoviruses
to identify outbreaks and transmission patterns. Additionally, we included
noroviruses and parechoviruses in patients with gastro-intestinal complaints.
For surveillance purposes, a national structure called TYPENED is available
through the Dutch National Institute of Health (RIVM). Our main objective is to
provide a rapid regional sequencing service that create a source of clinical
information and to correlate this with epidemiological patterns.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>A regional
sequencing and epidemiological strategy for surveillance, called REGIOTYPE, was
implemented by including GP’s, regional hospitals and regional Public Health
Services which covers the Northern part of the Netherlands, for a rapid&nbsp;
flow of information within our region. Sequencing is performed locally and
results are available within one week. Clinical data are exchanged and
phylogenetic analysis is performed.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Phylogenetic
analysis has contributed to the knowledge of circulating viral strains and the
relatedness within and between multiple institutions. This rapid sequencing
strategy in combination with the clinical data enables the application of
infection control measures in the participating (local) health institutions.
Enterovirus D68 and recently the ‘new’ C-group enteroviruses were identified.
This not only indicated a possible outbreak, but it also improved detection
methodologies in our region. Besides, a vaccine derived poliovirus strain was
recently identified in a refugee, using this local strategy.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>REGIOTYPE
has proven to be an adequate strategy for European surveillance on local
circulating viruses and not only for the hospitalized population with severe
(underlying) illnesses. This underlines the importance for regional
surveillance by sequencing, of which several networks are available within
Europe but still acting independently from each other.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO43 - Detection Of An
Expanding <i style='mso-bidi-font-style:normal'>Streptococcus pneumoniae</i>
Serotype 12F Clone With Acquired Antibiotic Resistance, Circulating In Africa
And Asia<o:p></o:p></span></b></p>

<p class=MsoNormal><u><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Rebecca A. Gladstone</span></u><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>; Brenda Kwambana-Adams<sup>2</sup>; Mignon Du
Plessis<sup>3</sup>; Maaike Alaerts<sup>4</sup>; Paulina Hawkins<sup>5</sup>;
Rama Kandasamy<sup>6</sup>; Ben J. Metcalf<sup>7</sup>; Somporn Srifuengfung<sup>8</sup>;
Stuart C. Clarke<sup>9</sup>; Abdullah W. Brooks<sup>10</sup>; Veeraraghavan
Balaji<sup>11</sup>; Diederik van de Beek<sup>12</sup>; Sarah E. Burr<sup>2</sup>;
Bernard Beall<sup>7</sup>; Andrew J. Pollard<sup>6</sup>; Paul Turner<sup>13</sup>;
Dean B. Everett<sup>4</sup>; Martin Antonio<sup>2</sup>; Anne von Gottberg<sup>14</sup>;
Keith P. Klugman<sup>5</sup>; Lesley McGee<sup>7</sup>; Robert F. Breiman<sup>15</sup>;
Stephen D. Bentley<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>1</span></sup></i><i style='mso-bidi-font-style:normal'><span
style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Pathogen Genomics, Wellcome Trust Sanger Institute, Hinxton, UK; <sup>2</sup>Vaccines
and Immunity Theme, MRC Unit The Gambia; <sup>3</sup>National Institute for
Communicable Diseases,Johannesburg, South Africa; <sup>4</sup>Malawi-Liverpool-Wellcome-Trust,
Malawi; <sup>5</sup>Rollins School Public Health, Emory University, Atlanta,
USA; <sup>6</sup>Oxford Vaccine Group, Department of Paediatrics, University of
Oxford, and the NIHR Oxford Biomedical Research Centre, Oxford UK; <sup>7</sup>Centers
for Disease Control and Prevention, Atlanta, USA. The findings and conclusions
in this report are those of the authors and do not necessarily represent the
official position of the Centers for Disease Control and Prevention.; <sup>8</sup>Siriraj
Hospital, Mahidol University, Bangkok, Thailand; <sup>9</sup>Faculty of
Medicine and Institute of Life Sciences, University of Southampton, UK; <sup>10</sup>International
Centre for Diarrheal Diseases Research, Dhaka, Bangladesh; <sup>11</sup>Christian
Medical College, Vellore, India; <sup>12</sup>Department of Neurology, Academic
Medical Center, Center of Infection and Immunity Amsterdam (CINIMA), University
of Amsterdam, Amsterdam, The Netherlands; <sup>13</sup>Centre for Tropical
Medicine and Global Health, Nuffield Department of Medicine, University of
Oxford, Oxford, UK; <sup>14</sup>National Institute for Communicable Diseases,
Johannesburg, South Africa; <sup>15</sup>The Emory Global Health Institute,
Atlanta, USA. On behalf of all Global </span></i><i style='mso-bidi-font-style:
normal'><span style='mso-bidi-font-size:9.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Pneumococcal
Sequencing project partners. </span></i><span lang=DE-CH><a
href="http://www.pneumogen.net/gps"><span lang=EN-US style='mso-bidi-font-size:
9.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-fareast-language:PT'>www.pneumogen.net/gps</span></a></span><i
style='mso-bidi-font-style:normal'><span style='mso-bidi-font-size:9.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Background:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> The Global Pneumococcal Sequencing project
(GPS) aims to study evolution in 20,000 pneumococcal genomes. Isolates were
collected from around the world, focusing on resource limited nations over the
introduction of pneumococcal conjugate vaccines (PCVs). Current PCVs target
serotypes associated with disease and antibiotic resistance. Over 40
globally-disseminated clones have been designated by the Pneumococcal Molecular
Epidemiology Network (PMEN). We describe an expanding clone (clonal complex
CC989) with acquired antibiotic resistance expressing non-vaccine serotype (NVT)
12F. Serotype 12F is found more frequently in disease than carriage and has
been increasing in the PCV era. The CC989 clone represents a lineage distinct
from the 12F PMEN clone, Denmark12F-34.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Methods:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> Whole genome data for ~10,000 pneumococcal
isolates were screened for known acquired antibiotic resistance determinants
using ARIBA. A phylogenetic tree of all taxa was produced using RAxML from a
concatenated MLST alignment. Metadata were visualised on the tree using
Phylocanvas.org and Microreact.org. Phenotypic data, where available, were used
to confirm the inferred resistant phenotype.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> A cluster of isolates with a shared antibiotic
resistance profile was identified in the phylogeny; &gt;90% were of sequence
type 989, serotype 12F. CC989 isolates (172) represent the largest 12F clone in
GPS and were predominantly from The Gambia (69) and South Africa (61) but also
from Malawi (12), Thailand (10), Ghana (4), Bangladesh (4), the USA (4),
Senegal (2), India (2), The Netherlands (2), Niger (1) and Nepal (1). The
earliest isolate in the GPS collection was from The Gambia in 2002 (95% CI
-2.7% to 8.4%), in South Africa the earliest isolates (n=2) were from 2005 (95%
CI -0.3% to 1.6%). In 2014, 11 (95% CI 6.5% to 22.1%) and 19 (95% CI 3.7% to
9.5%) were isolated in The Gambia (p=0.09) and South Africa (p=0.0001)
respectively. In 2015 CC989 cases were observed in the USA Active bacterial
core surveillance for the first time. CC989 isolates were more often from
disease than expected (95% CI 75.5% to 87.5%) when compared to GPS collection
overall (95% CI 57.2% to 58.8%).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Almost all
isolates had the tet(M) gene and a chloramphenicol acetyltransferase gene (cat
pC194). Phenotypic tetracycline minimum inhibitory concentrations (MICs) were
available for 56 isolates, 95% had a MIC of ≥2</span><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>μ</span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>g/ml (resistant)
with a mode MIC of 8</span><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT'>μ</span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>g/ml. Phenotypic chloramphenicol MICs were available for 38 isolates, 92%
had a MIC of ≥8</span><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT'>μ</span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>g/ml (resistant) with a mode MIC of 8</span><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>μ</span><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>g/ml. Pbp1a, 2b
and 2x allele profiles and phenotypic data suggest that the isolates are fully
susceptible to penicillin.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusions:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> Isolates of ST989 usually expressing serotype
12F have previously been reported in small numbers; the MLST database includes
sporadic CC989 isolates (n=55) from African, Asian and European countries with
the oldest isolate that we know of from 1998 in Kenya. The global perspective
of the GPS collection, combined with genomics and resistance profiles,
highlighted the intercontinental dissemination of an expanding NVT clone with
acquired resistance. The clone’s prevalence in Africa and 12F´s association
with disease make it a potential threat that may become important in vaccine
serotype replacement. Global genomic surveillance allows early detection of
expanding lineages expressing serotypes not currently targeted by vaccines,
providing vital information for future control of pneumococcal disease.<o:p></o:p></span></p>

<span style='font-size:8.0pt;mso-bidi-font-size:10.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:"Times New Roman";text-transform:
uppercase;mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:
AR-SA'><br clear=all style='mso-special-character:line-break;page-break-before:
always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:10.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;text-transform:uppercase;mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO44 - Two Decades Of
Epidemiological Surveillance <i style='mso-bidi-font-style:normal'>Of
Burkholderia cepacia</i> Complex Bacteria In Respiratory Infections At A
Portuguese Cystic Fibrosis Center<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Carla P. Coutinho<sup>1</sup>; Ana S. Moreira<sup>1</sup>; A. Amir Hassan<sup>1</sup>;
Sandra C. dos Santos<sup>1</sup>; Celeste Barreto<sup>2</sup>; Luísa Pereira<sup>2</sup>;
Pilar Azevedo<sup>3</sup>; Luís Lito<sup>4</sup>; José Melo-Cristino<sup>4</sup>;
Isabel Sá-Correia<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>iBB-Institute for Bioengineering and Biosciences,
Department of Bioengineering, Instituto Superior Técnico (IST), Universidade de
Lisboa, Av. </span></i><i style='mso-bidi-font-style:normal'><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>Rovisco Pais, 1049-001
Lisbon, Portugal,<sup>2</sup> Cystic Fibrosis Center, Department of
Paediatrics, Hospital de Santa Maria, Centro Hospitalar Lisboa Norte, Av. Prof.
Egas Moniz, 1649-028 Lisbon, Portugal,<sup>3</sup> Cystic Fibrosis Center,
Adults Pulmonology Unit, Hospital de Santa Maria, Centro Hospitalar Lisboa
Norte, Av. Prof. Egas Moniz, 1649-028 Lisbon, Portugal,<sup>4</sup> Laboratório
de Patologia Clínica, Hospital de Santa Maria, Centro Hospitalar Lisboa Norte,
Av. </span></i><i style='mso-bidi-font-style:normal'><span style='mso-ansi-language:
EN-US;mso-fareast-language:PT'>Prof. Egas Moniz, 1649-028 Lisbon, Portugal<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The <i>Burkholderia
cepacia</i> complex (Bcc) comprises 20 species of Gram-negative opportunistic
bacterial pathogens that generally lead to long term respiratory infections and
a more rapid decline in cystic fibrosis (CF) patients lung function and, in
some cases, to a fatal necrotizing pneumonia (cepacia syndrome). Bcc bacteria
can be transmitted between patients with or without CF but the environment,
including contaminated medications, may act as reservoir for novel infections.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials &amp;
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>The epidemiological surveillance of Bcc respiratory infections at the major
Portuguese CF Center at Hospital de Santa Maria (HSM) has been carried out over
the past 2 decades [1]. More than 800 clinical isolates were retrieved from
sputum samples collected from 52 CF patients (32 paediatric and 20 adult), as
part of routine monitoring at this Center [1]. The identification and strain
discrimination was performed using PCR amplification with specific primers, <i>recA</i>,
<i>hisA</i>, MLST and RFLP-PFGE analyses [1-3]. Extensive phenotypic analyses
of clonal variants obtained during long-term lung infection were performed
[1-3].<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>This
systematic and longitudinal survey of a CF population for Bcc-associated
respiratory infections (in 5% of the patients) covered different Bcc species
and disease outcomes, including chronically colonized patients (56%) between 7
months and 16 years.The infecting bacteria belongs not only to the most
frequent species worldwide, <i>B. cenocepacia</i> (36% of Bcc-infected
patients) and <i>B. multivorans</i>, but also species that are rarely isolated
in CF and therefore poorly studied, in spite of their damaging effects, as it
is the case of<i> B. dolosa</i> and <i>B. stabilis</i> [1,3]. Remarkably, the
CF population under analysis showed an abnormal prevalence of <i>B. cepacia</i>
[4]. Recently, after re-examination of the <i>B. cepacia </i>isolates belonging
to <i>recA</i> RFLP type K, theywere reclassified as <i>B. contaminans </i>[2].
The current data indicates that <i>B. cepacia</i> and <i>B. contaminans</i>
were isolated from 28% and 17% of the Bcc-infected patients, respectively. The
systematic phenotypic characterization (e.g. susceptibility to antimicrobials,
cell motility and hydrophobicity, colony morphology, fatty acid composition,
growth under iron limitation/load conditions, exopolysaccharide production,
size of biofilms) of several clonal variants obtained during long-term
infections suggested the occurrence of clonal expansionwhich ultimately proves
beneficial by providing an adaptive evolution leverage [1,5].<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: Although
rarely isolated from CF patients,<i> B. cepacia, B. contaminans and B. dolosa</i>
were highly represented during this retrospective study. The importance of the
surveillance of unexplained outbreaks involving poorly represented species [4]
is highlighted. Thecollection of isolates gathered is essential to get clues on
the adaptive strategiesadopted by Bcc species during chronic infectionand how
these may affect clinical outcome and disease progression.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><i><span style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Acknowledgements: </span></i></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>National
funding received from FCT by iBB (UID/BIO/04565/2013) PhD (ASM, AAH/FCT
BIOTECnico doctoral program) and Post-Doc fellowships (CPC, SS), is
acknowledged.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>References: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>[1].
Coutinho, <i>et al.</i> Front. Cell. Infect. Microbiol. 1(12), 2011. [2].
Coutinho, <i>et al.</i> J. Med. Microbiol. </span><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>64: 927-35, 2015. [3]. Moreira, <i>et
al.</i> J. Med. Microbiol. 63: 594–601, 2014. [4]. Cunha, <i>et al.</i> J.
Clin. </span><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Microbiol. 45: 1628-33, 2007. [5] Coutinho, <i>et al.</i> Infect. Immun. </span><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>79: 2950–60,
2011</span><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO45 - Transforming
Public Health Microbiology For <i style='mso-bidi-font-style:normal'>Campylobacter</i>
With Whole Genome Sequencing: PulseNET And Beyond<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Collette Fitzgerald<sup>1</sup>; Vikrant Dutta<sup>1</sup>; Jan Pruckler<sup>1</sup>;
Darlene Wagner<sup>1</sup>; Grant Williams<sup>1</sup>; Heather Carleton<sup>1</sup>;
Christy Bennett<sup>1</sup>; Lavin Joseph<sup>1</sup>; Eija Trees<sup>1</sup>;
Andrew Huang<sup>1</sup>; Lee S. Katz<sup>1</sup>; Lori Gladney<sup>1</sup>;
Martin C. J. Maiden<sup>2</sup>; William Miller<sup>3</sup>; Y. Chen<sup>4</sup>;
S. Zhao<sup>4</sup>; Pat McDermott<sup>4</sup>; J. Whichard<sup>1</sup>; E. M.
Ribot<sup>1</sup>; H. Pouseele<sup>5</sup>; P. Gerner-Smidt<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>CDC,<sup>2</sup> Department of Zoology, University of
Oxford,<sup>3</sup> Agricultural Research Service, United States Department of
Agriculture,<sup>4</sup> Center for Veterinary Medicine, United States Food and
Drug Administration,<sup>5</sup> Applied Maths, Inc.<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conventional
phenotypic and genotypic methods employed for identification and subtyping of <i>Campylobacter</i>
are labor intensive, expensive, and imprecise. We have begun development of
Enteric Reference Identification and <i>Campylobacter</i> subtype
characterization whole genome sequence (WGS) databases. The PulseNet
infrastructure (BioNumerics) will be used in conjunction with the existing
BIGSdb platform to build the databases, with the goal of characterizing <i>Campylobacter</i>
in a single workflow using WGS. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>&nbsp;
Reference genomes (n=103) provided by FDA and USDA and strains (n=100)
sequenced at CDC were used to develop the database. Assemblies and annotations
were performed using the Computational Genomics Pipeline v0.4. These genomes
cover the known members of the species and genera within <i>Campylobacteraceae </i>and
the known genetic diversity of <i>C. jejuni</i>.&nbsp; The data will be used to
set criteria for the current PubMLST.org/campylobacter locus definitions.
Multiple subschema are being&nbsp; set up within the databases to perform
identification and scalable, hierarchical subtyping that will include seven
locus, ribosomal, core genome and whole genome (MLST, rMLST, cgMLST and
wgMLST).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: To date
203 reference genomes have been sequenced, annotated and used for development
of the BioNumerics databases. The <i>Campylobacter</i> allele database 2.0
contains approximately 25,000 loci. Overall the average number of bases covered
was 87% and the average number of bases in coding sequences covered was
93%.&nbsp;An additional 600 isolates are in the process of being used to
validate the prototype databases. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusions: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>These WGS
BioNumerics databases will provide a single, unified, cost-effective approach
for accurate species identification and subtyping to aid the surveillance of
sporadic and outbreak related <i>Campylobacter</i> infections. Through
continued collaboration with domestic and international partners, we will test
and refine the nomenclature databases and&nbsp;validate the reference
identification subschema for diagnsotic accreditation within the next year.<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO46 - Carbapenemase
Genes, Oxy Types And MLST Types Of Four Danish Meropenem Non-susceptible
Clinical <i style='mso-bidi-font-style:normal'>Klebsiella oxytoca</i> Strains<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Hansen DS<sup>1</sup>; Littauer P<sup>2</sup>; Justesen US<sup>3</sup>;
Hammerum AM<sup>4</sup>; Hasman H<sup>4</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Herlev Hospital,<sup>2</sup> Hvidovre Hospital,<sup>3</sup>
Rigshospitalet,<sup>4</sup> Statens Serum Institut<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>: <i>K. oxytoca</i> is
amongst the top ten enterobacteria causing infections, particularly UTI, RTI
and sepsis. Due to mutations in the regulator sequence of the class A broad
spectrum chromosomal betalactamase (OXY) <i>K. oxytoca</i> frequently becomes
cephalosporin resistant and does in some clinical settings appear quite common.
In recent years a number of OXY-types have been described. Institut Pasteur
hosts a webpage that keep track of the presently known 37 OXY alleles (</span><span
lang=DE-CH><a
href="https://www.eventgest.com/GEST/%C2%B4%C2%B4http:/bigsdb.web.pasteur.fr/klebsiella/%C2%B4%C2%B4"><span
lang=EN-US style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:windowtext;mso-ansi-language:EN-US;mso-fareast-language:
PT;text-decoration:none;text-underline:none'>http://bigsdb.web.pasteur.fr/klebsiella/</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>). In 2014 a MLST typing
scheme for <i>K. oxytoca</i> was suggested by Herzog et al, which already
comprise 170 sequence types (ST) (</span><span lang=DE-CH><a
href="https://www.eventgest.com/GEST/%C2%B4%C2%B4http:/pubmlst.org/koxytoca/%C2%B4%C2%B4"><span
lang=EN-US style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:windowtext;mso-ansi-language:EN-US;mso-fareast-language:
PT;text-decoration:none;text-underline:none'>http://pubmlst.org/koxytoca/</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>). August 2015 a
collection of 68 expanded-spectrum cephalosporin resistant <i>K. oxytoca</i>
isolates were characterized with regard to betalactamase, OXY type and MLST
type. A number of (also new) ST and OXY-types were described. Beside a number
of ESBL and AmpC enzymes two isolates with the VIM-1 carbapenemase were found.
Denmark have an active surveillance of clinical meropenem non-suceptible
enterobacteria isolates; we here present data on four OXA-48 <i>K. oxytoca</i>
strains characterized with regard to other acquired resistance genes, OXY types
and MLST types.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Material
and Methods:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;mso-ansi-language:EN-US;
mso-fareast-language:PT'> The four isolates were from four patients in four
hospitals and with no known epidemiological link. Isolates were from urine
(N=3) and wound (N=1). Primary identification was by MALDI-TOF and
susceptibility testing by disc-diffusion (EUCAST). Due to non-susceptibility to
meropenem isolates were submitted to Statens Serum Institut. Whole-genome
sequencing (WGS) was performed by Illumina 251-bp paired-end sequencing. From
WGS data, acquired antimicrobial resistance genes were retrieved using
ResFinder 2.1 (</span><span lang=DE-CH><a
href="https://www.eventgest.com/GEST/%C2%B4%C2%B4https:/cge.cbs.dtu.dk/services/ResFinder/%C2%B4%C2%B4"><span
lang=EN-US style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:windowtext;mso-ansi-language:EN-US;mso-fareast-language:
PT;text-decoration:none;text-underline:none'>https://cge.cbs.dtu.dk/services/ResFinder/</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>) with a threshold of
100% identity for betalactamase genes and 98% identity for other genes.
OXY-types and MLST types were found using the above web pages. Mutations and
other events in the <i>bla</i>OXY gene and in the MLST alleles were checked
using CLCbio’s Genomics Workbench. Identification of single nucleotide
polymorphisms (SNPs) was performed using CSI Phylogeny 1.1 (</span><span
lang=DE-CH><a
href="https://www.eventgest.com/GEST/%C2%B4%C2%B4https:/cge.cbs.dtu.dk/services/CSIPhylogeny%C2%B4%C2%B4"><span
lang=EN-US style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:windowtext;mso-ansi-language:EN-US;mso-fareast-language:
PT;text-decoration:none;text-underline:none'>https://cge.cbs.dtu.dk/services/CSIPhylogeny</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>) with <i>K. oxytoca</i>
KCTC 1686 (CP003218.1) as the reference.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> The four strains had in addition to the OXA-48
carbapenemase also from 1-11 of the following acquired antimicrobial resistance
genes: <i>bla</i>CTX-M-15, <i>bla</i>SHV-2, <i>bla</i>SHV-12, <i>bla</i>TEM-1B,
<i>aad</i>A2, <i>aad</i>B, <i>aph</i>(3´)-Ia,<i> aac</i>(6´)Ib-cr, <i>aac</i>A4,<i>
str</i>A, <i>str</i>B, QnrA1, QnrS1, <i>mph</i>(A), <i>sul</i>1,<i> dfr</i>A12,
<i>dfr</i>A18. MLST showed that one isolate was ST-85, one ST-9-like
(1-7-2-1-9mut-1-2, one mutation in phoE) and two strains each showed new allele
combinations (3-5-21-19mut-24-6-20 and 1-7-2-15-16mut-1-14) and additional
mutation in an allele. OXY-types were OXY-1-1 (N=2), OXY-2-4 and OXY-2-11. By
SNP phylogeny isolate were grouped in two pairs, corresponding to OXY-1 or
OXY-2, and with a pairwise distance of 15.414 and 11.941 SNPs, respectively.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusion: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>The four OXA-48 <i>K. oxytoca</i> strains were
quite diverse by all parameters looked at, as number of acquired resistance
genes, OXY types, MLST types and SNP phylogeny. One isolate (ST-9) belonged to
the dominating and international spread, clonal complex CC2. When it is
possible in four isolates to find two new STs, a much larger diversity of STs
(and OXY-types) can be expected.</span><span style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO47 - <i
style='mso-bidi-font-style:normal'>Staphylococcus epidermidis</i> Nosocomial
Strain In Breast Milk Of Mothers Of Preterm Neonates<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Hiie Soeorg<sup>1</sup>; Tuuli Metsvaht<sup>2</sup>; Imbi Eelmäe<sup>2</sup>;
Sirli Treumuth<sup>1</sup>; Mirjam Merila<sup>3</sup>; Mari-Liis Ilmoja<sup>4</sup>;
Irja Lutsar<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Department of Microbiology, University of Tartu,<sup>2</sup>
Pediatric Intensive Care Unit, Tartu University Hospital,<sup>3</sup>
Department of Neonatology, Children’s Clinic, Tartu University Hospital,<sup>4</sup>
Pediatric Intensive Care Unit, Tallinn Children´s Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Staphylococcus
epidermidis</span></i><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'> is the commonest causative agent of late-onset
sepsis (LOS) in preterm neonates. Mothers of preterm neonates visit neonatal
intensive care unit (NICU) and often require hospitalization or antimicrobial
therapy. Therefore, breast milk (BM) may become colonized with nosocomial
strains rendering BM as a potential source of infecting strains. We aimed to
describe the prevalence of nosocomial <i>S. epidermidis</i> strains in BM of
mothers of preterm neonates.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Material and Methods:
</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>From January 2014 to March 2015 preterm neonates (gestational age (GA)
&lt;37 w=&quot;&quot; n=&quot;30)&quot; hospitalized=&quot;&quot;
to=&quot;&quot; nicu=&quot;&quot; of=&quot;&quot; tartu=&quot;&quot;
university=&quot;&quot; hospital=&quot;&quot; within=&quot;&quot;
the=&quot;&quot; first=&quot;&quot; week=&quot;&quot; life=&quot;&quot;
and=&quot;&quot; their=&quot;&quot; mothers=&quot;&quot; were=&quot;&quot;
studied=&quot;&quot; healthy=&quot;&quot; term=&quot;&quot;
neonates=&quot;&quot; ga=&quot;&quot;&gt;37 w) (n=20) and their mothers were
included as controls. Stool samples and skin swabs from neonates and BM from
mothers were collected weekly in the first month of life and cultured onto
mannitol salt agar. Five colonies typical to staphylococci were identified to
the species level by MALDI-TOF MS. All <i>S. epidermidis</i> isolates were
typed by multilocus variable-number tandem-repeats analysis (MLVA), each
distinct MLVA-profile was considered as a separate MLVA-type (MT) and was
arbitrarily assigned an integer. The presence of the <i>mecA</i> gene, the <i>ica</i>-operon,
IS<i>256</i> was determined. A randomly chosen representative isolate of each
MT was typed by multilocus sequence typing. Maternal and neonatal
characteristics potentially influencing colonization of mother were assessed in
logistic regression model.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>: </span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Of 30 preterm
neonates (median GA 29 w (IQR 27-33 w), median birth weight 1306 g (IQR
906-1889 g) LOS caused by <i>S. epidermidis</i> (2 by MT1, 1 by MT2) developed
in 3 neonates. All preterm neonates and their mothers were colonized with <i>S.
epidermidis</i>, yielding a total of 1239 isolates representing 174 different
MTs. MT1 was isolated from 22 neonates (skin of 21, gut of 15) and MT2 from 9
neonates (skin of 8, gut of 6). Median age at the first isolation of MT1 was 6
d (IQR 3-13 d) and MT2 10 d (IQR 8-19 d). MT1 was isolated from BM of 8 of 30
(26.7%) mothers of preterm neonates at median of 14.5 d (IQR 11-19.5 d)
postpartum. Neonates of all 8 mothers were colonized with MT1 (7 in gut, 7 on
skin) median of 7.5 d (IQR 6.5-14 d) prior to isolation from BM. Of neonatal
and maternal characteristics potentially influencing colonization with
bacteria, only neonatal gut colonization with MT1 was a significant factor
(p=0.0375) increasing the odds of colonization of BM with MT1 according to
multivariate logistic regression model (Table). BM from mothers of the neonates
with LOS caused by MT1 did not contain MT1. MT2 was not isolated from any BM
sample. All term neonates and their mothers were colonized with <i>S.
epidermidis</i>, yielding a total of 981 isolates representing 190 different
MTs. None of the term neonates or their mothers were colonized with MT1 or MT2.
Representative isolates of MT1 and MT2 were of multilocus sequence type 5,
carried the <i>mecA</i> gene and IS<i>256</i>, but not <i>ica-</i>operon.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>: </span></b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Our results
indicate that BM of mothers of preterm neonates can become colonized with
nosocomial <i>S. epidermidis</i> strains. Preceding neonatal gut colonization
as the only factor associated with the occurrence of the nosocomial strain in
BM suggests that neonate could be a source of maternal colonization. Therefore,
reducing transmission of nosocomial strains to microbiota of preterm neonates
could prevent transfer of the strains to mother’s BM.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape id="Imagem_x0020_55"
 o:spid="_x0000_i1029" type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/48imagem_2560108707_0.jpg"
 style='width:333pt;height:113pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S8_files/image003.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/48imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=115
src="public/abstracts/Webpages/PO_S8_files/image004.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/48imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_55"><![endif]></span><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO48 - Implementation
Of Whole Genome Sequencing (WGS) For Surveillance And Outbreak Detection Of
Shiga Toxin-producing <i style='mso-bidi-font-style:normal'>Escherichia coli </i>(STEC)
In The United States<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Rebecca L. Lindsey<sup>1</sup>; Heather Carleton<sup>1</sup>; Sung Im<sup>1</sup>;
Morgan Schroeder<sup>1</sup>; Katrine Joensen<sup>2</sup>; Flemming Scheutz<sup>2</sup>;
Lisley Garcia-Toledo<sup>1</sup>; Nancy Strockbine<sup>1</sup>; Efrain Ribot<sup>1</sup>;
Eija Trees<sup>1</sup>; Kelley Hise<sup>1</sup>; Hannes Pouseele<sup>3</sup>;
Peter Gerner-Smidt<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Centers for Disease Control and Prevention,<sup>2</sup>
Statens Serum Institut,<sup>3</sup> Applied Maths<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Shiga
toxin-producing <i>Escherichia coli</i> (STEC) is an important foodborne
pathogen capable of causing mild to severe disease in humans.&nbsp; Current
methods for characterization of STEC are expensive and time-consuming.&nbsp;
Work has begun to replace traditional methods with those using whole genome
sequence (WGS) data by developing an allele database of individual <i>Escherichia</i>
genes in BioNumerics 7.5 (Applied Maths, Austin, TX).&nbsp; This will allow
characterization of <i>Escherichia</i> in a single workflow using a multi-locus
sequence typing (MLST) approach.&nbsp; <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials and
Methods:</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'> The <i>Escherichia</i> allele database version 2.0 was built with 20
annotated reference genomes from a geographically diverse collection of human,
animal and environmental strains. Included in BN 7.5 is a genotyper tool that
uses genes encoding virulence factors, antimicrobial resistance and O and H
antigens from databases at the Center for Genomic </span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Epidemiology (</span><span
lang=DE-CH><a href="http://genomicepidemiology.org"><span lang=EN-US
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>http://genomicepidemiology.org</span></a></span><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>, DTU,
Lyngby, Denmark).&nbsp; The reference genomes represent eleven <i>Escherichia
coli </i>(<i>E. coli)</i> serogroups and two <i>Shigella</i> species. Multiple
subschema will be built within the database to perform identification,
characterization and subtyping, including MLST capabilities. To test the
ability of the BioNumerics-based whole genome MLST approach to correctly
identify, characterize and cluster strains, we analyzed 50 <i>Escherichia</i>
isolates from sporadic and outbreak-related infections and compared the
findings to those obtained previously with phenotypic and molecular subtyping
methods.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> The <i>Escherichia</i>
allele database 2.0 contains 9,494 loci. Overall the average number of bases
covered was 77% and the average number of bases in coding sequences covered was
90.8%.&nbsp; Identification by serotype, virulence profile characterization and
wgMLST analysis was completed for 50 <i>E coli</i> isolates. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> The
BioNumerics-based wgMLST approach provides a single, cost effective strategy to
identify and characterize isolates for surveillance and outbreak
investigations. The analysis tools in BioNumerics will enable end-users in
public health laboratories to analyze WGS data they generate with little
bioinformatics expertise, making the system equally efficient for local and
central investigations. The system will be refined through continued
collaboration with domestic and international partners.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO49 - Molecular
Characterization Of Vancomycin-resistant Enterococci In Hong Kong.<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Margaret Ip<sup>1</sup>; Christopher Lai<sup>2</sup>; CY Chiu<sup>1</sup>; Kitty
Fung<sup>3</sup>; Raymond Lai<sup>4</sup>; Dominic Tsang<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Chinese University of Hong Kong,<sup>2</sup> Queen
Elizabeth Hospital,<sup>3</sup> United Christian Hospital,<sup>4</sup> Prince
of Wales Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Vancomycin-resistant
enterococcus (VRE), particularly <i>Enterococcus faecalis</i> and <i>E faecium</i>,
causes nosocomial infections and raises infection control concerns because of
their prolonged survival in the environment and from asymptomatic carriers. In
Hong Kong, VRE was first detected in 1997 and sporadically reported and small
clusters of cases have been noted at various acute care hospitals between 2009
to 2014. A VRE screening program was thus commenced in all public hospitals and
some private hospitals.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>240 strains of VRE collected from six hospitals
across three of 7 clusters of public hospitals in Hong Kong were examined. They
were isolated from clinical specimens of infected patients or in the
surveillance screening program during the periods from 2013 - 2014.
Colonization screen were performed on inpatients who had a history of hospital
admission in the previous 3 months and from renal patients on hemodialysis in
2013 to early 2014. All VRE were identified as <i>E. faecium</i>, either by
biochemical methods or by MALDI-TOF MS. All strains were analyzed by pulse
field gel electrophoresis (PFGE) of <i>Sma</i>I-digested DNA. The similarity of
the PFGE fingerprints were analysed using the BioNumerics software (Applied
Maths, Belgium). A PFGE dendrogram was drawn by the Dice method and by
clustering by UPGMA with 1% position and 2% band tolerance. MLST was performed
on representative strains by obtaining the allelic profiles through sequences
of internal fragment of seven housekeeping genes – atpA, ddl, gdh, purK, gyd,
pstS and adk. Van genes were performed by PCRs.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Results: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>A dendrogram was drawn and two main clusters
were obtained using a cut-off at 90% similarity. The major Cluster A consists
of 177 strains and another 21 strains belong to cluster B. Each cluster of
isolates consists of strains from various hospitals, and the predominant
cluster A were identified ST761 and that of cluster B was ST555. Both were
members of clonal complex-17 of <i>E. faecium</i>. An additional 42 strains did
not belong to these two clusters but were not further typed. All strains
carried Van A genes.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Conclusion: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>Two predominant clones of VREs were present in
these hospitals though a number of various heterogenous strain types were also
identified among asymptomatic carriers. An active surveillance program
effectively identified these carriers and with extensive infection control
practice, prevented nosocomial transmission of VRE.<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO50 - Biocide
Resistance Genes In <i style='mso-bidi-font-style:normal'>Staphylococcus aureus</i>
From Portuguese Ex-colonies In Africa: Very High Prevalence And Antimicrobial
Cross-resistance<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Teresa Conceição<sup>1</sup>; Céline Coelho<sup>1</sup>; Hermínia de
Lencastre<sup>2</sup>; Marta Aires-de-Sousa<sup>3</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>Instituto de Tecnologia Química e Biológica António
Xavier,<sup>2</sup> Laboratory of Microbiology and Infectious Diseases, The
Rockefeller University,<sup>3</sup> Escola Superior de Saúde da Cruz Vermelha
Portuguesa<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Biocide
resistance in <i>Staphylococcus aureus</i> is mediated by multidrug efflux
pumps, which frequently promote cross-resistance to antibiotics. The overuse of
biocides in infection control in hospitals led to the emergence of <i>S. aureus</i>
strains with decreased antiseptic susceptibility, which became a problem in
different regions of the world. Geographic distribution of biocide resistance
genes has been poorly studied and, to date, there are no data from the African
continent where MRSA prevalence is not negligible. In the present work, we
assessed for the first time the prevalence of six biocide resistance genes and
its association with antimicrobial resistance, among a collection of <i>S.
aureus</i> isolates from three former Portuguese African colonies.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials and Methods:
</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>A total of 301 <i>S. aureus</i> isolates [including 82 methicillin
resistant <i>S. aureus</i> (MRSA) and 219 methicillin susceptible <i>S. aureus</i>
(MSSA)] were selected from a collection of 519 <i>S. aureus</i> nasal carriage
isolates recovered from patients and healthcare workers in Angola, São Tomé and
Príncipe and Cape Verde. Isolates were previously characterized by different
molecular typing techniques, including PFGE, <i>spa</i> typing, MLST, and SCC<i>mec</i>
typing.Antimicrobial susceptibility was determined by disk diffusion and the
presence of biocide resistance genes was detected by PCR amplification of
internal fragments of six efflux pumps genes (<i>qacAB, smr, norA</i>, <i>lmrS</i>,
<i>mepA</i> and<i> sepA</i>) in all isolates.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The global
prevalence of antiseptic resistance genes observed in the three African countries
was very high for <i>sepA</i> (95.3%), <i>mepA</i> (89.4%) and <i>norA</i>
(86.4%), intermediate for <i>lmrS</i> (60.8%) and <i>qacAB</i> (40.5%) and low
for <i>smr</i> (3.7%). A similar distribution was observed when comparing MRSA
and MSSA except for <i>qacAB </i>and<i> sepA</i> genes that were more prevalent
among MRSA (p=0.0137 and p&lt;0.0001, respectively). The genes <i>norA</i>, <i>lmrS</i>
and <i>mepA </i>were widespread in São Tomé and Príncipe (97.4%, 87% and 98.7%,
respectively) and Cape Verde (83.8%, 54.3% and 96.2%, respectively), while <i>qacAB</i>
was mainly detected in Angola (56.3%, p&lt;0.0001). Except for <i>smr</i>, the
remaining other five biocide resistance genes were very common (range: 41-100%)
among the major MRSA clonal lineages circulating in these countries: A-ST5-IVa,
B-ST88-IVa and C-ST8-IV/V. Moreover, the presence of biocide resistance genes
was significantly associated to antibiotic resistance: (i) <i>qacAB</i> was
associated to cefoxitin (p&lt;0.0001), gentamicin (p=0.0013), rifampicin
(p&lt;0.0001), trimethoprim-sulfamethoxazole (SXT) (p&lt;0.0001) and cloramphenicol
(p=0.0244); (ii) <i>mepA</i> was highly associated to tetracycline (p=0.0024)
and rifampicin (p=0.0007); (iii) and <i>smr</i> and <i>sepA</i> were associated
to gentamicin (p=0.0236) and SXT (p=0.0021), respectively.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Biocide
resistance genes are highly prevalent in Angola, São Tomé and Príncipe and Cape
Verde. The use of antiseptics might be selecting for antibiotic-resistant
strains in Africa and assisting their survival in the healthcare environment,
which is of major concern for future infection control programs.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO51 - Emergence Of
Pan-resistance In Kpc-2 Carbapenemase-producing <i style='mso-bidi-font-style:
normal'>Klebsiella pneumoniae</i> In Crete, Greece<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>E Bathoorn<sup>1</sup>; C Tsioutis<sup>2</sup>; JM da Silva Voorham<sup>1</sup>;
EV Scoulica<sup>3</sup>; E. Ioannidou<sup>4</sup>; K. Zhou<sup>1</sup>; JW
Rossen<sup>1</sup>; A Gikas<sup>2</sup>; AW Friedrich<sup>1</sup>; H Grundmann<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Department of Medical Microbiology, University of
Groningen, University Medical Center Groningen,<sup>2</sup> Department of
Internal Medicine, Infectious Diseases Unit, University Hospital of Heraklion,<sup>3</sup>
Laboratory of Clinical Bacteriology and Molecular Microbiology, Faculty of
Medicine, University of Crete,<sup>4</sup> Department of Internal Medicine,
Rethymnon General Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>:
KPC-2-producing <i>Klebsiella pneumoniae</i> (KPC-KP) sequence type (ST) 258
has been rapidly expanding and is often associated with serious nosocomial
infections. Last-line antibiotics such as colistin and tigecycline often remain
the only treatment option. We here describe the characteristics of KPC-KP
isolates from Crete and present the results of molecular fingerprinting by core
genome multilocus sequence typing (cgMLST).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials/Methods:
</span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Clinical isolates from sputum, urine, blood, ascetic fluid or pus samples
had randomly been collected from 34 unique patients hospitalised in three
hospitals on Crete in 2010 and 2013/2014. The antibiotic susceptibility of
these isolates was tested and their whole genome sequences were analysed for
acquired resistance genes and gene mutations. CgMLST analysis was performed
using an <i>ad hoc</i> scheme including 3664 genes. The DNA sequence of <i>mgrB</i>
(GI:695277517), and <i>phoQ</i> (GI:6938724) were used as reference for
detecting gene mutations associated with colistin resistance. Insertion
sequences were identified using Isfinder<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>: All KPC-KP
isolates belonged to ST258 with exception of one ST147 isolate. From 2014, 6
out of 23 isolates were pan-resistant to all antibiotics, compared to none out
of 11 isolates from 2010. The characteristics and antibiotic susceptibility of
pan-resistant isolates are presented in Table 1. Colistin-resistance was
associated with mutations in <i>mgrB</i>, which were present in all
pan-resistant isolates. CgMLST analysis showed that isolates in general
clustered by the hospital in to which patients were admitted (see Figure 1).
Pan-resistant isolates were closely related and appeared in a cluster of four
isolates, and a separate set of 2. Identical insertion sequences ISL3 inserted
in nucleotide position 12 in <i>mgrB</i> were detected in the isolates of the
cluster of four.&nbsp;<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> KPC-KP is
rapidly evolving to pan-resistance in Crete. We identified molecular resistance
markers for pan-resistant isolates, and showed that cgMLST is a promising tool
for molecular fingerprinting of KPC-KP ST258.<o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<table class=MsoNormalTable border=0 cellpadding=0 width="100%"
 style='width:100.0%;mso-cellspacing:1.5pt;mso-yfti-tbllook:1184'>
 <tr style='mso-yfti-irow:0;mso-yfti-firstrow:yes'>
  <td width="5%" style='width:5.48%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><b style='mso-bidi-font-weight:normal'><span
  lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-fareast-language:PT'>Isolate<o:p></o:p></span></b></p>
  </td>
  <td width="7%" style='width:7.12%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><b style='mso-bidi-font-weight:normal'><span
  lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-fareast-language:PT'>Hospital<o:p></o:p></span></b></p>
  </td>
  <td width="7%" style='width:7.78%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><b style='mso-bidi-font-weight:normal'><span
  lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-fareast-language:PT'>Isolation
  date<o:p></o:p></span></b></p>
  </td>
  <td width="10%" style='width:10.16%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><b style='mso-bidi-font-weight:normal'><span
  lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-fareast-language:PT'>MIC
  MER(mg/L)<o:p></o:p></span></b></p>
  </td>
  <td width="9%" style='width:9.38%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><b style='mso-bidi-font-weight:normal'><span
  lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-fareast-language:PT'>MIC
  IMP(mg/L)<o:p></o:p></span></b></p>
  </td>
  <td width="18%" style='width:18.0%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><b style='mso-bidi-font-weight:normal'><span
  style='font-size:5.0pt;mso-bidi-font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Susceptiblity
  GN/SXT/CIPX/FOS<o:p></o:p></span></b></p>
  </td>
  <td width="9%" style='width:9.86%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><b style='mso-bidi-font-weight:normal'><span
  lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-fareast-language:PT'>MIC
  COL(mg/L)<o:p></o:p></span></b></p>
  </td>
  <td width="9%" style='width:9.18%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><b style='mso-bidi-font-weight:normal'><span
  lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-fareast-language:PT'>MIC
  TIG(mg/L)<o:p></o:p></span></b></p>
  </td>
  <td width="8%" style='width:8.24%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><b style='mso-bidi-font-weight:normal'><i><span
  lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:8.0pt;mso-fareast-font-family:
  "Times New Roman";mso-bidi-font-family:Arial;mso-fareast-language:PT'>Bla-</span></i></b><b
  style='mso-bidi-font-weight:normal'><span lang=DE-CH style='font-size:5.0pt;
  mso-bidi-font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'>genes<o:p></o:p></span></b></p>
  </td>
  <td width="11%" style='width:11.2%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><b style='mso-bidi-font-weight:normal'><span
  style='font-size:5.0pt;mso-bidi-font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:PT'>Mutations
  in <i>mgrB</i> or <i>phoQ</i><o:p></o:p></span></b></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:1'>
  <td width="5%" style='width:5.48%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>545<o:p></o:p></span></p>
  </td>
  <td width="7%" style='width:7.12%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>C<o:p></o:p></span></p>
  </td>
  <td width="7%" style='width:7.78%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>31-08-13<o:p></o:p></span></p>
  </td>
  <td width="10%" style='width:10.16%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>&gt;16 R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.38%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>&gt;16 R<o:p></o:p></span></p>
  </td>
  <td width="18%" style='width:18.0%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>I/R/R/R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.86%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>16 R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.18%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=DE-CH
  style='font-size:5.0pt;mso-bidi-font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'><br>
  1.5 I<o:p></o:p></span></p>
  </td>
  <td width="8%" style='width:8.24%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>KPC-2 OXA-9 SHV-12<o:p></o:p></span></p>
  </td>
  <td width="11%" style='width:11.2%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>IS5 91<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:2'>
  <td width="5%" style='width:5.48%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>551<o:p></o:p></span></p>
  </td>
  <td width="7%" style='width:7.12%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>C<o:p></o:p></span></p>
  </td>
  <td width="7%" style='width:7.78%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>17-03-14<o:p></o:p></span></p>
  </td>
  <td width="10%" style='width:10.16%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>&gt;16 R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.38%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>&gt;16 R<o:p></o:p></span></p>
  </td>
  <td width="18%" style='width:18.0%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>I/R/R/R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.86%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>12 R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.18%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='text-align:center'><span lang=DE-CH
  style='font-size:5.0pt;mso-bidi-font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
  mso-bidi-font-family:Arial;mso-fareast-language:PT'><br>
  1.5 I<o:p></o:p></span></p>
  </td>
  <td width="8%" style='width:8.24%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>KPC-2 OXA-9 SHV-11<o:p></o:p></span></p>
  </td>
  <td width="11%" style='width:11.2%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>ISL3 11<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:3'>
  <td width="5%" style='width:5.48%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>541<o:p></o:p></span></p>
  </td>
  <td width="7%" style='width:7.12%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>A<o:p></o:p></span></p>
  </td>
  <td width="7%" style='width:7.78%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>14-04-14<o:p></o:p></span></p>
  </td>
  <td width="10%" style='width:10.16%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>&gt;16 R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.38%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>4 I<o:p></o:p></span></p>
  </td>
  <td width="18%" style='width:18.0%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>I/R/R/R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.86%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>6 R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.18%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>2 I<o:p></o:p></span></p>
  </td>
  <td width="8%" style='width:8.24%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>KPC-2 OXA-9 SHV-12<o:p></o:p></span></p>
  </td>
  <td width="11%" style='width:11.2%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>ISL3 12<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:4'>
  <td width="5%" style='width:5.48%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>543<o:p></o:p></span></p>
  </td>
  <td width="7%" style='width:7.12%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>A<o:p></o:p></span></p>
  </td>
  <td width="7%" style='width:7.78%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>14-04-14<o:p></o:p></span></p>
  </td>
  <td width="10%" style='width:10.16%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>&gt;16 R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.38%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>8 I<o:p></o:p></span></p>
  </td>
  <td width="18%" style='width:18.0%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>I/R/R/R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.86%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>4 R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.18%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>2 I<o:p></o:p></span></p>
  </td>
  <td width="8%" style='width:8.24%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>KPC-2 OXA-9 SHV-12<o:p></o:p></span></p>
  </td>
  <td width="11%" style='width:11.2%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>ISL3 12<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:5'>
  <td width="5%" style='width:5.48%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>531<o:p></o:p></span></p>
  </td>
  <td width="7%" style='width:7.12%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>A<o:p></o:p></span></p>
  </td>
  <td width="7%" style='width:7.78%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>16-04-14<o:p></o:p></span></p>
  </td>
  <td width="10%" style='width:10.16%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>&gt;16 R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.38%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>8 I<o:p></o:p></span></p>
  </td>
  <td width="18%" style='width:18.0%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>I/R/R/R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.86%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>32 R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.18%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>2 I<o:p></o:p></span></p>
  </td>
  <td width="8%" style='width:8.24%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>KPC-2 OXA-9 SHV-12<o:p></o:p></span></p>
  </td>
  <td width="11%" style='width:11.2%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>ISL3 12<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:6;mso-yfti-lastrow:yes'>
  <td width="5%" style='width:5.48%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>552<o:p></o:p></span></p>
  </td>
  <td width="7%" style='width:7.12%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>B<o:p></o:p></span></p>
  </td>
  <td width="7%" style='width:7.78%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>03-07-14<o:p></o:p></span></p>
  </td>
  <td width="10%" style='width:10.16%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>&gt;16 R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.38%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>&gt;16 R<o:p></o:p></span></p>
  </td>
  <td width="18%" style='width:18.0%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>I/R/R/R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.86%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>12 R<o:p></o:p></span></p>
  </td>
  <td width="9%" style='width:9.18%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>2 I<o:p></o:p></span></p>
  </td>
  <td width="8%" style='width:8.24%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>KPC-2 OXA-9 SHV-12<o:p></o:p></span></p>
  </td>
  <td width="11%" style='width:11.2%;padding:.75pt .75pt .75pt .75pt'>
  <p class=MsoNormal align=center style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
  auto;text-align:center'><span lang=DE-CH style='font-size:5.0pt;mso-bidi-font-size:
  8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
  mso-fareast-language:PT'>ISL3 12<o:p></o:p></span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;mso-ansi-language:EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Imagem_x0020_56" o:spid="_x0000_i1028" type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/20imagem_2560108707_0.jpg"
 style='width:272pt;height:351pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S8_files/image005.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/20imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=274 height=353
src="public/abstracts/Webpages/PO_S8_files/image006.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/20imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_56"><![endif]></span><span style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO52 - Characterization
Of Uncommon Clones Of Carbapenemase-producing <i style='mso-bidi-font-style:
normal'>Acinetobacter baumannii</i> From Nosocomial Settings In Tunis (Tunisia)<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Aymen Mabrouk<sup>1</sup>; Filipa Grosso<sup>2</sup>; Liliana Silva<sup>3</sup>;
Clara Sousa<sup>4</sup>; Wafa Achour<sup>1</sup>; Assia Ben Hassen<sup>1</sup>;
Luísa Peixe<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>Laboratories UR12ES02-The National Bone Marrow
Transplant Centre,<sup>2</sup> University of Carthage, Faculty of Sciences of
Bizerte,<sup>3</sup> UCIBIO-REQUIMTE- Laboratório de Microbiologia, Faculdade
de Farmácia, Universidade do Porto,<sup>4</sup> UCIBIO-REQUIMTE, Laboratório de
Microbiologia, Faculdade de Farmácia, Universidade do Porto,<sup>5</sup> ESALD,
Instituto Politécnico de Castelo Branco,<sup>6</sup> CEB-Centro de Engenharia
Biológica, Universidade do Minho<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>Keywords: CHDL, FTIR-ATR, MLST</span><span lang=EN-GB style='font-size:
8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-GB;mso-fareast-language:
PT'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><i><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Acinetobacter
baumannii&nbsp;</span></i><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'>is commonly associated with nosocomial
infections and severe outbreaks. Worldwide dissemination of carbapenem
resistant <i>A. baumannii</i>, has been mainly associated with
carbapenem-hydrolyzing class D-beta lactamases (CHDL) limiting the therapeutic
options. In Northern Africa, and in particular in Tunisia, the occurrence of
carbapenem-resistant <i>A. baumannii</i> (CRAB) has also been associated with
CHDLs, mainly OXA-23 and OXA-58-like enzymes. In addition, GES-11 and GES-14
have also been reported. However, little is known about the clones driving the
carbapenemases dissemination, with the exception of GES-14/ST499 (personal
communication, Abstr. 25<sup>th</sup> ECCMID, 2015).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>In this study we
assessed the occurrence of CHDL and associated clones on a CRAB collection from
nosocomial settings in Tunis, Tunisia.The suitability of FTIR and chemometric
analysis for <i>A. baumannii</i> clonal characterization in a barely studied
region was also evaluated.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Methods: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>This study
included 192 CRAB clinical isolates obtained from different biological samples
from inpatients attending the<b> </b>Neonatal Intensive Care Unit of the
Rabta´s Hospital or The National Bone Marrow Transplant Centre in Tunis,
Tunisia (2004-2013). Antimicrobial susceptibility was determined by disc
diffusion method. CHDLs search was performed by PCR and confirmed by
sequencing. &nbsp;Fourier transform infrared spectroscopy with attenuated total
reflectance (FTIR-ATR) spectra were acquired and modeled by partial least
squares discriminant analysis (PLSDA) for all isolates, a methodology
previously validated for depicting <i>A. baumannii</i> clones common in our
region. Representative isolates from the resulted PLSDA clusters were selected
for MLST analysis, according with the Oxford MLST database.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Isolates<b> </b>were
resistant to all b-lactams, presenting variable susceptibility to quinolones
and aminoglycosides. OXA-23 was detected in 88 isolates (45,8%, 2007-2013) and OXA-58
was detected in 104 isolates (54,2%, 2004-2010). PLSDA regression model using
the 1500–900 cm−1 spectral region revealed two main clusters, explained by the
first latent variable (LV), which represents the higher spectral diversity.
MLST analysis of representative isolates from each cluster revealed the
association of OXA-23 producing isolates with ST945 while OXA-58 were
associated with ST441. These are unusual STs that have been described in Saudi
Arabia (ST441) and Iraq (ST945), according with MLST database. They are single
locus variant (SLV), differing only on <i>gpi</i> allele. The remaining LVs
allowed the description of small sub-cluster that included both OXA-23 and
OXA-58 producing isolates and associated also with an infrequent clone, ST282.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>&nbsp;This
is the first study characterizing CHDL <i>A. baumannii</i> clones in Tunisia.
FTIR-ATR spectroscopy coupled with chemometric tools allowed a preliminary,
quick, and low cost, discrimination of the different clones. These results
which were further supported by MLST confirm this methodology as an important
tool for epidemiology and outbreak monitoring. Despite the long period of time
analyzed, it was observed the persistence of very few clones. Interestingly,
they are not associated with the most disseminated STs worldwide, being
previously identified only in the Middle East, which unveils a regional ecology
that worth to be explored.</span><span lang=EN-GB style='font-size:8.0pt;
mso-bidi-font-size:11.0pt;mso-ansi-language:EN-GB;mso-fareast-language:PT'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
<o:p></o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO53 - Phylogenetic
Analysis Of Severe Fever With Thrombocytopenia Syndrome Virus And Migratory
Bird Routes Between China, South Korea, And Japan<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Ji-Hoon Kang<sup>1</sup>; Sang Taek Heo<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Jeju National University<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Severe fever with
thrombocytopenia syndrome (SFTS) is a tick-borne viral disease. The SFTS virus
(SFTSV) has been detected in the <i>Haemaphysalis longicornis</i>, which acts as
a transmission host between animals and humans. SFTSV was first confirmed in
China in 2009 and has also been circulating in Japan and South Korea. However,
it is not known if a genetic connection exists between the viruses in these
regions and, if so, how SFTSV is transmitted across China, South Korea, and
Japan. We therefore hypothesize that the SFTSV in South Korea share common
phylogenetic origins with samples from China and Japan. Further, we postulate
that migratory birds, well-known carriers of the tick <i>H. longicornis</i>,
are a potential source of SFTSV transmission across countries. Our phylogenetic
analysis results show that the SFTSV isolates in South Korea were similar to
isolates from Japan and China. We connect this with previous work showing that
SFTSV-infected <i>H. longicornis</i> were found in China, South Korea, and
Japan. In addition, <i>H. longicornis</i> were found on migratory birds. The
migratory bird routes and the distribution of <i>H. longicornis</i> are
concurrent with the occurrence of SFTSV. Therefore, we suggest that migratory
birds play an important role in dispersing <i>H. longicornis</i>-borne SFTSV.</span><b
style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></b></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:Cambria;
mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;line-height:115%;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO54 - Burden
Evaluation Of Hand Soap’s Contamination By <i style='mso-bidi-font-style:normal'>Pseudomonas
aeruginosa</i> In A Tertiary Care Hospital Using Whole Genome Sequencing<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Bárbara Magalhães<sup>1</sup>; Mohamed M. H. Abdelbary<sup>1</sup>; Guy
Prod´Hom<sup>2</sup>; Gilbert Greub<sup>2</sup>; Jean-Blaise Wasserfallen<sup>3</sup>;
Patrick Genoud<sup>4</sup>; Giorgio Zanetti<sup>1</sup>; Laurence Senn<sup>1</sup>;
Dominique S. Blanc<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Service of Hospital Preventive Medicine, Lausanne
University Hospital,<sup>2</sup> Institue of Microbiology, Lausanne University
Hospital,<sup>3</sup> Medical directorate,<sup>4</sup> Nurse directorate<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Background: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>An
investigation of potential sources of <i>Pseudomonas aeruginosa</i> in the
intensive-care unit (ICU) showed that the liquid hand soap was highly
contaminated with this bacterium. From August 2012 to January 2013, all
isolates retrieved from clinical specimens and from contaminated soap
containers were analysed by Double Locus Sequence Typing (DLST). This molecular
typing method showed that three patients harboured the same DLST type, 13-31,
as the one observed in the majority of the tested contaminated soap containers
(17/20). Whole Genome Sequencing (WGS) was used to confirm or rule out the
burden of hand soap’s contamination on patients’ infection.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Material and
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>A set of ten <i>P. aeruginosa</i> isolates belonging to DLST type 13-31 was
analyzed with WGS. Among these, two were retrieved from two different soap
batches, three from the patients suspected to be contaminated by the soap, and
5 from unrelated strains retrieved by routine surveillance of <i>P. aeruginosa</i>
in the hospital. The isolates’ sequence type (ST) was assigned from the short
reads data. With Snippy software, the core genome alignment of all ten isolates
was acquired, which in turn was used to construct the maximum likelihood tree.
Subsequently, regions of high SNPs density (suggestive of recombination) were
excluded using the Gubbins software.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>All ten
sequenced isolates belonged to ST 155, which was fully concordant with DLST
results. The phylogenetic tree showed the occurrence of three major clades (A,
B, and C). Clade A comprised half of the analysed isolates, which were
retrieved from 2003 to 2014. Two isolates that were retrieved from infected
patients clustered in clade A, while the third patient’s isolate was part of clade
B. In contrast, both soap isolates were clustered in clade C. A high number of
core SNPs differences was detected between the three patients of the study, and
the two soap isolates. No close genetic relation was observed between
contaminated soap isolates and the patients included in the investigation.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>The three
patients suspected to be contaminated by the soap, clustered in different
clades from the one harbouring the soap isolates. Additionally, a high number
of SNPs differences was observed between patient and soap isolates, hence, it
was possible to exclude patients’ infection due to contamination of the soap.
WGS enabled the exclusion of a possible burden on our hospital due to the
soap’s contamination, which would not be achieved using other classical
molecular typing methods.</span><span style='font-size:8.0pt;mso-bidi-font-size:
11.0pt;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;line-height:115%;
font-family:Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-US;mso-fareast-language:PT;mso-bidi-language:AR-SA'><br
clear=all style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><span style='font-size:8.0pt;mso-bidi-font-size:11.0pt;
line-height:115%;mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO55 - Molecular
Surveillance And Emergence Of Carbapenemases At A University Hospital In
Germany Over A Five Year-period<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Andreas F. Wendel<sup>1</sup>; Colin R. MacKenzie<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Institute of Medical Microbiology and Hospital Hygiene<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Introduction:</span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'> Resistance
to the clinically relevant broad-spectrum antibiotic group of carbapenems
driven by carbapenemase enzymes is emerging in Gram-negative bacteria. Germany
is a low-prevalence country, though there is some regional spread (e.g. of KPC
enyzmes) and numbers and diversity of carbapenemase genes detected are
increasing. Little surveillance data is available at the local level. A
molecular surveillance of Gram-negative multidrug-resistant
carbapenem-non-susceptible organisms was established at the Institute of
Medical Microbiology and Hospital Hygiene in Düsseldorf (Germany) in 2010.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Materials |
Methods: </span></b><span style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:
PT'>We analyzed prospectively all Gram-negative isolates (<i>Enterobacteriaceae</i>,
<i>Acinetobacter</i> spp. and <i>Pseudomonas</i> spp.) non-susceptible to
imipenem and/or meropenem (<i>Pseudomonas</i> spp., only if additionally
non-susceptible to ceftazidime) from clinical and screening specimens.
Real-time and conventional PCRs targeting relevant carbapenemase genes (<i>bla</i><sub>IMP-1</sub>,
<i>bla</i><sub>VIM-1-like</sub>,<i> bla</i><sub>VIM-2-like</sub>,<i> bla</i><sub>GIM-1</sub>,
<i>bla</i><sub>NDM</sub>, <i>bla</i><sub>KPC</sub>, <i>bla</i><sub>OXA-48</sub>
and <i>bla</i><sub>GES</sub>; for <i>Acinetobacter</i> spp. (since 2012)
additonally: <i>bla</i><sub>OXA-23</sub>, <i>bla</i><sub>OXA-24</sub>,<sub> </sub><i>bla</i><sub>OXA-51</sub>,
<i>bla</i><sub>OXA-58</sub>,<i> bla</i><sub>OXA-143</sub> and <i>bla</i><sub>OXA-235</sub>)
were used. Epidemiologically related isolates were further genotyped by several
methods (RAPD, DiversiLab, PFGE and MLST) and subjected to S1-nuclease for
plasmid analysis.&nbsp;<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Results: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Approximately
500 isolates were analyzed, of which 112 were positive as follows: <i>Pseudomonas</i>
spp., <i>bla</i><sub>GIM-1</sub> (n=32), <i>bla</i><sub>VIM-2</sub> (n=14), <i>bla</i><sub>VIM-1</sub>
(n=1); <i>Enterobacteriaceae</i>, <i>bla</i><sub>GIM-1</sub> (n=23), <i>bla</i><sub>OXA-48</sub>
(n=9), <i>bla</i><sub>VIM</sub> (n=4),<sub> </sub><i>bla</i><sub>NDM-1</sub>
(n=3), <i>bla</i><sub>KPC</sub> (n=3), <i>bla</i><sub>KPC-1</sub>/<i>bla</i><sub>VIM
</sub>(n=1); <i>Acinetobacter </i>spp., <i>bla</i><sub>OXA-23</sub>/<i>bla</i><sub>OXA-51
</sub>(n=17),<sub> </sub><i>bla</i><sub>OXA-24</sub>/<i>bla</i><sub>OXA-51 </sub>(n=2),
<i>bla</i><sub>NDM-1</sub>/<i>bla</i><sub>OXA-51 </sub>(n=3), This data reveals
an endemic situation involving GIM-1-producing organisms and local acquisition
of at least three different clusters of VIM-2-producing <i>P. aeruginosa</i>.
Detection of OXA-23, OXA-24, OXA-48, KPC or NDM-1 was clearly associated with a
previous hospital stay abroad, Two patients were colonized/infected with
several species carrying the same carbapenemase gene (<i>bla</i><sub>VIM-1</sub>resp.
<i>bla</i><sub>NDM-1</sub>) proven to be plasmid mediated. Two patients with a
hospital stay abroad (Marocco and Greece) carried three different
carbapenemase-producing organisms. The first emergence of NDM-1 was in 2012
(interestingly community-acquired) and of KPC in 2013 (patient transferred from
Greece). We observed a shift in multi-drug resistant <i>A. baumanii</i> from <i>bla</i><sub>OXA-58
</sub>to <i>bla</i><sub>OXA-23 </sub>compared to strains collected before 2010
and analysed retrospectively..<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>Conclusion: </span></b><span
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-US;mso-fareast-language:PT'>This data
gives insight to the changing epidemiology of carbapenemases in a tertiary care
center in Germany over the last 5 years. Carbapenemase-mediated carbapenem resistance
on the whole is still uncommon, but is increasing and becoming more diverse. We
were able to demonstrate a regional spread of <i>bla</i><sub>GIM-1 </sub>and<sub>
</sub><i>bla</i><sub>VIM-2 </sub>and the “genetic import” of several other
carbapenemase genes (<i>bla</i><sub>VIM</sub>,<sub> </sub><i>bla</i><sub>OXA-48</sub>,
<i>bla</i><sub>NDM-1</sub> and <i>bla</i><sub>KPC</sub>) from abroad. The
results support the need for a local molecular surveillance system.</span><span
style='font-size:8.0pt;mso-bidi-font-size:11.0pt;mso-ansi-language:EN-US;
mso-fareast-language:PT'><o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO57 - A Molecular
Survey Of <i style='mso-bidi-font-style:normal'>Campylobacter jejuni</i> And <i
style='mso-bidi-font-style:normal'>Campylobacter coli</i> Virulence And
Diversity<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Mahdi Ghorbanalizdgan<sup>1</sup>; BitaBakhshi<sup>2</sup>;
ShahinNajar-Peerayeh<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Department of Bacteriology, Faculty of Medical
Sciences, TarbiatModares University, Tehran, IR Iran,<sup>2</sup> Department of
Bacteriology, Faculty of Medical Sciences,Baqiyatallah University, Tehran,IR
Iran<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT;mso-bidi-font-weight:bold'>Keywords:</span><i><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> C. jejuni</span></i><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>,<i>C.coli</i>, genotyping, virulence genes</span><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal><b><span style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>The aim of this study was to determinethe prevalence of
virulence-associated genesandenterobacterial repetitive intergenic
consensus-PCR (ERIC-PCR) analysis of<i>Campylobacter </i>spp. isolated from
children with diarrhea in Iran.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Methods: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>A total of 200 stool specimens were obtained from children under 5 years
duringJuly 2012 to July 2013. Detection of <i>C. jejuni</i> and <i>C.coli</i>
wasperformed by standard biochemical and molecular methods.The presence of
virulence-associatedgenes and genetic diversity of isolateswas examined with
PCR andERIC-PCR analysis.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results</span></b><b
style='mso-bidi-font-weight:normal'><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>A total of 12 (6%) <i>Campylobacter</i> spp. were isolated frompatients
including10 (4.5%) <i>C. jejuni</i>and 2 (1.5%)<i>C.coli</i>.The <i>fla</i>A, <i>cad</i>F
and <i>cia</i>Bgenes were present in 100% of isolates while no Plasmid<i> vir</i>B11
gene was present their genome. The prevalence of <i>iam</i> was 100% among<i>C.coli</i>and
was not detected in<i>C.jejuni</i>isolates. The distribution of both<i>pld</i>Aand
the genes associated with CDTwas 58.3% in<i>C. jejuni</i> isolates. Seven
distinct ERIC-PCR profiles in three clusters were distinguished by ERIC-PCR
analysis. Genotyping analysis showed a relative correlation with geographic
location of patients and virulence gene content of isolates.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion</span></b><b
style='mso-bidi-font-weight:normal'><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>To our knowledge, this is the first molecular survey of <i>Campylobacter</i>
spp. in Iran which concerns genotyping and virulence gene content of both <i>C.
jejuni</i> and <i>C. coli</i>. ERIC-PCR revealed appropriate discriminatory
power for clustering of <i>C. jejuni</i> isolates with identical virulence gene
content. However, more studies are needed to clearlyunderstand thepathogenesis
properties of specific genotypes.<o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO58 - Emergence And
Dissemination Of A Linezolid-resistant <i style='mso-bidi-font-style:normal'>Staphylococcus
capitis </i>Clone In Europe<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>P Martins Simoes<sup>1</sup>; M. Butin<sup>1</sup>; C. Dupieux<sup>1</sup>;
L. d´Anthouard<sup>1</sup>; D. Leyssene<sup>2</sup>; S. Bordes-Couecou<sup>3</sup>;
B. Pichon<sup>4</sup>; H. Meugnier<sup>1</sup>; N. Lemaitre<sup>5</sup>; F.
Schramm<sup>5</sup>; F. Vandenesch<sup>1</sup>; I. Spiliopoulou<sup>6</sup>;
H-L Hyyryläinen<sup>7</sup>; A. Kearns<sup>4</sup>; O. Dumitrescu<sup>1</sup>;
F. Laurent<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>National Reference Center for Staphylococci, Hospices
Civils de Lyon, CIRI INSERM U1111, Lyon 1 University,<sup>2</sup> Department of
Bacteriology, Centre Hospitalier de la Côte Basque,<sup>3</sup> Infection
Control Committee, Centre Hospitalier de la Côte Basque,<sup>4</sup> Public
Health England,<sup>5</sup> Department of Bacteriology,CHRU,,<sup>6</sup>
Department of Bacteriology,<sup>7</sup> National Institute for Health and
Welfare/ THL<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background</span></b><b
style='mso-bidi-font-weight:normal'><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> <i>Staphylococcus capitis</i> is a low virulence staphylococcus coagulase
negative (SCoN) seldom involved in human diseases and classically considered as
a contaminant when isolated from clinical samples. We investigated clinical,
microbiological and genetic characteristics of linezolid-resistant (LNZ-R) <i>S.
capitis</i> isolates from France, Greece and Finland.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Material/methods:
</span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>All LZD-R resistant <i>S. capitis</i> isolates
available in the collection of the French National Reference Center for
Staphylococci collection were included in this study (n=9, Bayonne, Strasbourg,
Lille). These isolates were compared with international LZD-R <i>S. capitis</i>
strains collected from the Staphylococcocal Research Laboratories of Greece
(n=10, Patras) and Finland (n=1, Turku). Antimicrobial susceptibility tests were
performed using the standard agar diffusion method and Etest (for daptomycin,
LZD, teicoplanin, vancomycin) for all isolates. Presence of the cfr gene and
mutations in the 23S rRNA loci were investigated by PCR and sequencing. Genetic
relationships of isolates were determined by PFGE using <i>Sma</i>I restriction
enzyme. Finally, for the French (n=9) and Greek isolates (n=1), whole genome
sequencing (Illumina HiSeq) and genomic comparison were performed.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>The 9 french isolates were isolated between June 2013 and December 2014
from 9 adult patients hospitalized in 3 distant ICUs (2 to 5 strains per
center). Only 5 of the 9 patients had previously received LZD therapy. These 9
French isolates as well as the 10 Greek ones and the isolate from Finland were all
resistant to methicillin, aminoglycosides and linezolid (MIC &gt;24mg/L).
Additional resistances to vancomycin, teicoplanin and daptomycin were observed
in 1, 3 and 4 isolates, respectively. The <i>cfr</i> gene was absent in all
isolates while two mutations (T2319C and G2576T) in the 23S rRNA were found in
all isolates. Moreover, they belong to a same clone as defined by their shared
PFGE profile. Additionally, for the 9 French and 1 Greek isolates, their high
genetic relationship was further confirmed by WGS: these isolates show at most
216 SNPs (over 1,484,801 shared positions) between the 10 isolates, versus a
mean of 17,851 SNPs when compared with 7 publicly available, unrelated<i> S.
capitis</i> genomes.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusions:
</span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>In this study we highlight for the first time
the European dissemination of a LZD-R <i>S. capitis</i> clone, harboring 2
mutations in the 23s rRNA, and present in at least 3 French, 1 Greek and 1
Finnish hospitals. The G2576T mutation, observed in all isolates, is
classically found in coagulase negative staphylococci. Although the role of
mutation T2319C (not previously described in the literature) in the LZD-R
phenotype remains to be definitly established. The emergence and inter-hospital
diffusion of this LZD-R clone involved in infections in several European hospitals
is both worrisome and unexpected as <i>S. capitis</i> is usually known as a
very low-virulent species. The absence of linezolid prior therapy in 4 of the
French patients suggests a nosocomial transmission within hospital. However,
the inter-hospital dissemination pathways of this clone at such a large
geographical scale is still unexplained and requires further investigations.
Finally, we actively continue to collect LZD-R S. capitis isolates from other
European countries, to better determine the european distribution of this
clone.</span><b style='mso-bidi-font-weight:normal'><span style='mso-ansi-language:
EN-US;mso-fareast-language:PT'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
<o:p></o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO59 - Whole Genome
Sequencing Data Confirm A Transmission Of <i style='mso-bidi-font-style:normal'>Salmonella
enterica</i> Serovar Typhimurium Phage Type Dt41 In Danish Poultry Production<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Ann-Sofie Hintzmann<sup>1</sup>; Gitte Sørensen<sup>1</sup>; Dorte Lau
Baggesen<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>National Food Institute<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Introduction</span></b><b
style='mso-bidi-font-weight:normal'><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Salmonella is the causes numerous foodborne outbreaks every year in both
developing and developed countries. It spreads through the entire food
production chain and thus can be difficult to manage. One specific serotype <i>Salmonella
enterica</i> serovar Typhimurium phage type DT41 (S. Typhimurium DT41) has
previously been isolated from broiler breeder flocks in Denmark, particularly
in older poultry.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Materials |
Methods: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>Danish S. Typhimurium DT41 isolates from 1999 to
2014 covering poultry (N=50), other birds (N=17), other animals (N=6) and other
sources (N=3) were whole genome sequenced on an Illumina MiSeq platform
(Illumina) and compared to Multi Locus Variable number tandem repeats Analysis
(MLVA) results. Isolates were either epidemiologically linked or considered as
“background” to increase the resolution of the analysis. The Multi Locus
Sequence Type (MLST) and Single Nucleotide Polymorphism (SNP) phylogeny was
constructed using tools available at </span><span lang=EN-GB style='font-size:
8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-GB;mso-fareast-language:PT'>Center for Genomic
Epidemiology (</span><span lang=DE-CH><a
href="https://www.eventgest.com/GEST/%C2%B4%C2%B4http:/www.genomicepidemiology.org%C2%B4%C2%B4"><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:windowtext;mso-ansi-language:EN-GB;mso-fareast-language:
PT;text-decoration:none;text-underline:none'>www.genomicepidemiology.org</span></a></span><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;mso-ansi-language:EN-GB;mso-fareast-language:PT'>)
and BioNumerics 7.1 (Applied <span style='color:black'>Maths).<o:p></o:p></span></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Based on epidemiological data it is possible that a transmission had
occurred in 2001 from layers to hatchery and broiler breeding farm. In the SNP
tree a cluster of 9 poultry isolates were identified, originating from 2001.
All isolated were MLST ST19 and had a minimum and maximum distance of 0 and 6
SNP, respectively, and a maximum distance of 123 SNPs over the entire tree. The
isolates shared a MLVA profile that was otherwise not seen in the entire
dataset, confirming a link between the isolates. Within the cluster there are
two identical isolates, where one is from a hatchery and the other is from a
broiler breeding flock, suggesting that a direct transmission event has
occurred.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>A natural source to S. Typhimurium DT41 has still not been found; however,
just outside the 2001 cluster is an isolate from a Swedish herring gull. It
would therefore be interesting to study this link further to see if there is a
link to reservoir of wild birds. More research is planned in an attempt to
identify a source to this specific Salmonella phage type.</span><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO60 - Dominance Of <i
style='mso-bidi-font-style:normal'>Escherichia coli</i> ST131 H30 Subclone In A
Portuguese Long Term Care Facility<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Ana Constança Mendes<sup>1</sup>; Carla Rodrigues<sup>1</sup>; Helena Ramos<sup>2</sup>;
Luísa Peixe<sup>1</sup>; Ângela Novais<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>UCIBIO@REQUIMTE. Laboratório Microbiologia, Faculdade
de Farmácia, Universidade do Porto,<sup>2</sup> Centro Hospitalar do Porto<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>The <i>Escherichia coli</i> B2-ST131 clone is nowadays one of the
predominant lineages causing extraintestinal infections in healthcare
associated settings, and long-term care facilities (LTCF) have been established
as important reservoirs of ST131 in some countries. Genome wide studies
revealed a subclonal structure within <i>E. coli</i> ST131, but the prevalence
and distribution of the different subclones (being the <i>fimH30</i> or the
more extensively resistant <i>fimH30</i>-Rx the predominant ones) in different
settings and hosts is still poorly understood. We aim to assess the faecal
carriage rate of ESBL- and/or carbapenemase-producing <i>E. coli</i> clones and
subclones among LTCF residents in Portugal, where different ST131 <i>E. coli</i>
subclones have been associated with hospital-acquired infections but situation
in LTCF is still underexplored.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Materials |
Methods: A pilot study was designed in whichfaecal samples from </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>all residents (n=24, 17 females and 7 males) of a Portuguese LTCF were
collected in July 2015. Eighty-three percent of the residents were &gt;70 years
old, 46% were previously hospitalized and 42% received recent (&lt;3
months=&quot;&quot; antibiotic=&quot;&quot; treatment=&quot;&quot;
samples=&quot;&quot; were=&quot;&quot; suspended=&quot;&quot; in=&quot;&quot;
2ml=&quot;&quot; of=&quot;&quot; saline=&quot;&quot; solution=&quot;&quot;
and=&quot;&quot; 200=&quot;&quot; l=&quot;&quot; seeded=&quot;&quot;
chromid=&quot;&quot; sup=&quot;&quot;&gt;® ESBL directly or ChromID<sup>®</sup>
Carba SMART plates after enrichment of 100µl of faecal suspension in APP
supplemented with a meropenem disk (37ºC/18h) and further incubated
(37ºC/24-48h). Thirty presumptive ESBL-producing <i>E. coli </i>isolates
presenting different morphotypes/ESBL chromID plate (n=1-6) were selected, and
its identification assessed by MALDI-TOF MS or PCR. ESBL production was
confirmed in all isolates by PCR (<i>bla</i><sub>CTX-M</sub>) and sequencing,
and <i>E. coli</i> ST131 clone and subclones (H30, H30 Rx) were identified by
PCR.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>ESBL-producing <i>E. coli</i> were detected in 63% (n=15/24) of the
samples, whereas carbapenemase producers were not detected. Isolates produced
almost exclusively CTX-M-15 (n=29/30; 97%) and rarely CTX-M-1 (n=1/30; 3%). All
CTX-M-producing <i>E. coli</i> belonged to the H30 lineage of the pandemic
B2-ST131 O25b:H4. The <i>E. coli</i> ST131 clade H30-Rx represented 87%
(n=26/30) of the ST131 population, all of them being CTX-M-15 producers, while
variability (H30 Rx and H30 non-Rx) was observed in two samples. ESBL carriers
were previously hospitalized in different hospitals or had previously been
admitted to different long-term care facilities, whereas only 22% (n=2/9) of
non-carriers had been previously hospitalized. Recent antibiotic treatment
existed in 33% (n=5/15) and 56% (n=5/9) of carriers and non-carriers,
respectively.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Despite the reduced sample size, this pilot study suggests a high
colonization rate by ESBL-producing <i>E. coli</i> among Portuguese LTCF
residents, dominated by the CTX-M-15-producing B2-ST131 H30 lineage. Our data
further support the role of LTCFs as reservoirs of the ST131 <i>E. coli</i> in
Portugal and suggest a high potential for transmissibility and persistence of
the H30 lineage in this population, anticipating the need for further studies.</span><span
lang=EN-GB style='mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span lang=EN-GB style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-GB;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span lang=EN-GB
style='mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO61 - Salmonella
Surveillance By Whole Genome Sequencing: WG-MLST And WG-SNP Approaches Using
The Bionumerics Platform.<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Hannes Pouseele<sup>1</sup>; Katrien De Bruyne<sup>1</sup>; Johan Goris<sup>1</sup>;
Bruno Pot<sup>1</sup>; Koen Janssens<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Applied Maths NV<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Introduction</span></b><b
style='mso-bidi-font-weight:normal'><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>The standard for molecular surveillance of bacterial foodborne diseases has
been PFGE for many years. With the advent of high-throughput Whole Genome
Sequencing (WGS), the key challenge remains the rapid and automated processing
of WGS data, obtaining higher resolution compared to traditional methods. Whole
genome Multi Locus Sequence Typing (wgMLST) and the availability of reliable
and easy to use workflows allow for outbreak cluster detection in WGS-based
surveillance. Similar to MLST, the wgMLST method requires a schema describing
the loci and alleles. In this study, we report on the development and
validation of a species-wide wgMLST schema for&nbsp;<i>Salmonella enterica</i>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Material and
methods: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>The&nbsp;<i>Salmonella enterica</i>&nbsp;wgMLST
schema was created based on 163 genome and plasmid sequences, representing the
major serotypes. An in-house developed schema creation procedure used a
sampling-based multi-reciprocal BLAST search to determine those sets of alleles
that make up the stable loci in the accessory, i.e. pan genome. Next, a
per-locus allele assessment procedure determined the central prototype allele
and thus the locus definition. The final schema contained 11636 loci, covering
94% on average of the CDS defined on the reference genomes.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Allele calls
for each of the loci in the schema were a consensus of two approaches: (1) an
assembly-based allele calling, done through a BLAST based procedure on de novo
assembled genomes and (2) an assembly-free allele calling, using a k-mer
approach directly on the reads.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Validation
of the wgMLST schema was performed on publicly available whole genome
sequences, described in outbreak reports or WGS studies. Since in the latter
studies mainly whole genome single nucleotide polymorphism (wgSNP) based
approaches were used, we re-analyzed the data with an in-house developed wgSNP
tool as additional reference.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>The&nbsp;<i>Salmonella enterica</i>&nbsp;schema validation includes
analysis of CDS and base recovery rates, and within and between locus diversity
statistics. After validation, wgMLST analysis is performed on data from public
studies and outbreak reports. Without exception, this analysis reliably
identified isolates belonging to documented&nbsp;<i>Salmonella</i>&nbsp;outbreaks.
wgMLST overall exhibited a very high resolution, obviously much higher than
PFGE but also surpassing core SNP approaches. Moreover, the resolution of
wgMLST can be tuned by switching to (local) typing schemes e.g. a core MLST
schema or even the traditional 7-gene MLST schema. Using a set of quality
criteria, problematic WGS runs could be clearly identified. The allele calling
procedure proved to be robust, since the number of called alleles remains
relatively unaffected by noticeable drops in sequence coverage or N50. For a
few samples, the analysis provided clear evidence for contamination (mixed
cultures).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>A good
correlation was observed between wgMLST and wgSNP results, with the resolution
of wgSNP only surpassing that of wgMLST when a cluster-specific reference
sequence was chosen.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>The developed wgMLST schema for&nbsp;<i>Salmonella</i>&nbsp;offers ample
resolution for outbreak cluster detection and, in combination with BioNumerics’
analysis tools, enables a species-wide sample comparison in routine
surveillance. While wgMLST and wgSNP essentially capture the same signal,
wgMLST has the advantage of being reference-independent and hence can form the
basis for fast screening and more important, for a stable nomenclature to be
used internationally.</span><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO62 - Molecular
Epidemiology Of <i style='mso-bidi-font-style:normal'>Klebsiella pneumoniae</i>:
Multiclonal Dissemination Of CTX-M-15 Extended Spectrum Β-lactamase<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>C. Caneiras<sup>1</sup>; L. Lito<sup>2</sup>; J. Melo-Cristino<sup>3</sup>;
A. Duarte<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>iMed.UL - Research Institute for Medicines and
Pharmaceutical Sciences, Faculty of Pharmacy, University of Lisbon,<sup>2</sup>
Laboratory of Microbiology, Centro Hospitalar Lisboa Norte,<sup>3</sup>
Laboratory of Microbiology, Centro Hospitalar Lisboa Norte,<sup>4</sup>
Institute of Microbiology, Institute of Molecular Medicine, Faculty of
Medicine, University of Lisbon<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT;mso-bidi-font-weight:
bold'>Keywords</span><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>: CTX-M-15, <i>Klebsiella pneumoniae, </i>epidemiology<o:p></o:p></span></p>

<p class=MsoNormal><b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background: </span></b><i><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Klebsiella pneumoniae</span></i><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'> is now recognized
as an urgent threat to public health because of the emergence of
multidrug-resistant strains associated with hospital outbreaks and severe
community-acquired infections. However, little is known about the emergence of
clinically important clones. The aim of this study was to verify the clonal
relationship among <i>Klebsiella pneumoniae </i>isolates producing CTX-M-15
extended spectrum </span><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT'>β</span><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>-lactamase (ESBL) since 2003, in Portugal.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Materials |
Methods</span></b><b style='mso-bidi-font-weight:normal'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> This retrospective surveillance study included 740 <i>K. pneumoniae</i>
clinical isolates collected from 5 different hospitals and 1 community
laboratory in Portugal, between 2003 and 2013. Antimicrobial susceptibility
testing was performed by disk diffusion and the results were interpreted
according to EUCAST guidelines. Genes encoding </span><span lang=DE-CH
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-fareast-language:PT'>β</span><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>-lactamases
including CTX-M, TEM, SHV, DHA, FOX and CMY were screened by PCR and confirmed
by sequencing. The common elements involved in the mobilization and expression
of <i>bla</i>CTX-M-15 gene, ISEcp1 and IS903, were analyzed by PCR with
specific primers. Plasmid replicon was used to identify the plasmids
incompatibility groups. The clonal relationship was evaluated by M13
fingerprinting and multilocus sequence typing (MLST).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>All isolates had resistance to gentamicin, ciprofloxacin, cefepime,
ceftazidime, cefotaxime and showed synergic effect among clavulanic acid and
cephalosporins indicative of CTX-M-15 extended spectrum </span><span
lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-fareast-language:PT'>β</span><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>-lactamase. In three isolates were also found the carbapenemase KPC-3 which
confer resistant to carbapenems. The gene <i>bla</i>CTX-M-15 was associated
with an upstream-inserted insertion sequence ISEcp1. The isolates CTX-M-15
producers belonged to the IncF, IncN and IncA/C groups. The relationship
between isolates by M13 fingerprinting showed different profiles and MLST was
performed for representative isolates from each cluster. 6 different clones
were found: ST-15; ST-76; ST-133; ST-147; ST-276 and ST-307. The ST15-CTX-M-15
accounted for 69% of the isolates analysed by MLST and was identified in
community and in all five hospitals studied.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusions:
</span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>Our data indicate a widespread of ST15-CTX-M-15 <i>K.
pneumoniae</i>which should be considered as clinical relevant for public
health. The dissemination of ESBL CTX-M-15 was due both to carriage of related
plasmids and to multiclonal spread of <i>K. pneumoniae</i>.The tracking of
multidrug resistant organisms is one of the four core actions proposed in
European Center for Disease Prevention and Control action plan to limit the
emergence and spread of this increasingly important pathogen.<o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO63 - Molecular
Characterization Of The <i style='mso-bidi-font-style:normal'>Staphylococcus
aureus</i> CC398 Population In Italy<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Fabiola Feltrin<sup>1</sup>; Patricia Alba<sup>1</sup>; Roberta Amoruso<sup>1</sup>;
Maria de los Angeles Argudín<sup>2</sup>; Birgit Lauzat<sup>3</sup>; Manuela
Iurescia<sup>1</sup>; Luigi Sorbara<sup>1</sup>; Samuele Dottarelli<sup>1</sup>;
Virginia Carfora<sup>1</sup>; Erika Menichini<sup>1</sup>; Fiorentino Stravino<sup>1</sup>;
Beatriz Guerra<sup>3</sup>; Antonio Battisti<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span lang=PT
style='mso-ansi-language:PT;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span lang=PT style='mso-ansi-language:PT;
mso-fareast-language:PT'>Istituto Zooprofilattico Sperimentale del Lazio e
della Toscana ´´M. Aleandri´´,<sup>2</sup> Veterinary and Agrochemical Research
Centre (VAR-CODA-CERVA),<sup>3</sup> Hôpital Erasme,<sup>4</sup> Federal
Institute for Risk Assessment (BfR)<o:p></o:p></span></i></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Background: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>Methicillin-resistant <i>Staphylococcus aureus</i>
(MRSA) Complex Clonal (CC) 398 have been reported as the most prevalent LA-MRSA
lineage in primary productions in Italy. The aim of this study was to provide
molecular characterisation of MRSA and methicillin-susceptible <i>S. aureus</i>
(MSSA) CC 398, isolated in primary productions in Italy in the last five years,
in order to describe the CC398 population structure in Italy.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Materials and Methods: </span></b><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>A total of 60
CC398 isolates were characterized by using <i>spa</i>-typing, Multilocus
Sequence Typing (MLST), <i>SCCmec</i> typing and a microarray genotyping kit
(Alere GmbH, Germany) that includes the main pathogenicity, virulence-, host-,
clone-associated and antimicrobial resistance genes. The majority (n=59) were
from different food producing animal holdings: 31 from swine (6 MSSA), 27 from
bovine (1 MSSA), 1 from broiler chicken, and one (MSSA) from wild boar.
Thirty-six representative isolates were further characterized using
Pulsed-Field Gel Electrophoresis (PFGE).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Results: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>All CC398 isolates were Sequence type (ST)398
except one ST2485 (Single Locus Variant, SLV, of ST398), distributed in 16
different <i>spa</i>-types, with spa-type t899 and related <i>spa</i>-types as
the most frequent, followed by <i>spa</i>-type t034 and t011. Twenty-seven MRSA
isolates carried SCC<i>mec</i> cassette type V while the remainders were type
IV. No significant differences in host distribution of the SCC<i>mec</i> type
or <i>spa</i>-type were observed. While the genetic pattern was almost the same
in all the isolates, the antimicrobial-resistance genes were found in different
proportions among the isolates except for the <i>tem</i>(M) gene encoding
tetracycline resistance, &nbsp;that was presented in all but the MSSA wild boar
isolate (fig.1). None of the isolates presented the human-specific virulence
PVL or immune evasion cluster genes. The macro-restriction profiles observed
showed a high heterogeneity (fig.1) in agreement with previous observations.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Conclusion: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>These observations confirm that <i>S. aureus</i>
CC398 circulating in livestock in Italy have undergone adaptation to animal
host(s), with loss of phage-carried human virulence genes, and that the
population is well established. However, as observed in other studies,
&nbsp;the selection pressure by the use of tetracyclines, beta-lactams and
other classes of antimicrobials, contributed to the emergence and radiation
worldwide of the methicillin-resistant, multi-drug resistant CC398 lineage in
food producing animals.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Acknowledgments: </span></b><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>This work was
supported by grants from the EMIDA ERA-NET (FP7-KBBE) first transnational
research call, Project “LA-MRSA” (acronym title) ID 68.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape id="Imagem_x0020_16"
 o:spid="_x0000_i1027" type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/217imagem_2560108707_0.jpg"
 style='width:333pt;height:250pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S8_files/image007.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/217imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=252
src="public/abstracts/Webpages/PO_S8_files/image008.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/217imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_16"><![endif]></span><span lang=EN-GB style='mso-ansi-language:
EN-GB;mso-fareast-language:PT'><o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span lang=EN-GB style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-GB;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span lang=EN-GB
style='mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO64 - <i
style='mso-bidi-font-style:normal'>Staphylococcus aureus</i> USA300 In
Sub-saharan Africa<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Lena Strauß<sup>1</sup>; Marc Stegger<sup>2</sup>; Anders Rhod Larsen<sup>2</sup>;
Geoffrey Coombs<sup>3</sup>; Sebastien Breurec<sup>4</sup>; Alexander Mellmann<sup>1</sup>;
Frieder Schaumburg<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>University of Münster,<sup>2</sup> Statens Serum
Institut,<sup>3</sup> Murdoch University,<sup>4</sup> University of Antilles<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>B<b>ackground:</b>
USA300 is a hypervirulent community-associated methicillin-resistant <i>Staphylococcus
aureus</i> (CA-MRSA) lineage. The classic USA300 clone is the dominant CA-MRSA
in North America. It is characterized by specific molecular features, which
include the affiliation to multilocus sequence type (ST) 8 and <i>spa</i> type
t008, the possession of the arginine catabolic mobile element (ACME) and a SCC<i>mec</i>
IV element, and the production of Panton-Valentine leukocidin (PVL). In South
America, a distinct lineage of USA300 is dominant, which lacks ACME and harbors
a different SCC<i>mec </i>IV element. Since a few years, both lineages are also
increasingly reported in all other continents - except for Africa. However,
there have been recent reports of MRSA with USA300-like molecular
characteristics in sub-Saharan Africa; but due to differing <i>spa</i> types,
they were not regarded as true USA300. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Materials |
Methods:</span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'> Our objective was to clarify the phylogenetic
relationship of 52 ST8 MRSA and MSSA (methicillin-susceptible <i>S. aureus</i>)
from different countries of sub-Saharan Africa and various published ST8
isolates, including two NCBI reference genomes of USA300 and its assumed
ancestor, USA500. The African isolates were whole genome sequenced using
Illumina technology and analysed with regard to both their core genome and
USA300-specific molecular characteristics using the software SeqSphere<sup>+</sup>
(Ridom GmbH, Münster, Germany).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> Most African MSSA isolates were found to cluster together in one seperate
branch of the phylogenetic tree. Seven MSSA and MRSA isolates from Nigeria and
three MRSA from Gabon were found to form a different group, which is related to
the USA500 reference genome. Moreover, twelve African isolates were identified
that cluster in the same branch like the “classic” USA300 isolates; despite
their differing <i>spa</i> types (t112 or t121), which display single <i>spa</i>
repeat locus variants of t008. Only these African isolates possess all
USA300-specific accessory molecular features; including PVL, ACME and SCC<i>mec</i>
IV. Interestingly, they display a monophyletic clade within the group of
classic USA300 isolates, which suggests an independent evolution of the African
USA300 lineage.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>In conclusion, the data shows that the USA300-related MRSA isolates
detected in West and Central Africa are “true” USA300 clones. This suggests
that USA300 is more dispersed than previously assumed and raises the question
about the origin of this pandemic clone.</span><span style='mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO65 - Evergreen; A
Webtool For Surveillance Of Bacterial Outbreaks<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Johanne Ahrenfeldt<sup>1</sup>; Martin Christen Frølund Thomsen<sup>1</sup>;
Jose Bellod Cisneros<sup>1</sup>; Ole Lund<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Technical University of Denmark<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Background: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>The cost of whole genome sequencing is falling,
and therefore its use in diagnostics is increasing. We wish to make a platform
for surveillance of pathogenic bacteria, where everything that is uploaded to
our servers automatically will be compared to everything else in our databases,
so that in case of an outbreak or an epidemic we will be able to see this
pattern very fast.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Methods: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>When an isolate is uploaded to our servers,
first it will be compared to a homology-reduced database of reference
sequences, where the closest reference sequence will be found. Afterwards the
isolate will be mapped to the found reference. Then the mapped sequence will be
compared to all other uploaded sequences that previously have been found to
match to the same reference. The phylogenetic distances will then be reported,
together with the number of isolates, which are within the following distances
0, 1, 5,10, 100 and 1000 SNPs.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Once a day a
tree will be calculated for each of the references. And whenever a new isolate
is uploaded, it will be placed in the tree.</span><span style='mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO66 - Demography And
Intercontinental Spread Of The USA300 CA-MRSA Lineage<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>P Martins Simoes<sup>1</sup>; P Glaser<sup>2</sup>; A. Villain<sup>2</sup>;
M Barbier<sup>3</sup>; A Tristan<sup>1</sup>; C Bouchier<sup>2</sup>; L Ma<sup>2</sup>;
M Bes<sup>1</sup>; F Laurent<sup>1</sup>; D Guillemot<sup>4</sup>; T Wirth<sup>3</sup>;
F Vandenesch<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>National Reference Center for Staphylococci, Hospices
Civils de Lyon, CIRI INSERM U1111, Lyon 1 University,<sup>2</sup> Institut
Pasteur,<sup>3</sup> Institut de Systématique, Evolution, Biodiversité,
UMR-CNRS 7205, Muséum National d’Histoire Naturelle, Université Pierre et Marie
Curie, Ecole Pratique des Hautes Etudes, Sorbonne Universités,<sup>4</sup>Inserm
UMR 1181 « Biostatistics, Biomathematics, Pharmacoepidemiology and Infectious
Diseases (B2PHI),<sup>5</sup> Institut Pasteur,<sup>6</sup> Univ. Versailles
Saint Quentin<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> Since the end of 1990´s, several genetically distinct community-acquired
methicillin-resistant<i> Staphylococcus aureus</i> (CA-MRSA) lineages have
emerged worldwide. In the US, the ST8-IV clone USA300, has become highly
prevalent, outcompeting MSSA and other MRSA both in the community and hospital
settings. In Europe, we observe the co-existence of the predominant ST80-IV European
CA-MRSA clone with the USA300 clone as well as other lineages. The presence of
USA300 CA-MRSA in Europe rises the question of this lineage capacity to
successfully expand in Europe, similarly to the USA´s worrisome scenario, by
replacing the current European ST80 CA-MRSA.</span><span lang=EN-GB
style='font-size:8.0pt;font-family:MingLiU;mso-bidi-font-family:MingLiU;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Material/Methods:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> We performed whole-genome sequencing to a collection of French USA300
CA-MRSA strains responsible for sporadic cases and micro-outbreaks (n=67) over
the past decade and US ST8 MSSA and MRSA isolates (n=431). Genome-wide
phylogenetic relationships and coalescence-based analyses were performed to
trace the origin, evolution, and dissemination pattern of the USA300 CA-MRSA
clone in France.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> Genome-wide phylogenetic analysis demonstrated that the population structure
of the French isolates is the product of multiple introductions dating back to
the onset of the USA300 CA-MRSA clone in North America. Coalescent-based
demography of the USA300 lineage shows that a strong expansion occurred during
the 90s concomitant to the acquisition of the ACME element and antibiotics
resistance, followed by a sharp decline initiated around 2008. This scenario is
reminiscent of the rise and fall pattern previously observed in the ST80
lineage.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusions:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> Here we show that in most instances, the USA300 CA-MRSA sporadic and
micro-outbreaks cases identified in France, in the last decade, corresponded to
sporadic and independent imports from the USA (USA300-NA MRSA clone) without
further indication of spreading on the French territory. The reasons behind the
apparent lack of success of these USA300-NA CA-MRSA isolates to diffuse within
the French community are unknown but they argue against the hypothesis that the
USA300-NA MRSA clone is intrinsically more successful than the European CA-MRSA
ST80 and, thus, that it could be a potential threat in Europe.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>At a global
level, in the transition from a MSSA lineage to a successful CA-MRSA clone, the
USA300 lineage first became resistant to multiple antibiotics and it acquired
the ACME element and, subsequently, it acquired a resistance to
fluoroquinolones. These two steps seem to be associated with a dramatic
demographic expansion. This expansion was followed by the current stabilization
and expected decline of this lineage. Our findings highlight the significance
of horizontal gene acquisitions and point mutations in the success of such
disseminated clones and illustrate their cyclic and sporadic life cycle.</span><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO67 - In Vivo
Characterisation Of Human Cytomegalovirus Genome Diversity Directly From
Clinical Specimens Sampled Over Time Or From Different Compartments<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Elias Hage<sup>1</sup>; Tina Ganzenmueller<sup>1</sup>; Gavin Wilkie<sup>2</sup>;
Silvia Linnenweber-Held<sup>3</sup>; Julius Schmidt<sup>3</sup>; Eva
Mischak-Weissinger<sup>4</sup>; Anke Schwarz<sup>3</sup>; Albert Heim<sup>1</sup>;
Thomas F. Schulz<sup>1</sup>; Andrew J. Davison<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Institute of Virology, Hannover Medical School,<sup>2</sup>
Centre for Virus Research, University of Glasgow,<sup>3</sup> Department of
Nephrology, Hannover Medical School,<sup>4</sup> Department of Haematology,
Haemostasis and Oncology, Hannover Medical School<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> Human cytomegalovirus (HCMV) belongs to the subfamily <i>Betaherpesvirinae
</i>and is an important pathogen causing a wide spectrum of disease in
immunocompromised individuals or congenitally infected children. Introduction
of high-throughput next-generation sequencing (NGS) has opened up opportunities
to study the intrahost structure of HCMV populations. To gain insights into
HCMV diversity among different anatomical compartments or longitudinally (i.e.
over time), we performed NGS of complete HCMV genomes directly from clinical
samples. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Materials
&amp; Methods</span></b><b style='mso-bidi-font-weight:normal'><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>:</span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'> We sequenced 37 HCMV-positive diagnostic
specimens such as urine, whole blood, plasma, vitreous body fluid or
respiratory material obtained from three children with congenital HCMV disease,
six renal transplant recipients (RTR-1 to RTR-6) and two haematopoietic stem
cell recipients (SCTR-1 and SCTR-2). For six out of these patients, blood
samples obtained longitudinally at different time-points were available (2-6
time-points per patient). The median HCMV load determined by quantitative HCMV
PCR was 3.0 x 10<sup>5</sup> copies/ml (range 7.5 x 10<sup>3</sup> - 4.2 x 10<sup>6</sup>)
for whole blood and 1.4 x 10<sup>6</sup> copies/ml (range 1.3 x 10<sup>5</sup>
- 1.9 x 10<sup>8</sup>) for other specimens. NGS was performed on an Illumina
MiSeq platform after preparation of target-enriched libraries. Consensus viral
genomes (“major genome types”) were created by <i>de novo</i> assembly of
sequence reads.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results</span></b><b
style='mso-bidi-font-weight:normal'><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> We were able to assemble HCMV genomes from 34 out of 37 samples with an
average read depth ranging from 23 to 5,450 (median 218) for whole blood and
from 159 to 4,677 (median 1,391) for other specimens. Samples with ≤1.0 x 10<sup>4</sup>
HCMV copies/ml did not yield enough HCMV reads for sufficient assembly.
Analysis of HCMV diversity between longitudinally collected blood samples
showed different patterns: while no major differences between different
time-points were detected in three patients, we observed a complete change of
the HCMV genome type in SCTR-1 (six time-points within 154 days), with the
second genome type being already detectable at a low frequency at time-point 1
and becoming dominant at time-point 4. Similarly, longitudinal samples (five
time-points within one year) from RTR-1 showed low HCMV diversity among the
first three samples, but marked differences located especially in the RL11 and
US6 gene families between the populations present at t1 and t4/t5. Major HCMV
genome types from different anatomical compartments were highly similar in
blood and urine of congenitally infected children, a corresponding breast milk
sample from one mother, and in blood, bronchioalveolar lavage fluid and a gut biopsy
from RTR-5. In contrast, high diversity was observed between vitreous body
fluid and blood from RTR-6 who suffered from HCMV retinitis. Furthermore,
low-frequency variants (i.e. present in 2-49% of the reads at a nucleotide
position) were observed particularly in blood from several patients (e.g.
SCTR-1, -2 or RTR-6). These might reflect mixed infections or a small fraction
of the dominant HCMV type being present in another body compartment.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion</span></b><b
style='mso-bidi-font-weight:normal'><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> Our data show that application of NGS to sequence HCMV directly from a
variety of diagnostic specimens is technically feasible and gives promising
insights into the viral diversity and evolution of HCMV <i>in vivo</i>.
Detailed analysis of different variability patterns observed in clinical
samples will further elucidate the pathogenic role of HCMV diversity <i>in vivo</i>.</span><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO68 - Frequency And
Predisposing Factors Of Upper Respiratory Tract Colonization By Gram-negative
Rods In Untreated Chronic Lymphocytic Leukaemia Patients<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Ewelina Grywalska<sup>1</sup>; Izabela Korona-Glowniak<sup>1</sup>; Anna
Malm<sup>1</sup>; Jacek Rolinski<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Medical University of Lublin<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Infectious complications are still one of the major causes of morbidity and
mortality in patients with chronic lymphocytic leukaemia (CLL). Infections
affect mainly the respiratory tract, skin, or urinary tract. The most common
respiratory infections are acute and chronic sinusitis and pneumonia. In the
past pneumonia was caused mainly by Streptococcus pneumoniae, but with current
chemotherapeutic regimens, the spectrum of pathogens includes Gram-negative
rods (GNR), Nocardia species, Legionella species, and Pneumocystis carinii. An
increase in infections caused by GNR, particularly bacteraemia and pneumonia,
may reflect more advanced disease and profound myelosuppression in these
patients.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Aim of the
study:</span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'> The aim of this study was to assess the
frequency and predisposing factors of colonization of upper respiratory tract
by GNR in previously untreated CLL patients. Antimicrobial susceptibility of
the isolated strains was determined.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Material and
methods: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>This prospective study included 30 previously
untreated patients with CLL and 24 healthy volunteers. Throat and nasal
specimens were taken using sterile alginate-tipped swabs. Swabs were placed
directly into Stuart’s trans-medium, and samples were delivered to the
laboratory, where specimens were streaked onto nonselective medium (blood agar)
and selective medium (MacConkey agar). Plates were incubated for 24–48 h at
35°C under aerobic conditions. Presence of GNR in sample from nostrils and/or
throat was called colonization. The identification of isolates was determined
and antimicrobial susceptibility of the isolates was tested by the disc
diffusion method according to the European Committee on Antimicrobial
Susceptibility Testing recommendations. ESBL production screening was detected
by double-disk synergy test. Peripheral blood (PB) specimens for flow
cytometric studies were obtained from 30 CLL patients and 24 healthy donors.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>A significantly higher frequency of GNR colonization in CLL patients (36.7%)
was observed in comparison to healthy volunteers (8.3%). This difference was
statistically significant (</span><span lang=DE-CH style='font-size:8.0pt;
font-family:"Cambria Math";mso-fareast-font-family:"Cambria Math";mso-bidi-font-family:
"Cambria Math";color:black;mso-fareast-language:PT'>𝑃</span><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>=0.02; RR
4.4; 95%CI 1.1–18.0). The overall 16 isolates of GNR (from 3 patients, 2
different species of GNR were isolated) were cultured: 8 (50%) isolates were
obtained from throat, 6 (37.5%) from nostrils, and 2 (12.5%) isolates colonized
both throat and nostrils. GNR isolates mainly belonged to the
Enterobacteriaceae family, and only 1 isolate, Pseudomonas aeruginosa, belonged
to nonfermentative Gram-negative rod-Pseudomonadaceae.ESBL-producing isolate,
Proteus vulgaris, was obtained from one CLL patient. Four isolates, Citrobacter
koseri, Serratia marcescens, and two isolates of Hafnia alvei, demonstrated the
presence of AmpC </span><span lang=DE-CH style='font-size:8.0pt;font-family:
"Cambria Math";mso-fareast-font-family:"Cambria Math";mso-bidi-font-family:
"Cambria Math";color:black;mso-fareast-language:PT'>𝛽</span><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>-lactamases.
Colonization rate was higher among CLL patients with lower level of IgG in
serum (</span><span lang=DE-CH style='font-size:8.0pt;font-family:"Cambria Math";
mso-fareast-font-family:"Cambria Math";mso-bidi-font-family:"Cambria Math";
color:black;mso-fareast-language:PT'>𝑃</span><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>=0.017) and
patients with higher number of lymphocytes (</span><span lang=DE-CH
style='font-size:8.0pt;font-family:"Cambria Math";mso-fareast-font-family:"Cambria Math";
mso-bidi-font-family:"Cambria Math";color:black;mso-fareast-language:PT'>𝑃</span><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>=0.005). It was shown that the longer the time elapsed since diagnosis of
CLL, the higher frequency of GNR colonization was observed (</span><span
lang=DE-CH style='font-size:8.0pt;font-family:"Cambria Math";mso-fareast-font-family:
"Cambria Math";mso-bidi-font-family:"Cambria Math";color:black;mso-fareast-language:
PT'>𝑃</span><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>=0.025).<br>
<b>Conclusions:</b> Awareness of risk factors predisposing to pathogens
colonization, allows to identify group of patients which should be considered
for immunoglobulin or antibiotic prophylaxis. Moreover, knowledge about
antibiotic resistance of the colonizing pathogens is important to propose not
only optimal antibiotic prophylaxis scheme but also empiric and targeted
therapy with greater likelihood of clinical success.</span><span
style='mso-bidi-font-family:Arial;mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO69 - Re-evaluation Of
<i style='mso-bidi-font-style:normal'>Streptococcus pneumoniae</i> Carriage By
Real-time PCR In Adults Over 60 Years Of Age<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Sónia T. Almeida<sup>1</sup>; Tânia Pedro<sup>1</sup>; Hermínia de
Lencastre<sup>2</sup>; Raquel Sá-Leão<sup>1</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Laboratory of Molecular Microbiology of Human
Pathogens, ITQB/UNL,<sup>2</sup> Laboratory of Microbiology and Infectious
Diseases, The Rockefeller University<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> <i>Streptococcus pneumoniae</i> (or pneumococcus) is a human commensal
that frequently colonizes the nasopharynx. It is also a major cause of
infectious diseases worldwide, especially at the extremes of age. Currently,
most studies on pneumococcal carriage among the elderly are based on
culture-based methods followed by identification of presumptive colonies. More
recently, new strategies to detect pneumococcal carriage, based on real-time
PCR, have been proposed. The aim of this study was to evaluate the prevalence
of nasopharyngeal and oropharyngeal pneumococcal carriage in adults over 60
years of age, using real-time PCR.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Materials
and Methods: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>Two collections of samples previously
characterized by routine procedures based on culture-based methods were
studied: one corresponding to samples of 78 individuals living in a retirement
home and other corresponding to samples of 200 individuals living in their
family homes. Pneumococcal carriage was re-evaluated by real-time PCR,
targeting two pneumococcal genes: <i>lytA</i> (major pneumococcal autolysin)
and <i>piaA</i> (iron uptake ABC transporter lipoprotein PiaA). <i>S.
pneumoniae</i> capsular types present in positive samples were determined by
multiplex PCR and DNA sequencing.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> By real-time PCR <i>S. pneumoniae</i> was more frequently detected in
oropharyngeal samples than in nasopharyngeal samples: 6.0% vs 4.0% (p=0.49),
respectively, in the family home collection and 16.7% vs 15.4% (p=1.0),
respectively, in the retirement home collection. Use of real-time PCR when
compared with culture-based methods increased pneumococcal carriage in
oropharyngeal samples from 0% to 6.0% (p=0.001) in the family home collection,
and from 3.8% to 16.7% (p=0.01) in the retirement home collection. By contrast,
no significant differences were observed in carriage detection in nasopharyngeal
samples when both methods were compared. In the retirement home collection 10
of 18 isolates were of serotype 19A. Overall, the use of real-time PCR
increased significantly the detection of <i>S. pneumoniae</i> from 3.5% to 9.0%
(p=0.05) in the family home collection and non-significantly from 16.7% to
20.5% (p=0.38) in the retirement home collection.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> The results of this study indicate that the use of real-time PCR increases
the detection of <i>S. pneumoniae</i> carriage among adults over 60 years of
age where colonization density is often low. The use of molecular methods is
particularly valuable when sampling the oropharynx.</span><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO70 - Whole Genome
Sequencing Of Methicillin-resistant And Sensitive <i style='mso-bidi-font-style:
normal'>Staphylococcus aureus</i> Strains Isolated In The Cardiosurgical Area
Of The Italian L. Sac<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=PT style='mso-ansi-language:PT;mso-fareast-language:
PT'>Gentile Bernardina<sup>1</sup>; Rimoldi Sara Giordana<sup>2</sup>; Pagani
Cristina<sup>2</sup>; Anselmo Anna<sup>1</sup>; Palozzi Anna Maria<sup>1</sup>;
Fortunato Antonella<sup>1</sup>; Di Gregorio Aannamaria<sup>2</sup>; Lista
Florigio<sup>1</sup>; Gismondo Maria Rita<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Histology and Molecular Biology Section, Army Medical
Research Center,<sup>2</sup> Microbiology Unit, L.Sacco University Hospital<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Background: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Methicillin-resistant <i>Staphylococcus aureus</i> (MRSA) in a intensive
care unit (ICU) may be acquired from colonized or infected patients and
healthcare workers (HCWs). Community reservoirs have been implicated in MRSA
introduction in health care settings by increasing the colonization prevalence
among patients, visitors and workers. However, identifying reservoirs and
tracking the source of implicated strains has proven to be difficult due to the
limitation of genotyping methods for clinical practice. The study was designed
to retrospectively evaluate the molecular characterization of <i>Staphylococcus
aureus </i>strains collected in the Cardiosurgical Area at the “Luigi Sacco”
Hospital, Milan (Italy). <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Material/methods</span></b><b
style='mso-bidi-font-weight:normal'><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> During April 2015 a total of 4 patients with deep sternal wound infection
positive for <i>S. aureus</i> were identified. Moreover, during the previous
months other 3 cases were reported. To evaluate putative transmission events
between patients and operators, 98 nasal swabs were collected from HCWs.
Microorganisms identification and susceptibility test were performed using
Vitek2 (Biomerieux, Marcy l’Etoille, France). <i>S. aureus&nbsp;</i>isolates
were analyzed by real time PCR (RealCycler SAMAPV, Progenie Molecular,
Valencia, Spain) to detect the presence of the <i>mecA</i> gene, coding for
methicillin resistance, and the Panton-Valentine leucocidine toxin (PVL) gene.
The whole-genome sequencing (WGS) approach was used to perform MLST (MultiLocus
Sequence Typing) and SNPs (Single Nucleotide Polymorphisms) <i>in silico</i>
analysis of the <i>S. aureus&nbsp;</i>isolates.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Results: </span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Among seven patients, 2 out of 7 <i>S. aureus</i> strains collected were
MSSA (Methicillin-sensitive <i>Staphylococcus aureus, mecA </i>negative), 5
were MRSA (<i>mecA</i> positive). The epidemiological surveillance carried out
on the staff showed that 12 out of 98 operators (12%) were positive to <i>S.
aureus: 7</i> resulted as MSSA and 5 as MRSA. The PVL toxin gene was not
detected. <i>In silico</i> extrapolation of MLST profiles from the 12 operators
and the 7 patients positive to <i>S. aureus </i>(n=19), revealed the precence
of 10 different sequence types (STs): the most represented was the ST22 (7
strains), followed by ST121, ST30 and ST8 (2 strains). ST5, ST10, ST15, ST45
were each representative of one single strain. Two new STs, differing from the
ST22 and the ST5 for a single MLST gene, were also identified. The SNPs
analysis outcome was in agreement with the MLST results but allowed us to
discriminate between samples with the same ST, as in the case of the most
predominant ST22 group (Fig.1).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b><span lang=EN-GB
style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:
Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>Conclusion:</span></b><span
lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'> Discrimination among genetically related strains is indispensable for
clinical and epidemiological investigation and WGS is emerging as the gold
standard in bacterial genotyping. The diversity of isolates detected during the
epidemiological evaluation in the Cardiosurgical Area of the “Luigi Sacco” Hospital
may reflect a larger epidemiology within the hospital and/or the community.
Further molecular investigations are request considering that the ST5, 8, 15,
22, 30 are reported in other Italian hospital settings while the ST121 and ST45
have been identified also in animals.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape id="Imagem_x0020_15"
 o:spid="_x0000_i1026" type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/202imagem_2560108707_0.jpg"
 style='width:349pt;height:255pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S8_files/image009.png" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/202imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=351 height=257
src="public/abstracts/Webpages/PO_S8_files/image010.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/202imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_15"><![endif]></span><span style='mso-ansi-language:
EN-US;mso-fareast-language:PT'><o:p></o:p></span></p>

<b style='mso-bidi-font-weight:normal'><span style='font-size:10.0pt;
mso-bidi-font-size:11.0pt;line-height:115%;font-family:Arial;mso-fareast-font-family:
Cambria;mso-fareast-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:
PT;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span></b>

<p class=MsoNormal align=left style='margin-bottom:10.0pt;text-align:left;
line-height:115%'><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>PO71 - DNA Structural
Properties At Origin-of-transfer Regions Determine Hosts Of Mobile Plasmids<o:p></o:p></span></b></p>

<p class=MsoNormal><span style='mso-ansi-language:EN-US;mso-fareast-language:
PT'>Jan Zrimec<sup>1</sup>; Aleš Lapanje<sup>2</sup><o:p></o:p></span></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><sup><span
style='mso-ansi-language:EN-US;mso-fareast-language:PT'>1</span></sup></i><i
style='mso-bidi-font-style:normal'><span style='mso-ansi-language:EN-US;
mso-fareast-language:PT'>Faculty of Health Sciences, University of Primorska,<sup>2</sup>
Institute of Metagenomics and Microbial Technologies,<sup>3</sup> Institute of
Metagenomics and Microbial Technologies,<sup>4</sup> Saratov state university<o:p></o:p></span></i></p>

<p class=MsoNormal><span style='mso-bidi-font-family:Arial;color:black;
mso-themecolor:text1;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Background: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>Horizontal gene transfer of conjugative plasmids
that carry antimicrobial resistance (AMR) genes is a major health concern, as
they spread AMR to infected patients from the large environmental pool. Since
the repertoire of bacterial hosts of such plasmids is characterized by their
MOB group, it is essential to analyze and enable fast prediction of MOB systems.
MOB systems are defined by relaxases, essential enzymes that nick plasmid DNA
and initiate transfer. Since each relaxase is specific to one type of
origin-of-transfer (oriT), DNA in each oriT region must have specific
conformational and physicochemical properties that most efficiently attract and
enable activity of the particular relaxase. We therefore analysed the
conservation of oriT structural properties according to their relaxase defined
MOB groups.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Materials | Methods: </span></b><span lang=EN-GB style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
color:black;mso-ansi-language:EN-GB;mso-fareast-language:PT'>DNA structural
properties of oriT regions from conjugative elements from 4 MOB groups were
predicted with parametric models and included nucleotide sequence,
conformational and physicochemical properties. Conformational and
physicochemical properties were predicted using parametric models as well as
new models based on Nearest Neighbor and Peyrard-Bishop-Dauxois models. The
bioinformatic framework incorporated prediction of oriT structural properties,
analysis of oriT nucleotide sequences and machine learning algorithms.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Results: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>Our computational analysis of oriT regions
showed that oriT structural properties enable accurate characterization of
plasmids, as we were able to successfully sort conjugative elements into their
corresponding MOB groups (classification accuracy over 99%, Figure 1). This was
possible, because predicted properties distinguished known experimentally
determined structural features of the oriT regions. Conventional classification
based on oriT nucleotide sequences was inaccurate (estimated classification
accuracy was approx. </span><span lang=DE-CH style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-fareast-language:
PT'>58%).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><b style='mso-bidi-font-weight:
normal'><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:Arial;color:black;mso-ansi-language:EN-GB;mso-fareast-language:
PT'>Conclusion: </span></b><span lang=EN-GB style='font-size:8.0pt;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-family:Arial;color:black;mso-ansi-language:
EN-GB;mso-fareast-language:PT'>Mobile plasmids can be correctly classified into
MOB groups not only according to conservation of the amino acid sequence of
relaxase, but also as a function of DNA structural properties of the relaxase
substrate oriT. Our results show that within particular MOB groups oriT regions
have relatively conserved structural features (Figure 1). We propose that
especially structural properties of oriT regions are connected to and coevolve
with the proteins involved in DNA recognition, nicking and transfer reactions
within their particular MOB group.&nbsp;Since our method can help to determine
the most probable hosts and AMR pathways based on merely 240 bp of DNA,
&nbsp;it can help clinicians apply promising approaches to treat AMR infections
by using antibiotic combinations that inhibit the transfer of AMR genes, or to
develop new ways to inhibit AMR gene transfer within polymicrobial infections.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:6.0pt'><span style='font-size:8.0pt;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:Arial;
mso-ansi-language:EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape id="Imagem_x0020_13"
 o:spid="_x0000_i1025" type="#_x0000_t75" alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/236imagem_2560108707_0.jpg"
 style='width:333pt;height:226pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="PO_S8_files/image011.jpg" o:title="//www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/236imagem_2560108707_0.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=335 height=228
src="public/abstracts/Webpages/PO_S8_files/image012.png"
alt="Description: https://www.eventgest.com/CLIENTES/veranatura/uploads/abstracts2/236imagem_2560108707_0.jpg"
v:shapes="Imagem_x0020_13"><![endif]></span><span style='mso-bidi-font-family:
Arial;color:black;mso-themecolor:text1;mso-ansi-language:EN-US'><o:p></o:p></span></p>

</div>

<span lang=EN-GB style='font-size:10.0pt;mso-bidi-font-size:11.0pt;font-family:
Arial;mso-fareast-font-family:Cambria;mso-fareast-theme-font:minor-latin;
color:black;mso-themecolor:text1;mso-ansi-language:EN-GB;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'><br clear=all style='page-break-before:always;
mso-break-type:section-break'>
</span>

<div class=WordSection2>

<p class=MsoNormal><span lang=EN-GB style='mso-bidi-font-family:Arial;
color:black;mso-themecolor:text1;mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=DE-CH><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
